<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Report on Polygenic Traits | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/polygenic-report"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Report on Polygenic Traits | Genomenal"><meta data-react-helmet="true" name="description" content="Polygenic traits are traits, such as height or weight, that are caused by the action of multiple genes."><meta data-react-helmet="true" property="og:description" content="Polygenic traits are traits, such as height or weight, that are caused by the action of multiple genes."><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/polygenic-report"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/polygenic-report" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/polygenic-report" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.7b4d41ea.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.643c0577.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Workflow Details</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Report on Polygenic Traits</h1></header><p><strong>Polygenic traits</strong> are traits, such as height or weight, that are caused by the action of multiple genes.
In addition to genetics, polygenic traits are influenced by many other factors, such as environment, socioeconomic
factors, and lifestyle. The genetic component of polygenic traits is assessed using <strong>polygenic risk score</strong> (PRS).
PRS incorporates the effects of many genetic variants into one number that predicts the genetic predisposition
for trait. Each PRS value can be plotted on <strong>a PRS distribution frequency plot</strong> (the bell-shaped curve).
For most people, their PRS values will be in the middle region, but for some, these values may deviate to the
left or right of the average, which will indicate the presence of a lower or higher value of the polygenic trait.</p><p>To calculate polygenic traits, <strong>predictive models</strong> (logistic regression model and Cox regression model) are
used that take into account PRS values, as well as parameters such as sex and age (allowable range: from 18 to
80 years). Polygenic risk models are a vector of variant effect scores identified for a trait based on the
results of genome-wide association studies and taking into account the structure of linkage disequilibrium.
The models are constrained by a list of common genetic variants represented
in <a href="https://doi.org/10.1038/nature09298" target="_blank" rel="noopener noreferrer">the HapMap 3 data set</a>.
<strong>The logistic regression model</strong> allows you to determine the relationship between the presence or absence
of a disease and predictor variables. <strong>The Cox regression model</strong> takes into account the age of onset of
the disease and, accordingly, allows you to see the dynamics of the risk of developing the disease with age.
More about comparing these models: <a href="https://pubmed.ncbi.nlm.nih.gov/2727473/" target="_blank" rel="noopener noreferrer">Ingram DD, Kleinman JC. Empirical comparisons of proportional hazards and logistic regression models. Stat Med. 1989 May;8(5):525-38</a>.
The final prediction of the models is not absolutely accurate, since the list of genetic variants influencing
polygenic traits is constantly being supplemented by the scientific community.
How well a statistical model predicts the presence or absence of a disease in a person is determined
by the <strong>AUC</strong> (Area Under the Receiver Operating Characteristic Curve) index. The AUC value ranges from 50% to
100%, where higher values indicate that the model has more predictive power.</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="generate-report"></a>Report generation<a class="hash-link" href="#generate-report" title="Direct link to heading">#</a></h2><p>The report is based
on <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">the &quot;Polygenic traits&quot; report template block</a>,
which can only be applied to non-tumor samples. The block regulates polygenic risk score calculation results
for which traits will be included in the report.</p><p>Report on polygenic traits is generated for a sample if the following conditions are met:</p><ol><li>The sample is uploaded as a non-tumor sample (a sample of the &quot;NORMAL&quot; type).</li><li>The sample analysis has been successfully completed (i.e. all stages included in the workflow
have the &quot;Complete&quot; status).</li><li>The &quot;Polygenic risk scores calculation&quot; task of the &quot;Phenotype prediction&quot; analysis stage has been successfully
completed for the sample. By default, the task is not included in the analysis workflow, so it must be included in the
parameters by activating <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#polygenic-risk-scores-calculation">the &quot;Run polygenic risk scores calculation&quot; option</a>.
Please note that to include polygenic risk scores calculation in the analysis workflow of a sample uploaded
in VCF or GT format, you must select <a href="/ngsw-docs-en/settings/settings/settings">the setting preset</a>, which includes the
&quot;Run polygenic risk scores calculation&quot; parameter, at the stage of composing a sample set.</li><li>The report template, which includes the &quot;Polygenic traits&quot; block, is active (adjusted
on <a href="/ngsw-docs-en/settings/report-templates/report-templates">the &quot;Report templates&quot; page</a>).</li><li>The report template was added to the system before the sample processing has been completed.</li></ol><p>After a report on polygenic traits has been successfully generated, open the report tab from the sample page.
To calculate a report, you must specify the patient&#x27;s age (date of birth) and sex, which are taken into account
when building a predictive model for calculating polygenic risk scores. This can be done on the patient page
or on the report page itself. If the patient is under 18 years old, the extreme lower limit will be used as
the age value of 18 y.o. for calculations, since the predictive model uses a range from 18 to 80 years old.
If the patient is over 80 years old, the extreme upper limit will be used as the age value of 80 y.o. for
calculations, since the predictive model uses a range from 18 to 80 years old.
After specifying the patient&#x27;s age and sex, you need to wait a bit while the values required for generating
the report are calculated.</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="results"></a>Results<a class="hash-link" href="#results" title="Direct link to heading">#</a></h2><p>Which polygenic traits will be included in the report is defined in the report template block.
Quantitative and binary polygenic traits are available for selection.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="quantity-traits-results"></a>Prediction results for quantitative polygenic traits<a class="hash-link" href="#quantity-traits-results" title="Direct link to heading">#</a></h3><p>A quantitative trait defined by a numerical value, often determined by multiple genes at different genomic
locations that have small effects and are influenced by environmental conditions. For such traits, the predicted
value is estimated from a predictive model using the PRS value and the patient&#x27;s sex and age. This dimension
of this value is of the given trait, so it can be easily interpreted. Since the predictive model determines a trait
value with a certain error, a confidence interval is provided for each trait, giving the information about the range
of possible values. The confidence interval is calculated from the parameters of the predictive model.</p><p>Quantitative polygenic risks calculated:</p><ul><li>Height</li><li>Weight</li><li>Body Mass Index (BMI)</li></ul><p>As prediction results for quantitative polygenic traits, the following information is provided:</p><ul><li><strong>Predicted trait value</strong> is the absolute trait value determined using a logistic regression model that takes
into account the patientâ€™s genetic data and data on sex and age. In the diagram below, the predicted trait value
is the patient&#x27;s genetic height (162 cm).</li><li><strong>The range of possible trait values</strong> is the 95% confidence interval, which includes the true value of the
trait with a high degree of confidence (the logistic regression model is used for calculation). In the diagram
below, the range of possible values is a height from 153 cm to 172 cm.</li><li><strong>A comparison of the predicted value with the mean value of the trait</strong>. The result of the comparison is a
conclusion about how much the patient&#x27;s predicted value of a given trait is less or more than the average value
of the trait, determined taking into account the patient&#x27;s ethnicity, as well as his sex and age. In the diagram
below, the average value of the trait is the average genetic height (166 cm).</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/confidence_interval-5336f095da0f58f59aa51c49b480f110.png" width="600"></p><ul><li>A display of a patient&#x27;s polygenic risk score (PRS) value on <strong>a PRS distribution graph</strong>.
The patientâ€™s PRS value area is marked green (if values fall in the low or average PRS range) or orange (if values
fall in the high PRS range) and the remaining area is blue. In the graph below, the patient&#x27;s PRS values deviate to
the left of the average PRS range, indicating lower values of the polygenic trait:</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/prs_height-7bbe64be3d79bf8a30a82fd22f16dc40.png" width="600"></p><p>We also determine how much the absolute value of the patient&#x27;s trait differs from the average value of the trait of
people from a group of close ethnic origin. As the result of comparison, the percentage of the number of people, who
have a genetic prognosis of the trait lower or greater than the patient, is given.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="quality-traits-results"></a>Prediction results for binary polygenic traits<a class="hash-link" href="#quality-traits-results" title="Direct link to heading">#</a></h3><p>Binary traits are characteristics with two possible outcomes: presence or absence (e.g. susceptibility or resistance
to a certain infection).</p><p>Binary polygenic risks calculated:</p><ul><li>Predisposition to being overweight</li><li>Predisposition to Prostate Cancer (if the patient&#x27;s sex is indicated as male)</li><li>Predisposition to Breast Cancer (if the patient&#x27;s sex is indicated as female)</li><li>Predisposition to Coronary Artery Disease</li><li>Predisposition to Inflammatory Bowel Disease</li><li>Predisposition to Type 2 diabetes</li><li>Predisposition to Colorectal Cancer</li></ul><p>As prediction results for binary polygenic traits, the following information is provided:</p><ul><li><strong>Predicted value of the patient&#x27;s predisposition to the trait</strong> determined using a logistic regression model
that takes into account the userâ€™s genetic data and data on age and sex. It&#x27;s presented in the diagram
as a &quot;Patient risk&quot; block. The block is colored green if the patientâ€™s risk is less than the mean risk, or orange
if the patientâ€™s risk is greater than the mean risk. In the diagram below, the predicted trait value (the risk
of developing the disease) is 21%.</li><li><strong>A comparison of the predicted value with the mean risk</strong>. Comparison is presented as a diagram. The mean
risk of disease is determined by the patient&#x27;s ethnicity as well as sex and age, and is presented in a blue-coloured
&quot;Mean risk&quot; block. In the diagram below, the mean risk value of the trait is 12%. As a result, information is
provided on how many times the patientâ€™s genetic risk for the trait is greater or less than the average risk of people
from a group of close ethnic origin.</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/obesity_result-999112c2bcb53e9c6af6d29665c95a69.png" width="600"></p><ul><li><strong>The quality of model prediction</strong> (in percent) observed using the AUC (Area Under the Receiver
Operating Characteristic Curve) score for calculation of genetic predisposition to a certain disease.
In the diagram below, the model prediction quality is 75% for the trait under study and patient&#x27;s ethnic group.</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/auc-495354dd97503c6904906c546a09ddc8.png" width="600"></p><ul><li>A display of a patient&#x27;s polygenic risk score (PRS) value on <strong>a PRS distribution graph</strong>.
The patientâ€™s PRS value area is marked green (if values fall in the low or average PRS range) or orange (if values
fall in the high PRS range) and the remaining area is blue. In the graph below, the patient&#x27;s PRS values deviate to
the right of the average PRS range, indicating greater values of the polygenic risk:</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/prs_obesity-ab1d6fb05fe0cd925b290002752a2530.png" width="600"></p><p>It is also determined how much the patient&#x27;s polygenic risk score value differs from the distribution of the
values of the polygenic risk scores of people from a group of close ethnic origin. As the result of comparison,
the percentage of the number of people, who have a genetic risk of the trait lower or greater than the patient,
is given.</p><ul><li><strong>A graph of disease risk versus age</strong>, taking into account the patient&#x27;s genetic predisposition and comparing
this risk value with the average results of people of similar ethnic origin. The probability of having a disease
based on age is presented as a graph. The patientâ€™s disease risk curve is marked green (if the patientâ€™s disease
risk is below average) or orange (if the patientâ€™s disease risk is above average), and the average disease risk curve
is marked blue.</li></ul><p>An example of a graph showing a patient&#x27;s risk of having a disease based on age is below average:</p><p align="center"><img src="/ngsw-docs-en/assets/images/cancer_graph-8116f8fb0f424c90e7a814bc3d5d4740.png" width="700"></p><p>An example of a graph showing a patient&#x27;s risk of having a disease based on age is above average:</p><p align="center"><img src="/ngsw-docs-en/assets/images/obesity_graph-5cf6c63a37e472c66837ca1574fecfdc.png" width="700"></p></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/onco-genetic-testing-report"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Onco-Genetic Testing Report</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Mutations Suitable For The Specified Conditions Â»</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#generate-report" class="table-of-contents__link">Report generation</a></li><li><a href="#results" class="table-of-contents__link">Results</a><ul><li><a href="#quantity-traits-results" class="table-of-contents__link">Prediction results for quantitative polygenic traits</a></li><li><a href="#quality-traits-results" class="table-of-contents__link">Prediction results for binary polygenic traits</a></li></ul></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2024 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.7b4d41ea.js"></script>
<script src="/ngsw-docs-en/assets/js/main.643c0577.js"></script>
</body>
</html>