"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[81],{3905:function(e,t,n){n.d(t,{Zo:function(){return p},kt:function(){return g}});var r=n(7294);function o(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function a(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);t&&(r=r.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,r)}return n}function i(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?a(Object(n),!0).forEach((function(t){o(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):a(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,r,o=function(e,t){if(null==e)return{};var n,r,o={},a=Object.keys(e);for(r=0;r<a.length;r++)n=a[r],t.indexOf(n)>=0||(o[n]=e[n]);return o}(e,t);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);for(r=0;r<a.length;r++)n=a[r],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(o[n]=e[n])}return o}var c=r.createContext({}),l=function(e){var t=r.useContext(c),n=t;return e&&(n="function"==typeof e?e(t):i(i({},t),e)),n},p=function(e){var t=l(e.components);return r.createElement(c.Provider,{value:t},e.children)},u={inlineCode:"code",wrapper:function(e){var t=e.children;return r.createElement(r.Fragment,{},t)}},f=r.forwardRef((function(e,t){var n=e.components,o=e.mdxType,a=e.originalType,c=e.parentName,p=s(e,["components","mdxType","originalType","parentName"]),f=l(n),g=o,h=f["".concat(c,".").concat(g)]||f[g]||u[g]||a;return n?r.createElement(h,i(i({ref:t},p),{},{components:n})):r.createElement(h,i({ref:t},p))}));function g(e,t){var n=arguments,o=t&&t.mdxType;if("string"==typeof e||o){var a=n.length,i=new Array(a);i[0]=f;var s={};for(var c in t)hasOwnProperty.call(t,c)&&(s[c]=t[c]);s.originalType=e,s.mdxType="string"==typeof e?e:o,i[1]=s;for(var l=2;l<a;l++)i[l]=n[l];return r.createElement.apply(null,i)}return r.createElement.apply(null,n)}f.displayName="MDXCreateElement"},9690:function(e,t,n){n.d(t,{ZP:function(){return c}});var r=n(7462),o=n(3366),a=(n(7294),n(3905)),i=["components"],s={toc:[]};function c(e){var t=e.components,n=(0,o.Z)(e,i);return(0,a.kt)("wrapper",(0,r.Z)({},s,n,{components:t,mdxType:"MDXLayout"}),(0,a.kt)("p",null,"When you first open Genomenal, you will see a login page. Enter your login and password if you already have an account or sign up."))}c.isMDXComponent=!0},4826:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return c},contentTitle:function(){return l},metadata:function(){return p},toc:function(){return u},default:function(){return g}});var r=n(7462),o=n(3366),a=(n(7294),n(3905)),i=n(9690),s=["components"],c={sidebar_position:3,sidebar_label:"Getting to know the service"},l="Getting to know the service",p={unversionedId:"overview",id:"overview",isDocsHomePage:!1,title:"Getting to know the service",description:"Next you will see the main page listing the patients. If you have not previously used the service, the page will be empty. There is a search bar at the top to help you navigate through the patient pages.",source:"@site/docs/overview.mdx",sourceDirName:".",slug:"/overview",permalink:"/ngsw-docs-en/overview",tags:[],version:"current",sidebarPosition:3,frontMatter:{sidebar_position:3,sidebar_label:"Getting to know the service"},sidebar:"tutorialSidebar",previous:{title:"Quick start for prenatal genetic testing",permalink:"/ngsw-docs-en/quick-start-for-prenatal-genetic-testing"},next:{title:"Samples for analysis",permalink:"/ngsw-docs-en/data-for-analysis"}},u=[],f={toc:u};function g(e){var t=e.components,c=(0,o.Z)(e,s);return(0,a.kt)("wrapper",(0,r.Z)({},f,c,{components:t,mdxType:"MDXLayout"}),(0,a.kt)("h1",{id:"getting-to-know-the-service"},"Getting to know the service"),(0,a.kt)(i.ZP,{mdxType:"Login"}),(0,a.kt)("p",null,"Next you will see the main page listing the patients. If you have not previously used the service, the page will be empty. There is a search bar at the top to help you navigate through the patient pages."),(0,a.kt)("p",{align:"center"},(0,a.kt)("img",{src:n(6971).Z,width:"1000"})),(0,a.kt)("p",null,"You can see various sections of the service in the navigation bar on the left. The bar contains the ",(0,a.kt)("strong",{parentName:"p"},"\u201cIn process\u201d")," section, where there will be only those patients with samples being currently processed; the ",(0,a.kt)("strong",{parentName:"p"},"\u201cProblems\u201d")," section, where there will be only those patients with problems during data processing. You can transfer irrelevant data or data for later deletion to the ",(0,a.kt)("strong",{parentName:"p"},"\u201cArchive\u201d")," section."),(0,a.kt)("p",null,"Patients can be grouped into ",(0,a.kt)("strong",{parentName:"p"},"cohorts"),", for example, by type of pathology or data source. By selecting a cohort in the navigation bar on the left, you can see only patients belonging to that cohort."),(0,a.kt)("p",null,"Various global configuration options can be accessed in the ",(0,a.kt)("strong",{parentName:"p"},"\u201cSettings\u201d")," section. Here you can add sets of genes, which you can later use to filter the analysis results (",(0,a.kt)("strong",{parentName:"p"},"\u201cGene panels\u201d")," section). When creating a panel, you can select genes from a list of all genes, or import the prepared gene list as text (for example TP53, BRCA1, EGFR). Alternatively, you can select a panel from a library of the most common clinical panels and the genes will be imported from there automatically. This can significantly speed up further data interpretation if you are only interested in a narrow range of genes within a specific pathology or group of pathologies."),(0,a.kt)("p",null,"Often the data analyzed is prepared by targeted sequencing (WES,\ndisease-specific gene panels, etc.).  NGS Wizard attempts to detect if\none of the widely adopted commercial kits was used. However, if it's not\nthe case or detection fails, you can upload sequenced\nregions (BED file) in the ",(0,a.kt)("strong",{parentName:"p"},"\u201cCapture kits\u201d")," section (make sure\nthat you have correctly specified the assembly of the reference genome\nwhen uploading the BED file)."))}g.isMDXComponent=!0},6971:function(e,t,n){t.Z=n.p+"assets/images/1-all-patients-page-783def9280a41b6da5269fb02c285e13.png"}}]);