"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[6590],{3905:function(e,r,t){t.d(r,{Zo:function(){return u},kt:function(){return f}});var n=t(67294);function o(e,r,t){return r in e?Object.defineProperty(e,r,{value:t,enumerable:!0,configurable:!0,writable:!0}):e[r]=t,e}function i(e,r){var t=Object.keys(e);if(Object.getOwnPropertySymbols){var n=Object.getOwnPropertySymbols(e);r&&(n=n.filter((function(r){return Object.getOwnPropertyDescriptor(e,r).enumerable}))),t.push.apply(t,n)}return t}function a(e){for(var r=1;r<arguments.length;r++){var t=null!=arguments[r]?arguments[r]:{};r%2?i(Object(t),!0).forEach((function(r){o(e,r,t[r])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(t)):i(Object(t)).forEach((function(r){Object.defineProperty(e,r,Object.getOwnPropertyDescriptor(t,r))}))}return e}function s(e,r){if(null==e)return{};var t,n,o=function(e,r){if(null==e)return{};var t,n,o={},i=Object.keys(e);for(n=0;n<i.length;n++)t=i[n],r.indexOf(t)>=0||(o[t]=e[t]);return o}(e,r);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(n=0;n<i.length;n++)t=i[n],r.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(e,t)&&(o[t]=e[t])}return o}var c=n.createContext({}),l=function(e){var r=n.useContext(c),t=r;return e&&(t="function"==typeof e?e(r):a(a({},r),e)),t},u=function(e){var r=l(e.components);return n.createElement(c.Provider,{value:r},e.children)},p={inlineCode:"code",wrapper:function(e){var r=e.children;return n.createElement(n.Fragment,{},r)}},d=n.forwardRef((function(e,r){var t=e.components,o=e.mdxType,i=e.originalType,c=e.parentName,u=s(e,["components","mdxType","originalType","parentName"]),d=l(t),f=o,g=d["".concat(c,".").concat(f)]||d[f]||p[f]||i;return t?n.createElement(g,a(a({ref:r},u),{},{components:t})):n.createElement(g,a({ref:r},u))}));function f(e,r){var t=arguments,o=r&&r.mdxType;if("string"==typeof e||o){var i=t.length,a=new Array(i);a[0]=d;var s={};for(var c in r)hasOwnProperty.call(r,c)&&(s[c]=r[c]);s.originalType=e,s.mdxType="string"==typeof e?e:o,a[1]=s;for(var l=2;l<i;l++)a[l]=t[l];return n.createElement.apply(null,a)}return n.createElement.apply(null,t)}d.displayName="MDXCreateElement"},8441:function(e,r,t){t.r(r),t.d(r,{frontMatter:function(){return s},contentTitle:function(){return c},metadata:function(){return l},toc:function(){return u},default:function(){return d}});var n=t(87462),o=t(63366),i=(t(67294),t(3905)),a=["components"],s={sidebar_position:5,sidebar_label:"Variant Discovery Preprocessing"},c="Variant discovery preprocessing",l={unversionedId:"results/workflow_details/variant-discovery-preprocessing",id:"results/workflow_details/variant-discovery-preprocessing",isDocsHomePage:!1,title:"Variant discovery preprocessing",description:"For the most accurate variant discovery, the alignment results are going through several stages of additional",source:"@site/docs/results/workflow_details/variant-discovery-preprocessing.mdx",sourceDirName:"results/workflow_details",slug:"/results/workflow_details/variant-discovery-preprocessing",permalink:"/ngsw-docs-en/results/workflow_details/variant-discovery-preprocessing",tags:[],version:"current",sidebarPosition:5,frontMatter:{sidebar_position:5,sidebar_label:"Variant Discovery Preprocessing"},sidebar:"tutorialSidebar",previous:{title:"Alignment",permalink:"/ngsw-docs-en/results/workflow_details/alignment"},next:{title:"Copy Number Variations Discovery",permalink:"/ngsw-docs-en/results/workflow_details/cnv-discovery"}},u=[{value:"1. Mark duplicates",id:"1-mark-duplicates",children:[]},{value:"2. Base recalibrate",id:"2-base-recalibrate",children:[]}],p={toc:u};function d(e){var r=e.components,t=(0,o.Z)(e,a);return(0,i.kt)("wrapper",(0,n.Z)({},p,t,{components:r,mdxType:"MDXLayout"}),(0,i.kt)("h1",{id:"variant-discovery-preprocessing"},"Variant discovery preprocessing"),(0,i.kt)("p",null,"For the most accurate variant discovery, the alignment results are going through several stages of additional\nprocessing. These steps are performed using ",(0,i.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us"},"GATK")," tools."),(0,i.kt)("h2",{id:"1-mark-duplicates"},"1. Mark duplicates"),(0,i.kt)("p",null,"During sequencing, the amount of the original fragmented DNA sequences increases in the polymerase chain reaction.\nSince the copy number count is based on the read count, it is very important to determine which of the reads come\nfrom the original sequences and which are copies. Further, only original reads are counted."),(0,i.kt)("h2",{id:"2-base-recalibrate"},"2. Base recalibrate"),(0,i.kt)("p",null,"The sequencer assigns a quality to each nucleotide, which is the logarithm of the probability that the nucleotide\nwas correctly determined during sequencing, but many sequencers tend to underestimate the sequencing quality and\nother errors. Computational methods make it possible to reduce the error, which further improves the quality of\nvariant discovery."))}d.isMDXComponent=!0}}]);