"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[2920],{3905:function(e,t,n){n.d(t,{Zo:function(){return d},kt:function(){return h}});var a=n(67294);function i(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function r(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function o(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?r(Object(n),!0).forEach((function(t){i(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function l(e,t){if(null==e)return{};var n,a,i=function(e,t){if(null==e)return{};var n,a,i={},r=Object.keys(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||(i[n]=e[n]);return i}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(i[n]=e[n])}return i}var s=a.createContext({}),p=function(e){var t=a.useContext(s),n=t;return e&&(n="function"==typeof e?e(t):o(o({},t),e)),n},d=function(e){var t=p(e.components);return a.createElement(s.Provider,{value:t},e.children)},m={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},c=a.forwardRef((function(e,t){var n=e.components,i=e.mdxType,r=e.originalType,s=e.parentName,d=l(e,["components","mdxType","originalType","parentName"]),c=p(n),h=i,f=c["".concat(s,".").concat(h)]||c[h]||m[h]||r;return n?a.createElement(f,o(o({ref:t},d),{},{components:n})):a.createElement(f,o({ref:t},d))}));function h(e,t){var n=arguments,i=t&&t.mdxType;if("string"==typeof e||i){var r=n.length,o=new Array(r);o[0]=c;var l={};for(var s in t)hasOwnProperty.call(t,s)&&(l[s]=t[s]);l.originalType=e,l.mdxType="string"==typeof e?e:i,o[1]=l;for(var p=2;p<r;p++)o[p]=n[p];return a.createElement.apply(null,o)}return a.createElement.apply(null,n)}c.displayName="MDXCreateElement"},30119:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return l},contentTitle:function(){return s},metadata:function(){return p},toc:function(){return d},default:function(){return c}});var a=n(87462),i=n(63366),r=(n(67294),n(3905)),o=["components"],l={sidebar_position:3,sidebar_label:"Annotation"},s="Annotation",p={unversionedId:"results/workflow-details/vcf-analysis/annotation",id:"results/workflow-details/vcf-analysis/annotation",isDocsHomePage:!1,title:"Annotation",description:"After successful upload, identification and verification, the sample analysis continues with the main stage of the",source:"@site/docs/results/workflow-details/vcf-analysis/annotation.mdx",sourceDirName:"results/workflow-details/vcf-analysis",slug:"/results/workflow-details/vcf-analysis/annotation",permalink:"/ngsw-docs-en/results/workflow-details/vcf-analysis/annotation",tags:[],version:"current",sidebarPosition:3,frontMatter:{sidebar_position:3,sidebar_label:"Annotation"},sidebar:"tutorialSidebar",previous:{title:"Upload, identify and verify",permalink:"/ngsw-docs-en/results/workflow-details/vcf-analysis/upload-identify-verify"},next:{title:"Phenotype prediction",permalink:"/ngsw-docs-en/results/workflow-details/vcf-analysis/phenotypes"}},d=[],m={toc:d};function c(e){var t=e.components,l=(0,i.Z)(e,o);return(0,r.kt)("wrapper",(0,a.Z)({},m,l,{components:t,mdxType:"MDXLayout"}),(0,r.kt)("h1",{id:"annotation"},"Annotation"),(0,r.kt)("p",null,'After successful upload, identification and verification, the sample analysis continues with the main stage of the\nanalysis, such as annotation. If the uploaded annotation contains SNVs/Indels, the "Germline SNVs/Indels annotation"\nstage (if the sample was uploaded as a non-tumor sample) or "Somatic SNVs/Indels annotation" stage (if the sample\nwas uploaded as a tumor sample) is launched. If SNVs/Indels are not found in the uploaded annotation, but there are\nstructural variations, the "Germline SV annotation" stage (if the sample was uploaded as a non-tumor sample) or\n"Somatic SV annotation" stage (if the sample was uploaded as a tumor sample) is launched. If any of the tasks\nlisted below fail, the sample analysis is stopped.'),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(49695).Z,width:"1000"})),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},'Tasks of the "SNVs/Indels annotation" stage:')),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Unify SNVs/Indels"),":")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Sorting a VCF file using ",(0,r.kt)("a",{parentName:"li",href:"https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html"},"GNU sort"),"\nby chromosome names."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://samtools.github.io/bcftools/bcftools.html#reheader"},"bcftools reheader")," modifies header of VCF file and\nchanges sample names."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037428851-FixVcfHeader-Picard-"},"GATK FixVCFHeader"),"\nreplaces or fixes a VCF header."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://samtools.github.io/bcftools/bcftools.html#norm"},"bcftools norm")," checks if reference alleles\nin the file match the reference; splits multiallelic sites into biallelic records; outputs only the first instance\nif a record is present multiple times."),(0,r.kt)("li",{parentName:"ul"},"Compressing a file into a GZIP archive using ",(0,r.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/bgzip.html"},"bgzip"),'. The resulting file\ncan be downloaded in the "',(0,r.kt)("em",{parentName:"li"},"Result files"),'" section in the "',(0,r.kt)("em",{parentName:"li"},"Unify SNVs/Indels"),'" task details ("',(0,r.kt)("em",{parentName:"li"},"Download VCF_GZ"),'").\nYou can also open this file in the IGV browser by clicking on the "',(0,r.kt)("em",{parentName:"li"},"Open in IGV Browser"),'" link.'),(0,r.kt)("li",{parentName:"ul"},"Indexing a file using ",(0,r.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/tabix.html"},"tabix"),'. The resulting index file can be downloaded\nin the same section ("',(0,r.kt)("em",{parentName:"li"},"Download VCF_TBI"),'").')),(0,r.kt)("p",null,"If the sample analysis workflow includes polygenic risk scores calculation and/or phenotypes prediction,\nthen ",(0,r.kt)("a",{parentName:"p",href:"/results/workflow-details/vcf-analysis/phenotypes"},'the "Phenotype prediction" stage'),' will be launched in\nparallel with annotation after the "Unify SNVs/Indels" task.'),(0,r.kt)("ol",{start:2},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Annotate SNVs/Indels"),":")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Annotation of SNVs/Indels in the file with data from the ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/refseq/"},"RefSeq"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://www.internationalgenome.org/"},"1000 Genomes"),", ",(0,r.kt)("a",{parentName:"p",href:"https://sites.google.com/site/jpopgen/dbNSFP"},"dbNSFP"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/snp/"},"dbSNP"),", ",(0,r.kt)("a",{parentName:"p",href:"https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/"},"gnomAD 3"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://gnomad.broadinstitute.org/"},"gnomAD 4"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),", ",(0,r.kt)("a",{parentName:"p",href:"https://cadd.gs.washington.edu/"},"CADD"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://spliceailookup.broadinstitute.org/"},"SpliceAI"),", ",(0,r.kt)("a",{parentName:"p",href:"https://enigmaconsortium.org/"},"ENIGMA")," databases.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Determination of the effect of SNVs/Indels on genes, transcripts, and protein sequence, as well as regulatory\nregions, using ",(0,r.kt)("a",{parentName:"p",href:"https://www.ensembl.org/info/docs/tools/vep/index.html"},"Ensembl Variant Effect Predictor (VEP)"),":"),(0,r.kt)("ul",{parentName:"li"},(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},(0,r.kt)("a",{parentName:"p",href:"http://genetics.bwh.harvard.edu/pph2/"},"PolyPhen")," predicts possible impact of an amino acid substitution on\nthe structure and function of a protein using straightforward physical and comparative considerations.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"A flag indicating if the transcript is the canonical transcript for the gene is added.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Allele number from VCF input is identified.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Affected exon and intron numbering is added.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},(0,r.kt)("a",{parentName:"p",href:"https://varnomen.hgvs.org/"},"HGVS nomenclature")," based on ",(0,r.kt)("a",{parentName:"p",href:"https://www.ensembl.org/index.html"},"Ensembl")," stable\nidentifiers is added.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Upstream variant consequences are determined if the upstream (5') distance between a variant and a transcript\nis more than 2000 bp. Downstream variant consequences are determined if the downstream (3') distance between\na variant and a transcript is more than 1000 bp."),(0,r.kt)("p",{parentName:"li"},'   The resulting file with raw annotated SNVs/Indels in TSV format can be downloaded in the "',(0,r.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,r.kt)("em",{parentName:"p"},"Annotate SNVs/Indels"),'" task details ("',(0,r.kt)("em",{parentName:"p"},"Download Raw annotated TSV"),'").\nYou can also open it in Google Sheets.',(0,r.kt)("br",null),'\nThe resulting file with annotated SNVs/Indels without duplicates in TSV format can be downloaded in the same section\n("',(0,r.kt)("em",{parentName:"p"},"Download All variants TSV"),'"). You can also open it in Google Sheets.\nThe same file in CSV format can be downloaded from the same place ("',(0,r.kt)("em",{parentName:"p"},"Download All variants CSV"),'").')))),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Converting TSV results to VCF.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},(0,r.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037428851-FixVcfHeader-Picard-"},"GATK FixVCFHeader"),"\nreplaces or fixes a VCF header.")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"The file is compressed into a GZIP archive using ",(0,r.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/bgzip.html"},"bgzip"),'.\nThe resulting file can be downloaded in the "',(0,r.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,r.kt)("em",{parentName:"p"},"Annotate SNVs/Indels"),'" task details ("',(0,r.kt)("em",{parentName:"p"},"Download All variants VCF_GZ"),'").\nYou can also open this file in the IGV browser by clicking on the "',(0,r.kt)("em",{parentName:"p"},"Open in IGV Browser"),'" link.')),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"The file is indexed using ",(0,r.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/tabix.html"},"tabix"),'.\nThe resulting index file can be downloaded in the same section ("',(0,r.kt)("em",{parentName:"p"},"Download All variants VCF_TBI"),'").')),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("p",{parentName:"li"},"Calculating variants' statistic."))),(0,r.kt)("ol",{start:3},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Store annotated variants for SNV Viewer"),":\nsaving results for display in ",(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/variant-viewer"},"SNV Viewer"),", i.e. an embedded service\nfor viewing and analyzing variants, and adding information\nabout ",(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-page#occurrences"},"the variant occurrence")," in other samples of the user.")),(0,r.kt)("p",null,'After the "Annotate SNVs/Indels" stage, the analysis can proceed\nwith ',(0,r.kt)("a",{parentName:"p",href:"/results/workflow-details/vcf-analysis/report-generation"},"report generation"),"."),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},'The "SV annotation" stage')," includes one task ",(0,r.kt)("strong",{parentName:"p"},'"Annotate SV VCF"'),":"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Annotation and ranking of structural variations using ",(0,r.kt)("a",{parentName:"li",href:"https://lbgi.fr/AnnotSV/"},"AnnotSV"),".",(0,r.kt)("br",null),'\nThe resulting file with structural variations in TSV format can be downloaded in the "',(0,r.kt)("em",{parentName:"li"},"Result files"),'" section\nin the "',(0,r.kt)("em",{parentName:"li"},"Annotate SV VCF"),'" task details ("',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants TSV"),'").\nYou can also open it in Google Sheets. The same file in CSV format can be downloaded from the same place\n("',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants CSV"),'").'),(0,r.kt)("li",{parentName:"ul"},"Compressing a file into a GZIP archive using ",(0,r.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/bgzip.html"},"bgzip"),'. The resulting file\ncan be downloaded in the same section ("',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants VCF_GZ"),'").\nYou can also open this file in the IGV browser by clicking on the "',(0,r.kt)("em",{parentName:"li"},"Open in IGV Browser"),'" link.'),(0,r.kt)("li",{parentName:"ul"},"Indexing a file using ",(0,r.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/tabix.html"},"tabix"),'. The resulting index file can be downloaded\nin the same section ("',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants VCF_TBI"),'").')),(0,r.kt)("p",null,'The "SV annotation" stage completes the sample analysis.'),(0,r.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"If you want to add a track with SNVs/Indels or structural variations discovered by the analysis of the sample you\nuploaded to Genomenal to your ",(0,r.kt)("a",{parentName:"p",href:"https://igv.org/doc/desktop/"},"desktop IGV"),', you can do so via a link.\nOpen the details of the necessary task ("',(0,r.kt)("em",{parentName:"p"},"Unify SNVs/Indels"),'", "',(0,r.kt)("em",{parentName:"p"},"Annotate SNVs/Indels"),'", "',(0,r.kt)("em",{parentName:"p"},"Annotate SV VCF"),'")\nand do the following:'),(0,r.kt)("ol",{parentName:"div"},(0,r.kt)("li",{parentName:"ol"},'Right-click the variant file link (depending on the selected task, this may be the link "',(0,r.kt)("em",{parentName:"li"},"Download VCF_GZ"),'",\n"',(0,r.kt)("em",{parentName:"li"},"Download All variants VCF_GZ"),'" or "',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants VCF_GZ"),'") and select the "',(0,r.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,r.kt)("li",{parentName:"ol"},"Upload the track via URL to your desktop IGV, as\ndescribed ",(0,r.kt)("a",{parentName:"li",href:"https://igv.org/doc/desktop/#UserGuide/loading_data/"},"here"),"."),(0,r.kt)("li",{parentName:"ol"},'Right-click on the download link for the index file corresponding to the annotation file ("',(0,r.kt)("em",{parentName:"li"},"Download VCF_TBI"),'",\n"',(0,r.kt)("em",{parentName:"li"},"Download All variants VCF_TBI"),'" or "',(0,r.kt)("em",{parentName:"li"},"Download Filtered variants VCF_TBI"),'")\nand select the "',(0,r.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,r.kt)("li",{parentName:"ol"},"Add the index via URL to the corresponding field in IGV."),(0,r.kt)("li",{parentName:"ol"},'Click "',(0,r.kt)("em",{parentName:"li"},"OK"),'". Done! The track with SNVs/Indels or structural variations discovered in the sample is added to IGV.')))))}c.isMDXComponent=!0},49695:function(e,t,n){t.Z=n.p+"assets/images/vcf_annotation-152f76a570f6958775422fdf049027de.png"}}]);