"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[9301],{3905:function(e,t,n){n.d(t,{Zo:function(){return d},kt:function(){return m}});var a=n(67294);function r(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function i(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function o(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?i(Object(n),!0).forEach((function(t){r(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):i(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,a,r=function(e,t){if(null==e)return{};var n,a,r={},i=Object.keys(e);for(a=0;a<i.length;a++)n=i[a],t.indexOf(n)>=0||(r[n]=e[n]);return r}(e,t);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(a=0;a<i.length;a++)n=i[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(r[n]=e[n])}return r}var l=a.createContext({}),c=function(e){var t=a.useContext(l),n=t;return e&&(n="function"==typeof e?e(t):o(o({},t),e)),n},d=function(e){var t=c(e.components);return a.createElement(l.Provider,{value:t},e.children)},p={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},u=a.forwardRef((function(e,t){var n=e.components,r=e.mdxType,i=e.originalType,l=e.parentName,d=s(e,["components","mdxType","originalType","parentName"]),u=c(n),m=r,f=u["".concat(l,".").concat(m)]||u[m]||p[m]||i;return n?a.createElement(f,o(o({ref:t},d),{},{components:n})):a.createElement(f,o({ref:t},d))}));function m(e,t){var n=arguments,r=t&&t.mdxType;if("string"==typeof e||r){var i=n.length,o=new Array(i);o[0]=u;var s={};for(var l in t)hasOwnProperty.call(t,l)&&(s[l]=t[l]);s.originalType=e,s.mdxType="string"==typeof e?e:r,o[1]=s;for(var c=2;c<i;c++)o[c]=n[c];return a.createElement.apply(null,o)}return a.createElement.apply(null,n)}u.displayName="MDXCreateElement"},90453:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return s},contentTitle:function(){return l},metadata:function(){return c},toc:function(){return d},default:function(){return u}});var a=n(87462),r=n(63366),i=(n(67294),n(3905)),o=["components"],s={sidebar_position:9,sidebar_label:"Structural variations discovery"},l="Structural variations discovery",c={unversionedId:"results/workflow-details/fastq-analysis/structural-variations",id:"results/workflow-details/fastq-analysis/structural-variations",isDocsHomePage:!1,title:"Structural variations discovery",description:'After the "Variant discovery preprocessing" stage',source:"@site/docs/results/workflow-details/fastq-analysis/structural-variations.mdx",sourceDirName:"results/workflow-details/fastq-analysis",slug:"/results/workflow-details/fastq-analysis/structural-variations",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations",tags:[],version:"current",sidebarPosition:9,frontMatter:{sidebar_position:9,sidebar_label:"Structural variations discovery"},sidebar:"tutorialSidebar",previous:{title:"Germline SNVs/Indels discovery and annotation",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels"},next:{title:"Copy number variations discovery",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery"}},d=[{value:"1. SV Discovery",id:"structural-variation-discovery",children:[]},{value:"2. Annotate SV VCF",id:"annotation",children:[]}],p={toc:d};function u(e){var t=e.components,s=(0,r.Z)(e,o);return(0,i.kt)("wrapper",(0,a.Z)({},p,s,{components:t,mdxType:"MDXLayout"}),(0,i.kt)("h1",{id:"structural-variations-discovery"},"Structural variations discovery"),(0,i.kt)("p",null,"After ",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"},'the "Variant discovery preprocessing" stage'),"\nhas been successfully completed, structural variations (SVs) discovery can be initiated for the samples if the stage\nis ",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#sv-discovery"},"included")," in the analysis workflow.\nGermline structural variations will be discovered for a non-tumor sample and a single tumor sample, and\nsomatic structural variations will be discovered for a tumor sample from the tumor/control sample set."),(0,i.kt)("p",{align:"center"},(0,i.kt)("img",{src:n(40426).Z,width:"650"})),(0,i.kt)("h3",{id:"structural-variation-discovery"},"1. SV Discovery"),(0,i.kt)("p",null,"Structural variations calling is performed using the ",(0,i.kt)("a",{parentName:"p",href:"https://github.com/dellytools/delly"},"Delly")," tool.\nDelly is an integrated structural variation prediction method that can discover, genotype and visualize deletions,\ntandem duplications, inversions and translocations at single-nucleotide resolution in short-read and long-read\nmassively parallel sequencing data. When calling structural variations, the telomere and centromere regions of\nchromosomes are not considered, since these repetitive regions cannot be analyzed using short-read data, as well\nas unplaced contigs."),(0,i.kt)("p",null,"To call somatic structural variations, at least one tumor sample and a matching normal sample are required.\nTherefore, somatic structural variations discovery is performed only in a tumor sample from a tumor/control\nsample set. In a single tumor sample, only germline structural variations are discovered."),(0,i.kt)("p",null,'Using the sample genome alignment (or the alignment of the tumor and non-tumor sample genomes in case of\nsomatic structural variations discovery in the tumor sample from the tumor/control sample set), Delly\ncalculates structural variations and outputs them as a BCF (binary variant call format) file, i.e. a binary\nversion of VCF. The resulting file can be downloaded in the "',(0,i.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,i.kt)("em",{parentName:"p"},"SV Discovery"),'"\ntask details ("',(0,i.kt)("em",{parentName:"p"},"Download BCF"),'"). The index file corresponding to this file can be downloaded in the same section\n("',(0,i.kt)("em",{parentName:"p"},"Download CSI"),'").'),(0,i.kt)("p",null,"View, subset and filter VCF or BCF files by position and filtering expression. Convert between VCF and BCF. Former bcftools subset."),(0,i.kt)("p",null,"The resulting BCF file is then converted to a VCF file\nusing ",(0,i.kt)("a",{parentName:"p",href:"https://samtools.github.io/bcftools/bcftools.html#view"},"bcftools view"),". The VCF file is then compressed\ninto a GZIP archive using ",(0,i.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/bgzip.html"},"bgzip"),'. The resulting file can be downloaded\nin the "',(0,i.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,i.kt)("em",{parentName:"p"},"SV Discovery"),'" task details ("',(0,i.kt)("em",{parentName:"p"},"Download VCF_GZ"),'"). You can also open\nthis file in the IGV browser by clicking on the "',(0,i.kt)("em",{parentName:"p"},"Open in IGV Browser"),'" link. In addition, the VCF file is indexed\nusing ',(0,i.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/tabix.html"},"tabix"),'. The resulting index file can be downloaded in the same\nsection ("',(0,i.kt)("em",{parentName:"p"},"Download VCF_TBI"),'").'),(0,i.kt)("h3",{id:"annotation"},"2. Annotate SV VCF"),(0,i.kt)("p",null,"Annotation and ranking of discovered structural variations is performed using the ",(0,i.kt)("a",{parentName:"p",href:"https://lbgi.fr/AnnotSV/"},"AnnotSV"),"\nprogram. This tool compiles functionally, regulatory and clinically relevant information and aims at providing\nannotations useful to interpret SV potential pathogenicity and filter out SV potential false positives."),(0,i.kt)("ul",null,(0,i.kt)("li",{parentName:"ul"},'The resulting file with annotated structural variations in TSV format can be downloaded\nin the "',(0,i.kt)("em",{parentName:"li"},"Result files"),'" section in the "',(0,i.kt)("em",{parentName:"li"},"Annotate SV VCF"),'" task details ("',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants TSV"),'").\nYou can also open it in Google Sheets. The same file in CSV format can be downloaded from the same\nsection ("',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants CSV"),'").'),(0,i.kt)("li",{parentName:"ul"},"The resulting file with annotated structural variations in VCF format is compressed\ninto a GZIP archive using ",(0,i.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/bgzip.html"},"bgzip"),'. It can be downloaded\nin the same section ("',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants VCF_GZ"),'"). You can also open this file in the IGV browser\nby clicking on the "',(0,i.kt)("em",{parentName:"li"},"Open in IGV Browser"),'" link. In addition, the VCF file is indexed\nusing ',(0,i.kt)("a",{parentName:"li",href:"http://www.htslib.org/doc/tabix.html"},"tabix"),'. The resulting index file can be downloaded in the same\nsection ("',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants VCF_TBI"),'").')),(0,i.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,i.kt)("div",{parentName:"div",className:"admonition-heading"},(0,i.kt)("h5",{parentName:"div"},(0,i.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,i.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,i.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,i.kt)("div",{parentName:"div",className:"admonition-content"},(0,i.kt)("p",{parentName:"div"},"If you want to add a track with structural variations discovered by the analysis of the sample you uploaded to\nGenomenal to your ",(0,i.kt)("a",{parentName:"p",href:"https://igv.org/doc/desktop/"},"desktop IGV"),', you can do so via a link.\nOpen the details of the necessary task ("',(0,i.kt)("em",{parentName:"p"},"SV Discovery"),'" or "',(0,i.kt)("em",{parentName:"p"},"Annotate SV VCF"),'") and do the following:'),(0,i.kt)("ol",{parentName:"div"},(0,i.kt)("li",{parentName:"ol"},'Right-click the variation file link (depending on the selected task, this may be the link "',(0,i.kt)("em",{parentName:"li"},"Download VCF_GZ"),'"\nor "',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants VCF_GZ"),'") and select the "',(0,i.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,i.kt)("li",{parentName:"ol"},"Upload the track via URL to your desktop IGV, as\ndescribed ",(0,i.kt)("a",{parentName:"li",href:"https://igv.org/doc/desktop/#UserGuide/loading_data/"},"here"),"."),(0,i.kt)("li",{parentName:"ol"},'Right-click on the download link for the index file corresponding to the annotation file ("',(0,i.kt)("em",{parentName:"li"},"Download VCF_TBI"),'"\nor "',(0,i.kt)("em",{parentName:"li"},"Download Filtered variants VCF_TBI"),'") and select the "',(0,i.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,i.kt)("li",{parentName:"ol"},"Add the index via URL to the corresponding field in IGV."),(0,i.kt)("li",{parentName:"ol"},'Click "',(0,i.kt)("em",{parentName:"li"},"OK"),'". Done! The track with structural variations discovered in the sample is added to IGV.')))),(0,i.kt)("p",null,'After the "Structural variations discovery" stage, the analysis can proceed\nwith ',(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/report-generation"},"report generation"),"."))}u.isMDXComponent=!0},40426:function(e,t,n){t.Z=n.p+"assets/images/structural_variants-6ff70e47381a575ec8c9c6bb66262313.png"}}]);