"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[101],{3905:function(e,t,n){n.d(t,{Zo:function(){return c},kt:function(){return h}});var a=n(67294);function i(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function r(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function s(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?r(Object(n),!0).forEach((function(t){i(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function l(e,t){if(null==e)return{};var n,a,i=function(e,t){if(null==e)return{};var n,a,i={},r=Object.keys(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||(i[n]=e[n]);return i}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(i[n]=e[n])}return i}var o=a.createContext({}),p=function(e){var t=a.useContext(o),n=t;return e&&(n="function"==typeof e?e(t):s(s({},t),e)),n},c=function(e){var t=p(e.components);return a.createElement(o.Provider,{value:t},e.children)},m={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},d=a.forwardRef((function(e,t){var n=e.components,i=e.mdxType,r=e.originalType,o=e.parentName,c=l(e,["components","mdxType","originalType","parentName"]),d=p(n),h=i,f=d["".concat(o,".").concat(h)]||d[h]||m[h]||r;return n?a.createElement(f,s(s({ref:t},c),{},{components:n})):a.createElement(f,s({ref:t},c))}));function h(e,t){var n=arguments,i=t&&t.mdxType;if("string"==typeof e||i){var r=n.length,s=new Array(r);s[0]=d;var l={};for(var o in t)hasOwnProperty.call(t,o)&&(l[o]=t[o]);l.originalType=e,l.mdxType="string"==typeof e?e:i,s[1]=l;for(var p=2;p<r;p++)s[p]=n[p];return a.createElement.apply(null,s)}return a.createElement.apply(null,n)}d.displayName="MDXCreateElement"},13169:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return l},contentTitle:function(){return o},metadata:function(){return p},toc:function(){return c},default:function(){return d}});var a=n(87462),i=n(63366),r=(n(67294),n(3905)),s=["components"],l={sidebar_position:4,sidebar_label:"Alignment"},o="Alignment",p={unversionedId:"results/workflow-details/fastq-analysis/alignment",id:"results/workflow-details/fastq-analysis/alignment",isDocsHomePage:!1,title:"Alignment",description:'After the "Check quality and cleanup" stage',source:"@site/docs/results/workflow-details/fastq-analysis/alignment.mdx",sourceDirName:"results/workflow-details/fastq-analysis",slug:"/results/workflow-details/fastq-analysis/alignment",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment",tags:[],version:"current",sidebarPosition:4,frontMatter:{sidebar_position:4,sidebar_label:"Alignment"},sidebar:"tutorialSidebar",previous:{title:"Check quality and cleanup",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup"},next:{title:"Variant discovery preprocessing",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"}},c=[],m={toc:c};function d(e){var t=e.components,l=(0,i.Z)(e,s);return(0,r.kt)("wrapper",(0,a.Z)({},m,l,{components:t,mdxType:"MDXLayout"}),(0,r.kt)("h1",{id:"alignment"},"Alignment"),(0,r.kt)("p",null,"After the ",(0,r.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/check-quality-and-cleanup"},'"Check quality and cleanup"')," stage\nhas been successfully completed, further sample analysis (SNVs/Indels, structural variations, or copy number\nvariations discovery, as well as phenotype prediction) requires alignment (mapping) of reads to the reference\ngenome. ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.40/"},"GRCh38.p14"),", the most recent version\nof the human genome is used for alignment. If alignment\nis ",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/alignment-settings#disable-alignment"},"excluded"),' from the workflow, the analysis is\ncompleted after the "Upload, identify and verify" and "Check quality and cleanup" stages have been successfully\ncompleted. If any of the tasks listed below fails, the sample analysis is stopped.'),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(74514).Z,width:"650"})),(0,r.kt)("p",null,'The "Alignment" stage of sample analysis may include the following tasks:'),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Re-pair")," is launched if the following conditions are met:",(0,r.kt)("ul",{parentName:"li"},(0,r.kt)("li",{parentName:"ul"},"A paired-end sequencing sample is being analyzed (i.e. sequencing in which both ends of a DNA fragment are\nsequenced);"),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"/settings/settings/alignment-settings#disable-repair"},'"Ensure 1st and 2nd read file consistency" parameter')," is\nenabled and/or the number of reads in the paired files of the sample does not match.")))),(0,r.kt)("p",null,(0,r.kt)("a",{parentName:"p",href:"https://github.com/BioInfoTools/BBMap/blob/master/sh/repair.sh"},"BBMap Repair")," is used to re-pair reads.\nIt includes the following:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"Sorting lines in files."),(0,r.kt)("li",{parentName:"ul"},'Restoring the order in paired files: the first read in the first file must match the first read in the\nsecond file, etc. The order is restored by analyzing the read names, which must either be in Illumina format\n(an identical prefix followed by "1:" and "2:" or "/1" and "/2") or completely identical for both reads in a pair.'),(0,r.kt)("li",{parentName:"ul"},"Restoring paired reads, which is necessary for alignment, so that each read from one file contains a pair\nin the other. Paired reads can be broken during cleanup, which leads to the forming of singletons (a singleton\nis a read with a sequence that is present exactly once, i.e. is unique among the reads) that have lost their\npair in the paired file. By default, singletons are removed from the analysis, but they can be used for genome\nalignment if ",(0,r.kt)("a",{parentName:"li",href:"/settings/settings/alignment-settings#use-singleton-reads"},"the corresponding parameter")," is enabled.")),(0,r.kt)("ol",{start:2},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Alignment")," is the process of matching sequence reads to a reference sequence (the version hg38 of the reference\ngenome is used). By default, the alignment is performed using the community-recommended\ntool ",(0,r.kt)("a",{parentName:"li",href:"https://github.com/bwa-mem2/bwa-mem2"},"BWA2 Burrows-Wheeler Aligner")," (BWA-MEM2), which is the latest and\nfaster version of the BWA-MEM algorithm, which is based on backward search with Burrows\u2013Wheeler Transform (BWT)\nto efficiently align short sequencing reads against a human genome, allowing mismatches and gaps",(0,r.kt)("sup",{parentName:"li",id:"fnref-1"},(0,r.kt)("a",{parentName:"sup",href:"#fn-1",className:"footnote-ref"},"1")),".\nThe aligner can be ",(0,r.kt)("a",{parentName:"li",href:"/settings/settings/alignment-settings#change-alignment-tool"},"changed")," in the parameters\nto ",(0,r.kt)("em",{parentName:"li"},"BWA Burrows-Wheeler Aligner")," (i.e. an earlier version of the BWA-MEM2 algorithm",(0,r.kt)("sup",{parentName:"li",id:"fnref-2"},(0,r.kt)("a",{parentName:"sup",href:"#fn-2",className:"footnote-ref"},"2")),"), ",(0,r.kt)("em",{parentName:"li"},"Bowtie2"),(0,r.kt)("sup",{parentName:"li",id:"fnref-3"},(0,r.kt)("a",{parentName:"sup",href:"#fn-3",className:"footnote-ref"},"3")),", ",(0,r.kt)("em",{parentName:"li"},"HISAT2"),(0,r.kt)("sup",{parentName:"li",id:"fnref-4"},(0,r.kt)("a",{parentName:"sup",href:"#fn-4",className:"footnote-ref"},"4")),"\nor ",(0,r.kt)("em",{parentName:"li"},"STAR"),(0,r.kt)("sup",{parentName:"li",id:"fnref-5"},(0,r.kt)("a",{parentName:"sup",href:"#fn-5",className:"footnote-ref"},"5")),". The alignment is written to a file in SAM format (i.e. a text-based format for storing biological\nsequences aligned to a reference sequence).")),(0,r.kt)("p",null,"After alignment, the mapped reads are sorted by leftmost coordinates\nusing ",(0,r.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/samtools-sort.html"},"samtools sort"),' and written to a file in BAM format\n(i.e. a binary form of the SAM format, which uses less disk space and is faster to process than SAM).\nYou can download the resulting file in the "',(0,r.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,r.kt)("em",{parentName:"p"},"Alignment"),'" task\ndetails ("',(0,r.kt)("em",{parentName:"p"},"Download BAM"),'"). You can also open this file in the IGV browser by clicking on the\n"',(0,r.kt)("em",{parentName:"p"},"Open in IGV Browser"),'" link. The BAM file is indexed\nusing ',(0,r.kt)("a",{parentName:"p",href:"http://www.htslib.org/doc/samtools-index.html"},"samtools index"),'. The resulting index file can be\ndownloaded in the same section ("',(0,r.kt)("em",{parentName:"p"},"Download BAI"),'").'),(0,r.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"If you want to add the alignment track obtained from your sample analysis in Genomenal to\nyour ",(0,r.kt)("a",{parentName:"p",href:"https://igv.org/doc/desktop/"},"desktop IGV"),", you can do so via a link. To do this, follow these steps:"),(0,r.kt)("ol",{parentName:"div"},(0,r.kt)("li",{parentName:"ol"},'Right-click the alignment file link "',(0,r.kt)("em",{parentName:"li"},"Download BAM"),'" and select the "',(0,r.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,r.kt)("li",{parentName:"ol"},"Upload the track via URL to your desktop IGV, as\ndescribed ",(0,r.kt)("a",{parentName:"li",href:"https://igv.org/doc/desktop/#UserGuide/loading_data/"},"here"),"."),(0,r.kt)("li",{parentName:"ol"},'Right-click the alignment index file link "',(0,r.kt)("em",{parentName:"li"},"Download BAI"),'" and select the "',(0,r.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,r.kt)("li",{parentName:"ol"},"Add the index via URL to the corresponding field in IGV."),(0,r.kt)("li",{parentName:"ol"},'Click "',(0,r.kt)("em",{parentName:"li"},"OK"),'". Done! The sample alignment track is added to IGV.')))),(0,r.kt)("p",null,'The "',(0,r.kt)("em",{parentName:"p"},"Metrics"),'" section in the "',(0,r.kt)("em",{parentName:"p"},"Alignment"),'" task details provides metrics for assessing the quality of alignment:'),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(3201).Z,width:"900"})),(0,r.kt)("p",null,"For each metric, the following are provided:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"metric name;"),(0,r.kt)("li",{parentName:"ul"},"description of the result of checking the compliance of a certain sample alignment indicator and the threshold\nof this metric: the indicator value for the sample (e.g. Mapped reads, i.e. the percent of mapped reads in the\nalignment file) and the used metric threshold, at which the alignment file is considered high-quality\n(can be changed in the ",(0,r.kt)("a",{parentName:"li",href:"/settings/settings/alignment-settings#change-metric-thresholds"},"parameters"),"), are provided;"),(0,r.kt)("li",{parentName:"ul"},"result of assessing the quality of alignment by\nmetric: ",(0,r.kt)("img",{src:n(57674).Z,width:"90"})," if the sample alignment satisfies the metric threshold, or ",(0,r.kt)("img",{src:n(55915).Z,width:"80"})," if it does not.")),(0,r.kt)("h4",{id:"alignment-metrics"},"The alignment quality metrics"),(0,r.kt)("table",null,(0,r.kt)("tr",null,(0,r.kt)("td",null,"Metric"),(0,r.kt)("td",null,"The metric threshold value at which the alignment file is considered high-quality (this is the default value, can be changed in ",(0,r.kt)("a",{href:"/settings/settings/alignment-settings#change-metric-thresholds"},"parameters"),")")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Mapped reads")),(0,r.kt)("td",null,"The alignment file contains at least 85% mapped reads.")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Multiple alignments")),(0,r.kt)("td",null,"The alignment file contains no more than 15% multiple alignments of the same read per genome.")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Forward/reverse balance")),(0,r.kt)("td",null,"The difference between forward and reverse strand reads is no more than 10% in the alignment file.")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Paired mapped reads")),(0,r.kt)("td",null,"The alignment file contains at least 80% paired mapped reads.")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Paired properly mapped reads")),(0,r.kt)("td",null,"The alignment file contains at least 75% paired properly mapped reads.")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"Coverage per nucleotide")),(0,r.kt)("td",null,"The minimum acceptable coverage per nucleotide is 0.1."))),(0,r.kt)("p",null,'If the metric value does not meet the threshold in the sample alignment file, the file is marked as not meeting\nthe metric criteria, i.e. the alignment is completed successfully, but the stage status indicates the number of\nmetrics that do not meet the criteria. If no sample reads are aligned to the genome, the "Alignment" stage completes\nwith an error and the analysis is stopped.'),(0,r.kt)("p",null,"If at least one of the following stages is included in the sample\nanalysis: ",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#somatic-snvs-discovery"},"somatic SNVs/Indels discovery"),",\n",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#germline-snvs-discovery"},"germline SNVs/Indels discovery"),",\n",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#sv-discovery"},"structural variations discovery"),",\n",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#cnv-discovery"},"copy number variations discovery"),",\n",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#enable-phenotype-prediction"},"phenotypes prediction")," or\n",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#polygenic-risk-scores-calculation"},"polygenic risk scores calculation"),',\nthen after the "Alignment" task is successfully completed,\n',(0,r.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"},'the "Variant discovery preprocessing" stage'),"\nwill start."),(0,r.kt)("ol",{start:3},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Determine Kit"))),(0,r.kt)("p",null,"If the user has defined the sample sequencing type as WGS (whole-genome sequencing), then the capture kit\ndetermination is not performed, and the sequencing type can remain WGS or be redefined as Low-pass WGS (whole genome\nsequencing with low coverage). WGS and Low-pass WGS are differentiated by the average sample coverage: the\nsequencing type of samples with an average coverage less than 4x is defined as Low-pass WGS, and above or\nequal to 4x as WGS."),(0,r.kt)("p",null,"If the user has defined the sample sequencing type as Targeted selection, the sequencing type can be defined as\nPanel (sequencing using a targeted panel) or WES (whole-exome sequencing). Next, the most suitable capture kit\n(i.e. the targeted panel used in targeted selection) for the sample is determined or the coverage statistics\nof the sample by the capture kit selected by the user are calculated:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/genomecov.html"},"bedtools genomecov"),"\ncomputes coverage summaries of aligned sequences for a reference genome and reports depth\nin ",(0,r.kt)("a",{parentName:"li",href:"http://genome.ucsc.edu/goldenPath/help/bedgraph.html"},"bedGraph format")," (File A)."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html"},"bedtools intersect"),"\nscreens for overlaps between File A and either the capture kit selected for the sample by the user,\nor each ",(0,r.kt)("a",{parentName:"li",href:"/settings/capture-kits"},"capture kit that is built into or uploaded into the system")," if the user has\nnot selected a capture kit. For a capture kit, the coverage depth, kit length, intersection length, complement\ncoverage depth, and complement intersection length of aligned sample sequences with capture kit intervals\nare calculated."),(0,r.kt)("li",{parentName:"ul"},"For a capture kit selected by the user, or for the best matched capture\nkit, ",(0,r.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/complement.html"},"bedtools complement")," returns a file\nwith all intervals in a genome that are not covered by at least one interval in the capture kit file (File B)."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html"},"bedtools intersect"),"\nscreens for overlaps between File A and File B by computing the coverage index, depth, and intersection length\nof these files.")),(0,r.kt)("p",null,'As a result, if the user has not selected a capture kit and if the best matched capture kit for the sample was\nfound, the sample is further analyzed with this kit. If a suitable capture kit could not be found, SNVs/Indels\nfiltration (variants discovery in the same genomic intervals in which targeted sequencing was performed)\nand copy number variation discovery will not be performed for the sample. The file with the statistics of the\ncapture kit selected by the user or determined by the system can be downloaded in the "',(0,r.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,r.kt)("em",{parentName:"p"},"Determine Kit"),'" task details ("',(0,r.kt)("em",{parentName:"p"},"Download Capture kit stats JSON"),'").'),(0,r.kt)("div",{className:"footnotes"},(0,r.kt)("hr",{parentName:"div"}),(0,r.kt)("ol",{parentName:"div"},(0,r.kt)("li",{parentName:"ol",id:"fn-1"},(0,r.kt)("a",{parentName:"li",href:"https://doi.org/10.1109/IPDPS.2019.00041"},"Vasimuddin M., Sanchit M., Heng L., Srinivas A. Efficient Architecture-Aware Acceleration of BWA-MEM for Multicore Systems. IEEE Parallel and Distributed Processing Symposium (IPDPS) (2019)"),(0,r.kt)("a",{parentName:"li",href:"#fnref-1",className:"footnote-backref"},"\u21a9")),(0,r.kt)("li",{parentName:"ol",id:"fn-2"},(0,r.kt)("a",{parentName:"li",href:"https://doi.org/10.48550/arXiv.1303.3997"},"Li H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2 (2013)"),(0,r.kt)("a",{parentName:"li",href:"#fnref-2",className:"footnote-backref"},"\u21a9")),(0,r.kt)("li",{parentName:"ol",id:"fn-3"},(0,r.kt)("a",{parentName:"li",href:"https://doi.org/10.1038/nmeth.1923"},"Langmead B., Salzberg S. Fast gapped-read alignment with Bowtie 2. Nature Methods 9, 357:359 (2012)"),(0,r.kt)("a",{parentName:"li",href:"#fnref-3",className:"footnote-backref"},"\u21a9")),(0,r.kt)("li",{parentName:"ol",id:"fn-4"},(0,r.kt)("a",{parentName:"li",href:"https://doi.org/10.1038/s41587-019-0201-4"},"Kim D., Paggi J.M., Park C. et al. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol 37, 907:915 (2019)"),(0,r.kt)("a",{parentName:"li",href:"#fnref-4",className:"footnote-backref"},"\u21a9")),(0,r.kt)("li",{parentName:"ol",id:"fn-5"},(0,r.kt)("a",{parentName:"li",href:"https://doi.org/10.1093/bioinformatics/bts635"},"Dobin A., Davis C.A., Schlesinger F. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29(1), 15:21 (2013)"),(0,r.kt)("a",{parentName:"li",href:"#fnref-5",className:"footnote-backref"},"\u21a9")))))}d.isMDXComponent=!0},74514:function(e,t,n){t.Z=n.p+"assets/images/alignment-84b2df0e13e9d1760c38cd38c6991837.png"},3201:function(e,t,n){t.Z=n.p+"assets/images/alignment_metrics-8cf2e606d7ac505043ce2db7bf72edf2.png"},55915:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAATwAAABXCAYAAACZQfkrAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAwHSURBVHhe7Z39s1dFHcf7DxnwIW5FJmDJFUms8THHYignigbTVHzAISy1sCx8GAkfIISUS4YUXZXgKl6exDsMQzWNk5vvO506c76f3bO737O7n/M97x9eA3Pv3Yfv9+59fT9n97O7n1v4+6eGEEKGAIVHCBkMFB4hZDBQeISQwUDhEUIGQ2+E99sX9phbb7/TrFp9nVmyZMki+D++hu+9O3dWLEcIIRXqhfeDH937P8G1se2nP6f4CCFW1AoP4kL0JonNBcpQeoQQCZXCg7AkmfmCR11KjxDSRKXwYiK7JpQeIaSJOuFhHk4SWAyoS2qDEDJMVAlv3EdZCUZ5hJAKVcILWZH1BXVKbRFChoca4aWI7ioY5RFCgBrh1ROKuwaLIFKbhJBhoUJ4XS5U2MBuDKltQshwUCE8SVBdwzQVQkhx4eWI7iqYpkLIsCkqvJQLFTYY5REyXIoKL2d0V8Eoj5DhUkx4JaK7CkZ5hAyTYsKLPQmlOvsOHDj0dlSyMtNUCBkmRYQHUUkicoEyUl0AEgzN42OaCiHDo4jwQuXkM+8WOh/INBVChkd24cWISapHIoVICSGTQ1bhxSxUhMy3xcwLTlqUd/7iv8wvf/2cuf/BR5Lz8GPbzfEPzov98OF3rx0Q6wUPPrzN/PVv82K5cXl+92tim3jf8P5JZeocnZ0Ty4/7foSCttCm1Jeu8XltXY29HU/uNHtfnzHvnDxjPrr0idhWLFmFFxrdgZAoLGYBY9KivDML/zB33b1BfK1d88UvrTBHjp0Q+9HG/IXL5rY7viXWW/HEUzvFsuOy5b4HxPbwvuH9k8rU2XfwsFh+nPcjBrSFNqW+dI3Pa0sx9pYtu8Lccusdix9Sp85fEtsNIZvwYqI7ECKkGKGCSYry+iK8mSOz5vPLl4v1VmCgz390WSw/DhReOKWEVwfyu/f+h8x7758T2/chm/Biz7rLIbxJOjOvD8L7+PK/zUOPPC7WWefKK68yBw8fFesYBwovHA3Cq4D4tm3/mTnnMf3QJIvwYqM7kEN4YFKivD4I7/0zF826m9aLdTaBGCFIqZ5YKLxwNAmvAk8AoXOmWYQXunpaJ5fwJiUZuQ/C23vgsFm6dKlYZxOIEYKU6omFwgtHo/DAypWrg97z5MIbR0Igl/DAJCQjuwYd/tClMjm5cOkTs3nLfWL/JCBGCFKqK5YhCA99lMqkxDX2Nm3eYuZOL7RyYv7jxU0G25942qz7+nqvD8YQ6SUXntTBEHIKbxKSkbULDxPOX7t+jdg/GxAkRCnVFwOFl4YUYw8rs5ivWz41JdZbgScByFKqo05S4Y0rIJBTeCCkPY1oF97uV/aLfbvmK9eaH/9kq/g9CHKclbkmFF4aUo69Ex9eMHd/Z6NYd8UDWx9r/WBMJrxxFirq5BYe6HOUp1l4SEzd+L3vi33D148d/9Csvu6r4vd3vfiyWGcMFF4aUo89rMpCarbH3KuuvtocnHGv6icTXlfyKSG8kDa1oVl4f3nv1GIkJ/UNkR8+nTf9cIv4/Q0b74lKQ5Cg8NKQY+xh5wV2Y0htgLZxkkR4XUV3oITwQF+jPM3Ce/b53WK/ENVVj6zYbiZ9gkOUEGazzhgovDTkGnsfnL1o1t/8TbEd5G6+fuiIWA4kEV7MnlYbpYTX1zQVrcJzbSVDVFfNvbgWNZ5+ZtdIvTFQeGnIOfZe3X9oUW5SW9iNYcvd7Fx4MWfduSglPNDHNBWtwvvj0XfN1NQXRvqEaA5RXfVzrrQVCBPirNcbA4WXhpxjD4+teHyV2lozvdacnF8Qy3UuvHGSjCVKCq+PaSpahYe8KqlP0gqs7bEWwoQ46z8bA4WXhtxjz7bij7Gz/40/iWU6FV7XwgElhQdC2teARuGdOnfJ3HTzN8Q+STl2+HTGp7T08xBn/WdjoPDSkHvs4fiwa1euEtuzjZPOhNflQkWd0sIDfYryNAoPk8jSfAs+iaVdFJh/wTxM8+cBxAmBNsuEQOGlIffYc7WHNCfpbMPOhJdKNhqEF9KH0mgTnutkFNdci02SbatwPlB4aSgx9rY+ul1sz/bB2InwUkV3QIPwQF+iPG3Ccz2euk5CcZ2o4lqF84HCS0OJsffMb14Q28PcsLTVrBPhxZ5154MW4fXlzDzXoJu+4caR47RjCDnK3LYA0RapuSLDcbeaDUF4397wXfF3FwNOG5bab1JCeKG/i7GFlzK6A1qEB/oQ5bkGXVf4/mG7Ukx8jn3C4Z+2ub96KksoQxBel/jKahDC6zoNpYkm4fUhGVmT8FyraJh7kcrUca3u1pOVQ6HwwtAsPNt7kER4qQUDNAkPaE9G1iQ821YyRG2+R7fbJqUh0thbzSi8MCi8/9JsJAUhc2c5hKc9GVmL8FyZ8CGX88w4LvuBUKUybVB4YWgWHhKMpTnizoWXQy4g5DGyyz28LkKiztzkEB7AH73UfoXrjzEkeRhihCClemJPUKHwwtAsvNDfRZTwUi9UNPGNqKSyqdAa5ZUYdBK4U1bqA6K1mc+iNqmMDVtdsYIZgvDaPpBSUGLs4ZxEqb36CTx1ooSXK7qr8ImoNPapBBqE5zoZJeYAANvBAyDmsm4KLw0lxp5tj/b02hvFLIBg4eWO7ipciwVdn9Dii8YoT4PwZjwu2e6KmMu6Kbw05B57rhO0cRz82YV/jpQJFl6ueTIJLGBAbpVoIMGS/dGYpqJBeLaV1RSErPhWUHhpyD32XLt4bGlPQcIrFUlpRluaSmnhhVyy3RWhl3VTeGnIPfZc9xvj6CipTJDwUicZ9xFtaSqlhRdyyXZXhF7WTeGlIefYc+3iceVoegsv96JAn9C0gFFSeK5BmBIINuSybgovDTnHnuu1246GAl7CK7VQ0Se0RHklhee6jwKLC3v2HjSv/P6NaJ597iXrIJcOErVB4aUh19jDzWW2A0va9ll7CY/RXTtaorySwrMduQ1id0XUce3eCDlBhcJLQ46xh7naX/xql3XaBLeZ4VYzqSxoFZ626A4ro7A7/sX8maZ5RQ1RXinhuVIExtn32sSWaApsE9VNKLw0pB57kN3L+/6weOG21AYk+FLLGGgVni10zA0Eh1XiZv/wNS3SC9n3m4pSwnNdsu2aUwnFdQKL71YzCi8NKcceHmMR2dlkB3AGYNvvzyk8LdEdhCb1r44W6ZWO8koJz3YyCvCNvHzAPB2OhpLa8b2sm8JLQ4qxh6jurT8fN7fcJu+nrsDv/u3Zk2IddZzC0yIRKbJrAtFo6G/pZOQSwnO1advTOA6uuUKfy7opvDR0NfYQzb1z8sxiRHfD2nVifXUQ9b26/02xriZW4WlZqPCJ7iq0CLpkMnIJ4bn2uoasnvriWg322aubSnjY9XHPps3iMekxPL7jKedrcQmvyyPeK47Ozon9qHCNveVTU5/9zqZbWbHiy2J5G5Ddi3v2eSeeW4UnVV6CkNVPLfONEG+pR9sSwrNt4G5LEYjFle/nc1l3KuF1zfVrps3caflWN+ASXgraokbX2EsBXjvyL0N22YjC05SGErIQoKnfpaK83MJzHcM+7mU7LmyXA4G28/YovDg0CQ9zerMnTov9cCEKT8ujIQiZE8PPSnWUoNRcXm7h2e6PBSkeZytcj7Vtl3VTeHFoEN7KVasXP+xCoro6I8LTsjJbx+fxsK/97pqcwsOgs12liOgrZLtXKGgb99NKbUPArisgKbw4Sglv2bIrzO133rXYPhY0pLZ9GRGepsfCCp85MU3RXUWJx9qcwnMdz4Ov4/tSua5wHVTgOkGFwosjh/CwCIHIHefZ7Xhyp3nzrWOd5XCCXggPQGiS9JCyolF2QEMiMiHk/4wIT8tKpw3IDX0EmuYaJSg8QnTRO+H1iZCUGkJIekaEh3kn6Y+XhEPhEaKLEeHxGPfu8NkSRwjJx4jwgPa5sT6A91B6bwkh5RCFp3Wltk/wcZYQfYjCA4zy4mF0R4hOrMJDzhulF0eJHRaEkHaswgOaThPuC1yoIEQvTuEBRnp+2HaCEEL00Cq8Cs1buEqCRG1GdYT0A2/hVSCKwR84EpSxEjlE8NrxHjCiI6RfBAuPEEL6CoVHCBkMFB4hZDBQeISQwUDhEUIGwqfmPxU1Q8Zc6H67AAAAAElFTkSuQmCC"},57674:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVMAAABCCAYAAADqtjX+AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAA/wSURBVHhe7Z1psxRFFkDnr8zvmB9huI6Mgoqyiiju+664IMOiICgwoCIqIoq4oIIiiMATZPWJgCA8FlnGcJuYmBhyOD1RYdHv5lpZzeuu++GEIa8zs7q76nRm3sybfzr92xmjKIqiVENlqiiKkgGVqaIoSgYaK9Odx4+bj/btMAu3rW79V3qNoihKKI2U6eT1S8yfZ118Dn9ZNK7179LrFUVRfDROppcuu3+QSMtMXDVTLKcoiuKiUTL1ibSAob9UXlEUxUZjZBoqUuC1Uh2Koig2GiHTGJEC86dSPYqiKDZ6XqaxIi2Q6lIURbHR0zJNFakGoRRFiaVnZZoqUtB1p4qixNKTMq0iUspKdSrN4Ojp382BI6fPgX+TXttJTv36X3Po+M/nXNfBoz+ZEz//R3y90nl6Tqa9LNIff/q3eeOtd82zs1+ozNsrPzKbtu42Ayd/FdvKzUeffiFeB8yeO998s29ALFcnCGrvDz+a15etMJNuuc1cetll5oILLhC56KKLzaSbbzULX37N9O8/0ior1ZkLJLll53dm1px5ZtTosebCCy8UrwsuHzbMPPDQo2blqk/NoR9/FuuLgc9k7ryF4neVG9qhPek6yvDepPIxzF+02Hz82Qaz67tDrWdJaqcKPSXTXu+R0kO674GHxQeqCjysCCXHgyiBsO+8+16x7YKXlywVy9YBIvzsi6/MxJsmidcSwvUTJppP1n6ZXapHTv1mFr/+lrniiivFdn0g/Ucee8Ls3ntYrD+Ebbv3myuuHC7WnxvaoT3pOspMmzlLLJ8KP04Tb5zUuu/3HTohthlLz8i010UKdcm0gJ7ZgkWvmuOZf7X7tvWbv15+udhmwe133G0GTtTfS6ZXefudd4vXkAJ1UafUVgxIGTmPuOoqsZ1YkOqcFxeaY6f/JbbnogkyLYNYH3z4MbNzz8FKP449IdMmiBTqlmkBPbZcw25uzudfWCC2U+aSSy41G76qL/BX9EbrkATD7E/Wbkx+EPnxenHBS86hfCpMTew5cExs10bTZFrA5//01Onm8IlfxGvw0fUybYpIoVMyhZEjrwu6yX0QJAkdTiPdKj0DF2vWbXLOiVaFumlDatsFc6PM50l15uKGiTdFDWWbKtOCMWPHm6937xOvw0VXy7RJIoVOyhRiH0IJeoOhPS6ki3yleqqwdddec9XV14htlrlu1GgzZeoM89KrS83qzze2ghUELSY/OcUMHz5CLFOGNmhLugYJfjiYs/N9PkXwa8Zzc1qBQ66NgAyBuzvvujfoR4IAVWiwsekyBUYba9ZtFq/FRtfKtGkiBZdMudmkMu0QxUSQH65e15rv8z3I0599Pnn5zcmz5Z6ZNlOsV4JrQb5SXakQVHPNkSIqhnbfHTwuli9AfDv6D7Tm1lyfGX/ne5LqaGfztn4zbNjfxHqAIBSy9c178p2yWmLsuPFiPcA1L12+Uizfjkumazfk/X5Cscn0rnvua01jlJeM2WD1Cp8n8/N871J9ZWKF2pUybaJIIYdM22HS3TUM52HnoZfK+kBQY8aOE+u1gXyRsFRfCu+897FVfvQk12/eFjW1wGvpFdp6g8z9hjyAzJM+9sTTYh1w7/0Pmv2HT4plbbASYOr0Z8X6YMLEG4NGGt0kU56H0B+vMvxALVvxgTfgxz2yvf97sY52uk6mTRUp1CFTYML9/oceEeuFVMGtOtv7lerjBmXIKv0N+fp6iaG4Pq/UOc4CIu+IU6o75PNyCavK9AqjCEYTUr38qNCDlcqVaYJMC/gBou4co42ukmmTRQp1yRR27z3UCjpJdTOXGCs4hp6PPv6kWB//zjpI6pX+vuL9T8Q6Y2FFwjUjrxXbmDlrbqUeMNJiekCqe/yEG7xzv7xHqWxoz9YFw95Ro8eI9fO+pTJlmiRTKOaubUN/RMsIRypbpmtk2nSRQp0y5YayLWHiZoqdy2SXiS3oQ48VkT01ZZr494cfnZxlreu6jV+LPY6U9yNhC64hIoQklSlAau3lYNz4Ceb7gdNimVD4Lm290xD5NE2mUAhV+j4hJDjaFTJVkf6fOmUKfY7F9UveWC6WsbF85SqxnnIvlyGndPMiYWTcXmcstmmGENmFsHPPD61tqCylKcPc5Ffb94hlwPVDkksO9KTarwv4ofKto2yiTME3j809LZUrcMp0KJzgqSL9g7plun/g1NkHTg4YxdTv2j6KRIrhtStA9dqbKwbVG8ubb78n1p1LpqkQ/GCpknRtOeWQSlNlCq737mvLKtOhcIKnivRc6pbp0VO/m3vue0Csn4c/dGvilh17xCU/9ELLARCkals6hYyrJmHhwZfqzjXMr4JNDjmG+VVpskxd012+H2FRpj6JdeJYDxXpYGqXqaP+mJuWzEpSHVKk3jbUR8ZIufzaWPoc0xYM53LnIIjB9hnxWYQEO+qkyTIF133DaEcqA4NkynBeElQ7dQpVRSpTu0wz9EzJuXnTzbeIdUhLhuiF0RuTXo9wyq+NxVU30pr3j1eSNyRUxRa8AkT2+ZdbxXKdoOkydd3D5WmqdgbJlCM7JElJ1CFUFamdumWaY86URfDS+kvb0NoVeeaG5sZuLxOKa8hW8PjZHmrs4vgc+HIWsA72tTffOS+956bLNPWeHCRTBCmJykZOoapI3dQt076K0XyXvFxzgTYB82/8TSoTimv9bAHi4vMjOs97kOqpA9furIKR117XWrJTR84CG02XKdjWAbNu2ZZRrbJMIYdQVaR+6pSpS4S2XmU7rmE1ddtE5eql0UOoKrgQaRWwLIvs9qQDrCMbexnfUpwyXD/JjJcsfbv1MNcpfZWpfY0y+RJsGaUGyTRValWEqiINo06ZunpwoVs8bcEkXw/TJfLQtl1QP0k+QpJblCkExtwtgqljfpUVC2TGl9p3QW+aLcD0oDj2I6dcXTJla2X7cSCpkONAal+i0zJN+UEZJFPWlErSCiFFqCrScOqSKQ+0bd0jhOw1dy1zCtk9Qk/QNtcasp/cB7JZu2FLpUz2COyhRx5vPUw5xUoPdf7CV6JlX6boUbNNt6pYXSLJScw925UyhZggVDsxQlWRxlGHTHn4br3tDrFOCM0a5doHz/EZUpkyqRHUWAg2sae+iriA4R5HvOQ6NwsBbtyyyxmUCoXzqcjFmip8lemZVso+W0rDKJmy86mK6EKEqiKNJ5dMqYdh91PPTPNKhd5OiMhs20fpbYYeR4J0pTpck/6p9O8faH1m9DalNkMh5yUnxubqqSJVlkWF5Jr1gVTJ4Sm140JlmlGmUKdQVaRpuGRaB6Gp4BimsudbqiPmoLw+x2oC377oVFg7y/lNDN+riJVpkgNH/ym2kQqfPZF85m1TxcqPJaedxsheZXrG7Pj2gBk+Qp4SipYp1CFUFWk6nZRpzBlQrocvZuE90kW+Uj25Mkm5oFdID5hrThEYR4uk5iH1wcoCRhPTZ862ZuOywftghBEqVJWp+zNIkinkFKqKtBqdkinzdjHDas68l+qhl0lvUypjw1YXN3ao3HNRCGzK32e0hvPSdbVDD7VqToEQCOgRDQ89ggOh0ssNCU6liKRuOi1TW0DUdU97ZQo5hKoirU7dMmWYO3f+ouCEJuDKEJWSrMSWJAUQrVSmE9Cr69v+bevMIVePlb+FnrWUCz5jxMoCf+maCujRhhzBoTK1p29k6M8UgFQmSKZQVaipqEj/oC6ZFrtsUiLT/Erb5jlzEzP/WhdF1J3coNI1Qo61sSkwDcIWVNfcb9UjVZoiU1sw1HWKQrBModNCVZGei0um46+/QVwYbQN5frF5e6W972C76eogZmVA3TA3yhypdJ1Az0Yq1wk49sQ2LRFyBE3TZep6zjhyx7YzLkqm0CmhqkgH4/qSudmkMnXi2gZaF65tqZ3GdVRzyFlLdcHns2jx6+J1MQ3BVkmpXEHTZep6/66AarRMoW6hqkhlhppMXWnk6iJkN1WncH0fdTzkMbiE4DvFoOkytQVBfSOjJJlCXUJVkdoZSjJ1bR+tE+R9vrPkl+nkQx6Da9G5ytQO0zec4SW15UsJmSxTyC1UFamboSRT5t1suU8JFLGdcfXnG5NZ/u6H1gc6JIhCykCpbMqx1S5iH3JXAu6c22ZVpvG4pkfAt5qkkkwhl1BVpH6GkkxtS0cgx24l166qkGi561z6XEGslJMJCF4QxJDKVE2GXYY0ceQPkNpxHb0BTZXpmnWbrCsh2MRCZjWpXEFlmUJVoapIwxgqMnUJIec+epsQwRct73Ms2SI4lKMHmBqAIoghlUH0ROKlMjFoACoejuZ27Szj8/QFPrPIFFKFqiINZ6jIlDPtbTeea+lILK5MVL7tpa6VBvQ+NvRV65360ha6ZM+uKml3DbDc6odj1QJsW3fttX4/ujTqXBAkU1K29wvcRxzpI5Uvk02mECtUFWkcQ0WmtgxRkHN9Jb1H5hGldpAFUpfKAQ+JLeE0UJ7eiFTWx+ETv5gnnp4q1gu+aQjfkjKSPtOGVNYHw3vXhgJdtP8HZA7jDDDXihR+eEMDnlllCqFCVZHGMxRk6rqG3MEdcM3N+gIpvvOfWNjOjqHQ7bMImh4tqe2k+gpChoS+o1Rog7Z89RTwHngvrhwCoblpe1WmfJbks2XrLTl8fcv6+HtoPgPILlNAqJPXLxElCirSNIaCTF1750N6PbEgZ9uqAd/efx4CHgbfQ0PvY8rUGS1RMJwrpikoz5CbYTOJoH173yF0SOibJiigTdrmGriW4sHmGmmHa+baXVtIC0Jz07pkmvPYkoId/fJe9zI2mfK+uT/ojbu4+pqRYnkb3DNE72NSF9Yi0wKOQCFrP4lOimQnnMsvvVbxMxRkaguecPPlOF6kHR5+23pWpI7cpXIFodLKAVMHSE+6DgnX3GZuYrJZuWRaByG9XZtM6wBBL1vxQXCPtKBWmSp5Od8ydR0tErJcKRXbQX0Qki8ViaQcWhcDw+uUSDzi8h1FXZXYPKtNlmnq6QSgMu0izrdMXVHoOob4Ba6hfujaTOYUZ7+wwDvkT4EH0Hb8bwj9+484z+FKhffKeVexwawmypQ1uRw/UyUBucq0izifMmXIU/Vc/VRom/PzpbaRO5KXyrVDPZu+/iabuOiNMieb4wQA6mDXV5XTU8sgeFIFxg5VoSky5b7lRIX3P14blcPXhsq0izifMv1+4LQZN36C2Db/zt+lcrlwJVWJzSTFa7fu3BuVQb8g9wPYDlJdtWZ9cAb9Msz1sc6XJV8pEi3oRZnyWY4aPaYVtJy/aHFr40LoHHIoKlOl0SAdlst8un5z6yEjujz5ySmtCPAtZ3uwM56bY2bPnd9aTkPAKNeGhBBoi7W0tM01cG3IgGsjoMT/c83kMthz4FglgSrVUZkqiqJkQGWqKIqSAZWpoihKBlSmiqIoGVCZKoqiZEBlqiiKkgGVqaIoSgZUpoqiKJU5Y/4H+6yctwgIkxcAAAAASUVORK5CYII="}}]);