"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[7096],{3905:function(e,t,n){n.d(t,{Zo:function(){return p},kt:function(){return g}});var r=n(67294);function a(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function o(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);t&&(r=r.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,r)}return n}function s(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?o(Object(n),!0).forEach((function(t){a(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):o(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function i(e,t){if(null==e)return{};var n,r,a=function(e,t){if(null==e)return{};var n,r,a={},o=Object.keys(e);for(r=0;r<o.length;r++)n=o[r],t.indexOf(n)>=0||(a[n]=e[n]);return a}(e,t);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(e);for(r=0;r<o.length;r++)n=o[r],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(a[n]=e[n])}return a}var c=r.createContext({}),l=function(e){var t=r.useContext(c),n=t;return e&&(n="function"==typeof e?e(t):s(s({},t),e)),n},p=function(e){var t=l(e.components);return r.createElement(c.Provider,{value:t},e.children)},u={inlineCode:"code",wrapper:function(e){var t=e.children;return r.createElement(r.Fragment,{},t)}},f=r.forwardRef((function(e,t){var n=e.components,a=e.mdxType,o=e.originalType,c=e.parentName,p=i(e,["components","mdxType","originalType","parentName"]),f=l(n),g=a,m=f["".concat(c,".").concat(g)]||f[g]||u[g]||o;return n?r.createElement(m,s(s({ref:t},p),{},{components:n})):r.createElement(m,s({ref:t},p))}));function g(e,t){var n=arguments,a=t&&t.mdxType;if("string"==typeof e||a){var o=n.length,s=new Array(o);s[0]=f;var i={};for(var c in t)hasOwnProperty.call(t,c)&&(i[c]=t[c]);i.originalType=e,i.mdxType="string"==typeof e?e:a,s[1]=i;for(var l=2;l<o;l++)s[l]=n[l];return r.createElement.apply(null,s)}return r.createElement.apply(null,n)}f.displayName="MDXCreateElement"},76985:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return i},contentTitle:function(){return c},metadata:function(){return l},toc:function(){return p},default:function(){return f}});var r=n(87462),a=n(63366),o=(n(67294),n(3905)),s=["components"],i={sidebar_position:9,sidebar_label:"Coverage Analysis"},c="Coverage Analysis",l={unversionedId:"results/coverage-analysis",id:"results/coverage-analysis",isDocsHomePage:!1,title:"Coverage Analysis",description:"Genome coverage (or target exome regions or target panel) allows you to assess how informative the CNV",source:"@site/docs/results/coverage-analysis.mdx",sourceDirName:"results",slug:"/results/coverage-analysis",permalink:"/ngsw-docs-en/results/coverage-analysis",tags:[],version:"current",sidebarPosition:9,frontMatter:{sidebar_position:9,sidebar_label:"Coverage Analysis"},sidebar:"tutorialSidebar",previous:{title:"Bioinformatic Report",permalink:"/ngsw-docs-en/results/bioinformatic-report"},next:{title:"Workflow Details",permalink:"/ngsw-docs-en/results/workflow-details/workflow-details"}},p=[],u={toc:p};function f(e){var t=e.components,i=(0,a.Z)(e,s);return(0,o.kt)("wrapper",(0,r.Z)({},u,i,{components:t,mdxType:"MDXLayout"}),(0,o.kt)("h1",{id:"coverage-analysis"},"Coverage Analysis"),(0,o.kt)("p",null,"Genome coverage (or target exome regions or target panel) allows you to assess how informative the CNV\ndiscovery is in general. In this section, you can estimate both the fraction of data covered in general,\nand how much the genes and transcripts of your interest are covered."),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"Depth of coverage of target regions (example)")),(0,o.kt)("p",null,(0,o.kt)("img",{alt:"Coverage analysis",src:n(42044).Z})),(0,o.kt)("p",null,"The plot shows which fraction of the genome or target regions is covered at different depths. So, in the example,\n96.79% of the target regions are covered with 30 reads, which is a good indicator. For a more accurate assessment\nof coverage in genes of interest, you can use the search below on the page and select a transcript. To select\na transcript, click on the gene name. By default, the transcript of the\n",(0,o.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/refseq/MANE/"},"MANE Clinical")," database is chosen, if there is no gene in the\ndatabase, the canonical transcript is chosen according to\n",(0,o.kt)("a",{parentName:"p",href:"https://www.ensembl.org/info/docs/tools/vep/index.html"},"Ensembl")," annotation.\nSearch results can be exported as a table."),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},(0,o.kt)("em",{parentName:"strong"},"TP53"))," ",(0,o.kt)("strong",{parentName:"p"},"gene coverage")),(0,o.kt)("p",null,(0,o.kt)("img",{alt:"Gene coverage",src:n(82856).Z})))}f.isMDXComponent=!0},42044:function(e,t,n){t.Z=n.p+"assets/images/36-coverage-analysis-676dd4481abaf0d87234dca051cb763a.png"},82856:function(e,t,n){t.Z=n.p+"assets/images/37-gene-coverage-1adfcb4e2a445b0aeca95c4925bea90b.png"}}]);