"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[622],{3905:function(e,t,n){n.d(t,{Zo:function(){return m},kt:function(){return g}});var a=n(67294);function i(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function r(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function o(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?r(Object(n),!0).forEach((function(t){i(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,a,i=function(e,t){if(null==e)return{};var n,a,i={},r=Object.keys(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||(i[n]=e[n]);return i}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(a=0;a<r.length;a++)n=r[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(i[n]=e[n])}return i}var l=a.createContext({}),p=function(e){var t=a.useContext(l),n=t;return e&&(n="function"==typeof e?e(t):o(o({},t),e)),n},m=function(e){var t=p(e.components);return a.createElement(l.Provider,{value:t},e.children)},c={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},h=a.forwardRef((function(e,t){var n=e.components,i=e.mdxType,r=e.originalType,l=e.parentName,m=s(e,["components","mdxType","originalType","parentName"]),h=p(n),g=i,d=h["".concat(l,".").concat(g)]||h[g]||c[g]||r;return n?a.createElement(d,o(o({ref:t},m),{},{components:n})):a.createElement(d,o({ref:t},m))}));function g(e,t){var n=arguments,i=t&&t.mdxType;if("string"==typeof e||i){var r=n.length,o=new Array(r);o[0]=h;var s={};for(var l in t)hasOwnProperty.call(t,l)&&(s[l]=t[l]);s.originalType=e,s.mdxType="string"==typeof e?e:i,o[1]=s;for(var p=2;p<r;p++)o[p]=n[p];return a.createElement.apply(null,o)}return a.createElement.apply(null,n)}h.displayName="MDXCreateElement"},32969:function(e,t,n){n.d(t,{Z:function(){return i}});var a=n(67294);function i(e){var t=e.children,n=e.color;return a.createElement("span",{style:{backgroundColor:n,borderRadius:"2px",color:"#fff",padding:"0.2rem"}},t)}},75692:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return l},contentTitle:function(){return p},metadata:function(){return m},toc:function(){return c},default:function(){return g}});var a=n(87462),i=n(63366),r=(n(67294),n(3905)),o=n(32969),s=["components"],l={sidebar_position:3,sidebar_label:"Variant Details Panel"},p="Variant Details Panel",m={unversionedId:"results/main/snvs-indels/snv-details-panel",id:"results/main/snvs-indels/snv-details-panel",isDocsHomePage:!1,title:"Variant Details Panel",description:"Click on the variant row to see a variant details panel under the table:",source:"@site/docs/results/main/snvs-indels/snv-details-panel.mdx",sourceDirName:"results/main/snvs-indels",slug:"/results/main/snvs-indels/snv-details-panel",permalink:"/ngsw-docs-en/results/main/snvs-indels/snv-details-panel",tags:[],version:"current",sidebarPosition:3,frontMatter:{sidebar_position:3,sidebar_label:"Variant Details Panel"},sidebar:"tutorialSidebar",previous:{title:"SNV Viewer",permalink:"/ngsw-docs-en/results/main/snvs-indels/variant-viewer"},next:{title:"Basic Filtering of Variants",permalink:"/ngsw-docs-en/results/main/snvs-indels/basic-filtering"}},c=[{value:"Details Panel Tabs",id:"panel-tabs",children:[{value:"Common",id:"common-tab",children:[]},{value:"Gene",id:"gene",children:[]},{value:"ClinVar",id:"clinvar-tab",children:[]},{value:"gnomAD 3",id:"gnomad-3-tab",children:[]},{value:"gnomAD 4",id:"gnomad-4-tab",children:[]},{value:"ExAC",id:"exac-tab",children:[]},{value:"Other Frequencies",id:"other-frequencies-tab",children:[]},{value:"Conservation",id:"conservation-tab",children:[]},{value:"Protein function effect",id:"protein-function-effect-tab",children:[]},{value:"Other",id:"other-tab",children:[]},{value:"Custom annotation sources",id:"custom-annotation-sources-tab",children:[]}]}],h={toc:c};function g(e){var t=e.components,l=(0,i.Z)(e,s);return(0,r.kt)("wrapper",(0,a.Z)({},h,l,{components:t,mdxType:"MDXLayout"}),(0,r.kt)("h1",{id:"variant-details-panel"},"Variant Details Panel"),(0,r.kt)("p",null,"Click on the variant row to see a variant details panel under the table:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(9152).Z,width:"900"})),(0,r.kt)("p",null,"The panel can be resized by hovering over the top border of the panel and dragging down or up.\nYou can also collapse or expand the panel by\nclicking on ",(0,r.kt)("img",{src:n(55236).Z,width:"30"}),"or ",(0,r.kt)("img",{src:n(73365).Z,width:"30"}),", respectively."),(0,r.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"On ",(0,r.kt)("a",{parentName:"p",href:"/settings/profile#variant-details-bottom-panel"},'"Profile settings" page'),", you can adjust whether\nthe variant details panel is expanded or collapsed in SNV Viewer by default."))),(0,r.kt)("h2",{id:"panel-tabs"},"Details Panel Tabs"),(0,r.kt)("p",null,"The panel is divided into 12 tabs (or 11 if there are no ",(0,r.kt)("a",{parentName:"p",href:"/settings/custom-annotations"},"custom annotations")," in\nthe system):"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#common-tab"},"Common")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#gene"},"Gene")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#clinvar-tab"},"ClinVar")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#gnomad-3-tab"},"gnomAD 3")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#gnomad-4-tab"},"gnomAD 4")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#exac-tab"},"ExAC")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#other-frequencies-tab"},"Other frequencies")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#conservation-tab"},"Conservation")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#protein-function-effect-tab"},"Protein function effect")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#protein-function-effect-tab"},"Protein function effect (additional)")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#other-tab"},"Other")),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-panel#custom-annotation-sources-tab"},"Custom annotation sources")," (if\nat least one custom annotation was ",(0,r.kt)("a",{parentName:"li",href:"/settings/custom-annotations#add-custom-annotation"},"added")," to the system\nbefore the variants in the sample were annotated)")),(0,r.kt)("p",null,'On the panel, only those tabs are visually highlighted that include at least one non-empty variant annotation\nfield. For example, "ClinVar" tab filled with data looks\nlike this: ',(0,r.kt)("img",{src:n(66982).Z,width:"80"}),", and empty - like this: ",(0,r.kt)("img",{src:n(69239).Z,width:"80"}),"."),(0,r.kt)("p",null,"You can navigate through the panel tabs using keyboard shortcuts. The navigation will be carried out only\nthrough non-empty tabs of the panel."),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},"Available keyboard shortcuts")),(0,r.kt)("table",null,(0,r.kt)("tr",null,(0,r.kt)("td",null,"Press this key"),(0,r.kt)("td",null,"To do this")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"D")," or \u27a1\ufe0f"),(0,r.kt)("td",null,"Move to the next tab that has at least one non-empty variant annotation field")),(0,r.kt)("tr",null,(0,r.kt)("td",null,(0,r.kt)("b",null,"A")," or \u2b05\ufe0f"),(0,r.kt)("td",null,"Move to the previous tab that has at least one non-empty variant annotation field"))),(0,r.kt)("p",null,"As you move between variants in the table, the selected panel tab is retained."),(0,r.kt)("h3",{id:"common-tab"},"Common"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(52358).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains basic information about the\nvariant: ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#external-links"},"external links"),",\n",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#sample-info"},"samples' info")," and\n",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#variant-interpretation"},"variant interpretation"),".\nThere may also be information\nabout ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#normalized-variant"},"the normalized variant"),"\nand ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel/#phasing-group"},"the phasing group"),"."),(0,r.kt)("h4",{id:"external-links"},"External Links"),(0,r.kt)("p",null,"Links to pages with variant information in ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/snp/"},"dbSNP")," (rsId),\n",(0,r.kt)("a",{parentName:"p",href:"https://varsome.com/"},"VarSome"),", ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://cancer.sanger.ac.uk/cosmic/"},"COSMIC")," (if the database\nwas ",(0,r.kt)("a",{parentName:"p",href:"/settings/custom-annotations"},"uploaded as a custom annotation"),"),\n",(0,r.kt)("a",{parentName:"p",href:"https://franklin.genoox.com/clinical-db/home"},"Franklin"),", ",(0,r.kt)("a",{parentName:"p",href:"https://genome-euro.ucsc.edu/"},"UCSC"),",\n",(0,r.kt)("a",{parentName:"p",href:"https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/"},"gnomAD 3"),", ",(0,r.kt)("a",{parentName:"p",href:"http://ruseq.ru/#/"},"RUSeq")," (if the\ndatabase was ",(0,r.kt)("a",{parentName:"p",href:"/settings/custom-annotations"},"uploaded as a custom annotation"),")."),(0,r.kt)("h4",{id:"sample-info"},"Samples Info"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Table with samples")," with the following columns:")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Name"))," is a name of the sample, for which the analysis was performed."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Genotype"))," is an allele combination for this sample (0 - reference allele, 1 - first alternative allele,\n2 - second alternative allele etc.), divided by \u201d/\u201d (for unphased genotype) or \u201c|\u201d (for phased genotype).\nThe number of alleles suggests ploidy of the sample."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Read depth"))," is a total number of reads of the sequence overlapping the variant position for this sample."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Ref count"))," is a reference allele reads count for this sample."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Alt count"))," is an alternative allele reads count for this sample."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"AF"))," is an allelic frequency for this sample.")),(0,r.kt)("ol",{start:2},(0,r.kt)("li",{parentName:"ol"},"Information about ",(0,r.kt)("strong",{parentName:"li"},"the variant occurrences in other samples"),' uploaded into the system:\nnumber of variant/number of samples.\nHere is the number of variants that completely coincide with the variant in terms of origin (somatic or germline),\nchromosome, start position, reference and alternative allele and gene. These variants contain at least one\nalternative allele (i.e. have genotype 1/1, 1|1, 0/1, 1/0, 0|1, 1|0, 1/., 1|., ./1, .|1, 1, 0/0/1, 0 /2, 1/2, etc.)\nThese variants are searched among all samples uploaded into the system (including deleted ones), for which the\nstage of variant discovery (or annotation for samples uploaded in VCF format) has been successfully completed.\nTo calculate the occurrences of the somatic variant, those samples are taken into account in which the stage\n"Somatic SNVs/Indels discovery" (or "Somatic SNVs/Indels annotation" stage for VCF samples) has been\nsuccessfully completed. To calculate the occurrences of the germline variant, samples with a successfully completed\nstage "Germline SNVs/Indels discovery" (or "Germline SNVs/Indels annotation" for VCF samples) are taken into account.\nBy clicking on the link, you will see ',(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-page#occurrences"},'"Occurrences" tab'),"\nof the variant details page."),(0,r.kt)("li",{parentName:"ol"},"Information about ",(0,r.kt)("strong",{parentName:"li"},"the variant occurrence in run samples"),".\nOnly presented on the variant panel in ",(0,r.kt)("a",{parentName:"li",href:"/runs#run-variant-viewer"},"run SNV Viewer"),"\nand run sample SNV Viewer. For example, in the following case, a variant was discovered in two of the\nthree run samples:")),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(69829).Z,width:"450"})),(0,r.kt)("h4",{id:"variant-interpretation"},"Variant Interpretation / Comment"),(0,r.kt)("p",null,"A variant interpretation is a comment with important information about the variant added by the user.\nWhere to find it after adding is described ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/variant-interpretation#where"},"here"),".\nTo add or edit a variant interpretation, click on the input box, enter the text and click outside\nthe input box to save the interpretation:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(39716).Z,width:"400"})),(0,r.kt)("p",null,"The author of the added interpretation (the name of the user who interpreted the variant) will be displayed in\nbrackets next to the section name:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(1371).Z,width:"500"})),(0,r.kt)("h4",{id:"normalized-variant"},"Normalized Variant"),(0,r.kt)("p",null,"Start position, reference allele, or alternative allele of some variants in the uploaded VCF file may\nnot be left-normalized. In this case, the variant will be written in SNV Viewer table in the same form\nas in the original VCF file, and the normalized data will be shown on the details panel:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(43892).Z,width:"800"})),(0,r.kt)("h4",{id:"phasing-group"},"Phasing Group Info"),(0,r.kt)("p",null,"Phasing group is a merging of homozygous variants located within a specific genomic interval\n(window of size N, which can be changed as\ndescribed ",(0,r.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#pg-merge-distance"},"here"),").\nIn addition, the phasing group is merged in such a way that at least one variant from the group\nhas an effect on the protein-coding part of at least one gene."),(0,r.kt)("p",null,"The phasing group details panel shows the variants that the group contains\n(you can open the variant details pages by clicking on the links):"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(3166).Z,width:"280"})),(0,r.kt)("p",null,"The details panel of a variant included in the phasing groups shows these groups\n(you can open the phasing group details pages by clicking on the links):"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(58020).Z,width:"300"})),(0,r.kt)("p",null,"If a variant is a part of phasing group with discordant amino acid substitutions, then that variant is\nmarked with ",(0,r.kt)("img",{src:n(35690).Z,width:"28"}),". If the variant is a\npart of phasing group with concordant amino acid substitution, then the variant is not marked with such icon."),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},"Example of phasing group representation in IGV:"),' phasing groups are located in "phasing groups" track.\nThere are three phasing groups on the left in the track: chr15:48505102 CA>TT (contains 2 variants),\nchr15:48505102 CAA>TTG (contains 3 variants), chr15:48505103 AA>TG (contains 2 variants).\nThere are one phasing group in the middle of the track: chr15:48505109 CA>TG (contains 2 variants),\nand one phasing group on the right in the track: chr15:48505117 CA>AG (contains 2 variants).'),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(9674).Z,width:"1000"})),(0,r.kt)("h3",{id:"gene"},"Gene"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(432).Z,width:"900"})),(0,r.kt)("p",null,"The tab contains basic information about ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#gene-info"},"the gene"),"\nand ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#transcripts"},"the transcripts")," in which the variant is located.\nIf you are granted access to OMIM database, there may also be information\non ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#omim-phenotypes"},"gene-phenotype relations"),"."),(0,r.kt)("h4",{id:"gene-info"},"1. Gene"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Official full name")),". When hovering over the gene name, you can see from which databases it was taken:")),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(96618).Z,width:"300"})),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Links"))," to pages with information about the gene in various databases\n(",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/Homo_sapiens/Info/Index"},"Ensembl"),", ",(0,r.kt)("a",{parentName:"li",href:"https://www.uniprot.org/uniprot/"},"UniProt"),",\n",(0,r.kt)("a",{parentName:"li",href:"https://www.gtexportal.org/home/"},"GTEx"),", ",(0,r.kt)("a",{parentName:"li",href:"https://www.omim.org/"},"OMIM"),", ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/gene/"},"NCBI"),")."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Search for all variants in gene")),': search for all variants discovered in the sample and located in the same\ngene. To search, click on "',(0,r.kt)("em",{parentName:"li"},"All variants in gene"),'" link and new tab with SNV Viewer filtered by this gene\nwill open.')),(0,r.kt)("div",{className:"admonition admonition-note alert alert--secondary"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"}))),"note")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"If you make any changes on the new tab, they will be saved in SNV Viewer. In this case, if variants\nwere filtered and/or sorted on the original SNV Viewer page, these filtering and sorting may be reset\nor changed, and the state of SNV Viewer on the new tab will retain."))),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Genetic Constraint Metrics"))," \u2014 these are metrics that reflect the degree to which a gene is intolerant to\nfunctionally significant mutations and help assess the likelihood of pathogenicity:",(0,r.kt)("ul",{parentName:"li"},(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"pLI"))," (Probability of Loss-of-Function Intolerance) \u2014 the probability that a transcript is intolerant to\nloss-of-function (LoF) variants. This metric indicates the probability that a transcript falls into distribution\nof haploinsufficient genes, meaning that the loss of one copy leads to functional consequences. It is computed\nfrom high-confidence pLoF (predicted LoF) gnomAD data. Range: 0 to 1. Genes with pLI \u2265 0.9 are considered\nLoF-intolerant."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"LOEUF"))," (Loss-of-function Observed/Expected Upper bound Fraction) \u2014 the upper bound of the 90% confidence\ninterval for the observed/expected ratio of high-confidence pLoF (predicted loss-of-function) variants.\nLower values (LOEUF < 0.35) indicate a gene is more constrained against LoF variants."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Missense O/E ratio"))," \u2014 the upper bound of the 90% confidence interval for the observed/expected ratio of\nmissense variants. Lower values suggest a gene is more constrained against missense variation."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Missense Z-score"))," \u2014 a Z-score representing the deviation of the observed number of missense variants in a\ntranscript from the expected number under a model of neutral mutation distribution.\nGreater positive values (Z-score \u2265 3.09) indicate that the transcript is more intolerant of missense variation.\nLower or negative Z-scores indicate that the gene tolerates more missense mutations than expected.")))),(0,r.kt)("h4",{id:"transcripts"},"2. Transcripts"),(0,r.kt)("p",null,"The panel contains gene transcripts from two databases: ",(0,r.kt)("a",{parentName:"p",href:"https://www.ensembl.org/Homo_sapiens/Info/Index"},"Ensembl"),"\nand ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"RefSeq"),".\nEnsembl transcript models are annotated directly on the reference genome, while RefSeq annotates on mRNA sequences.\nDue to sequence differences between the reference genomes and individual mRNAs, some of the RefSeq mRNAs may not\nmap perfectly to the reference genome."),(0,r.kt)("p",null,"By default, a table with transcripts from Ensembl is shown. To open the table with transcripts from RefSeq,\nclick on the corresponding switch button:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(45659).Z,width:"700"})),(0,r.kt)("div",{className:"admonition admonition-note alert alert--secondary"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"}))),"note")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},'On "',(0,r.kt)("a",{parentName:"p",href:"/settings/profile#transcripts"},'Profile Settings" page'),", you can choose which of the gene transcript\nmodels will be shown by default."))),(0,r.kt)("p",null,"The gene transcript table from ",(0,r.kt)("a",{parentName:"p",href:"https://www.ensembl.org/Homo_sapiens/Info/Index"},"Ensembl")," has the following columns:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Transcript"))," is a transcript ID from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/Homo_sapiens/Info/Index"},"Ensembl")," database\n(ENSTxxxxxxxxxxx). For the first, main transcript, a transcript from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"RefSeq"),"\n(NM_xxxxxx.x) is also given. You can open the transcript page by clicking on the ID.\nThe main gene transcript, according to ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/refseq/MANE/#Select"},"MANE Select"),",\n",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/refseq/MANE/#:~:text=MANE%20Plus%20Clinical%3A%20The%20MANE,variants%20available%20in%20public%20resources."},"MANE plus clinical"),"\nor ",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/info/genome/genebuild/canonical.html"},"Ensembl canonical"),", is marked\nwith ",(0,r.kt)("img",{src:n(51194).Z,width:"20"}),". The gene transcript chosen\nas the main one for this variant is framed. Impact, HGVSp, and HGVSc values for this transcript are shown\nfor the variant in SNV Viewer table.")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"IMPACT"))," is the predicted effect of the variant on the protein\n(see ",(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/variant-viewer#variant-impacts"},"table with variant impacts values in the annotation"),").\nHIGH impact value is colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),", MODERATE\nis colored ",(0,r.kt)(o.Z,{color:"#E2CC81",mdxType:"Highlight"},"yellow"),", and LOW and MODIFIER are\ncolored ",(0,r.kt)(o.Z,{color:"#999AAB",mdxType:"Highlight"},"gray"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"HGVSp"))," is an amino acid substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"HGVS notation"),": \u201cp.\u201d prefix\n(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"HGVSc"))," - is a nucleotide substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"HGVS notation"),": \u201cc.\u201d (coding; for a\nsubstitution in the coding sequence) or \u201cn.\u201d (non-coding; for a substitution in the non-coding sequence) prefix +\ngenomic position of the substituted nucleotide + reference allele > alternative allele.")),(0,r.kt)("p",null,"The gene transcript table from ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"RefSeq")," has the following columns:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Transcript"))," is a transcript ID from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"RefSeq")," database\n(NM_xxxxxx.x). You can open the transcript page by clicking on the ID.\nThe main gene transcript, according to ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/refseq/MANE/#Select"},"MANE Select"),",\n",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/refseq/MANE/#:~:text=MANE%20Plus%20Clinical%3A%20The%20MANE,variants%20available%20in%20public%20resources."},"MANE plus clinical"),"\nor ",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/info/genome/genebuild/canonical.html"},"Ensembl canonical"),", is marked\nwith ",(0,r.kt)("img",{src:n(51194).Z,width:"20"}),". The gene transcript chosen\nas the main one for this variant is framed."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"HGVSp"))," is an amino acid substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"HGVS notation"),": \u201cp.\u201d prefix\n(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"HGVSc"))," - is a nucleotide substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"HGVS notation"),": \u201cc.\u201d (coding; for a\nsubstitution in the coding sequence) or \u201cn.\u201d (non-coding; for a substitution in the non-coding sequence) prefix +\ngenomic position of the substituted nucleotide + reference allele > alternative allele.")),(0,r.kt)("h4",{id:"omim-phenotypes"},"3. OMIM Gene-Phenotype Relationships"),(0,r.kt)("p",null,"Access to data from ",(0,r.kt)("a",{parentName:"p",href:"https://www.omim.org/"},"OMIM"),' database inside Genomenal is provided if the user has a\nlicense to access the database. If you have been granted access to OMIM, there will be information on the\nphenotypes associated with the gene in which the variant is located on the right side of "Gene" tab:'),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(17738).Z,width:"500"})),(0,r.kt)("p",null,"The information is presented in the form of a table with the following columns:"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Phenotype")),' is the full name of the phenotype associated with the gene. For some phenotypes,\nthere is a question mark before the name (e.g. "',(0,r.kt)("b",null,"?"),'Bleeding disorder, platelet-type, 22"), which\nindicates that the relationship between the phenotype and gene is provisional. Some phenotypes are enclosed\nin square brackets (e.g. "',(0,r.kt)("b",null,"["),"Urate oxidase deficiency",(0,r.kt)("b",null,"]"),'"), which indicates mainly genetic variations\nthat lead to apparently abnormal laboratory test values. Some phenotypes are enclosed in curly brackets\n(e.g. "',(0,r.kt)("b",null,"{"),"Prostate cancer/brain cancer susceptibility, somatic",(0,r.kt)("b",null,"}"),'"), which indicates mutations\nthat contribute to susceptibility to multifactorial disorders or to susceptibility to infection.'),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Phenotype MIM number"))," is a phenotype ID in OMIM database.\nYou can open the phenotype page by clicking on the link."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Inheritance"))," is a pattern of inheritance phenotype: ",(0,r.kt)("b",null,"AD")," - autosomal\ndominant; ",(0,r.kt)("b",null,"AR")," - autosomal recessive; ",(0,r.kt)("b",null,"DD")," - digenic dominant; ",(0,r.kt)("b",null,"DR")," - digenic\nrecessive; ",(0,r.kt)("b",null,"IC")," - isolated cases; ",(0,r.kt)("b",null,"Mi")," - mitochondrial; ",(0,r.kt)("b",null,"Mu")," - multifactorial; ",(0,r.kt)("b",null,"PADom")," -\npseudoautosomal dominant; ",(0,r.kt)("b",null,"PARec")," - pseudoautosomal recessive; ",(0,r.kt)("b",null,"SomMos")," - somatic\nmosaicism; ",(0,r.kt)("b",null,"SMu")," - somatic mutation; ",(0,r.kt)("b",null,"XL")," - X-linked; ",(0,r.kt)("b",null,"XLD")," - X-linked\ndominant; ",(0,r.kt)("b",null,"XLR")," - X-linked recessive; ",(0,r.kt)("b",null,"YL")," - Y-linked."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Phenotype mapping key"))," is an OMIM phenotype map key (the map displays relationships of the\nphenotype with associated phenotypes, phenotypic series and genes): ",(0,r.kt)("b",null,"1")," - the phenotype is placed on\nthe map due to its association with a gene, but the underlying defect is not known; ",(0,r.kt)("b",null,"2")," - the phenotype\nhas been placed on the map by statistical methods, no mutation has been found; ",(0,r.kt)("b",null,"3")," - the molecular basis of the\ndisorder is known; ",(0,r.kt)("b",null,"4")," - a contiguous gene duplication or deletion syndrome in which multiple genes are involved.")),(0,r.kt)("h3",{id:"clinvar-tab"},"ClinVar"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(75448).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains information\nabout ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#haplotype-phenotypes"},"haplotype phenotype significance"),",\nif the variant is part of a haplotype, and\nabout ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#phenotypes"},"variant phenotype significance"),"\nfrom ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," database. Both sections contain a table of\nphenotypes that have been observed in individuals with this haplotype/variant and reported to ClinVar.\nThe table has the following columns:"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Phenotype"))," is a phenotype name (with links to various databases)."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Clinical Significance"))," is a clinical significance of the phenotype (you can find the description of\nsignificances ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv"},"here"),").\nBenign significance is colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"dark green"),"; Likely\nbenign - ",(0,r.kt)(o.Z,{color:"#0EB555",mdxType:"Highlight"},"light green"),"; Likely\npathogenic - ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),";\nPathogenic - ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"; all other values, including ones mixed from\ndifferent clinical significances, are colored ",(0,r.kt)(o.Z,{color:"#141E29",mdxType:"Highlight"},"black"),"."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Review status"))," is an assessment of the data source in which the clinical significance of the phenotype\nwas reported (you can find the description of review\nstatuses ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status"},"here"),").")),(0,r.kt)("h4",{id:"haplotype-phenotypes"},"Haplotype Phenotypes"),(0,r.kt)("p",null,"The section is present on the tab if the variant is part of a haplotype. The section includes:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Variant Haplotype ID"))," is a haplotype ID in which the variant is included.\nYou can open the haplotype page in ClinVar by clicking on the link."),(0,r.kt)("li",{parentName:"ul"},"A table with phenotypes that have been observed in individuals with this haplotype and reported in\nClinVar (see table column descriptions above).")),(0,r.kt)("h4",{id:"phenotypes"},"Phenotypes"),(0,r.kt)("p",null,"The section on the variant phenotype significance includes the following information:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Clinical significance"))," is the overall clinical significance of all phenotypes that were observed in\nindividuals with this variant and were reported in ClinVar. You can find the description of\nsignificances ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv"},"here"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"ClinVar ID"))," is a variant ID in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),".\nYou can open the variant page in ClinVar by clicking on the link."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Review status"))," is the total assessment of all data sources in which the clinical significance\nof the phenotypes associated with the variant was reported. You can find the description of review\nstatuses ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status"},"here"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele ID"))," is a variant allele ID in ClinVar."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele origin"))," is a total allele origin of the variant in different sources provided by ClinVar."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Cross references"))," are links to a variant in various databases."),(0,r.kt)("li",{parentName:"ul"},"A table with phenotypes that have been observed in individuals with this variant and reported in\nClinVar (see table column descriptions above).")),(0,r.kt)("h3",{id:"gnomad-3-tab"},"gnomAD 3"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(15839).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains the information about allele frequencies of the variant\nfrom the ",(0,r.kt)("a",{parentName:"p",href:"https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/"},"gnomAD v3")," database.\nOn the left, there is a general information about the variant:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele frequency"))," is a total alternative allele frequency of the variant in high quality genotypes.\nFor AF, there are ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"AC")),", that is the alternative allele count in high quality genotypes, and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"AN")),", that is\nthe total number of called high quality genotypes."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Coverage"))," is a median per-base depth of coverage binned to rages ","<","10, 10-100, \u2a7e100."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Number of Homozygotes"))," is a number of individuals homozygous for alternative allele."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XY samples AF"))," is a total alternative allele frequency in XY subpopulation."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XX samples AF"))," is a total alternative allele frequency in XX subpopulation.")),(0,r.kt)("p",null,"On the right, there are variant frequencies in different populations: ",(0,r.kt)("em",{parentName:"p"},"Ashkenazi Jewish"),", ",(0,r.kt)("em",{parentName:"p"},"Amish"),",\n",(0,r.kt)("em",{parentName:"p"},"African and African American (Afr. Am.)"),", ",(0,r.kt)("em",{parentName:"p"},"Finnish"),", ",(0,r.kt)("em",{parentName:"p"},"South Asian"),", ",(0,r.kt)("em",{parentName:"p"},"Latino"),", ",(0,r.kt)("em",{parentName:"p"},"Non-finnish european"),",\n",(0,r.kt)("em",{parentName:"p"},"East Asian"),", ",(0,r.kt)("em",{parentName:"p"},"Middle Eastern")," and ",(0,r.kt)("em",{parentName:"p"},"Other")," (includes the samples that were not uniquely assigned to other\nancestries by the random forest model). For each population, there are total alternative allele frequency (AF)\nin high quality genotypes in the population, AFs in XY and XX subpopulations, and the number of homozygotes\nin the population:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(585).Z,width:"260"})),(0,r.kt)("h3",{id:"gnomad-4-tab"},"gnomAD 4"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(18194).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains the information about allele frequencies of the variant\nfrom the ",(0,r.kt)("a",{parentName:"p",href:"https://gnomad.broadinstitute.org/"},"gnomAD v4")," database.\nThe tab presents three columns with information about allele frequencies: on the left, there is ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"total"))," information\n(in exomes and genomes); in the middle, there is only data in ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"exomes")),"; on the right, there is only data\nin ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"genomes")),". The fields in these columns are identical:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele frequency (AF)"))," is the alternative allele frequency of the variant in high quality genotypes.\nFor AF, there are ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"AC")),", that is the alternative allele count in high quality genotypes, and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"AN")),", that is\nthe total number of called high quality genotypes."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Number of Homozygotes"))," is the number of individuals homozygous for alternative allele. For number\nof homozygotes, ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XY samples"))," and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XX samples"))," are given, that are the numbers of individuals homozygous\nfor alternative allele in the XY subpopulation and the XX subpopulation, respectively."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XY samples AF (XY AF)"))," is the alternative allele frequency in the XY subpopulation. For AF, there\nare ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XY samples AC (XY AC)")),", that is the alternative allele count in high quality genotypes in the XY\nsubpopulation, and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XY samples AN (XY AN)")),", that is the total number of called high quality genotypes in\nthe XY subpopulation."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XX samples AF (XX AF)"))," is the alternative allele frequency in the XX subpopulation. For AF, there\nare ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XX samples AC (XX AC)")),", that is the alternative allele count in high quality genotypes in the XX\nsubpopulation, and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"XX samples AN (XX AN)")),", that is the total number of called high quality genotypes in\nthe XX subpopulation.")),(0,r.kt)("h3",{id:"exac-tab"},"ExAC"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(56936).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains information about the allele frequencies of the variant\nin ",(0,r.kt)("a",{parentName:"p",href:"http://exac.broadinstitute.org/"},"ExAC")," database:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele frequency"))," is a total alternative allele frequency of the variant."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Adjusted AF"))," is an alternative allele frequency of the variant in the population with\ngenotype quality \u226520 and depth of coverage \u226510."),(0,r.kt)("li",{parentName:"ul"},"Alternative allele frequencies in ",(0,r.kt)("em",{parentName:"li"},"African and African American"),", ",(0,r.kt)("em",{parentName:"li"},"American"),", ",(0,r.kt)("em",{parentName:"li"},"East Asian"),", ",(0,r.kt)("em",{parentName:"li"},"Finnish"),",\n",(0,r.kt)("em",{parentName:"li"},"Non-Finnish European"),", ",(0,r.kt)("em",{parentName:"li"},"South Asian")," populations."),(0,r.kt)("li",{parentName:"ul"},"Alternative allele frequencies in ",(0,r.kt)("em",{parentName:"li"},"non-TCGA")," (cohort from which normal tissue samples from cancer patients\nwere excluded) and ",(0,r.kt)("em",{parentName:"li"},"non-Psych")," (cohort from which normal tissue samples from patients with mental disorders\nwere excluded) cohorts: a total alternative allele frequency (AF) in cohort, AF in cohort of the population with\ngenotype quality \u226520 and depth of coverage \u226510 (adjusted AF), and AFs in cohort of different populations:\nAfrican and African American, American, East Asian, Finnish, Non-Finnish European, South Asian.")),(0,r.kt)("p",null,"For each frequency, there is an alternative allele count in high quality genotypes (",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"Allele count")),"):"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(60809).Z,width:"170"})),(0,r.kt)("h3",{id:"other-frequencies-tab"},"Other Frequencies"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(51412).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains information about the allele frequencies of the variant\nfrom ",(0,r.kt)("a",{parentName:"p",href:"https://www.internationalgenome.org/"},"1000 Genomes")," (1000G), ",(0,r.kt)("a",{parentName:"p",href:"https://www.uk10k.org/"},"UK10K"),"\nand ",(0,r.kt)("a",{parentName:"p",href:"https://esp.gs.washington.edu/drupal/"},"NHLBI GO Exome Sequencing Project")," (ESP) projects."),(0,r.kt)("p",null,"From ",(0,r.kt)("strong",{parentName:"p"},"1000 Genomes project"),", there are:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele frequency"))," is a total alternative allele frequency of the variant;"),(0,r.kt)("li",{parentName:"ul"},"Alternative allele frequencies in East Asian, European, African, American, South Asian populations.")),(0,r.kt)("p",null,"From ",(0,r.kt)("strong",{parentName:"p"},"UK10K project"),", there are:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Allele frequency"))," is a total alternative allele frequency of the variant in combined genotypes\nin UK10K cohort (TwinsUK + ALSPAC);"),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"TWINSUK AF"))," is an alternative allele frequency in TwinsUK, that is a cohort of adult twins living in the UK;"),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"ALSPAC AF"))," is an alternative allele frequency in ALSPAC, that is a cohort of fetuses.")),(0,r.kt)("p",null,"From ",(0,r.kt)("strong",{parentName:"p"},"ESP project"),", there are alternative allele frequencies in African American and European American populations."),(0,r.kt)("p",null,"For frequencies from UK10K and ESP projects, there is an alternative allele count in high quality\ngenotypes (",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"Allele count")),"):"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(6672).Z,width:"170"})),(0,r.kt)("h3",{id:"conservation-tab"},"Conservation"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(13562).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains the information on the variant conservation:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Ancestral allele"))," inferred from\nthe ",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/info/genome/compara/epo_anchors_info.html"},"Enredo-Pecan-Ortheus (EPO)")," multiple\nalignments using a probabilistic method Ortheus. ",(0,r.kt)("strong",{parentName:"li"},"ACTG")," - high-confidence alleles, ancestral state supported\nby the other two sequences; ",(0,r.kt)("strong",{parentName:"li"},"actg")," - low-confidence alleles, ancestral state supported by one sequence\nonly; ",(0,r.kt)("strong",{parentName:"li"},"N")," - failure, the ancestral state is not supported by any other sequence; ",(0,r.kt)("strong",{parentName:"li"},"-")," - the extant species\ncontains an insertion at this position; ",(0,r.kt)("strong",{parentName:"li"},".")," - no coverage in the alignment."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Altai Neanderthal"))," is a genotype of a deep-sequenced Altai Neanderthal."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Denisova"))," is a genotype of a deep-sequenced Denisova."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Vindija Neanderthal"))," is a genotype of a deep-sequenced Vindija Neanderthal."),(0,r.kt)("li",{parentName:"ul"},"Conservation scores for the variant position in the genome estimated\nusing ",(0,r.kt)("a",{parentName:"li",href:"http://compgen.cshl.edu/fitCons/"},"fitCons")," method. fitCons is a computational method that estimates\nthe likelihood of a mutation at any genomic position to affect fitness. It integrates functional assays\n(such as ChIP-Seq) with selective pressure inferred using the INSIGHT method. The result of the prediction\nis a score \u03c1 in the range ","[0-1]"," that indicates the fraction of genomic positions evincing a particular pattern\nof functional assay results, that are under selective pressure.\nFrom fitCons results, there are ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Integrated FitCons rankscore")),", that is an integrated score obtained by\ncombining scores from three cell lines (HUVEC, H1 hESC and GM12878), and ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"GM12878 FitCons rankscore")),",\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"H1-hESC FitCons rankscore")),", ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"HUVEC fitCons rankscore")),", that are conservation scores in GM12878, H1 hESC\nand HUVEC cell lines, respectively. For each rankscore, there are also score and confidence value."),(0,r.kt)("li",{parentName:"ul"},"Conservation scores for the variant position in the genome estimated\nusing ",(0,r.kt)("a",{parentName:"li",href:"http://mendel.stanford.edu/sidowlab/downloads/gerp/index.html"},"GERP++"),' program. GERP (Genomic Evolutionary\nRate Profiling) identifies constrained elements in multiple alignments by quantifying substitution deficits.\nThese deficits represent substitutions that would have occurred if the element were neutral DNA, but did not\noccur because the element has been under functional constraint. GERP authors refer to these deficits as "Rejected\nSubstitutions" (RS). RS score is defined as the number of substitutions expected under neutrality minus the\nnumber of substitutions "observed" at the position. From GERP++ results, there are ',(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"GERP++ RS rankscore")),",\nneutral rate and score."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhyloP vertebrate conservation rankscore")),", ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhyloP mammalian conservation rankscore")),"\nand ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhyloP primate conservation rankscore")),", and corresponding scores are conservation scores\nderived from alignment of vertebrate, mammalian, and primate species, respectively, by\n",(0,r.kt)("a",{parentName:"li",href:"https://ccg.epfl.ch/mga/hg19/phylop/phylop.html"},"PhyloP")," (phylogenetic P-values) program.\nInterpretations of the scores are compared to the evolution that is expected under neutral drift."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhastCons vertebrate conservation rankscore")),", ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhastCons mammalian conservation rankscore")),"\nand ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"PhastCons primate conservation rankscore")),", and corresponding scores are conservation scores\nderived from alignment of vertebrate, mammalian, and primate species, respectively, by\n",(0,r.kt)("a",{parentName:"li",href:"http://compgen.cshl.edu/phast/phastCons-HOWTO.html"},"phastCons")," program.\nPhastCons is a program for identifying evolutionarily conserved elements in a multiple alignment,\ngiven a phylogenetic tree. It is based on a phylogenetic hidden Markov model."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"SiPhy rankscore"))," and score are conservation scores for the variant position in the genome estimated\nby ",(0,r.kt)("a",{parentName:"li",href:"http://portals.broadinstitute.org/genome_bio/siphy/documentation.html"},"SiPhy")," using a simple generalization\nof the substitution pattern estimation."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"BStatistic rankscore"))," is a conservation score for the variant position in the genome,\nderived from analysis of the genomic distributions of human polymorphisms and sequence differences\namong five primate species relative to the locations of conserved sequence features\n(you can read more ",(0,r.kt)("a",{parentName:"li",href:"//doi.org/10.1371%2Fjournal.pgen.1000471"},"here"),").")),(0,r.kt)("h3",{id:"protein-function-effect-tab"},"Protein function effect"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(8533).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains the results of the prediction of amino acid substitution effect on protein function,\nobtained using the following algorithms and programs:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://sift.bii.a-star.edu.sg/"},"SIFT"))," (Sorting Intolerant From Tolerant) predicts whether\nan amino acid substitution affects protein function basing on sequence homology and the physical properties\nof amino acids. The result of the prediction is ",(0,r.kt)("strong",{parentName:"li"},"rankscore")," and ",(0,r.kt)("strong",{parentName:"li"},"score"),", the values of which\nvary from 0 to 1. An amino acid substitution is considered damaging if score \u2264 0.05 and tolerated if score > 0.05."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://genetics.bwh.harvard.edu/pph2/"},"PolyPhen"))," (Polymorphism Phenotyping) predicts possible\nimpact of an amino acid substitution on the structure and function of a protein using\nsequence homology and different databases and tools. The result is:",(0,r.kt)("br",null),"1. ",(0,r.kt)("strong",{parentName:"li"},"prediction"),":\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Benign"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),"); ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Possibly damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),"); ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Probably damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),");",(0,r.kt)("br",null),"2. ",(0,r.kt)("strong",{parentName:"li"},"score")," whose values vary from 0\nto 1: the lower, the more likely that the variant is benign, the higher - damaging."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://sift.bii.a-star.edu.sg/sift4g/AboutSIFT4G.html"},"SIFT4G"))," (SIFT Databases for Genomes) is a\nfaster version of SIFT. The result of SIFT4G is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),"), or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752137/"},"LRT"))," (Likelihood Ratio Test)\nidentifies deleterious mutations that disrupt highly conserved amino acids within protein-coding\nsequences basing on the evolutionary models of the DNA sequence. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),":\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Neutral"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),"), or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Deleterious"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"; ",(0,r.kt)("strong",{parentName:"li"},"omega"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://fathmm.biocompute.org.uk/"},"FAtHMM"))," (Functional Analysis through Hidden Markov Models)\npredicts the functional consequences of both coding variants and non-coding variants\nbasing on sequence homology. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),"), or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://spliceailookup.broadinstitute.org/"},"SpliceAI"))," is a deep neural network that predicts\nthe probability of the position to be a splice acceptor or donor from a pre-mRNA sequence.\nThe result is: ",(0,r.kt)("br",null),"1. ",(0,r.kt)("strong",{parentName:"li"},"Delta score")," (DS) is the probability of the variant being splice-altering\nat any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default).\nDelta scores range from 0 to 1:",(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Acceptor gain"))," is the probability of a position within a\ngiven interval to be splice acceptor; ",(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Acceptor loss"))," is the probability of a position within a\ngiven interval not to be splice acceptor; ",(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Donor gain"))," is the probability of a position within a\ngiven interval to be splice donor; ",(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Donor loss"))," is the probability of a position within a\ngiven interval not to be splice donor. ",(0,r.kt)("br",null),"DS is defined as the maximum of these four scores.\nValues 0 ","\u2264"," DS < 0.2 are colored ",(0,r.kt)(o.Z,{color:"#141E29",mdxType:"Highlight"},"black"),",\n0.2 ","\u2264"," DS < 0.5 - ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),",",(0,r.kt)("br",null),"0.5 ","\u2264"," DS <\n0.8 - ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),", 0.8 ","\u2264"," DS ","\u2264","\n1 - ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),". ",(0,r.kt)("br",null),"2. ",(0,r.kt)("strong",{parentName:"li"},"Delta position")," conveys information about\nthe location where splicing changes relative to the variant position (positive values are downstream of\nthe variant, negative values are upstream). Delta positions corresponds to the DS\nscores.",(0,r.kt)("br",null),"3. Link to ",(0,r.kt)("a",{parentName:"li",href:"https://spliceailookup.broadinstitute.org/"},"SpliceAI Lookup")," website for\nmore detailed information about the scores for the variant."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://cadd.gs.washington.edu/"},"CADD"))," (Combined Annotation-Dependent Depletion) scores the\ndeleteriousness of a variant by integrating multiple annotations into one metric by contrasting variants\nthat survived natural selection with simulated mutations. The result is: ",(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},"Raw"),' score has\nrelative meaning and is interpretable as the extent to which the annotation profile for a variant\nsuggests that the variant is likely to be "observed" (negative values) vs "simulated" (positive values).\nHigher values indicate that a variant is more likely to have deleterious\neffect. ',(0,r.kt)("br",null),"- ",(0,r.kt)("strong",{parentName:"li"},"Phred")," is a Phred-scaled score. Phred < 10 is\ncolored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),", 10 ","\u2264"," Phred <\n20 - ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),", Phred ","\u2265","\n20 - ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html"},"MaxEntScan"))," predicts\nthe fitness of a variant position as either a donor or an acceptor splice site based on a maximum entropy model.\nThe result of prediction is scores for alternative (",(0,r.kt)("strong",{parentName:"li"},"Alt score"),") and reference (",(0,r.kt)("strong",{parentName:"li"},"Ref score"),") alleles necessary\nto predict the loss of a native splice site for a variant. The difference between these scores (",(0,r.kt)("strong",{parentName:"li"},"Diff"),") is also\ngiven.",(0,r.kt)("br",null),"Alt score < -19 is colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),", -19 ","\u2264","\nAlt score < -3 - ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),", Alt\nscore \u2265 -3 - ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://mutationassessor.org/r3/"},"MutationAssessor"))," predicts the functional impact of variant amino acid\nsubstitution in proteins basing on evolutionary conservation of the affected amino acid in protein homologs.\nThe result is:",(0,r.kt)("br",null),"1. ",(0,r.kt)("strong",{parentName:"li"},"Prediction"),": predicted functional, i.e. ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"High")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),") or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Medium")),"\n(colored ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),"), or predicted non-functional, i.e.\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Low"))," or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Neutral"))," (both are colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),").",(0,r.kt)("br",null),"2.\n",(0,r.kt)("strong",{parentName:"li"},"Score"),". Score \u2265 3,5 corresponds to the high impact, 3,5 < Score ","\u2264"," 1,935 - to the medium impact,\n1,935 < Score ","\u2264"," 0,8 - to the low impact, 0,8 < Score - to the neutral impact.")),(0,r.kt)("p",null,"Additional results of the prediction of amino acid substitution effect on protein function are located\non ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"Protein function effect (additional)"))," tab:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(48580).Z,width:"1000"})),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://bjfenglab.org/BayesDel"},"BayesDel"))," is a variant deleteriousness meta-score. It works for coding and\nnon-coding variants, single nucleotide variants and small indels. The higher the score, the more likely the variant\nis pathogenic. The result is:",(0,r.kt)("br",null),"1. ",(0,r.kt)("strong",{parentName:"li"},"addAF prediction")," is a prediction of deleteriousness meta-score\nwith inclusion of MaxAF (the maximum allele frequency across populations): ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"). For the prediction value, ",(0,r.kt)("strong",{parentName:"li"},"addAF score")," and ",(0,r.kt)("strong",{parentName:"li"},"addAF rankscore"),"\nare given, on the basis of which the prediction was made. The score cutoff between damaging and tolerated prediction\nvalues is 0.0692655.",(0,r.kt)("br",null),"2. ",(0,r.kt)("strong",{parentName:"li"},"noAF prediction")," is a prediction of deleteriousness meta-score\nwithout inclusion of MaxAF: ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"). For the prediction value, ",(0,r.kt)("strong",{parentName:"li"},"noAF score")," and ",(0,r.kt)("strong",{parentName:"li"},"noAF rankscore"),"\nare given, on the basis of which the prediction was made. The score cutoff between damaging and tolerated prediction\nvalues is -0,0570105."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://provean.jcvi.org/index.php"},"PROVEAN"))," (Protein Variation Effect Analyzer) predicts whether\nan amino acid substitution has an impact on the biological function of a protein. The result\nis ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Neutral"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5270233/"},"Meta SVM")),"\nis a meta-analytic support vector machine (SVM) that accommodates 10 component scores (SIFT, PolyPhen-2 HDIV,\nPolyPhen-2 HVAR, GERP++, MutationTaster, MutationAssessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum\nfrequency observed in the 1000 genomes populations. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),":\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"; ",(0,r.kt)("strong",{parentName:"li"},"reliability index"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://academic.oup.com/hmg/article/24/8/2125/651446"},"Meta LR"))," is a meta-analytic\nlogistic regression (LR) that accommodates 10 component scores (SIFT, PolyPhen-2 HDIV,\nPolyPhen-2 HVAR, GERP++, MutationTaster, MutationAssessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum\nfrequency observed in the 1000 genomes populations. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),":\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"; ",(0,r.kt)("strong",{parentName:"li"},"reliability index"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://bejerano.stanford.edu/mcap/"},"M-CAP"))," (Mendelian Clinically Applicable Pathogenicity) is a\npathogenicity classifier for rare missense variants in the human genome that is tuned to the high sensitivity\nrequired in the clinic. It combines various pathogenicity scores (including SIFT, Polyphen-2 and CADD)\nwith novel features and a powerful model. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging")),"\n(colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://academic.oup.com/bioinformatics/article/25/21/2744/227925"},"MutPred"))," classifies an\namino acid substitution as disease-associated or neutral and predicts molecular cause of disease.\nThe result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),", ",(0,r.kt)("strong",{parentName:"li"},"score"),", ",(0,r.kt)("strong",{parentName:"li"},"transcript ID")," (from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"UniProtKB"),"),\n",(0,r.kt)("strong",{parentName:"li"},"amino acid change"),", ",(0,r.kt)("strong",{parentName:"li"},"top 5 features"),". Higher score values indicate that a variant is more\nlikely to have deleterious effect."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.nature.com/articles/s41467-020-20847-0"},"MVP"))," (Missense Variant Pathogenicity)\nis a prediction method that uses deep residual network to leverage large training data sets and\nmany correlated predictors. The result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore")," and ",(0,r.kt)("strong",{parentName:"li"},"score"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.biorxiv.org/content/10.1101/148353v1"},"MPC"))," (Missense badness, PolyPhen-2, and Constraint)\nis a ",(0,r.kt)("em",{parentName:"li"},"de novo")," missense deleteriousness metric based on identifying sub-genic regions that are depleted\nof missense variation. The result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore")," and ",(0,r.kt)("strong",{parentName:"li"},"score"),". Higher score values indicate that\na variant is more likely to be pathogenic."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://illumina.github.io/NirvanaDocumentation/3.14/data-sources/primate-ai/"},"Primate AI")),"\nis a deep residual neural network for classifying the pathogenicity of missense mutations\nbased on the common missense variants in non-human primate species. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),":\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Tolerated"))," (colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"score"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570203/"},"DEOGEN2")),"\nis a method that predicts whether a protein-coding variant likely affects the carrier individual's health.\nThe method incorporates heterogeneous information about the molecular effects of the variants,\nthe domains involved, the relevance of the gene and the interactions in which it participates.\nThe result of the prediction is ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://cbcl.ics.uci.edu/public_data/DANN/"},"DANN"))," (Domain-Adversarial Neural Network)\nis a deep neural network trained on the same feature set and training data as CADD.\nThe result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore")," and ",(0,r.kt)("strong",{parentName:"li"},"score"),". Higher score values indicate that a variant is more\nlikely to have deleterious effect."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://fathmm.biocompute.org.uk/fathmmMKL.htm"},"FAtHMM-MKL"))," (Multiple Kernel Learning)\nis a machine learning approach that integrates functional annotations from ENCODE with nucleotide-based\nsequence conservation measures. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Neutral")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"p-value"),"; ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"; ",(0,r.kt)("strong",{parentName:"li"},"group of features"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://fathmm.biocompute.org.uk/fathmm-xf/"},"FAtHMM-XF"))," (eXtended Feature)\nis a method for predicting pathogenic point mutations in the human genome based on an extensive feature set.\nIt outperforms competitors particularly in non-coding regions. The result is ",(0,r.kt)("strong",{parentName:"li"},"prediction"),": ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Neutral")),"\n(colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"green"),") or\n",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"Damaging"))," (colored ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"); ",(0,r.kt)("strong",{parentName:"li"},"p-value")," (lower p-value indicate\nthat a variant is more likely to have deleterious effect); ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"http://www.columbia.edu/~ii2135/eigen.html"},"Eigen"))," is a spectral approach integrating functional\ngenomic annotations for coding and non-coding variants. The result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore")," and ",(0,r.kt)("strong",{parentName:"li"},"score"),"."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4731313/"},"Eigen-PC"))," is a conceptually simpler meta-score,\nwhich is based on the decomposition of the annotation covariance matrix, and using the lead vector\nto weight the individual annotations. The result is ",(0,r.kt)("strong",{parentName:"li"},"rankscore"),", ",(0,r.kt)("strong",{parentName:"li"},"score")," and ",(0,r.kt)("strong",{parentName:"li"},"score in phred scale"),".")),(0,r.kt)("h3",{id:"other-tab"},"Other"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(7817).Z,width:"1000"})),(0,r.kt)("p",null,"The tab contains additional information about the variant:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Coding sequence position")," is a genomic position of a variant reference allele."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Protein position")," is a reference amino acid position in protein."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Amino acids"),": reference amino acid / new amino acid resulting from the substitution."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Codons")," is an amino acid substitution written as codons (e.g. GAG/CAG = E/Q (Glutamic acid -> Glutamine))."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Amino acid position")," is a reference amino acid position in protein."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("a",{parentName:"strong",href:"https://appris.bioinfo.cnio.es/#/"},"APPRIS")," annotation")," is an annotation of principal splice isoforms of human.\nThe isoform for each gene is selected as the principal one by combining protein structural information,\nfunctionally important residues and evidence from cross-species alignments.\nPossible annotation values: PRINCIPAL1-5 (principal isoforms, with 1 being the most reliable),\nALTERNATIVE1 (candidate transcript model conserved in at least three tested non-primate species),\nALTERNATIVE2 (candidate transcript model conserved in fewer than three tested non-primate species)."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"Geuvadis EQTL target gene")," is target gene IDs from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ensembl.org/index.html"},"Ensembl"),"\nidentified by the analysis of expression quantitative trait loci (eQTL)\nin ",(0,r.kt)("a",{parentName:"li",href:"https://www.biorxiv.org/content/10.1101/008797v1"},"GEUVADIS")," (Genetic European Variation in Disease) project."),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("strong",{parentName:"li"},"ENIGMA clinical significance")," is a clinical significance of the variant if it's located in the\nknown or suspected breast and/or ovarian predisposition genes (such as ",(0,r.kt)("em",{parentName:"li"},"BRCA1"),", ",(0,r.kt)("em",{parentName:"li"},"BRCA2"),")\ndetermined by ",(0,r.kt)("a",{parentName:"li",href:"https://enigmaconsortium.org/"},"ENIGMA")," consortium.\nBenign significance is colored ",(0,r.kt)(o.Z,{color:"#219653",mdxType:"Highlight"},"dark green"),"; Likely\nbenign - ",(0,r.kt)(o.Z,{color:"#0EB555",mdxType:"Highlight"},"light green"),"; Likely\npathogenic - ",(0,r.kt)(o.Z,{color:"#F2994A",mdxType:"Highlight"},"orange"),";\nPathogenic - ",(0,r.kt)(o.Z,{color:"#E95726",mdxType:"Highlight"},"red"),"; all other values, including ones mixed from\ndifferent clinical significances, are colored ",(0,r.kt)(o.Z,{color:"#141E29",mdxType:"Highlight"},"black"),".\nWhen hovering over a field value, you will see a comment on clinical significance.")),(0,r.kt)("h3",{id:"custom-annotation-sources-tab"},"Custom annotation sources"),(0,r.kt)("p",null,"The tab is present in the details panel if at least one custom annotation\nwas ",(0,r.kt)("a",{parentName:"p",href:"/settings/custom-annotations#add-custom-annotation"},"added")," to the system\nbefore the variants in the sample were annotated. The tab contains the column values from the\ncustom annotations that the variant was annotated with:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(21074).Z,width:"1000"})),(0,r.kt)("p",null,'Deleted annotation data are marked "',(0,r.kt)("em",{parentName:"p"},"deleted"),'" for variants that were annotated with\nit before it was deleted.'))}g.isMDXComponent=!0},35690:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFYAAABQCAYAAACDD4LqAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAOKSURBVHhe7ZpBjhUxDEQ5H7fgFpyCPXvEBWbJgguwYscBkFiPGIEQqFENtKYlnMROXLa/SEm1GI1+T/J+dexY8+zYomiDJWmDJWmDJWmDJak82J+fPx4/Pr07vn9482j8/Ov+y9/f1lVZsID59e2L4/71c9EPdy8fIVdVObBII6BJMCV/e/+qZIJLgUUCJXgjI9nV4JYBCzASNK2rJbcMWMvr3zKKWxWVAItCJYGacZXUlgArAZp1ldSmgwUICdCKK6Q2FexqwWoZ53W2UsEy0noa53am0sCy0no6u7dNA+vRXo2cWchSwM7esGacldoUsL3hirezUhsO1lqwcGSgECF5SDo+b/1iMlIbCtZasHD/bwm/kz4jufcclkLBWtKq6UUtyY2e3YaBtaZVA8LyRUW3X2FgLa8urJG1u4gsZCFgrWlFujSaeW5UakPAWi8DLLBwVGrpYGdmrUywcERq6WCljY3MBhuRWipYS9W+mg0WZqeWBnZl0xFg2TNbGtjZtMIRYGHmzJYCdnXDUWDxd/AMhihgV2etUWBhViFzB+sxa40ECzNS6w4WUKTFWxwNlpFaV7ArBevqaLCwd2rdwHpuMgOs98zWDaxXWuEMsLDnzNYFrPcGs8Di7+KZHnIBa521jpwFFvYqZMtgGZvLBOuV2mWwq5cByZlgYY/ULoH1/L/Wq7PBwqupXQIrLcjDFcCupnYarGd7JVkjJlh4JbVTYNkbgjWbYn+5KzPbKbDsDcGaVxFHhvRZT8/ObM1gI9J6urcp79655dn2ywyW0V71DIC4ap6bA+zoNcwUMhNYj1nrrdqaWhPYiDOtqq2pVYONKFjVbUmtCmxkwapsy8xWBXan9cname0QbHZa0QGgE8A6sCk486zXtl9DsFH9ouReOjLXpSlkXbCZadVcJ7OSq0ltF2x0I3615ixjjS01HqW2CTZz0bBGgC99Nsq91DbBZl8GNMo8quBeakWw2QuGR2cYVKENbK1TBFthwZrKm/1Wwa0JnAg2s2hd3Vo0lNluXd26jYlgpQdkGQtHkTpfOcCu8sXDJrAVXrFbcavf3mAX3aoFItgq59ct2AQ2+3JwS25JBAvt42DsXkvYBFuhl61sBK+nJlhow217NCTqgoU23H+tmbwNwUIoZvvM/dOzaqBCKrCn8FA8HJD/F9DY73n7s8gEdkuvDZakDZakDZakDZai4/gNwz9pnVGlZ00AAAAASUVORK5CYII="},69829:function(e,t,n){t.Z=n.p+"assets/images/occurrence-17afdb45dcb88ade1120442a02490bb0.png"},585:function(e,t,n){t.Z=n.p+"assets/images/af_example-ae9a0de8dacb18db22aa1a667c6b5c9f.png"},60809:function(e,t,n){t.Z=n.p+"assets/images/allele_count-c3e86c5bde0ebd24644505b0f45612b1.png"},6672:function(e,t,n){t.Z=n.p+"assets/images/allele_count_uk10k-bed4a3c4a7d84f08014666024fb8c65b.png"},69239:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAiQAAAC3CAYAAAAinkW0AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAABOISURBVHhe7d17c9NIFsbh+f4fa4CBhUnIhZgkBHJ1CBBCCIH5U6tXVHY9yrHcp3U5sv1T1VO1UxtasqOoX/X1j4KDg4ODg4ODI/ggkHBwcHBwcHCEHwQSDg4ODg4OjvCDQMLBwcHBwcERfvxxcvapmOf65g4AAKBTVuYgkAAAgEFZmYNAAgAABmVlDgIJAAAYlJU5/ri7uyse1P/Pf/75BwAAoJV6IJnNHg8IJAAAoFcEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAjdPv9rjg9uyz2JkfF0dFZ8enTtflzALAqCCTAiNzd/Si2tvaLP59sFH/++W/PX2wXl5f8zQFYTQQSjNL373fFly9fi5PT6SMXF1fFt2/fi/v7e/PfLit9ppcvdx8FkVlPnm4Up+V3YP17AFhmBBKE+/nzVzGdfirevHlXPH++bbYOzPP06Wbx8tVucXh4UtzcfDPL79rd3X0xmRwVO7uHC52ffzTLqPv161exs3Ngfsa6Z39tlWHtxixnmV1efja/w3kiW4tub78Xb/bS7gE5Ojotf8d2WQB+I5AgjCrV7e2D6q3fqnhzqLI+ODyu7lXrnF1QZfT8+ZZ5/rqDg2OzjDp9F8/+em2WYXn79r1ZzjJToPwr8XuV16/fVkHOKqtvx8cX5jXNo0BilQPg/wgkGNzV1ZeqVcPTEuKlkKMWF92z1jW00UcgOTu7NP/9PH9v7BX39z/NspaVwoVChvV5LQovQ7WKzfJep4LmKrZoAV0jkGAwGiOxqQd5j0GkTl06796dmNeTawyB5NXfb1ZuDI2cnExd94daKqxy+uRtydnYnBQ/f65WeAT6QCDBIM7OL4tnz9K7JLqmFoXb225aS/oIJNPplasiVleXVc6yW4Zum2UITcAyIpCgVxqwur//YdBWkXkUIrpYz6OPQKKBslU3llHGI+V3ucozbRS2zM9tGLrbRuGnmpZtXIslqlsJWEYEEvTm/ufP5JkjQ1ErzcePX8zrTdVHIBF126QM8NWsDQU9q4xVUHVfjbQFwtuCo/ASNfAWWDYEEvRCFaYqTushHU2hRANrretO0VcgEU0Tfvps0yxLlfTW1tvy73D1xo7M0iq1L15s29+BYchuG1d3TflzCldWOQAeI5CgcwojuwojjrfcoanCu7m5Na9/kT4DiejvTNN6tTKrWkwUoDY3J61C1LLZfZMeZofqFvF212hNHQ3ktsoC8BiBBJ3T+IZWa4uUQUaBQQtP6Y1UFbFCgEwvPxXv358VW9v71Zoj5r9PpEpe3UrWZ2jSdyCBc5Bv+XO6T6xyuuTtrlGossoBYCOQoFMaNJo7m0b/zruomd5AFVw0vdcqs1FZkWnjOqvcJgSS/mlPn0XL6M8aYqyGt7vm/OLKLAeAjUCCzqi1Qa0O5gO6gcKEgkhOa8WD3DVO1C3ibe4nkAxjMnlvfqeWvrttvN01L/6zU+3HZJUFwEYgQWeOj8/9gaCsSLocG/Hhw7m7u2jiXIadQDKMj+V9kdzyVd53fXbbeLtrFKascgDMRyBBJ/Q2qLdC6+E8j34+d2BpE1VMnlDifbsmkAzDtTZLqc9uG89UZIWoyI3/gGVFIEEntHmY9XCep80slxTqAvK01ng2P1v3QKJZVJqaq26yi4ur/w067qOLQt+f9b1a+uy28SzWphDV19RsLUGv71o0wFtBSX9H+u9lnA6ua9b9c3I6reh+0n216lPbYSOQoDXvm+wQb5Da52VjY888v2VjI33GTV+BRANsVe4ir169Sa789XP6eaucWYvKVCWhri1NZW0KemqZ0r2g7rsu9trRIOm567LU9dRto89efW7rnIYud2JWwDo8PKm+06Tuq/I70LgobSypmUp97KGjQeTWPVQ3b8C46pG9yVHj4PdV3asJzQgkaE2zCTytEd4xG7kuygdyateNZ0fWvgJJamuAzq1rsMqoS73WeWXmDhYWVaB7ZeX140d+xaKQqLBolW/po9vG1V1Thqe22xOoItaGkG2ntYvuf7XuXF93t9twamtR/d5Xy5paLlOCFYFkPRFI0JqnOVtvb6rkrHK65q3MUpcgX5dA8uH4PO2tfAFVrNNp/rPC0x3YR7eN5/72tLTVqcLWZ82dNt+oDFQKll387eUEErWivt4qg63xcxYCyXoikKAVb3O2Z6xGF47en5nXYVEzt1VG3aoHElWM6nbIaRWZR2/qmgFVv74UarlSC5ZV7iPlNXfZbTPU/a0QpW4zq8wuqQWn7XL23kCiMKLdtq2fmYdAsp4IJGjFs6Jm32tFWDxjEDY2J0l97qseSLwDglMplOTsUqwuGO1XY5Vp6bLbxtNdo5aNL1++muU00fTmLrpnUun30ObFwBtIUu/rWQSS9UQgQSuexav0ILPK6JOaz7Vh3cMo/iaatZBSka1yINFg4y66aeZRpZ2z7oy606zyLF3uIePqrkkMtLP0XfTSRbNAbjgUTyD53brlD1sEkvVEIEE2PXz1ELYeKI+Ub5ltm4rHYlUDyfX1t6oisP7/Lmn2k7eyUcta8sJkHd1r3u6a1DFIDzRd17OrcdcUFFIHcs/yBBLPJomzCCTriUCCbFqLQoNUrQdKnWcWy9itYiBRZa+ZHX101TxSnsP7dq6WK8/S7V20xnm6a7zdkRqn42l9+ZfymhQm9Ht9kNuqpYGmuhbrGudJve6dnYOke89CIFlPBBJk+/Q5fSM9DdhrM/1zTFYxkCyi37P2KdrZPaxokGLy+iCGnNkonoDQRbeNJzB4x614pqRXys+9VV7P5/Jvbl6AUCDaLX83qd+RKMhoDItV3jzZQcqBQLKeCCTIprEZqQ+/Lt5Yx2JdAokqK60jopYwq2xRq1e1oaKjEpSc9TpcXSjl9bTpttFnTu5OKc/lmdnjnY6u359n3I13kGzq7LIHbQOJWlU1cFpdhLNjbvS/db9qLJemnHvH42D5EUiQzTPQ0FMxj906BBJt/e/pgshZs0RTsq2ymnjGJLQJwZ7F/rytMZ5NA3O3WPC0wOh3fXf3wyzHkhtI1Mp2fOIbZ4P1QiBBttQKVLwD/sZs1QOJmstvb/370mhciKcbYmt73yynSZ9BYZYn+OhnrTLmSZ2ZljsjSTxjbrzju3ICSe49hfVCIEE2TyBZlRk2ssqBRJVg7tLn3oGaOeOKXLtKZ3bbeLtrFJKscubR96SgpI3k1HWh70znq7eatN0TJ7kFs/wMWk/IKsPiDSS5rTxYPwQSZNMoeusBZCGQLDaGQNJ2nyFPd4Tnc8zqe+0bTyuMKtumMTZeD2FFS+17ulEsal1J/V14/j49gURjhXJbebB+CCTIlvpgyhnAOGarGkjUOpKz0ugszYxQ87xVfl3VGvPZf194Fm/L6bbxdNd4B4QOybNKcV+BRC8tXa2ai9VHIEE2Aon9eWctUyDpaqqlKmmr/Lrc+0J7o2g7fqvMR5zdNp7uGoUihSOrnDHwBLdeAomzKwggkCDbuj6YVjWQ6G3WKsMr9XO0CarV5n9GmRbdp1YZFk93jUKRwpFVTjR1/Wi9GOu6LX0EEi0WlzuoGOuJQIJs+4kVj3geeGO3qoHEc61N9Lu2yq9rE0g83RGebhtPd03bQad90KBfrYmiqbzWNc/TRyBhcTN4EUiQLbUCFQLJYgSSdJ6xKmrxSJkJ45nBo64Q7wqnXVCLzMMMHc2iUfeYFqZTa4Rr5dcaAgnGgECCbFrYynoQWbqq7MaAQNJsiEAi2kLfKteSslaIuhVTu2v6rmzV5aK1QTQtWIGjWnk18dpyEEgwBgQSZEuteGTMsxG8CCTNhgokmhGkmTpW2XUp03NTB+OKwpBVRhuqvI+Pz393t/QYPiwEEowBgQTZPOscbJRveauyNwWBpNlQgUT3k+4rq+xHFnTbeLprupgePUuLw2nPoNS/pT4QSDAGBBJkU1+2+q6th1GdNtTqcgGpSASSZkMFEvHsp9TUbePprukyXGtvl9RWnj4RSDAGBBJk05udlv+2HkZ1Y1+zwYNA0mzIQKINAFNDcVO3jau7JmNTwDqNEdnVtNweu2aeqMUlsXwCCcaAQIJWPMvHj3GaZA4CSbMhA4lnE7l53Tae7hqFH88uyBbvGiEeam3Z2npbzcKplvEvv2Pr5+oIJBgDAgla8TSZRywkpQpZrTiqlBfZfP02abM3AkmzIQOJaN2N1JYAq9vG012j8NN2KXTP9ZrKf6tgpNk3e5Oj4uR0Wlxff3vUjaTvlkCCZUIgQSuamqjty60H0iNz3lD75KlsXpeBJKWyIZA0GzqQaCyTFj+zzlFnddskd9eU95HCxOy/9XLtVvygPO/fG3vVDBzPyqcEEiwbAgla0QNHDx7rgWTRW939gLNtPGMDUitkAkmzoQOJpFaS9VDsCQg5G/XVuVpHyp9Tq13uOQkkWDYEErTmWaBKq0l6Hn5teAY8egbdEkiaRQQShYzUin6228azAZ0q4tlzeqn1Ta1wVtmPlJ9Fi6JZ5aQikGDZEEjQmqfiF72R6s3UKqtLE8cGbJ5rIpA0iwgknl16Z3/Xk0niPVJrWcnhaY1Rpa/Br1Y5qQgkWDYEEnTCsymZaJZB2wduEy3a5lnfwTMDiEDSLCKQiGcsiMYW3d39SN6ELmWl10VSx1t1NUXe0/pDIMEYEEjQCc/bWEVN0h1VgHU3N7fJb8viXXmTQNIsKpB4KmCFF+/PW+f0SP0bUWtj27Eq4ulKJZBgDAgk6IxnAGmlDCVqmeiypeTr12/JzeIPtJaKZyongaRZVCDxtHhoLFNqGOmqxSI1kHh+v/Noer2m2VvlWwgkGAMCCTqjMOAZS/JAUxq7eCO8mF793hXVOMc8OZUigaRZVCARBVzrXG0o5CjsWOfzSA0k6tZR945VRirvWicEEowBgQSd0loJngfhA72Fvt3/UN1jVrlN9PDWdOKc82rgq1VmEwJJs8hAUq1OmtjykaqrFYaTB3+X93Gb9U68XZZCIMEYEEjQqdbLYpcP40WLQGlFSo35UAWqTfvMchL8fvP1ByACSbPIQKIKUBWhdb4cCjcKOda5vDyzbHLvzZwuSyGQYAwIJOhczhtaE/X3682y6o7JaAWxaCCrZuJY178IgaRZZCCR1O8xRdeVamplLjq3pyuzzc7BBBKMAYEEvVA3SpvWiz4p4Jye5jeJE0iaRQcStZ7lVsx1mqlinSOXd2yHWmje7B1VLR9WeQos2n247d8agQRjQCBBb64+fXEPMu2bwsiHD+fm9aYikDSLDiTammBjY2Ke08M7HTyFZwG3R8ogo78n/T5TWgt1rz9JHE9DIMEYEEjQqzG1lOhts+3maEIgaRYdSEStBtY5PRRq+th36eiovLaOuh6baMD2VmJ4IJBgDAgk6N3t7d3vWTDGQ2soqpRzx4zUEUiajSGQeLczsCjUWGW3pZDT99+DBoZrLZLU8EAgwRgQSDCYd+9OOp+SuVD5JqodUxWKrGvKQSBpNoZA4trIzqAwo1Bjld0F3Y+qsK1zt6UuIQ0s13kIJFgmBBIMSoPwXm+VFcUATdbqKppOu79HCSTNxhBIxDuAdJbCjGf13hwKJWrJsM6fS2FkdlE1AgmWCYEEIfT2ub1zUA28sx5m2coKSGs4nJ9/NM/bBQJJs7EEkuxum/Ie6mKsUQp13xwcHrdvOSyvWUHhx49/BwACCZYJgQSh9MDSw3+jfFPMDiflw/j58+2qQu1iCfpFCCTNxhJIJLXynNV3d41Fz9b9/Q/+6crlva/Bt/OWmieQYJkQSDAqChSn5cNxb3JUPfjUBK0KdZaauXd3D4v378+qgapaudUqC1hGWnNEA2o18FX3e9XKUwYPhRX9939e7lZrk0ynV9z7WCkEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIRLCiT1EAIAADA0AgkAAAhHIAEAAOEIJAAAINwfBQcHBwcHBwdH8EEg4eDg4ODg4Ag/CCQcHBwcHBwcwUdR/BfmwLilMgtjGQAAAABJRU5ErkJggg=="},66982:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAiUAAAC2CAYAAAAGAP0vAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAABXVSURBVHhe7d0JVxRXGodxv/83mIlZzMyYmbjjgoCCRokbuOKGIZqIGEBNzBeo4d85JJ3y7XrvW8vt6uapc37nTM5gdQNN19P3Vt06VLCxsbGxsbGx9WAjStjY2NjY2Nh6sRElbGxsbGxsbL3YDr36ebsAAAAYN6IEAAD0AlECAAB6gSgBAAC9cOj3338vyspfZH0NAABAhNcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkwJltvt4vVu4+Lufnl4vry3WJ9/WXx8eNH82sBYBp4fUGUAJltb+8Wp88sFv/87Hjxj38e+5uv/zVTPH78wvx3ADDpvL4gSjAWGhF4+3a72Nj4qVhZXfvEw4fPis03b4t3796b/35S6Xs6+s2FT2Jk2GeHjxerez8D698DwCTz+oIoQRa//vpbsba2XsxevFYc+XrGHCUY5fDnJ4pv/nuhuPrdneLnn9+Y+2/bzs67Yn5+uTh3/qrr/oOn5j7Kfvvtt+LcuSvm91j25Veni40ffzb3M8mePPnB/BmOMs5Ro62tX4qLc9fN52W5fn1173fM9BtQxesLogSd+nHvwDpzdmnw6d86+NahA/aVq7eL7e0d8zHboAPSka/PmI9fduXKbXMfZfpZfPnVKXMflkuXb5j7mWSKyq+OnDa/X8upU5cGMWftq2u3bj80n9MoihJrPwD+4vUFUYJOPHu2MRjdiIyIRCl0ZmevdRInXUTJvXtPzH8/yrfH5or37z+Y+5pUCgyFhvX9WhQwuUbHhkWfp2JT0WntC8BfvL4gStCqzc23xcm9N/MuY6RM0zvXrq20euVKH6Lkf99enLpzauTOnUeh14dGLKz9dCk6onP8xHzx4cOv5r4A/MXrC6IErbl3/0nxxRfp0xNt08jC1lY7r9cuouTRo+ehg/HZs0vmfibdHwf8tJ+tjGMKZxLCCZhEXl8QJWhMJ7EuLt7MOjoyikLixYtX5vOM6CJKdPKsprSsfZTpZznNV+AouKzv25J7CkcBpEu2rediUWCNY4oJmEReXxAlaOT9hw/JV5Tk8sWXp4qnTzfM55uqiygRTeGknPSrqzkUe9Y+poF+Dn0diYiO5ChgxnUyLjBpvL4gSlCbDpo6eFpv1OOmMHn+/EfzeafoKkrk/v2nxedfnDT3pQP16TOXB6Mq1r+dFlrN9l//Pmv+DCw5p3AiUzf6OgWWtR8An/L6gihBLQqS8xeuhj7t5qaD3uvXW+bz93QZJaIrhnTJr1Zw1ciJIurEiYXBVUvW10+jC7PfmT9PS64pnOjUjdbcefPmrbkvAJ/y+oIoQS0636HJ2iOKGUWDFqfSJ1ONaigEZO3xenHjxr3izMziYE0S69+nOnFyYTDFZH0PVbqOEsRO/NXX6XVi7adN0akbhZW1HwA2ry+IEoS9ePFy8MneepP26Oqc6MJnusxY8aJLf619VtHBbHn5rrnfKkRJ93Z2dt0l94flOHcjOnXz4OEzcz8AbF5fECUI0aiDRh+sN+kqCgrFSJ1Ri31110DRFEl06J8oyWNh4XvzZ2rp+iqX6NTNv/9zrvjll+5WFQamkdcXRAlCbt1+EI4CHdybnHRadvPm/fDU0cKl7819jUKU5PH02UbyCJhed11O4USnbhRU1n4AjOb1BVGCZLqrrz4dWm/Qo+jr655sWuXOyqNQmEQ/ZRMleUTWbpEup3AilykfPnyiePJkfDcLBCaV1xdECZLphmPWG/QoTa5+SaHpoMioTeSGaQc9SnR1lS7b3Xzztnj48NmfJyJ3MV2hn5/1c7V0OYUTWdBNIdXVZdtarl4/a9FJ34ol/R3pvyfxUnE9Z71+VlbXBvR60utq2i97h83rC6IESaKfaDUk/7jjT5K6L8yx43Pm41uOHZ9PPqelqyjRSbfar+e//5sdjExZ+yjT1+nrrf0M8/apc3Y0zaXLXKtiTyNUei1oKq+Ne/Osr78cuW5LWVdTOLqsV9+39ZiWNu/grMi6+t2dwc80ZSpLPwOdJzV78drgCqYu7rmjE8ut11DZqJPIdSL73Pxy5Qnx03pvJ1Tz+oIoQRJdZRAZlYiew1HXw7035dRpnMidXLuKktRRAT22noO1j7LU5zpqn01uoqiD6NzeAWx3t/6nXoWigtHav6WLKZzI1I0CSiFl7SeVDsa6iWTTS95Fr/+Zs0vFy5evzceqQ/uzHqus/NrXCJtGMFPiiig5mLy+IEqQJDK0rU9xOtBZ+2lb9ICWulz5QYmSm7ceJJ9oWkUH17W19b/tOyIyNdjFFE7k9R0ZcSvTQVvfaxc3rlRUKS7b+NurEyUaTT11+rL5dRai5GDy+oIogSs6tB05d6MNy9/fNZ+HZXb2mrmPsmmPEh0cNQVRZ3RkFH1i15VR5eeXQiNYGsmy9lum59zmFE6u17dCSlNo1j7bpJGcpkvfR6NEQaK7dFtfMwpRcjB5fUGUwBVZebPrtSQskXMSjp+YT5qDn/YoiZ4knEphUufuxpqO0f1trH1a2pzCiUzd6ByJjY2fzP1U0aXPbUzVpNLvocmHg2iULF25Zf7/VYiSg8nrC6IErsgCVxoGt/bRJQ2l33/w9M+z+6voaoaUg9k0R4lOQG5jymaUujdD1NSatT9Lm/eciUzdpEbtMP0s6q6A3ETdQJRIlGwMRrniwUWUHExeXxAlqKQ3YL0RW28qZfq02XTYuC+mNUpevdocHAys/79NuioqesDRCFvq4mVtvdaiUzep5yTt06W8kbsht02xoGiwnluV1ChZ2ns9R26sOIwoOZi8viBKUElrVejEVetNpSxydUvfTWOU6ICvKz66mLYp02NEP6VrBCuyzHsbo3KRqZvonYp13k5kFGaYnpOCQr/XfXVHt3TyqZ6L9RxHSY2Sc+euJL32LETJweT1BVGCSi9+eJV8pYBO4tvdnY43mWmMEo+mGE6eXCjOnb86oBMXU8/VsdS5SiUSCW1M4USiIXoeS+RyddH3fWZmqfhh729uVEQois5fuJr8MxLFjM5psfY3SmqUNEGUHExeXxAlqKRzNVLfAPVGZu1jEh2UKNEBS+uMaETM2rdo9Es3YYwcCKXOeh6R6RQ9nyZTOPqeU6dW9FiRK36il6rr9xc5Dyd64mzqVWf7mkaJRld1MrWmC4fPwdH/1utV53bpcvTo+TmYfF5fECWoFDn5MHJw7ruDECVHv7kQmo6os6aJLte29lUlco5CkymcyIKA0VGZyI0G696OITISo9/1zs6uuR9L3SjRqOrtOw+Ljx8/mvsFvL4gSlAp9SAq0ZMA+2zao0RD51tb8b9tnScSmZI4M7No7qdKl7EwLBI/+lprH6OkXrFW90oliZyDEz3fq06U1H1N4WDx+oIoQaVIlEzLlTcyzVGiA+GLF/WWSY+evFnnPCPd9C/1btR1p3CiUzcKJWs/o+jnpFjSzec0jaGDvB6vPHrS9B46qSOZ+h603pC1D0s0SuqO9uDg8fqCKEElnV1vvQlZiBJfH6Kk6X2JIlMTke9jWNdr40RGY3TArTrnJmo/WLQs//Z2+pSKRaMshz9POxk58vcZiRKdO/QseCItDi6vL4gSVEp9c6pzUmOfTWuUaJSkzoqkw3TFhIbqrf2X6RwDXcFl7afKEy3wdjg1fOJTOJGpm+hJojlFVjPuKkr0waWt1XUx/by+IEpQiSixv99hkxQlbV2GqQO1tf+yuq8L3UtFt/K39lkWncKJTN0ojBRI1n76IBJvXURJdFoI8PqCKEGlg/rmNK1Rok+11j6iUr+PJrGq8y2sfVoiUziRqRuFkQLJ2s+4aRpI68lYz9vSRZRoQb7Nmica42Dy+oIoQaXFpfQbbUXe9PpuWqMk8lyr6Hdt7b+sSZREpiYiUziRqZumJ6J2QScCr6ysFUePnjef8yhdRAkLoCHK6wuiBJVSD6JClPiIknSRc1c08pFyhUzkyh6dzBtdCbUNGpnR6IOu3NHVNZoq00q7GpWIXI5dRpSgD7y+IEpQSYtfWW9GlrYOeH1AlFTLESWi2+9b+7WkrCWiKcbUqZuuD7iaftHN8nTJsKJDK7SmPrc6iBL0gdcXRAkqpR58pM9XKUQRJdVyRYmuFEq97X/KpbupJ+iKgsjaRxM6gN+6/WCwwmqXAWIhStAHXl8QJagUWQfh+In5qbmXBVFSLVeU6PWk15W17zJvCicyddPGpdPDdnffDe4xFF2mv01ECfrA6wuiBJU0t625bOsNqUw34WpzkalxIkqq5YoSidx/qWoKJzJ101Zg6x4wuhdM6p22u0SUoA+8viBKUElLhGupcOsNqazvazpEECXVckaJbhr41ZG0O+JWTeFEpm7q3EiwTOeMnL9wtdNpms/2/uZS90+UoA+8viBK4IosNd/HSyjrIEqq5YySyI3nRk3hRKZuFECRuydbomuIRGhq6fSZy4Orc3R1kH7G1teVESXoA68viBK4IsPn41hsSgdljebowOw5eerSYH7f2s8woqRaziiRO3ceJY8IWFM4kakbBVDTZdPvrKQ/X4v+raZNdVXO3PxysbK6Vrx6tfnJlJJ+tkQJJonXF0QJXLrluW59br0plY36pNqlyAHn1F6UpBxwiJJquaNEC6NpgTTrMcqsKZzUqRu9jhRAw/826u3eY6eOyuzT4357bG5wZc6bN2m/dyFKMGm8viBK4NKbjt58rDcly4m9T3fvP3ww99WFyLkCqQdloqRa7igRLSVvPUZZOYwjUzd1bu5XFhnV0ddp9G5zs95jEiWYNF5fECVIElnESqtORt4Am/jjJMi0A3LkRFyipNo4okShkXqwH57Cidy0LnIPHYtG4TQaZ+27TN+LFk6z9pOKKMGk8fqCKEGSyMFf9MlUw9jWvtq0cOl78/Etek761Gztp4woqTaOKInc3Xf4d72wkPYaKY+w1BEZlVEA6YRYaz+piBJMGq8viBIki9zITHT1QdM33Spa2C11tU+JXBlElFQbR5RI5NwQnWu0s7M7WD3V+pqylBVhPannX7V1+XxkFIgoQR94fUGUIFnkU5nowLB05dZgASlrf028fr2V/KlZoit0EiXVxhUlkYOwAib69dZjRqT+jWjUsY1b/kemVYkS9IHXF0QJQmYvpp9UKgoTjVC0OWLy00+byUPk+7TWSuQyT6Kk2riiJDLyoXObUm+R0NbIRWqURH6/o+jSe12Cb+3fQpSgD7y+IEoQoiCInFuyT5c71r3CYNjDR88Hd1O1HmOUOgdGoqTauKJEFLnWYzWh0FHwWI8XkRolmuLRVI+1j1TRtVCIEvSB1xdECcK0lkLkzXCfbkZ2efFmsb2ddrLpML2B61LjOo+rk2GtfVYhSqqNM0q0imnbN7ZrayXi1CXx9Tpush5KdPpSiBL0gdcXRAnCmi6hrTfk/YWiNHpinXOilSt1DsjS0q3Bjf6s/aT4Zu8TcJ0IIkqqjTNKdBDUwdB6vDoUOAod67GiIlff1H1t1pm+FKIEfeD1BVGCWup8Uqui+X9NC2lqps5oiEUnt+oKHev5e4iSauOMEkn9OaZo+8Causib6LFTpzWb3nGYKEEfeH1BlKA2Tak0GcXokiJndXXNfN4piJJq444SjaJFLgevoitYrMeoK7Kiq2ik5uLc9cEIiDVqqGXnddfipn9rRAn6wOsLogSNPF//MXziadcUJDdv3jefbyqipNq4o0S3MTh2fN58zIjopeIpIou8lSlm9Pek32fKqKFe65+xTgkmiNcXRAka69OIiT51rqzUHyHZR5RUG3eUiEYPrMeMUNh0cZ+m5eW7rU1DVtFJ3Gdm0gKCKEEfeH1BlKAVW1vbg6tjrDeuXHRgrnsOSRlRUq0PUZJ6pUsVhY2176YUOl3/Pehkca1VkhoQRAn6wOsLogSt0Xz4tWsrrV+u6dEnUt1pVWFkPa86iJJqfYiSyM3vLAoahY217zbo9aiDtvXYTWl6SCeb63GIEkwSry+IErROVxOcOn05y/C1po3W1tbN59EEUVKtD1Ei0ZNKhyloIqv81qEw0YiG9fh1KUiGF14jSjBJvL4gStAZfQo9e+7K4GQ86w2tLh2EtMbD/ftPO7mvjhAl1foSJdG7V+/Ta6jJ4mURmsq5cvV24xFEPWfFwu7uu7/tnyjBJPH6gihB5/SmpZNPjx2fqx0oekM+8vXMYDG1Npar9xAl1foSJRJZF2SfQqbLqRuLFkpbXLwZvpRZr32dkDtqWXqiBJPE6wuiBFlpZENRsXr3cTE3vzx489NwtA6qwzTkff7Cd8WNG/cGJ69qhVdrf8Ck0d+A1iTRSbYnTy4MXu+KJMWHgkX//Z+j5wdrlzx69JzXPqaK1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68vDpW/AAAAYByIEgAA0AtECQAA6AWiBAAA9MKhgo2NjY2NjY2tBxtRwsbGxsbGxtaLjShhY2NjY2Nj68FWFP8H8u7IyQlLeLkAAAAASUVORK5CYII="},75448:function(e,t,n){t.Z=n.p+"assets/images/clinvar_tab-17c4411e73abc08b2b2a696539b5001e.png"},73365:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACkAAAARCAYAAABASYU2AAAA+UlEQVRIDeWUvQ3CQAxGMwQ1PS0lNT0lJRNQ0bEBDRUjISHWAPY49IqLLo4d7reIsBQlciJ/z5/P6dwMopsBo/tPyNf7446ns7tcb9WGVNVJwBbLVX/t9gcHdGlUgQRkvdn2cCEo+VJXiyCBk+4BSE5CcwRyIxvy/niOnAvHqzWQ62oyJOLAxI6UZqSrqeNPgtRGGzNGy1UaiIkoSMu9WBEPIpuMHf9PSFnYL4YXTr1brpK3woTUFoPOp4pZIlpeNj/l6gjSGi1Fa4elJY0YQPIy3Fqe2eSWgabmanjeB5DA+N8L9ocftgSlNrBoasaMIH1nraG0+pb2F3Unu19fQerdAAAAAElFTkSuQmCC"},52358:function(e,t,n){t.Z=n.p+"assets/images/common_tab-ebad4fb10222689884a972f4c198900e.png"},13562:function(e,t,n){t.Z=n.p+"assets/images/conservation_tab-3b8fbf60d615f75e983811190c20a17f.png"},21074:function(e,t,n){t.Z=n.p+"assets/images/custom_annot_tab-852fbd1fad60b10ef07b42644315512e.png"},9152:function(e,t,n){t.Z=n.p+"assets/images/details_panel-c8f53cb731602a4fec1b29fa8edacb18.png"},56936:function(e,t,n){t.Z=n.p+"assets/images/exac_tab-dea3ea5647b223f1dfd3d553a82948a6.png"},55236:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAASCAYAAAAg9DzcAAABLUlEQVRIDdWVsQ3CMBBFMwQ1PS0lNT0lJRNQ0bEBDRU7IFEwBRJiDWAPoxfpR050sZ3ggHISuojg/1++z6FwI6tiZLxu3MDP19sdjqe/hY43DKGqJbxab9xkOnPzxfKn4EDKmx6qCvh2f5SwAOuz3e2jTxwST7lHqvJTD6VcASOukdBC+lBp+6nKD68QLIw1YCVyvlxLUAkJPCam9bFupZp6dkxgDIdIuy1VxjG1WoElgAlb1Uy7iwla7JqvwXVqqmKhR4H5kZW2DLkXqhyp+vpJwFqAOW8OP6nQobTePH1SlT+9E7AWphzK5sHiwbqOkfz83gsYAWtMgGIHmjMf+zPwgWLXvYElDHgTUCOTK1V50b8GRsRKO2eq2YElCDig3x4s6Vk9S8KW8FDffQB0pjRcbUu4nQAAAABJRU5ErkJggg=="},96618:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAYQAAABQCAYAAAAZZQJeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAACNWSURBVHhe7Z2LcxTHncf3T0iuqKtKUpW6yuPu0L04VOW6yp1VRy6q2GUHGRyCbGKfijgofoJjgeKA5RyKkkAwBhuFJcg4ZwsfmMUGFoyRbAvJBoRsSUgGGWzJCGTQC8FKSGhBtn/X35numZ6ZntXualcSUnfVx5a6e3p6enq+3+6eHhEYGRmhGzdu0PXr1ykajVoMDw/TtWvXDIaGhgwGBwfp6tWrFgMDA0r6+/vjJhKJaDQajYah0kg/VNoLZI2GZgv9FnoObZe1HtoPD4AXBPCLbAbiIIDChAnIJ3RXDBdy5coVi8uXL4+Zvr4+jUajmbKodC8RZM1VmYms2cIcgMoYhA8YhuA2A7iJcBhhACrRFxd26dIlJb29vUnT09Oj0Wg0UxKV5sWLSmuBbDSyVguzEMYAxIzBbQoB/CCWh+RlIRyMgo6/X0/rn9tMjz7xJP384V8aPPDQ43HxsweXaTQaTUr4zx/erYyfbqi01sHDJoZeP/KEod3QcGg5NF2eMUDzof/CGAL4AZFiVgAngascOXacFvz0AZrxzZkajUajmQJA06HtwhiEKYjZQkDMCoAwg//dvlNZmEaj0WhufqDxYilJ6D9MIYAfxMwAa097w28oC9BoNBrN1AFajwmAbAoB2QzwwuKWrNuUB2s0Go1m6gCth+ZD++EB8IKAbAalW15QHqjRaDSaqQc0X5gC3ikEMGXAVqWuri7K+cn9yoM0Go1GM/WA5kP74QHwggCcobu7my5cuEDf/ed/Ux6k0Wg0mqkHNB/aDw+AFwQwXejs7KRz584pD9BoNBrN1AXa3/7xx9T26QUKwBk+++wz+vTTT5WZU86d+ZS/rMBk8Xx1Ho2XrDzKG692S8W59H2eXtxs/XPMzKfcuOuQSN6ZNGcxz/tgHs1WpKeaf/z6d+ir3/gu/dU3/pYCWD86f/48tba2KjOnjKxiCp0eIE/ob6PQymz1MVOF1bUUYZcaObpGnc4oOoq2aaNyd1pWAZUe7aCo0VhSGOygmq0Fng5T3momt+/Od8TbrKGafpahv5aK3Gl3+t+jyqBfeS4m232Oo+2TwWhnVRsmRDaVnTJah7qrVinSx4lp1T9R315eX8X1MOYUhekM6sCCt02yKf+lFoqMmOkiRDtqqXSxnC/RvDNp9kr7vFZg7VhZkt7n5qtf/zYzhO+YhnDx4kVqb2+n06dPKzOnhnzWEXALotReF6IS7pYlrzRSu3FnOiisaKApQ7IPXBZ7OHrQPuzY1loKbSpm7VZM63fXUjvvON01axwPnXjg/NvU54FbvJ2ajTKj1N1UTWVrcI82UFlVC3Ub9yhKZ3aO9tBNwvs8mQ0hi7W5EIsxm8sYmCb90xZ6/O0e/N99PZJZmBk8bTJ7Y6OZjgGOUQf0b1YH3Mc+Z50TyTsjaxs1I3OUGYDRjiz/Gl7fKKvnIilvinlyZ4PhAfCCQEdHh7Fc9NFHHykzpwTe4ZSjoIJq6mZpqX5gJxVJPnBmHOvou70jLeNh7GPJ1EuVBXa89cCh0/VU01L5GAPVA5dH5afRGweoTjXSEueKtlBplitNZjLe50lsCPP2dKBRqa6uzfz/OnW+tDMV+ydmqqxda14SdePl9jRS6SPZvB4uQ9hp3of2g8U0R9kmeRQ6iyze50CIf/uevCTyMl4xz90cdM0GxDNVWeCMTyH7mPbDA/DqwDCEtrY2amlpUWZOCUZDE3NwRZpYX3Otrc1+ZBtVtg5QlI+gon1sWuiegrLpWBRTqiIpDhRVU/sga9yXRFwxVXawUUFTyJzC4W6wzlWzmqdjKtrUy+NZ6O+l5t2sU8hlKvJFlfmyqTAYplCw2K5rUg8cGzHg2jsqKFfK54CXG20IWnGio4dZm6OaETZCcxyjeuCWmWKNcjwPNmf2iy0omN3DGNPXBO5z3h5Wv8FeqlGIYFEVG6Wx+xouxO/i3m2n3KA98kR/MJag+BKV6CeRjmoquVMqT2r7wt1tnnvsvd75LB8bxfXzjCMDbEQapkK5TIbXEBT3PSZcMFBGVojOsB+jdRsU+UB8dYovXz4bpYapfJ0krFOxfxoCy8JIG5UZeQpo/Vb7WVUawupthlmYP6vahN8n6XpseL1Ph/jvieRlg4P9GBxImiTIClM7S4l1b8bKDx5dSzuqPqCWNmYIeH/wySef0MmTJ5WZU8KyCuOGUk+t82H1YzFrTPSWaC+dOVpBod3V1NxjdnI0jNUpDAFSNCK/mbYwiU42QJERNuU8XUuVR2upfBlLs0Yy5lQ0tLuCapgRITjOJfKxh6y9Lka+IvPcxsgI5SMumQdOjCJ2xxBgBXZHt0dVNY41SNEW9gNndcYSu5ykSOQ+L6owOrpXBDdQ3SDKEKNHXt+eXupm/aG5ioluFRN2iBGbSp9hVY+craXw7jBVMrM2esnZMM0T5QlR6oNpDMTuT0zUzfvA4jsaqZKVGT5qn6tcWuLwGMLP+bWjHd0DFBVc5MzZFH+XMMJGk568dp2iPS3OOmHZwRp9xpnPEkrpXFOxf7LnNXya9Y/9ziUrgdIQZGIYgrqd3PVOJC9jndme3ZXO2XXubnMW2Ry041LNV77+Les9QgBbjj7++GP68MMPlZlTAxs9sZtqPIJckMMvbqBcpWjwkdNIB4Uca4zz2UODEqQpaKKG4Ol8M6mkTtUps2l9Az8XF3XzgfDmM+OlOrGOWNnBjIeNnKybLTpXwzZzbVBBaYPrgfO7tlFwdHSxLukQDm9n9DzsSZPEfR5spBI5nj8Y9nRa3DvnmvPsIJsdIpqJvz1CzabSJsRK18Lb3rtmLd539FJYGDc/t9Mk2LlW8hHqqW1WvMcQUB6bqWCGYre1P0vZLEjuN2KE2/yiS2D96sQF2VpKiDcf1uL7okZfdJvm1O+fNskZQgFV4p2J476bzC7h/cxKSyQvYFpSgz7BztmKAQ4bcJ42Bzjdddv8Z2EpIOf7/8QN4dsUwMuEM2fOUHNzszJzKplTsJ0q2UXKb92NN+5imgb4yFHprHwd2urciRoCE2nrITAImqPR1pB3FGFsm3yM5hkd1cynnP7x+sZc47NEabRgd1BTMBTXtixEdRj5yNSFKJ+nuzu6EARbKLwPXNlp5zFjJa77zJht3L8o1W2040yDZoMB6yWaz71TPrCsTGNEJbWbaHsm5nI+OU3cO3MQwNrBI+jeEbzXEBKBXxObBYn7Zi3BnNru6Iv+dTKX4PIWmm0abz4l06x/guQMgc1W2AwL9Y2ereBLcdk0r4RpACJx/6R6J5IXaflba/nmCylgVqxc2kwdmAx88tFJqq/ZQ4GzZ8+OmyHYsIYpDFJITGkVD7B6HZp3FrH2lqghMOF35Is5rZMx82HUGx30ghCzDNG5UjEC4+9H5HNjhFXI070dXcxiILwQBe8Dl44RmEmM+wz4LhvbaLlBO0TR5975PLCeduP51IbtvP+xRN4yGr4cNCZDEMsDVWscfaDcmAHLZhj/ecZUn2nYP731cOHXv1DfSrFt1Q4R1mfDMC7HPYg/rzCPyGnpnc+dq6jMaPd4dvglD7QfHoDJgWEI2HLa1NSkzJx2FoepHWIhRJ7fiJiGIMRhvA2hr8M58pGoe2X0GUKs83g6PZ9Wtu+XdiJ48F6XsqM7dmF4HzhT7Jwj9ZTjvs+cUnnkzUeLzmUTn3vn16Y+hqBuR/O+ivcYMUXVVe5YBNicBfkHefdJvOcZS32mY/9U1kNmlDaZvXANle0OU2h3iNYXYKNEHoVRzbNhz2h+9Lx8ecm9fGqQT+GLLC3ZexsHz1eYhgAvGBdDWLoJjRFUbDEDrgsWL9tUIzr3G3fxkLpfNvGbObohiGm6YjnBARcO5Y6BOEjmgfO8XFXAl9DkF7N+HV2sW0ZP11Kd64GbschsV3mN3E08u4wSus8CPlpu3srNgT0U6+X0FBmC8t7xvhapKTZ+d5iTC/P+sNE7X5JJXoDt3TmFnpE4mzHxey6WWGLVSSbefEqmYv8sqDD2+0fZAES1/p6cIah3RBrwnWK2biWS10+fTMwBRIy6jpGvfO1b9NWv8XcI4huEdBqC2Zmi7KFXiMlicw3eFuVV9ssY13roUjb9QjnWfm3eiZz73sUUNB5DEGvD7hfYMynX2BYptj7yfcuKPcUzCkJUV1fNnN+Om73wMeeabVIPnLheHLfBuwU2q4BC/KWod5+3uvMU1ZjtYgSHmGUzQUFZPnvKxa4viLVnjdomsfss4MJyqtFcPvLsOkqNIajunaiv6E9CVDxf0YqXn9LupWQNYTRjNevkfdnsqRObcZ0ZxJ55s+/Hmw/gTyM4XvRPxf4pNh0oR93JGoLY8NJCZY6+JDRHXu5LJK89Q/A8X4oZU6pRGsKJEyeUmVPCIn7D2IPaflR8wSp/ZSg9wIzZW82bGb3YSOXGF37FtL7K3L0SZcJgOz4XE5bSXofthGGqxKftWL9ksaMbAkMIRn8bhfkXgiXYr444eY+1Il/hpgr+5aO0g4WPZgyTETc8yQfO3gnDQl8b1exnU85N2ym0v5HajetmZqj8EtSvo3OzRXB3MDFtZ8Hc5bCdyl6Uti4aMzG1iFkkeJ8F5gtKv/TUGEKkj7VvT4vVn8r4F6kYQdovq0X7DNCZg0FzBL8mRM08Tr5+lSGUNbF+1+ee4chIH0S5H3yB2DhhDXLEfZHrJL64lXdOxZmPz8hQd0sop2T/zKb8jUEq8nmZnpwhMG3ig1BLB4qCFOZ/TgN55WtNKC/XPOStfHGDqUMvVltfVqfzHUJT7TH64IN6av7wpGkIWDJKqyEw8KFZTQfvPHLAB0ZF3mlVbrDR/MRbCsYLF9eDNHslE2Xe+RCM3SxB80bEZQiMOUXOMhCi7g+cGMq/NcJuoONv9AhRlEekST9wAB8biU4vhZh/KyZGRxcjddWIQ/xdGte5ohdb4v47RIneZwMugurlh9QYAh6osibzYRRB1Z/woVtYiJwIaGvXF7JeQ+D1dOwcciEGC66dRE64KMoj2zvVdapc52rPePLx9nC09TTqn4JkDQEYH0jKzTzCBqRVillSMnldOuTXjqkEq0N4hwAvGDdDsJD+CmbMrXAG2TTvQeQV2z/9wTKNer97/Ii/MjhaOdZfI/T7y4VZ82nOKPVNBlxjPPUbO/G3uy+J3Od4tu6miLjbUNTf7x57CBnrwrFEdczEW6fR8t3J+qcqfozcVP0zBZg6kB9XWyaSV7Tj6PqYGqD98ADHS+VxMwSNxoFqTfUmxPhCPc07tTSaNIAZgscQJmzbqWbagr3t8ncc6ZwWpx38SYhkd/loNBOIryH89d9kKA/QaFIP/lAgvuFooZpXFH9I8CbDmObHvbyk0UwOoPniHYJhCOJPVyAyI/M/lAdpNBqNZuoBzReGgL9rZ/xxO2EIC376M+VBGo1Go5l6QPPFn66wDEH8tdPS4FblQRqNRqOZepQGy4x/+gD/hLJlCOLfQ2hoaKDbchYqD9RoNBrN1AFaD80/deqUYQj4t3EC+GfT8C+m4WvlxsZG2hc+QDNn/7uyAI1Go9Hc/EDjofXQfGg/PMD4JzQvXLhgfKEmdhq9//77tGPnLrol64fKgjQajUZz8wJth8ZD68X7A+wwghcE8C/tu5eNjh07RtXV1fTI4yvo7//1e8pCNRPD3/3Dv6QF1bk0Gs3UAVoOTa+pqaHa2lpD6+X3B52dnRTo7u4m+R/axyyhrq6Ojhw5YpjCW2+9RVu2vkBPF/+efrXqN1S48mkq/PXTtPzJpywKfrVKyROFKzUpZukThfT9//oBZWZmpgSUhTJV59JoJhO33ZmrjJ9OFBQybQUurZX1GECjodXQbGg3NBxaDk0/evSoMTuA1mNlCCtE8AB4QaCnp4cwS8D3CGKWgD9jIZvCO++8Q5WVlXTo0CF688036Y033qADBw7Q/v37DcLhsIN9+/Z52Lt376js2bNHEwc7d+6ku+66SynwiYAyUJbqHBrNZOOhh1co46cCKj2UUWmqW3eFHkObodHQamg2tBsaDi2HpkPbofGYAGB2AO2HB8ALApcuXaKuri7jhQKcAutJeOuMlw1wEbjJu+++axnD22+/bZxAGIQwCXDw4EFfUEFN6njttddo/vz5SqGPBxyLMlRlazSTkUcf+7Uyfqqj0lOB0F4h/sIAALRaGAE0HK8CoOnQdmg8PjeA5kP7MTvo6+ujwOXLlw1ngEOIpSNhCnCR+vp6w1FQGNwFBWMN6vDhwwZVVVUGOLEfqJgm9WAkcPfddysFPxY4BseqytRoJiuP/7JIGT+dERordFjoMjT6vffeMzQbg3poOLQcmg5th8ZD66H5eHfQ29tLV65coUAkEiGYAhwCpoC9qHANuAemFPhgDYXgBcQHH3xgFIwXEgAmgZMBnNgNKqRJL1gXXLBggVL4VSAvjlGVpdFMZpavKFbGTzdUWit0GJos9BlaDc2GdkPD8ToAmo5XA9hVBDMQS0XwAHhBYGBgwHAGTBfkmQLWlbC+BCfBPlW4CrYoARSOaQdOBOA8AlRgLGBKo0kMjAYWLlyoNAAZ5EFeVRkazWTn1yvXKONvRlTalyiy7gKhx9BnIPQa2g0NF7MCaLuYGUDzof0wA3hB4OrVq9Tf32+ZAqYOMAWsK2G2gINRCFwFBQqDgNPgRADOg5lEvIiKalLH8ePHKTc3V2kEAGnIozpWo7kZKCp6Rhk/lVHppxvoLxB6DG0WBgCg3cIIoOnQdmg8tB6aD+2HBwwODlIA/4EpyDMFZBSzBWEMANMMLCcBzB4AToblJT9EpTTpB7O2e++912MGiEOa6hiN5mah+LcblfHTBZW+ykCLhS4LnYZmQ7sxIxBGAG2XzQDaDw8wDGF4eNj4QZiCeKeA3Uc4CO8WMLXAV2wAhQJ8yCCA8whQAc3EgRHCokWLLDPAz4hT5dVobiZ+9/tSZfx0RdZdIGuy0Gmh29BwaDk0Hdou3hnIZgAvMAwBDA0NWaYgLyHJMwZsTwUoXADHAeLEiSIqrkkdGC3cd999BvhZlUejudn4w5otyviphkonR0PosNBlodXQbXlGIGYF0HhhBteuXTM8IBqNUgD/EaaABGEKsjHATcSsQcwcxIngOn6ISmnGHzFlVKVpNDcjf1y3VRk/nVDprEBosjAAoddCvzEjEEbgNgPLEK5fv278IJsCwBRCNgYUBnOQDQII1wGiAhqNRpNqnlm/TRk/3ZE1WNZmAL12GwG0Xei8bAY3btygAP4DECGMQWTGMpJ4vyAQhQJhEPHirqxGo9HEy4aNf1HGT1VUGhoPskbL2u1nBJgUCB8IjIyMWL/4zRYAChPIJ5FBBeBEGo1Gk2o2PveSMn66ohJ9GaHXGNjLWi70HXovmwG8wDAE2RRUxuA2ByBmDxqNRjMePPf8y8p4jRO3AQBZy91GAIQPBD7//HPrF5UxCHNwG4TAfWJURqPRaFLNptJyZfx0xq2/Ko0Wy0KxjAB88cUXFMB/gJwA5IOA2xxkVJXQaDSaVLKp9BVlvMbErct+JgDcei98wDIEgTsjcBcmiGUSGo1Gk0pK//R/yvjx5vzrT9CPfvSjGJTSB4rj0o2f+AtU2u7Wf48hCNxLSW5UJ3QjKqjRaDRjBYagih9vLjeEWF1KTdYXUC4zgdwVz9pxf3qbPlUclypUWutGpdkCaLtK84GvIciMZg4ajUaTbjYHdyjjJ5QLe6mAGULBns+8aUMRivRFaCjKfr6Kn4c8eYZYOvJErjrjDeRjjJ8ZQ648guiQme6TR2j5MHYmXWEMeXV+mMUHvvzySwLuxNGASbhxV0Iz3Riixr/cT5nfnEV3/eZNuqjMMwFErlDf5dgMqY4DQ13U8v4ROvyuSWO796HWjA+bgzuV8RPKxTAVzJ1LBfsueNIu7CuguXMLaPPWlXQ3yzN37mZqEOkdVfS7n93N4hAP7qYHgg0UkY5v+BPiN9JrB8TxZr6VB5znipx8iZb+xEzPycmhnB8/QL97+6JTs3veo40PLTDTDRbQw2UnaICnf/nlCdrC4i1DSNYYYqEyDZkh64EcUqZrxgfrPvSr0+OnntZlzqQZ3wRLaG+nKs94c4UOPSnq5McSerXDfVwXHX42l5mbN3/mfX+gwz3u/Jp0A0NQxU8okiG400xDmEs/XrqZ6tr7qf/yNZ7WTjseYvErdtAnl3neg7+hH8/9MW2ut49vZIYA8f7vZ6ro3OA1unapkcoeguD/kd6zRvgtVPbTHFqw+hBdvMF+vzFAjdseopz7NlLdgMhzkcIr2HGL11HVZ8MszzB1Vj9DecwUVr/Tx7WfGwKx4DYFGVngU0s3vfoL/pD94lXqVObRpJ96ekaI3YZ6RXpinD20mnKzb6UlL5yka4r08UfqZ74wQ/jMeczepbPs9Kw7aN6i+TQv5xY7LmcN1VoPnGY8CG7ZqYyfUDrDtJwJ6fJ9rhE54+K+5UzQf047zjrjLSDg4ufPXqNlrnIaN2Mkv04S/y9ouGYdi1tO4U4R10hBdty9G+toQC5P5qMX6F6W55l3hyVtv0rnm96nD9oUhiAH+4D4UVZiVLppl2UIu6hLmUeTfhokQ2hQpN/s2Nc3b3urIt3LtSOrKYMfc/uGY3RZSjsbyrfScneccxynSS/BLa8q4yeUrv2mIYQ7PWmdYRjCctrf5Yw3GO6kE+GX6bniZbTkF3m0gJWB2YBczgnDEIJ0Qj6uIcjiVjjK7AyvMI9fmEdLVq2jl8MnqHPE1ue+N1bwY+w4GTM0qQ1BDqqDU8HwhQaqPhKm3y7iQrRoDe07cozFNVDXMPIMU9cJ/A5a6YryeDk/Y/g8NRlxx6jpApywn9qP7KKtm5+lTTvC1NQtlXGplSp2sHiW9vLBU9R1Q0pzwOpx8jDt2mbmNcoxynblc5/7Rg8d37PNLL++35vfQLrGE+dpGHHdp2hfrHolcJ7hC6eogqdt2lxO+07yc0hc+QRlbaOlXOBmFG4z66Noc7TZ8YPlMctTXhNgx5rlHqPTlxAn3Zttu6j6E782kkmivcDwYSri15cf7vGmK2jfs9ycESxaTRURd7rTQJ1pmnQCQ1DFTyjMEFYwIV3BhNydBqFWCvG1ExRcjCWcp+iF8GF6v6mVOtte95RjGYJ8bMMWZZnRvlY6UbmTnvvtw3TvApS9hU5cM9OEIRzgx6hDHIYgB/nkY6VrT775QHnIp10XkKeHdj0o4tbScd/jRX7GhV2Ux8vJCx2jXcukKb/BHbS+fpiG69fS7Y54Rs4qqhDlCC4dppKF7jLALJq3OkztsvjEOHfeHj8Rkq7xwV10un4DzbPW3zlZebT1pGRAcZ2nh6rXzbdGsTIZC1fRvvPi/F/S8Y3ePCZymw/T6R2PKdfS0W5yee5r6hLxrM3FMeurT9HWxe52ZW36fIPCYGSSaC8gtdn6eik+aY5RCS8vg9VZnUeTDrb8eYoYgiHqObSlQYq78Do97jaEIAyBCbvIA9yGMBKlq5Hz1Nkn5al/3iy/kf/e/iotYb8/9dZlruYIXXT05S20paqD/56gIfgFqxIJcKV+O5UGiyn/Nv4g35ZPJcENLG47HTdGZD0UitMQQpYhhKwHf07OHZSReSvllxTTUse6Lxv1MRHJzHmMikoeo9uzeDwjY80xW5CuNtD6HJF2C+WuXMvqtoGKHrTLylh5yB5Fu8/N/p+Jc7FRZuGb/oZgXeNt91Aua4vM+5cb7VD06K22oGcupwpjVM0Y9TzDTOTvMI9DGi+vtCTPFnSpvPYKtPlyyhVpLD+uszR4iNp5PbvC9hKJVd46doxoO7l+8jU9GFIawu2s3liTz2NtWlRwh2Q0s6jkiGL2ZZFEe4GTQZpjpM2nMja7KJXyZmTfQ0V7TnlnQ36wGVnFGtG+d1DpSUUeTdrY8uddyvgJJRlDOP86PYxjtn9EndeiFO36iF59eoGx7DOqITSahiBG+1/2VdNqNiPIe7aazkeuGuZQ/Wwey/Mw7RFaT5ep+n8WUM6CFVTe2EGD/R3U9DKWmRbQM8eu8zwpMoR4g3VBFj7i4U5jhlDnSGMitTe2IczILKZKa6o/THXrsnl+JqTraiXhr6USYUqZG6iJx7fvvIfnn0VFVfJyBivLEtxsWxDkc2MmUhfPEoh8jUwoNzY6Rsjt7BqFcM3ZesqMH+08n263xD2jpNohdMMN9sxoTpCXZ9BI63n8DFYHO54xzNqHj8LRbg7hjFRTkUgT9fO7p+zcRhzIYffzKo9nXKkqtgV6k+v8DpJoLyCf2wd3WU4aqcxYPrLNKyM7n0rr4lt+0qSOyWkIB0xD2K8whP3OpRpH2sGnrPcGOQuW0PNHKw1BlssRhtDENdQIwhC6+e8sDDaX0+MLeVlg4eO0rV6eDbAw2EYH1i6RznkvrX6tjQZ58rgbgjf0OsRDuj4WpDQYAo8VoVs2hIs88qIkluwBd4Q6acmigcfxULeOH2Odp43KrXcb2+mcESeFvv3WmnuGOI98bibEUTN2lCBf42MU7uPRVmihUmFWS0JmPUY5z7lX5vPy5lP5WR5phQEKF/BjM+U2dRqCHKI1xTHKI2reyo3Waifpmm4LUrMRx4Ikynm7rWELD9L5Pf1ADkm0FwuRhiAtZYJ+e9Ysyt1UTWd60GpRinTU0npr6YoZf43fXZPqZ3AL3V6wlkINvTxdh/EKMIQpFT6/zkbrg3T9c/77GMN146+dihG/T7jO8sQ45/QwBEmQPIZgraPz80TZyNevHCNIhiHqHevcvkG6RpXxMAGvXMnTRd1inidKNSWu/K5gG4bUbjEMwc4/i7630FyacpAtBFWcz+e+yYaw1y2kSRhCvO01Whi0ZzkzVlZQhEe7Q7R/gCJsmt1cFaL11rLhLMrf6zY3HdIZppwhTMKgDcFtCKOKu6LeYzUEHyFMrG6x2wxB2W4xDME+/2iI8/nUIdWGEG97jRokE83cYM9oYoYO6RqXU2U/j9Yh7UEbQvqDNgSPiEgCpVz+YeliVLlsf2oMQSlGTHiWuNJHOY99LcWkWgFp3iRG9PkU7uGRMQxBbmc7f6wwToYQb3uxYI7ugXpJyFoyTOCYc7vzzGMY7v6kQ/qCNoT0h0lqCPLyh3u9WFoLT4shSEsPmUxY7bcuZjgldq3MpHmvtJlxYzUEvKA+xaNFkMsUyxmjnCdSuZyXp1oTl9bYHUsu/oaAdjNf1M6ikqNecYy0ttA5CKfVRuNkCPG2Fwv2O6JVVOm+l8olI9l4FPef9c3KIp7u825Fh/QEbQjpD5PDEDKZsLc6R2TnrJ0+Mylj8Qaq6WDpPS0UWi3vsU+HIbBH/qi98+X2ov10TlSrp5pKrO2ocZ7bN8gCx8hZReGz/ER9jVQqvfC0xHi080TtXUGO8kZ6qdLaLukWZUmQc9ZSTY8s8G1Ufr+iPBYip7dTPj9XhkpM02oIjHjaiwX5XmYs207NfWZatK+Fyq3vOJzHuPteZWuvOWPoaaPKTffY/e9+1bsMHdIVtCGkP0ygITgfPBNJRPoqqFCImwMmxLuFwKTHEDAKtLeXqriFlu6XXiiO2RAeo8KVQpycZCyzd8zEc54ou1bPh3cSmQXM4HheM7DZ2BrXuaVdOtQRsoQfZGTLL5MZMHOrKcbLEOJsLyOMdi/Nbae2HSB0UMjzYaMLx3XrMB5BG0L6w4QaAh7yuqD00ZRrCh5tDVGR9LUw9n+XnWKPriUw6TIEhCid2Vtsf4DFwcdQ5U0DPA8PYzYEdu5BNhovkD6iY6aTF6yl7hGeHSHO8xjtdr9cFiPrHirc2ajeScPO7Zx5ud5B9NQ6PugymUXfezRIdY7lvPEyhDjbywrsXr65lvJlI2MY3xMcdddHhAHlMTMyb6XckhCd0S+Txz1oQ0h/mGBDECFqTsmdwzQ7DDqXk8Y7WC8ZPevJYwk+4mlc6wBFlcKWYIjyesfbdiPmffA9N09PWf0SCilqL54/oXspjpnAPqiDNoTxCJPEEKZj8BE4HXyCbq/pHrQhpD9oQ5iwoAUusaDba7oHbQjpD9oQJixogUss6Paa7kEbQvqDNoQJDKN9AKWDM+j2mt5BG0L6gzYEHXTQ4aYI2hDSH7Qh6KCDDjdF0IaQ7kD0/ygCV5zT/vzbAAAAAElFTkSuQmCC"},432:function(e,t,n){t.Z=n.p+"assets/images/gene_tab-fd606789cce7ac42b137989ff2d35640.png"},18194:function(e,t,n){t.Z=n.p+"assets/images/gnomad_4_tab-2ae0cb39b7e2e491177acd64f29ea9c5.png"},15839:function(e,t,n){t.Z=n.p+"assets/images/gnomad_tab-9b9c2264f68ad6bdf8269762e3b34204.png"},58020:function(e,t,n){t.Z=n.p+"assets/images/in_pg-d31470c9f988df25fc30e40a821e914b.png"},1371:function(e,t,n){t.Z=n.p+"assets/images/interpretation_author_on_panel-ec170ae49a4579698ff17a398fd4f2fe.png"},39716:function(e,t,n){t.Z=n.p+"assets/images/interpretation_on_panel-00d44172e30b78fa4623242bd33a74fc.png"},51194:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAF0AAABdCAYAAADHcWrDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAQDSURBVHhe7ZrBjRQxEEXJiDiIgziQCIEbV0KACAiAAyFw4ohWQgghJNTshynJ9lS7v+2ye9uuL73b2O1501Plcc+zzTM8Lv2EUNK/PvzZPn35vb39+HN7+e779vz1N+fGizcP26v3P/65gSMmh9IxkXYxRwcfAj6AXHal4+7GYG1i55ic+F3p+MS0yRweONSiSve6bQfqfZo76V7D7fnw+dfN7v/cSdcGOW2kZSaSjk9EG+S0E24nI+m+W+lHuJuJpKPoawOcdrA5kUTSfZvYj7CuR9K1Fzt2SFz6QCQufSASE+moV2jCJWjznIG2thwtfU9iIh2LKQ3GaHONZPS6JadJfwrHDez5d5hLS0fOPFgL980lubz0M48d0kMoNpeXjgcl2nwjwLVrcnnpyBklpmW9U0g/o6HWNFDJFNKR0Xd7S6aRPrKh1jZQyTTSRzbU2gYqmUY6MqLEWKxzKukjGmpraUGmko6vfe+HKBaZSjrS8xmt1Rqnk96zobY2UMl00pEeDdVyfVNK79FQLRqoZErpPUqMZS4nHa9jzj1a3lhKydqYtKxNMlx67n/bEsu7nWmgKD9TSw//dJOLRUPFHEzkwTqTS0rH65kSY3EIxjRQuc700pk70KLEMJE1TS+dLTEtb5BdU+nrW9YkOUU6YL76LXv20hK2hPSeDbVm7iWkg14NteZbtIx0ZlxNQ2X25jVrQdJxJUhOld6jxLBrwbVrxl1eOmBKTElDrS1ZS0lnfzWydzsTba6lpAPm7mSeKjHnOog2djnpVodg7OGWNnY56RYNlV1D2kBLx08jHbQ2VGZvntvzLymdaai5EsMkd/0lpbMlRmuo7PXTcSFLSgdMidDu9pYGKiwrvaahstc+2ucvKx2U/qJkvh25BiwsLZ2ZKywxTKyuizBz7SF5ctLZEoO52Ovu7c1DLN/DHpInJx0wJYZpnshRAxWWl87s2dkcNVBheemAuduZaHNruPRH2BPDXJiTScGlP8I21FyYBiq49BstJYZtoIJLv9HSUEuv5dJvtJQYbb4cLj2A+ZmfprS0AJceUHO3s3vzEJeeUNJQmcMtDZeewM6P9L5G7fxAcgnpJSWmZG8e4tIVmBJT00AFl67A7NlrGqjg0nc4utu1MSyXk446isUcUVtvBYzfQ3t9CZhDW3NKy7UkJtIdDolLH4jEpQ9EEkm3qI3OPpJIesuWy8mDJiyJpJc83nLKCB8/RtJrD4ucY8Ij6kg6/kvidb0PYSLpSMv5haOT/pK+k46g6GuDnXLCBipRpXuZsQEOtajSEYj33Uw92h0u2ZUuQY33u54Hro5OQw+lI7jrMRE+gNaTttmAC/yoRFU4ki2hpHts49JPiEsfnm37CzH4jI5dvUxJAAAAAElFTkSuQmCC"},43892:function(e,t,n){t.Z=n.p+"assets/images/normalized_variant-02dd7fe64fdccf3055ea6effca518a5e.png"},17738:function(e,t,n){t.Z=n.p+"assets/images/omim_phenotypes-dee84bdc012e4850ec472bd00e5f5b08.png"},51412:function(e,t,n){t.Z=n.p+"assets/images/other_freq-427b31875cbd307215755d6c797c1c0f.png"},7817:function(e,t,n){t.Z=n.p+"assets/images/other_tab-097dc8a46c4b2c2e588401b70cd641b1.png"},9674:function(e,t,n){t.Z=n.p+"assets/images/pg_example_igv-9f49937d5b6131c32d576bdbd299b23e.png"},3166:function(e,t,n){t.Z=n.p+"assets/images/pg_includes-4f770691367ef4545b358eb9100ee1d4.png"},48580:function(e,t,n){t.Z=n.p+"assets/images/protein_func_add-f03782eae2a6ae64ca6ab56d91b1cee6.png"},8533:function(e,t,n){t.Z=n.p+"assets/images/protein_func_eff-bb55a57d581aae6be2d86002dbbddb7c.png"},45659:function(e,t,n){t.Z=n.p+"assets/images/refseq_transcripts-001899ce3fc2e8524fea5697d7cbeb57.png"}}]);