"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[3650],{3905:function(e,t,a){a.d(t,{Zo:function(){return p},kt:function(){return u}});var n=a(67294);function r(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function o(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var n=Object.getOwnPropertySymbols(e);t&&(n=n.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,n)}return a}function s(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?o(Object(a),!0).forEach((function(t){r(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):o(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function l(e,t){if(null==e)return{};var a,n,r=function(e,t){if(null==e)return{};var a,n,r={},o=Object.keys(e);for(n=0;n<o.length;n++)a=o[n],t.indexOf(a)>=0||(r[a]=e[a]);return r}(e,t);if(Object.getOwnPropertySymbols){var o=Object.getOwnPropertySymbols(e);for(n=0;n<o.length;n++)a=o[n],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(r[a]=e[a])}return r}var i=n.createContext({}),c=function(e){var t=n.useContext(i),a=t;return e&&(a="function"==typeof e?e(t):s(s({},t),e)),a},p=function(e){var t=c(e.components);return n.createElement(i.Provider,{value:t},e.children)},m={inlineCode:"code",wrapper:function(e){var t=e.children;return n.createElement(n.Fragment,{},t)}},d=n.forwardRef((function(e,t){var a=e.components,r=e.mdxType,o=e.originalType,i=e.parentName,p=l(e,["components","mdxType","originalType","parentName"]),d=c(a),u=r,f=d["".concat(i,".").concat(u)]||d[u]||m[u]||o;return a?n.createElement(f,s(s({ref:t},p),{},{components:a})):n.createElement(f,s({ref:t},p))}));function u(e,t){var a=arguments,r=t&&t.mdxType;if("string"==typeof e||r){var o=a.length,s=new Array(o);s[0]=d;var l={};for(var i in t)hasOwnProperty.call(t,i)&&(l[i]=t[i]);l.originalType=e,l.mdxType="string"==typeof e?e:r,s[1]=l;for(var c=2;c<o;c++)s[c]=a[c];return n.createElement.apply(null,s)}return n.createElement.apply(null,a)}d.displayName="MDXCreateElement"},6441:function(e,t,a){a.r(t),a.d(t,{frontMatter:function(){return l},contentTitle:function(){return i},metadata:function(){return c},toc:function(){return p},default:function(){return d}});var n=a(87462),r=a(63366),o=(a(67294),a(3905)),s=["components"],l={sidebar_position:6,sidebar_label:"Calculate coverage"},i="Calculate coverage",c={unversionedId:"results/workflow-details/fastq-analysis/calculate-coverage",id:"results/workflow-details/fastq-analysis/calculate-coverage",isDocsHomePage:!1,title:"Calculate coverage",description:"At this stage, the coverage of genes and target regions (capture kit intervals) by sample read alignments is",source:"@site/docs/results/workflow-details/fastq-analysis/calculate-coverage.mdx",sourceDirName:"results/workflow-details/fastq-analysis",slug:"/results/workflow-details/fastq-analysis/calculate-coverage",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage",tags:[],version:"current",sidebarPosition:6,frontMatter:{sidebar_position:6,sidebar_label:"Calculate coverage"},sidebar:"tutorialSidebar",previous:{title:"Variant discovery preprocessing",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"},next:{title:"Somatic SNVs/Indels discovery and annotation",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels"}},p=[],m={toc:p};function d(e){var t=e.components,l=(0,r.Z)(e,s);return(0,o.kt)("wrapper",(0,n.Z)({},m,l,{components:t,mdxType:"MDXLayout"}),(0,o.kt)("h1",{id:"calculate-coverage"},"Calculate coverage"),(0,o.kt)("p",null,"At this stage, the coverage of genes and target regions (capture kit intervals) by sample read alignments is\ncalculated. These metrics are important for assessing the quality of mapping in general and in the regions\nof interest (genes, transcripts). The coverage calculation results will be located\non ",(0,o.kt)("a",{parentName:"p",href:"/results/coverage-analysis"},'the "Coverage Analysis" tab')," on the sample page. The coverage calculation\nis launched\nafter ",(0,o.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"},'the "Variant discovery preprocessing" stage')," has\nbeen successfully completed."),(0,o.kt)("p",{align:"center"},(0,o.kt)("img",{src:a(12484).Z,width:"700"})),(0,o.kt)("p",null,'The "Calculate coverage" stage of sample analysis includes the following tasks:'),(0,o.kt)("ol",null,(0,o.kt)("li",{parentName:"ol"},(0,o.kt)("p",{parentName:"li"},(0,o.kt)("strong",{parentName:"p"},"Calculate coverage statistics"),". The workflow of the task depends on the sample sequencing type:"),(0,o.kt)("p",{parentName:"li"},"1.1. ",(0,o.kt)("em",{parentName:"p"},"If the sequencing type is defined as Targeted selection and a capture kit is defined for the sample,\nthe task includes the following steps:")))),(0,o.kt)("ul",null,(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html"},"bedtools coverage"),"\ncomputes the depth and breadth of coverage of sample read alignments on capture kit intervals and\nreports a histogram of coverage for each capture kit interval as well as a summary histogram for all intervals."),(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html"},"bedtools intersect"),"\nscreens for overlaps between exons from ",(0,o.kt)("a",{parentName:"li",href:"https://www.ensembl.org/index.html"},"Ensembl")," and capture kit intervals."),(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("a",{parentName:"li",href:"https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html"},"bedtools coverage"),"\ncomputes the depth and breadth of coverage of sample read alignments on exons\nfrom ",(0,o.kt)("a",{parentName:"li",href:"https://www.ensembl.org/index.html"},"Ensembl")," and reports a histogram of coverage for each exon as well\nas a summary histogram for all exons.")),(0,o.kt)("p",null,'The file with gene coverage on mapped reads can be downloaded in the "',(0,o.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,o.kt)("em",{parentName:"p"},"Calculate coverage statistics"),'" task details ("',(0,o.kt)("em",{parentName:"p"},"Download Gene coverage TSV"),'").\nYou can also open it in Google Sheets.'),(0,o.kt)("ul",null,(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("p",{parentName:"li"},(0,o.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360036856051-CollectHsMetrics-Picard-"},"GATK CollectHsMetrics"),"\ncollects hybrid-selection (HS) metrics for the sample alignment file within capture kit intervals.",(0,o.kt)("br",null),'\nThe file with the calculated metrics can be downloaded in the "',(0,o.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,o.kt)("em",{parentName:"p"},"Calculate coverage statistics"),'" task details ("',(0,o.kt)("em",{parentName:"p"},"Download HS Metrics TSV"),'").\nYou can also open it in Google Sheets.',(0,o.kt)("br",null),'\nThe file with information about the coverage of target regions (capture kit intervals) can be downloaded in\nthe same section ("',(0,o.kt)("em",{parentName:"p"},"Download Per target coverage TSV"),'"). You can also open it in Google Sheets.'),(0,o.kt)("p",{parentName:"li"},"   1.2. ",(0,o.kt)("em",{parentName:"p"},"If the sequencing type is defined as Targeted selection, but a capture kit is not defined for the sample,\nthe task includes the following step"),":")),(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("p",{parentName:"li"},(0,o.kt)("a",{parentName:"p",href:"https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html"},"bedtools coverage"),"\ncomputes the depth and breadth of coverage of sample read alignments on exons\nfrom ",(0,o.kt)("a",{parentName:"p",href:"https://www.ensembl.org/index.html"},"Ensembl"),' and reports a histogram of coverage for each exon as well\nas a summary histogram for all exons.\nThe file with gene coverage on mapped reads can be downloaded in the "',(0,o.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,o.kt)("em",{parentName:"p"},"Calculate coverage statistics"),'" task details ("',(0,o.kt)("em",{parentName:"p"},"Download Gene coverage TSV"),'").\nYou can also open it in Google Sheets.'),(0,o.kt)("p",{parentName:"li"},"   1.3. ",(0,o.kt)("em",{parentName:"p"},"If the sequencing type is defined as WGS (whole-genome sequencing), the task includes the following steps"),":")),(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("p",{parentName:"li"},(0,o.kt)("a",{parentName:"p",href:"https://bedtools.readthedocs.io/en/latest/content/tools/coverage.html"},"bedtools coverage"),"\ncomputes the depth and breadth of coverage of sample read alignments on exons\nfrom ",(0,o.kt)("a",{parentName:"p",href:"https://www.ensembl.org/index.html"},"Ensembl")," and reports a histogram of coverage for each exon as well\nas a summary histogram for all exons.")),(0,o.kt)("li",{parentName:"ul"},(0,o.kt)("p",{parentName:"li"},(0,o.kt)("a",{parentName:"p",href:"https://bedtools.readthedocs.io/en/latest/content/tools/genomecov.html"},"bedtools genomecov"),"\ncomputes histograms of aligned sample sequences coverage for the reference genome.",(0,o.kt)("br",null),'\nThe file with gene coverage on mapped reads can be downloaded in the "',(0,o.kt)("em",{parentName:"p"},"Result files"),'" section\nin the "',(0,o.kt)("em",{parentName:"p"},"Calculate coverage statistics"),'" task details ("',(0,o.kt)("em",{parentName:"p"},"Download Gene coverage TSV"),'").\nYou can also open it in Google Sheets.'))),(0,o.kt)("ol",{start:2},(0,o.kt)("li",{parentName:"ol"},(0,o.kt)("strong",{parentName:"li"},"Save coverage")," task is saving of gene and transcript coverage.")))}d.isMDXComponent=!0},12484:function(e,t,a){t.Z=a.p+"assets/images/calculate_coverage-4c4f95b4be856b7d168322d88b6ff823.png"}}]);