"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[569],{3905:function(e,n,t){t.d(n,{Zo:function(){return p},kt:function(){return m}});var r=t(7294);function a(e,n,t){return n in e?Object.defineProperty(e,n,{value:t,enumerable:!0,configurable:!0,writable:!0}):e[n]=t,e}function i(e,n){var t=Object.keys(e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);n&&(r=r.filter((function(n){return Object.getOwnPropertyDescriptor(e,n).enumerable}))),t.push.apply(t,r)}return t}function o(e){for(var n=1;n<arguments.length;n++){var t=null!=arguments[n]?arguments[n]:{};n%2?i(Object(t),!0).forEach((function(n){a(e,n,t[n])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(t)):i(Object(t)).forEach((function(n){Object.defineProperty(e,n,Object.getOwnPropertyDescriptor(t,n))}))}return e}function l(e,n){if(null==e)return{};var t,r,a=function(e,n){if(null==e)return{};var t,r,a={},i=Object.keys(e);for(r=0;r<i.length;r++)t=i[r],n.indexOf(t)>=0||(a[t]=e[t]);return a}(e,n);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(r=0;r<i.length;r++)t=i[r],n.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(e,t)&&(a[t]=e[t])}return a}var c=r.createContext({}),s=function(e){var n=r.useContext(c),t=n;return e&&(t="function"==typeof e?e(n):o(o({},n),e)),t},p=function(e){var n=s(e.components);return r.createElement(c.Provider,{value:n},e.children)},u={inlineCode:"code",wrapper:function(e){var n=e.children;return r.createElement(r.Fragment,{},n)}},g=r.forwardRef((function(e,n){var t=e.components,a=e.mdxType,i=e.originalType,c=e.parentName,p=l(e,["components","mdxType","originalType","parentName"]),g=s(t),m=a,f=g["".concat(c,".").concat(m)]||g[m]||u[m]||i;return t?r.createElement(f,o(o({ref:n},p),{},{components:t})):r.createElement(f,o({ref:n},p))}));function m(e,n){var t=arguments,a=n&&n.mdxType;if("string"==typeof e||a){var i=t.length,o=new Array(i);o[0]=g;var l={};for(var c in n)hasOwnProperty.call(n,c)&&(l[c]=n[c]);l.originalType=e,l.mdxType="string"==typeof e?e:a,o[1]=l;for(var s=2;s<i;s++)o[s]=t[s];return r.createElement.apply(null,o)}return r.createElement.apply(null,t)}g.displayName="MDXCreateElement"},4197:function(e,n,t){t.r(n),t.d(n,{frontMatter:function(){return l},contentTitle:function(){return c},metadata:function(){return s},toc:function(){return p},default:function(){return g}});var r=t(7462),a=t(3366),i=(t(7294),t(3905)),o=["components"],l={sidebar_position:4,sidebar_label:"Alignment"},c="Alignment",s={unversionedId:"pgt-pipeline/alignment",id:"pgt-pipeline/alignment",isDocsHomePage:!1,title:"Alignment",description:"To call variants, reads must be mapped to a reference genome. At Genomenal, we use the most recent version of GRCh38 genome.",source:"@site/docs/pgt-pipeline/alignment.md",sourceDirName:"pgt-pipeline",slug:"/pgt-pipeline/alignment",permalink:"/ngsw-docs-en/pgt-pipeline/alignment",tags:[],version:"current",sidebarPosition:4,frontMatter:{sidebar_position:4,sidebar_label:"Alignment"},sidebar:"tutorialSidebar",previous:{title:"Check file quality and cleanup",permalink:"/ngsw-docs-en/pgt-pipeline/check-quality-and-cleanup"},next:{title:"Variant discovery preprocessing",permalink:"/ngsw-docs-en/pgt-pipeline/variant-discovery-preprocessing"}},p=[{value:"1. Re-pair",id:"1-re-pair",children:[]},{value:"2. Alignment",id:"2-alignment",children:[]},{value:"3. Calculate coverage",id:"3-calculate-coverage",children:[]}],u={toc:p};function g(e){var n=e.components,t=(0,a.Z)(e,o);return(0,i.kt)("wrapper",(0,r.Z)({},u,t,{components:n,mdxType:"MDXLayout"}),(0,i.kt)("h1",{id:"alignment"},"Alignment"),(0,i.kt)("p",null,"To call variants, reads must be mapped to a reference genome. At Genomenal, we use the most recent version of ",(0,i.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26/"},"GRCh38")," genome.\nThe process goes through several stages."),(0,i.kt)("h2",{id:"1-re-pair"},"1. Re-pair"),(0,i.kt)("p",null,"When using paired-end sequencing, each read from one file should contain a pair in the second. When filtering, the pairing can be broken, so we delete the reads that became unpaired in the previous stages."),(0,i.kt)("h2",{id:"2-alignment"},"2. Alignment"),(0,i.kt)("p",null,"At this stage, the position of the reads on the reference genome is determined. Genomenal uses ",(0,i.kt)("a",{parentName:"p",href:"http://bio-bwa.sourceforge.net/"},"BWA Burrows-Wheeler Aligner")," (MEM algorithm), a de facto standard tool for this purpose."),(0,i.kt)("h2",{id:"3-calculate-coverage"},"3. Calculate coverage"),(0,i.kt)("p",null,"For the quality control of alignment of the regions of interest (genes, transcripts), we calculate the coverage in the genome, as well as the average coverage in the target area (complete genome, exome, panel). The results of this stage can be seen on the ",(0,i.kt)("strong",{parentName:"p"},"\u201cCoverage Analysis\u201d")," tab."))}g.isMDXComponent=!0}}]);