"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[6821],{3905:function(e,t,n){n.d(t,{Zo:function(){return m},kt:function(){return c}});var a=n(67294);function o(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function l(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function i(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?l(Object(n),!0).forEach((function(t){o(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):l(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,a,o=function(e,t){if(null==e)return{};var n,a,o={},l=Object.keys(e);for(a=0;a<l.length;a++)n=l[a],t.indexOf(n)>=0||(o[n]=e[n]);return o}(e,t);if(Object.getOwnPropertySymbols){var l=Object.getOwnPropertySymbols(e);for(a=0;a<l.length;a++)n=l[a],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(o[n]=e[n])}return o}var r=a.createContext({}),u=function(e){var t=a.useContext(r),n=t;return e&&(n="function"==typeof e?e(t):i(i({},t),e)),n},m=function(e){var t=u(e.components);return a.createElement(r.Provider,{value:t},e.children)},d={inlineCode:"code",wrapper:function(e){var t=e.children;return a.createElement(a.Fragment,{},t)}},p=a.forwardRef((function(e,t){var n=e.components,o=e.mdxType,l=e.originalType,r=e.parentName,m=s(e,["components","mdxType","originalType","parentName"]),p=u(n),c=o,h=p["".concat(r,".").concat(c)]||p[c]||d[c]||l;return n?a.createElement(h,i(i({ref:t},m),{},{components:n})):a.createElement(h,i({ref:t},m))}));function c(e,t){var n=arguments,o=t&&t.mdxType;if("string"==typeof e||o){var l=n.length,i=new Array(l);i[0]=p;var s={};for(var r in t)hasOwnProperty.call(t,r)&&(s[r]=t[r]);s.originalType=e,s.mdxType="string"==typeof e?e:o,i[1]=s;for(var u=2;u<l;u++)i[u]=n[u];return a.createElement.apply(null,i)}return a.createElement.apply(null,n)}p.displayName="MDXCreateElement"},96051:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return s},contentTitle:function(){return r},metadata:function(){return u},toc:function(){return m},default:function(){return p}});var a=n(87462),o=n(63366),l=(n(67294),n(3905)),i=["components"],s={sidebar_position:10,sidebar_label:"Copy number variations discovery"},r="Copy number variations discovery",u={unversionedId:"results/workflow-details/fastq-analysis/cnv-discovery",id:"results/workflow-details/fastq-analysis/cnv-discovery",isDocsHomePage:!1,title:"Copy number variations discovery",description:'After the "Variant discovery preprocessing" stage',source:"@site/docs/results/workflow-details/fastq-analysis/cnv-discovery.mdx",sourceDirName:"results/workflow-details/fastq-analysis",slug:"/results/workflow-details/fastq-analysis/cnv-discovery",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery",tags:[],version:"current",sidebarPosition:10,frontMatter:{sidebar_position:10,sidebar_label:"Copy number variations discovery"},sidebar:"tutorialSidebar",previous:{title:"Structural variations discovery",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations"},next:{title:"Phenotype prediction",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes"}},m=[{value:"Non-tumor or single tumor sample",id:"normal",children:[{value:"1. Estimate read counts or depths",id:"estimate-read-counts-or-depths",children:[]},{value:"2. Calculate coverage in the given regions",id:"calculate-coverage",children:[]},{value:"3. Compile a copy-number reference",id:"compile-cn-reference",children:[]},{value:"4. CNV discovery",id:"cnv-discovery",children:[]},{value:"5. Evaluating and visualizing biological features",id:"evaluate-and-visualize",children:[]},{value:"6. Annotate CNV VCF",id:"annotate-cnv",children:[]},{value:"7. Store annotated CNV bins for CNV Viewer",id:"generate-cnv-report",children:[]},{value:"8. Store annotated CNV segments for CNV Viewer",id:"generate-cnv-report",children:[]}]},{value:"Tumor sample from a tumor/control sample set",id:"tumor",children:[{value:"1. Preprocess intervals list",id:"intervals",children:[]},{value:"2. Collect read counts",id:"read-count",children:[]},{value:"3. Normalization of read counts",id:"normalization",children:[]},{value:"4. Model segments",id:"model-segments",children:[]},{value:"5. Evaluating and visualizing biological features",id:"evaluate-and-visualize",children:[]},{value:"6. CNV discovery",id:"cnv-discovery-tumor",children:[]},{value:"7. Annotate CNV VCF",id:"annotate-cnv",children:[]},{value:"8. Store annotated CNV bins for CNV Viewer",id:"generate-cnv-report",children:[]},{value:"9. Store annotated CNV segments for CNV Viewer",id:"generate-cnv-report",children:[]}]}],d={toc:m};function p(e){var t=e.components,s=(0,o.Z)(e,i);return(0,l.kt)("wrapper",(0,a.Z)({},d,s,{components:t,mdxType:"MDXLayout"}),(0,l.kt)("h1",{id:"copy-number-variations-discovery"},"Copy number variations discovery"),(0,l.kt)("p",null,"After ",(0,l.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"},'the "Variant discovery preprocessing" stage'),"\nhas been successfully completed, copy number variations (CNVs) discovery, which determines how many copies of a\nDNA region were, on average, present in the sequenced genome, can be initiated for a sample. To run the stage,\nit must be ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#cnv-discovery"},"included")," in the analysis workflow\nand the sequencing type and capture kit must be defined for the sample (either automatically or manually by the\nuser). The copy number variations discovery pipeline differs for non-tumor or single tumor samples, and tumor\nsample from a tumor/control sample set."),(0,l.kt)("p",{align:"center"},(0,l.kt)("img",{src:n(5423).Z,width:"650"})),(0,l.kt)("h2",{id:"normal"},"Non-tumor or single tumor sample"),(0,l.kt)("p",null,"In case of analysis of a tumor sequencing sample only without control or prenatal diagnosis (analysis of a non-tumor\nsequencing sample), ",(0,l.kt)("a",{parentName:"p",href:"https://cnvkit.readthedocs.io/en/stable/"},"CNVkit")," tools are used to discover copy number\nvariations."),(0,l.kt)("h3",{id:"estimate-read-counts-or-depths"},"1. Estimate read counts or depths"),(0,l.kt)("p",null,"In the first step, ",(0,l.kt)("a",{parentName:"p",href:"https://cnvkit.readthedocs.io/en/stable/pipeline.html#autobin"},"CNVkit autobin"),"\nestimates read counts or depths in an alignment file to estimate reasonable on- and off-target bin (i.e. a coverage\nassessment interval) sizes. In addition, it adds gene names to target regions. Depending on the coverage depth and\nthe sequencing type, different ways of dividing the genome into bins, within which reads will be counted, are applied:"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'If the sample sequencing type is defined as "Panel" (sequencing using a targeted panel) or "WES" (whole-exome\nsequencing), autobin is provided with the uniquely mapped genome regions and the capture kit intervals\ncorresponding to the sample (i.e. the targeted panel used for targeted selection).'),(0,l.kt)("li",{parentName:"ul"},'If the sample sequencing type is defined as "WGS" (whole-genome sequencing) or "Low-pass WGS" (whole genome\nsequencing with low coverage), autobin is provided with the genomic regions available for sequencing.\nFirst, a target bin size is calculated in the range from 10 to 1000000000 bp. Then, the estimated bin size is used\nto limit the given bin size, which is defined in ',(0,l.kt)("a",{parentName:"li",href:"/settings/settings/variant-discovery-settings#bin-size"},"parameters"),".\nThis size is used for generating an output.")),(0,l.kt)("p",null,"As a result, autobin generates target and antitarget BED files, and prints a table of estimated average read depths\nand recommended bin sizes."),(0,l.kt)("p",null,"If a reference samples panel ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#wgs-reference-panel"},"for WGS"),"\nor ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#targeted-capture-reference-panel"},"for Targeted capture"),"\nis selected for sample analysis, the bin size corresponding to the selected panel (it can be customized for some\npanels) will be used instead of estimating target bin sizes."),(0,l.kt)("p",null,'The resulting files with target and antitarget bins can be downloaded\nin the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Estimate read counts or depths"),'" task details ("',(0,l.kt)("em",{parentName:"p"},"Download target BED"),'"\nand "',(0,l.kt)("em",{parentName:"p"},"Download antitarget BED"),'", respectively).'),(0,l.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,l.kt)("div",{parentName:"div",className:"admonition-heading"},(0,l.kt)("h5",{parentName:"div"},(0,l.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,l.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,l.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,l.kt)("div",{parentName:"div",className:"admonition-content"},(0,l.kt)("p",{parentName:"div"},"If the sample analysis reveals that the sequencing sample has too low coverage with the capture kit (more than 10%\nof capture kit regions have zero coverage), then the copy number variations discovery is not further performed\nfor the sample. This can happen if the capture kit was incorrectly selected or if the sample is of low quality."))),(0,l.kt)("h3",{id:"calculate-coverage"},"2. Calculate coverage in the given regions"),(0,l.kt)("p",null,"At this step, ",(0,l.kt)("a",{parentName:"p",href:"https://cnvkit.readthedocs.io/en/stable/pipeline.html#coverage"},"CNVkit coverage"),'\ncalculates the number of reads in each of the bins determined in the previous step. Coverage is\ncalculated via mean read depth from a pileup. The resulting files with coverage information in target and\nantitarget bins can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Calculate coverage in the given regions"),'"\ntask details ("',(0,l.kt)("em",{parentName:"p"},"Download target coverage CNN"),'" and "',(0,l.kt)("em",{parentName:"p"},"Download antitarget coverage CNN"),'", respectively).'),(0,l.kt)("h3",{id:"compile-cn-reference"},"3. Compile a copy-number reference"),(0,l.kt)("p",null,"To discover copy number variations by read depth, it is necessary to estimate the depth deviation from the copy-number\nreference. Normal coverage in regions is calculated taking into account the GC content and repeat-masked proportion of\neach region. This is done using the ",(0,l.kt)("a",{parentName:"p",href:"https://cnvkit.readthedocs.io/en/stable/pipeline.html#reference"},"CNVkit reference"),"\ntool. Both target and antitarget bins are involved in the compilation."),(0,l.kt)("p",null,"If a reference samples panel ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#wgs-reference-panel"},"for WGS"),"\nor ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#targeted-capture-reference-panel"},"for Targeted capture"),"\nis selected for sample analysis, then this panel will be used instead of compiling a copy-number reference."),(0,l.kt)("p",null,'The resulting file with the expected read depth and the reliability of this estimate can be downloaded in\nthe "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Compile a copy-number reference"),'" task details ("',(0,l.kt)("em",{parentName:"p"},"Download CNN"),'").'),(0,l.kt)("h3",{id:"cnv-discovery"},"4. CNV discovery"),(0,l.kt)("p",null,"This is the key step where the copy number of each genome region is calculated based on the number of reads\nmapped to the genome. For this, ",(0,l.kt)("a",{parentName:"p",href:"https://cnvkit.readthedocs.io/en/stable/"},"CNVkit")," tools are used:"),(0,l.kt)("ol",null,(0,l.kt)("li",{parentName:"ol"},(0,l.kt)("a",{parentName:"li",href:"https://cnvkit.readthedocs.io/en/stable/pipeline.html#fix"},"CNVkit fix"),' combines the uncorrected target\nand antitarget coverage tables and correct for biases in regional coverage and GC content, according to the\nreference. The resulting table of copy number ratios can be downloaded in\nthe "',(0,l.kt)("em",{parentName:"li"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"li"},"CNV discovery"),'" task details ("',(0,l.kt)("em",{parentName:"li"},"Download CNR"),'").'),(0,l.kt)("li",{parentName:"ol"},(0,l.kt)("a",{parentName:"li",href:"https://cnvkit.readthedocs.io/en/stable/pipeline.html#segment"},"CNVkit segment"),' infers discrete copy number\nsegments from the coverage table. If the sample sequencing type is a targeted panel, or if a single tumor sample\nis analyzed, then the circular binary segmentation (CBS) is used as the segmentation algorithm. If the sample\nsequencing type is WGS, then the experimental method is used as the segmentation algorithm, that is a 3-state\nhidden Markov model with fixed amplitude for the loss, neutral, and gain states corresponding to absolute copy\nnumbers of 1, 2, and 3. The resulting file with segments can be downloaded in the same section ("',(0,l.kt)("em",{parentName:"li"},"Download CNS"),'").'),(0,l.kt)("li",{parentName:"ol"},(0,l.kt)("a",{parentName:"li",href:"https://cnvkit.readthedocs.io/en/stable/importexport.html#seg"},"CNVkit export seg")," exports the segmentation files\nto the standard SEG format to be loaded\nin ",(0,l.kt)("a",{parentName:"li",href:"https://software.broadinstitute.org/software/igv/"},"the Integrative Genomics Viewer (IGV)"),'.\nCalled segments in SEG format can be downloaded in the same section ("',(0,l.kt)("em",{parentName:"li"},"Download Called segments SEG"),'").')),(0,l.kt)("h3",{id:"evaluate-and-visualize"},"5. Evaluating and visualizing biological features"),(0,l.kt)("p",null,"At this step, the copy number variations are processed depending on the log2FC threshold value, that is\nthe logarithmic ratio of the detected copy number to the normal copy number. The default threshold value (-0.7)\ncan be changed in ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#cnv-call-threshold"},"the parameters"),". Using this\nvalue as an example, let's see how the thresholds for calling deletions and duplications on autosomes and sex\nchromosomes will be calculated:"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"Copy number variations on autosomes with log2FC \u2264 -0.7 are considered deletions. -0.7 is the default\nlog2FC threshold, below which only one autosome is called instead of the normal two."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on the X and Y chromosomes with log2FC \u2264 -2.11 are considered deletions."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on autosomes with log2FC \u2265 0.46 are considered duplications.\nThe threshold for duplication on an autosome is calculated using the following\nformula: ",(0,l.kt)("em",{parentName:"li"},"log2FC = log",(0,l.kt)("sub",null,"2"),"((2+D)/2)"),", where ",(0,l.kt)("em",{parentName:"li"},"D")," is the \u201cnumber\u201d of copy number that\nis \u201clost\u201d at log2FC = -0.7, and ",(0,l.kt)("em",{parentName:"li"},"2")," is the number of sister chromatids in the autosome."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on the X and Y chromosomes with log2FC \u2265 0.82 are considered duplications."),(0,l.kt)("li",{parentName:"ul"},"Variations that do not satisfy the thresholds, i.e. variations on autosomes with -0.7 < log2FC < 0.46\nand variations on sex chromosomes with -2.11 < log2FC < 0.82 do not pass filtering.")),(0,l.kt)("details",null,(0,l.kt)("summary",null,"Click here to see how log2FC thresholds are calculated for deletions and duplications on autosomes and sex chromosomes"),"The threshold of calling a single copy instead of two on autosomes is calculated as follows: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"21")," = log",(0,l.kt)("sub",null,"2"),"(C/2)"),", where ",(0,l.kt)("i",null,"C")," is the number of called copies.",(0,l.kt)("br",null),"Delta of copy number: ",(0,l.kt)("i",null,"D = 2 \u2013 C = 2 \u2013 2 \xd7 2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21"))),".",(0,l.kt)("br",null),"Then the corresponding threshold of calling three copies instead of two on autosomes is calculated as follows: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"23")," = log",(0,l.kt)("sub",null,"2"),"((2+D)/2) = log",(0,l.kt)("sub",null,"2"),"(2-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Threshold of calling deletions on autosomes:")," ",(0,l.kt)("i",null,"lo = T",(0,l.kt)("sub",null,"21")),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Threshold of calling duplications on autosomes:")," ",(0,l.kt)("i",null,"hi = T",(0,l.kt)("sub",null,"23")),".",(0,l.kt)("br",null),"To calculate the threshold of calling three copies instead of two on the X chromosome, we normalize to XY genotype by dividing it by copy number of 1: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"23X"),"= log",(0,l.kt)("sub",null,"2"),"((2+D)/1) = log",(0,l.kt)("sub",null,"2"),"(2+2-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + log",(0,l.kt)("sub",null,"2"),"(2-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + T",(0,l.kt)("sub",null,"23")),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling duplications on the X chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"hi")," = 1 + T",(0,l.kt)("sub",null,"23")),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"hi")," = log",(0,l.kt)("sub",null,"2"),"(3-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),"The threshold of calling one copy instead of two on the X chromosome: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"21X")," = log",(0,l.kt)("sub",null,"2"),"((2-D)/1) = log",(0,l.kt)("sub",null,"2"),"(2-2+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + T",(0,l.kt)("sub",null,"21")),".",(0,l.kt)("br",null),"Threshold of calling no copies instead of one on the X chromosome: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"10X")," = log",(0,l.kt)("sub",null,"2"),"((1-D)/1) = log",(0,l.kt)("sub",null,"2"),"(1-2+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = log",(0,l.kt)("sub",null,"2"),"(-1+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling deletions on the X chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"lo")," = 1 + T",(0,l.kt)("sub",null,"21")),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"lo")," = log",(0,l.kt)("sub",null,"2"),"(-1+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),"The threshold of calling one copy instead of none on the Y chromosome in the case of genotype XX: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"01")," = log",(0,l.kt)("sub",null,"2"),"((0+D)/1) = log",(0,l.kt)("sub",null,"2"),"(2-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = log",(0,l.kt)("sub",null,"2"),"(2\xd7(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")) =  1 + log",(0,l.kt)("sub",null,"2"),"(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling duplication ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi"))," and deletions ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo"))," on the Y chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi")," = 1 + log",(0,l.kt)("sub",null,"2"),"(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),"))"),"; ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo")," = -\u221e"),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi")," = X",(0,l.kt)("sub",null,"hi")),"; ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo")," = X",(0,l.kt)("sub",null,"lo")),"."),(0,l.kt)("p",null,"If ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#germline-aneuploidies-only"},"the CNV aneuploidies calling only")," is\nenabled for a non-tumor sample, then only aneuploidies (copy number variations on chromosomes) will be presented\nin the resulting files."),(0,l.kt)("p",null,'As a result of processing the variations, the following files are created, which can be downloaded in the\n"',(0,l.kt)("em",{parentName:"p"},"Results files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Evaluating and visualizing biological features"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'The CNV coverage file in TSV format, that is an unannotated file convenient for studying copy number variations\nof large regions (chromosome arms, chromosomes). File download link: "',(0,l.kt)("em",{parentName:"li"},"Download CNV coverage TSV"),'".\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with copy number bins in BED format ("',(0,l.kt)("em",{parentName:"li"},"Download CNV bins BED"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with copy number segments in BED format ("',(0,l.kt)("em",{parentName:"li"},"Download CNV segments BED"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with log2FC thresholds, i.e. the logarithmic ratio of the detected copy number to the normal copy number,\nbased on which copy number variations are filtered. The log2FC values at which variations on autosomes, the X\nor Y chromosome are considered deletions or duplications are provided, too.\nFile download link: "',(0,l.kt)("em",{parentName:"li"},"Download CNV thresholds JSON"),'".')),(0,l.kt)("p",null,"In addition, files with genome-wide visualization and karyogram are generated, which are convenient for studying\nlarge copy number variations. You can open them on ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnvs"},'the "Main" tab'),' or\ndownload in the "',(0,l.kt)("em",{parentName:"p"},"Results files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Evaluating and visualizing biological features"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'Karyogram-like graph with denoted chromosome-level CNVs in JPG format ("',(0,l.kt)("em",{parentName:"li"},"Download Karyogram-like graph JPG"),'").'),(0,l.kt)("li",{parentName:"ul"},"Genome-wide CNV segments' plot in JPG format (\"",(0,l.kt)("em",{parentName:"li"},"Download JPG"),'").'),(0,l.kt)("li",{parentName:"ul"},'PDF file with CNV segment graphs for all chromosomes ("',(0,l.kt)("em",{parentName:"li"},"Download PDF"),'").'),(0,l.kt)("li",{parentName:"ul"},'Files with CNV segment graphs for each chromosome separately ("',(0,l.kt)("em",{parentName:"li"},"Download ","[chromosome]"," JPG"),'").')),(0,l.kt)("h3",{id:"annotate-cnv"},"6. Annotate CNV VCF"),(0,l.kt)("p",null,"To completely and effectively interpret the CNV impact on the phenotype, each detected variation is annotated.\nAnnotation and ranking of discovered copy number variations is performed using the ",(0,l.kt)("a",{parentName:"p",href:"https://lbgi.fr/AnnotSV/"},"AnnotSV"),"\nprogram. This tool compiles functionally, regulatory and clinically relevant information and aims at providing\nannotations useful to interpret CNV potential pathogenicity and filter out CNV potential false positives.\nSegments and bins are annotated separately."),(0,l.kt)("p",null,'The resulting files with the annotated copy number variations can be downloaded in\nthe "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Annotate CNV VCF"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'A file with all copy number segments in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments all TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number segments in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments all CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number segments in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments filtered TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number segments in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments filtered CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number bins in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins all TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number bins in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins all CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number bins in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins filtered TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number bins in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins filtered CSV"),'").')),(0,l.kt)("p",null,"You can find the description of annotation format and fields\nin ",(0,l.kt)("a",{parentName:"p",href:"https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf"},"the AnnotSV documentation"),"."),(0,l.kt)("h3",{id:"generate-cnv-report"},"7. Store annotated CNV bins for CNV Viewer"),(0,l.kt)("p",null,"Saving the results of copy number variations discovery in bins for display in the embedded service for viewing\nand analyzing copy number variations ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnv-viewer"},"CNV Viewer"),"."),(0,l.kt)("h3",{id:"generate-cnv-report"},"8. Store annotated CNV segments for CNV Viewer"),(0,l.kt)("p",null,"Saving the results of copy number variations discovery in segments for display in the embedded service for viewing\nand analyzing copy number variations ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnv-viewer"},"CNV Viewer"),"."),(0,l.kt)("p",null,'After the "Copy number variations discovery" stage, the analysis can proceed\nwith ',(0,l.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/report-generation"},"report generation"),"."),(0,l.kt)("h2",{id:"tumor"},"Tumor sample from a tumor/control sample set"),(0,l.kt)("p",null,"In case of analyzing a tumor sequencing sample from a tumor and matched normal sample\npair, ",(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/"},"the GATK pipeline")," is used to to discover copy number variations."),(0,l.kt)("h3",{id:"intervals"},"1. Preprocess intervals list"),(0,l.kt)("p",null,(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/13832754597915-PreprocessIntervals"},"GATK PreprocessIntervals"),"\nprepares bins for coverage collection."),(0,l.kt)("p",null,"The way of bin preparation varies depending on the sample sequencing type:"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'If the sample sequencing type is defined as "Panel" (sequencing using a targeted panel) or "WES" (whole-exome\nsequencing), the input intervals (capture kit) are first checked for overlapping intervals, which are merged.\nThen the resulting intervals are padded and binned. Bins containing only N (unknown nucleotides) are filtered out.\nBinning (grouping intervals into a smaller number of bins) is not performed. Abutting intervals (i.e. intervals\nthat are directly side-by-side but do not actually overlap) are not merged, but treated as separate intervals.'),(0,l.kt)("li",{parentName:"ul"},'If the sample sequencing type is defined as "WGS" (whole-genome sequencing) or "Low-pass WGS" (whole genome\nsequencing with low coverage), each contig is taken as one interval and grouped accordingly. Abutting intervals\nare merged into a single continuous interval. This creates bins suitable for whole genome sequencing analysis.\nThe length of the generated bins is controlled\nby ',(0,l.kt)("a",{parentName:"li",href:"/settings/settings/variant-discovery-settings#bin-size"},"the corresponding parameter"),".\nBins containing only N (unknown nucleotides) are filtered out.")),(0,l.kt)("p",null,'The resulting preprocessed Picard interval-list file can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the\n"',(0,l.kt)("em",{parentName:"p"},"Preprocess intervals list"),'" task details ("',(0,l.kt)("em",{parentName:"p"},"Download INTERVAL_LIST"),'").'),(0,l.kt)("h3",{id:"read-count"},"2. Collect read counts"),(0,l.kt)("p",null,(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037592671-CollectReadCounts"},"GATK CollectReadCounts"),'\ncollects read counts at specified intervals. The count for each interval is calculated by counting the number\nof read starts that lie in the interval. Abutting intervals (i.e. intervals that are directly side-by-side but\ndo not actually overlap) are not merged, but treated as separate intervals. The command is executed for the\nalignment files of the tumor and matched normal samples. The resulting counts files in HDF5 format (Hierarchical\nData Format, Version 5) can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Collect read counts"),'" task\ndetails (the result for the normal sample is "',(0,l.kt)("em",{parentName:"p"},"Download Normal file read counts HDF5"),'", the result for the tumor\nsample is "',(0,l.kt)("em",{parentName:"p"},"Download Tumor file read counts HDF5"),'").'),(0,l.kt)("p",null,(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360040510031-CreateReadCountPanelOfNormals"},"GATK CreateReadCountPanelOfNormals"),'\ncreates a panel of normals (PoN) for read-count denoising given the read counts for samples in the panel.\nIt is carried out by counting the number of reads in the normal sample. Genomic intervals with a median (across\nsamples) of fractional coverage less than or equal to 5th percentile are filtered out. The resulting\npanel-of-normals file can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Collect read counts"),'" task\ndetails ("',(0,l.kt)("em",{parentName:"p"},"Download Generated panel of normal HDF5"),'").'),(0,l.kt)("h3",{id:"normalization"},"3. Normalization of read counts"),(0,l.kt)("p",null,(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360040508731-DenoiseReadCounts"},"GATK DenoiseReadCounts"),'\ndenoises read counts to produce denoised copy ratios. A panel of normals produced in the previous step is\nused to standardize and denoise read counts in the tumor sample. Standardization includes (a) transforming\nto fractional coverage, (b) filtering intervals to those contained in the panel, (c) dividing by interval\nmedians contained in the panel, (d) dividing by the sample median, and (e) transforming to log2 copy ratio.\nThe result is then denoised by subtracting the projection onto the specified number of principal components\nfrom the panel. The output is the following files, which can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in\nthe "',(0,l.kt)("em",{parentName:"p"},"Normalization of read counts"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'Denoised-copy-ratios file in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Denoised copy ratios TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'Standardized-copy-ratios file in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Standardized copy ratios TSV"),'").\nYou can open this file in Google Sheets, too.')),(0,l.kt)("h3",{id:"model-segments"},"4. Model segments"),(0,l.kt)("p",null,(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037225892-ModelSegments"},"GATK ModelSegments"),"\nmodels segmented copy ratios from denoised read counts. Segmentation is done\nusing ",(0,l.kt)("a",{parentName:"p",href:"https://javadoc.io/static/org.broadinstitute/gatk/4.1.4.0/org/broadinstitute/hellbender/tools/copynumber/utils/segmentation/KernelSegmenter.html"},"kernel segmentation"),".\nThen Markov-chain Monte Carlo (MCMC) is applied to determine posteriors for segmented models for the log2 copy ratio.\nFinally, smoothing of the segmented posteriors is performed by merging adjacent segments whose posterior credible\nintervals sufficiently overlap sufficiently (number of 10% equal-tailed credible-interval widths must be equal to 2).\nAdditional rounds of segmentation smoothing are performed until convergence, at which point a final round of MCMC is\nperformed."),(0,l.kt)("p",null,'Copy-ratio segment file can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Model segments"),'" task\ndetails ("',(0,l.kt)("em",{parentName:"p"},"Download Copy ration segments SEG"),'"). A file with modeled segments after segmentation smoothing\ncan be downloaded in the same section ("',(0,l.kt)("em",{parentName:"p"},"Download Modeled segments SEG"),'").'),(0,l.kt)("h3",{id:"evaluate-and-visualize"},"5. Evaluating and visualizing biological features"),(0,l.kt)("p",null,"At this step, the copy number variations are processed depending on the log2FC threshold value, that is\nthe logarithmic ratio of the detected copy number to the normal copy number. The default threshold value (-0.7)\ncan be changed in ",(0,l.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#cnv-call-threshold"},"the parameters"),". Using this\nvalue as an example, let's see how the thresholds for calling deletions and duplications on autosomes and sex\nchromosomes will be calculated:"),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},"Copy number variations on autosomes with log2FC \u2264 -0.7 are considered deletions. -0.7 is the default\nlog2FC threshold, below which only one autosome is called instead of the normal two."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on the X and Y chromosomes with log2FC \u2264 -2.11 are considered deletions."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on autosomes with log2FC \u2265 0.46 are considered duplications.\nThe threshold for duplication on an autosome is calculated using the following\nformula: ",(0,l.kt)("em",{parentName:"li"},"log2FC = log",(0,l.kt)("sub",null,"2"),"((2+D)/2)"),", where ",(0,l.kt)("em",{parentName:"li"},"D")," is the \u201cnumber\u201d of copy number that\nis \u201clost\u201d at log2FC = -0.7, and ",(0,l.kt)("em",{parentName:"li"},"2")," is the number of sister chromatids in the autosome."),(0,l.kt)("li",{parentName:"ul"},"Copy number variations on the X and Y chromosomes with log2FC \u2265 0.82 are considered duplications."),(0,l.kt)("li",{parentName:"ul"},"Variations that do not satisfy the thresholds, i.e. variations on autosomes with -0.7 < log2FC < 0.46\nand variations on sex chromosomes with -2.11 < log2FC < 0.82 do not pass filtering.")),(0,l.kt)("details",null,(0,l.kt)("summary",null,"Click here to see how log2FC thresholds are calculated for deletions and duplications on autosomes and sex chromosomes"),"The threshold of calling a single copy instead of two on autosomes is calculated as follows: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"21")," = log",(0,l.kt)("sub",null,"2"),"(C/2)"),", where ",(0,l.kt)("i",null,"C")," is the number of called copies.",(0,l.kt)("br",null),"Delta of copy number: ",(0,l.kt)("i",null,"D = 2 \u2013 C = 2 \u2013 2 \xd7 2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21"))),".",(0,l.kt)("br",null),"Then the corresponding threshold of calling three copies instead of two on autosomes is calculated as follows: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"23")," = log",(0,l.kt)("sub",null,"2"),"((2+D)/2) = log",(0,l.kt)("sub",null,"2"),"(2-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Threshold of calling deletions on autosomes:")," ",(0,l.kt)("i",null,"lo = T",(0,l.kt)("sub",null,"21")),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Threshold of calling duplications on autosomes:")," ",(0,l.kt)("i",null,"hi = T",(0,l.kt)("sub",null,"23")),".",(0,l.kt)("br",null),"To calculate the threshold of calling three copies instead of two on the X chromosome, we normalize to XY genotype by dividing it by copy number of 1: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"23X"),"= log",(0,l.kt)("sub",null,"2"),"((2+D)/1) = log",(0,l.kt)("sub",null,"2"),"(2+2-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + log",(0,l.kt)("sub",null,"2"),"(2-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + T",(0,l.kt)("sub",null,"23")),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling duplications on the X chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"hi")," = 1 + T",(0,l.kt)("sub",null,"23")),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"hi")," = log",(0,l.kt)("sub",null,"2"),"(3-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),"The threshold of calling one copy instead of two on the X chromosome: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"21X")," = log",(0,l.kt)("sub",null,"2"),"((2-D)/1) = log",(0,l.kt)("sub",null,"2"),"(2-2+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = 1 + T",(0,l.kt)("sub",null,"21")),".",(0,l.kt)("br",null),"Threshold of calling no copies instead of one on the X chromosome: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"10X")," = log",(0,l.kt)("sub",null,"2"),"((1-D)/1) = log",(0,l.kt)("sub",null,"2"),"(1-2+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = log",(0,l.kt)("sub",null,"2"),"(-1+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling deletions on the X chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"lo")," = 1 + T",(0,l.kt)("sub",null,"21")),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"X",(0,l.kt)("sub",null,"lo")," = log",(0,l.kt)("sub",null,"2"),"(-1+2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),"The threshold of calling one copy instead of none on the Y chromosome in the case of genotype XX: ",(0,l.kt)("i",null,"T",(0,l.kt)("sub",null,"01")," = log",(0,l.kt)("sub",null,"2"),"((0+D)/1) = log",(0,l.kt)("sub",null,"2"),"(2-2\xd72",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),") = log",(0,l.kt)("sub",null,"2"),"(2\xd7(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")) =  1 + log",(0,l.kt)("sub",null,"2"),"(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),")"),".",(0,l.kt)("br",null),(0,l.kt)("b",null,"Calculating the threshold of calling duplication ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi"))," and deletions ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo"))," on the Y chromosome:"),(0,l.kt)("br",null),"- if there is no Y chromosome: ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi")," = 1 + log",(0,l.kt)("sub",null,"2"),"(1-2",(0,l.kt)("sup",null,"T",(0,l.kt)("sub",null,"21")),"))"),"; ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo")," = -\u221e"),";",(0,l.kt)("br",null),"- if there is Y chromosome: ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"hi")," = X",(0,l.kt)("sub",null,"hi")),"; ",(0,l.kt)("i",null,"Y",(0,l.kt)("sub",null,"lo")," = X",(0,l.kt)("sub",null,"lo")),"."),(0,l.kt)("p",null,'As a result of processing the variations, the following files are created, which can be downloaded in the\n"',(0,l.kt)("em",{parentName:"p"},"Results files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Evaluating and visualizing biological features"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'The CNV coverage file in TSV format, that is an unannotated file convenient for studying copy number variations\nof large regions (chromosome arms, chromosomes). File download link: "',(0,l.kt)("em",{parentName:"li"},"Download CNV coverage TSV"),'".\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with copy number bins in BED format ("',(0,l.kt)("em",{parentName:"li"},"Download CNV bins BED"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with copy number segments in BED format ("',(0,l.kt)("em",{parentName:"li"},"Download CNV segments BED"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with log2FC thresholds, i.e. the logarithmic ratio of the detected copy number to the normal copy number,\nbased on which copy number variations are filtered. The log2FC values at which variations on autosomes, the X\nor Y chromosome are considered deletions or duplications are provided, too.\nFile download link: "',(0,l.kt)("em",{parentName:"li"},"Download CNV thresholds JSON"),'".')),(0,l.kt)("p",null,"In addition, files with genome-wide visualization and karyogram are generated, which are convenient for studying\nlarge copy number variations. You can open them on ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnvs"},'the "Main" tab'),' or\ndownload in the "',(0,l.kt)("em",{parentName:"p"},"Results files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Evaluating and visualizing biological features"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'Karyogram-like graph with denoted chromosome-level CNVs in JPG format ("',(0,l.kt)("em",{parentName:"li"},"Download Karyogram-like graph JPG"),'").'),(0,l.kt)("li",{parentName:"ul"},"Genome-wide CNV segments' plot in JPG format (\"",(0,l.kt)("em",{parentName:"li"},"Download JPG"),'").'),(0,l.kt)("li",{parentName:"ul"},'PDF file with CNV segment graphs for all chromosomes ("',(0,l.kt)("em",{parentName:"li"},"Download PDF"),'").'),(0,l.kt)("li",{parentName:"ul"},'Files with CNV segment graphs for each chromosome separately ("',(0,l.kt)("em",{parentName:"li"},"Download ","[chromosome]"," JPG"),'").')),(0,l.kt)("h3",{id:"cnv-discovery-tumor"},"6. CNV discovery"),(0,l.kt)("p",null,"This is the key step where the copy number of each genome region is calculated based on the number of reads\nmapped to the genome. In case of analyzing a tumor sequencing sample from a tumor and matched normal sample\npair, ",(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/"},"the GATK pipeline")," is used."),(0,l.kt)("p",null,"Copy-ratio segments are called as amplified, deleted, or copy-number neutral using\n",(0,l.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360041850791-CallCopyRatioSegments"},"GATK CallCopyRatioSegments"),".\nThis is a relatively naive caller that takes the segments modeled\nin ",(0,l.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/cnv-discovery#model-segments"},'the "Model segments" step')," and\nperforms a simple statistical test on the segmented log2 copy ratios to call amplifications and deletions, given\na specified range for determining copy-number neutral segments."),(0,l.kt)("p",null,'The output is the following files, which can be downloaded in the "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in\nthe "',(0,l.kt)("em",{parentName:"p"},"CNV discovery"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'Called copy-ratio-segments file in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Called segments SEG"),'").'),(0,l.kt)("li",{parentName:"ul"},'CBS-formatted (circular binary segmentation) file with information on segments ("',(0,l.kt)("em",{parentName:"li"},"Download Statistics SEG"),'").\nIt is compatible with ',(0,l.kt)("a",{parentName:"li",href:"https://software.broadinstitute.org/software/igv/"},"the Integrative Genomics Viewer (IGV)"),".\nYou can find a description of the file format and its\ncolumns ",(0,l.kt)("a",{parentName:"li",href:"https://www.genepattern.org/file-formats-guide#CBS"},"here"),".")),(0,l.kt)("h3",{id:"annotate-cnv"},"7. Annotate CNV VCF"),(0,l.kt)("p",null,"To completely and effectively interpret the CNV impact on the phenotype, each detected variation is annotated.\nAnnotation and ranking of discovered copy number variations is performed using the ",(0,l.kt)("a",{parentName:"p",href:"https://lbgi.fr/AnnotSV/"},"AnnotSV"),"\nprogram. This tool compiles functionally, regulatory and clinically relevant information and aims at providing\nannotations useful to interpret CNV potential pathogenicity and filter out CNV potential false positives.\nSegments and bins are annotated separately."),(0,l.kt)("p",null,'The resulting files with the annotated copy number variations can be downloaded in\nthe "',(0,l.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,l.kt)("em",{parentName:"p"},"Annotate CNV VCF"),'" task details:'),(0,l.kt)("ul",null,(0,l.kt)("li",{parentName:"ul"},'A file with all copy number segments in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments all TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number segments in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments all CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number segments in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments filtered TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number segments in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Segments filtered CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number bins in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins all TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with all copy number bins in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins all CSV"),'").'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number bins in TSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins filtered TSV"),'").\nYou can also open this file in Google Sheets.'),(0,l.kt)("li",{parentName:"ul"},'A file with filtered copy number bins in CSV format ("',(0,l.kt)("em",{parentName:"li"},"Download Bins filtered CSV"),'").')),(0,l.kt)("p",null,"You can find the description of annotation format and fields\nin ",(0,l.kt)("a",{parentName:"p",href:"https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf"},"the AnnotSV documentation"),"."),(0,l.kt)("h3",{id:"generate-cnv-report"},"8. Store annotated CNV bins for CNV Viewer"),(0,l.kt)("p",null,"Saving the results of copy number variations discovery in bins for display in the embedded service for viewing\nand analyzing copy number variations ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnv-viewer"},"CNV Viewer"),"."),(0,l.kt)("h3",{id:"generate-cnv-report"},"9. Store annotated CNV segments for CNV Viewer"),(0,l.kt)("p",null,"Saving the results of copy number variations discovery in segments for display in the embedded service for viewing\nand analyzing copy number variations ",(0,l.kt)("a",{parentName:"p",href:"/results/main/copy-number-variations/cnv-viewer"},"CNV Viewer"),"."),(0,l.kt)("p",null,'After the "Copy number variations discovery" stage, the analysis can proceed\nwith ',(0,l.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/report-generation"},"report generation"),"."))}p.isMDXComponent=!0},5423:function(e,t,n){t.Z=n.p+"assets/images/cnvs-9c6786a09c7f6d8d92185dfc2301398c.png"}}]);