"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[4846],{3905:function(e,t,n){n.d(t,{Zo:function(){return p},kt:function(){return m}});var i=n(67294);function a(e,t,n){return t in e?Object.defineProperty(e,t,{value:n,enumerable:!0,configurable:!0,writable:!0}):e[t]=n,e}function r(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);t&&(i=i.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,i)}return n}function o(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?r(Object(n),!0).forEach((function(t){a(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function s(e,t){if(null==e)return{};var n,i,a=function(e,t){if(null==e)return{};var n,i,a={},r=Object.keys(e);for(i=0;i<r.length;i++)n=r[i],t.indexOf(n)>=0||(a[n]=e[n]);return a}(e,t);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(e);for(i=0;i<r.length;i++)n=r[i],t.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(e,n)&&(a[n]=e[n])}return a}var l=i.createContext({}),c=function(e){var t=i.useContext(l),n=t;return e&&(n="function"==typeof e?e(t):o(o({},t),e)),n},p=function(e){var t=c(e.components);return i.createElement(l.Provider,{value:t},e.children)},d={inlineCode:"code",wrapper:function(e){var t=e.children;return i.createElement(i.Fragment,{},t)}},h=i.forwardRef((function(e,t){var n=e.components,a=e.mdxType,r=e.originalType,l=e.parentName,p=s(e,["components","mdxType","originalType","parentName"]),h=c(n),m=a,g=h["".concat(l,".").concat(m)]||h[m]||d[m]||r;return n?i.createElement(g,o(o({ref:t},p),{},{components:n})):i.createElement(g,o({ref:t},p))}));function m(e,t){var n=arguments,a=t&&t.mdxType;if("string"==typeof e||a){var r=n.length,o=new Array(r);o[0]=h;var s={};for(var l in t)hasOwnProperty.call(t,l)&&(s[l]=t[l]);s.originalType=e,s.mdxType="string"==typeof e?e:a,o[1]=s;for(var c=2;c<r;c++)o[c]=n[c];return i.createElement.apply(null,o)}return i.createElement.apply(null,n)}h.displayName="MDXCreateElement"},4172:function(e,t,n){n.r(t),n.d(t,{frontMatter:function(){return s},contentTitle:function(){return l},metadata:function(){return c},toc:function(){return p},default:function(){return h}});var i=n(87462),a=n(63366),r=(n(67294),n(3905)),o=["components"],s={sidebar_position:5,sidebar_label:"ACMG Secondary Findings Report"},l="ACMG Secondary Findings Report",c={unversionedId:"results/acmg-sf-report",id:"results/acmg-sf-report",isDocsHomePage:!1,title:"ACMG Secondary Findings Report",description:"The report includes the variants discovered in the sample, which are located in the genes",source:"@site/docs/results/acmg-sf-report.mdx",sourceDirName:"results",slug:"/results/acmg-sf-report",permalink:"/ngsw-docs-en/results/acmg-sf-report",tags:[],version:"current",sidebarPosition:5,frontMatter:{sidebar_position:5,sidebar_label:"ACMG Secondary Findings Report"},sidebar:"tutorialSidebar",previous:{title:"Sample Interpretation",permalink:"/ngsw-docs-en/results/sample-interpretation"},next:{title:"ClinVar Phenotypes Report",permalink:"/ngsw-docs-en/results/variants-by-phenotypes"}},p=[{value:"Conditions for the report generation",id:"acmg-sf-conditions",children:[]},{value:"Report content",id:"acmg-sf-content",children:[]},{value:"Potential findings",id:"potential-findings",children:[]},{value:"Report limitations in ClinVar as a database source",id:"acmg-sf-limitations",children:[]}],d={toc:p};function h(e){var t=e.components,s=(0,a.Z)(e,o);return(0,r.kt)("wrapper",(0,i.Z)({},d,s,{components:t,mdxType:"MDXLayout"}),(0,r.kt)("h1",{id:"acmg-secondary-findings-report"},"ACMG Secondary Findings Report"),(0,r.kt)("p",null,"The report includes the variants discovered in the sample, which are located in the genes\nfrom ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the Secondary Findings (SF) list")," of\nAmerican College of Medical Genetics and Genomics (ACMG). For these genes, specific mutations are known\nto be causative of disorders with defined phenotypes that are\nclinically actionable by an accepted intervention. The ACMG recommends that pathogenic or likely pathogenic\nvariants detected in any of these genes should be reported as they are of medical relevance and could be used\nin the future to inform clinical treatment."),(0,r.kt)("div",{className:"admonition admonition-caution alert alert--warning"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"16",height:"16",viewBox:"0 0 16 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"}))),"caution")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"ACMG secondary findings report generation has its limitations due to variant annotation\nusing ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),".\nThese limitations are described ",(0,r.kt)("a",{parentName:"p",href:"/results/acmg-sf-report#acmg-sf-limitations"},"below"),"."))),(0,r.kt)("h3",{id:"acmg-sf-conditions"},"Conditions for the report generation"),(0,r.kt)("p",null,"The report is generated for the sample if the following conditions are met:"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},"The sample is uploaded in FASTQ or BAM format or as a VCF file with a single sample."),(0,r.kt)("li",{parentName:"ol"},'The sample analysis was successfully completed (that is, all stages of the analysis have the status "Complete").'),(0,r.kt)("li",{parentName:"ol"},'For a sample, the variant annotation stage was successfully completed:\n"Germline SNVs/Indels annotation" stage if Germline origin was selected when creating a report template,\nor "Somatic SNVs/Indels annotation", if Somatic origin.'),(0,r.kt)("li",{parentName:"ol"},'The report template including "SNVs/Indels in ACMG SF genes" block\nwas ',(0,r.kt)("a",{parentName:"li",href:"/settings/report-templates/report-templates#add-template"},"created")," before the sample was processed."),(0,r.kt)("li",{parentName:"ol"},'The report template including "SNVs/Indels in ACMG SF genes" block is active.')),(0,r.kt)("h3",{id:"acmg-sf-content"},"Report content"),(0,r.kt)("p",null,"The report includes two sections with lists of variants discovered in the sample, which are filled in according\nto the following scheme:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(62833).Z,width:"560"})),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},"TIER I"),": The most important variants for interpretation for which the following conditions are met:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"The variant is located in a gene from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list"),";"),(0,r.kt)("li",{parentName:"ul"},"The variant is associated according to ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," with a\nphenotype only of pathogenic or likely pathogenic clinical significance that is\nlisted in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list")," for\nthe gene in which the variant is located;"),(0,r.kt)("li",{parentName:"ul"},"The allele frequency in ",(0,r.kt)("a",{parentName:"li",href:"https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/"},"gnomAD 3"),' is empty or\nbelow the frequency threshold for too frequent pathogenic variants, specified in the report template block\n"SNVs/Indels in ACMG SF genes" (recommended default value = 0.03);'),(0,r.kt)("li",{parentName:"ul"},"The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)\nin the case of a VCF file with a single sample;"),(0,r.kt)("li",{parentName:"ul"},"The variant meets all the conditions specified in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list"),"\nfor the gene in which the variant is located. These conditions may be associated with:",(0,r.kt)("ul",{parentName:"li"},(0,r.kt)("li",{parentName:"ul"},"the clinical significance of the phenotype (as noted above, the report primarily includes variants\nassociated with a phenotype only with pathogenic or likely pathogenic clinical significance\naccording to ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),"; this condition is not required only\nfor variants in ",(0,r.kt)("em",{parentName:"li"},"HFE")," gene);"),(0,r.kt)("li",{parentName:"ul"},"type of inheritance;"),(0,r.kt)("li",{parentName:"ul"},"the number of variants discovered in the sample and suitable for the conditions\nthat are located in one gene from the ACMG SF list (for example, for some variants with an autosomal\nrecessive inheritance to be included in the report, at least two such variants must be discovered in the sample);"),(0,r.kt)("li",{parentName:"ul"},"the variant origin;"),(0,r.kt)("li",{parentName:"ul"},"the genotype (for example, some variants are reported only if they are homozygous in alternative allele);"),(0,r.kt)("li",{parentName:"ul"},"specific nucleotide and amino acid substitution of the variant.")))),(0,r.kt)("p",null,(0,r.kt)("strong",{parentName:"p"},"TIER II"),": Variants that require further analysis for which the following conditions are met:"),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},"The variant is located in a gene from ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list"),";"),(0,r.kt)("li",{parentName:"ul"},"The variant is associated with at least one phenotype of pathogenic and/or likely pathogenic clinical\nsignificance according to ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),";"),(0,r.kt)("li",{parentName:"ul"},"The allele frequency in ",(0,r.kt)("a",{parentName:"li",href:"https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/"},"gnomAD 3"),' is empty or\nbelow the frequency threshold for too frequent pathogenic variants, specified in the report template block\n"SNVs/Indels in ACMG SF genes" (recommended default value = 0.03);'),(0,r.kt)("li",{parentName:"ul"},"The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)\nin the case of a VCF file with a single sample;"),(0,r.kt)("li",{parentName:"ul"},"The variant does not meet at least one of the conditions in Tier I:",(0,r.kt)("ul",{parentName:"li"},(0,r.kt)("li",{parentName:"ul"},"the variant is not associated according to ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," with a\nphenotype that is listed in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list")," for the gene\nin which the variant is located;"),(0,r.kt)("li",{parentName:"ul"},"according to ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar"),", the variant is associated with at\nleast one phenotype of benign, likely benign, or uncertain clinical significance, which is listed\nin ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list")," for the gene in which the variant is located;"),(0,r.kt)("li",{parentName:"ul"},"the variant does not meet at least one of the conditions specified\nin ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list")," for the gene in which the variant is located.")))),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(17853).Z,width:"1000"})),(0,r.kt)("p",null,"Each section contains a table of variants with the following columns:"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Gene")," is the common name of the gene in which the variant is located.\nIf you click on a gene, you will see a window with all the gene transcripts:")),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(27922).Z,width:"600"})),(0,r.kt)("p",null,"You can find the description of the transcripts' table\ncolumns ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/snv-details-panel#transcripts"},"here"),"."),(0,r.kt)("ol",{start:2},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Position")," is a coordinate of the variant in the genome (chromosome + start position)."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Genetic variant")," is the nucleotide and amino acid substitutions\nusing ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"the HGVS notation"),". Nucleotide substitution: \u201cc.\u201d (coding; for a\nsubstitution in the coding sequence) or \u201cn.\u201d (non-coding; for a substitution in the non-coding sequence) prefix +\ngenomic position of the substituted nucleotide + reference allele > alternative allele.\nAmino acid substitution: \u201cp.\u201d prefix (protein) + reference amino acid + amino acid position in protein + new\namino acid resulting from the substitution."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Transcript")," is the main transcript ID from the ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/nuccore/"},"RefSeq"),' database\n(NM_xxxxxx.x). If you click on a transcript, you will see the same window with all the gene transcripts, as when\nyou click on the "Gene" field.'),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Consequence")," is the effect of the variant on genes. A detailed description of the possible values\ncan be found ",(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/variant-viewer#variant-consequences"},"here"),"."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Allele frequency")," is an alternative allele frequency for the sample (in percentages)."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"Coverage depth")," is a sequencing depth; the total number of reads of the sequence overlapping the variant\nposition for the sample."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"External links")," are links to pages with variant information in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/snp/"},"dbSNP"),",\n",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," and ",(0,r.kt)("a",{parentName:"li",href:"https://cancer.sanger.ac.uk/cosmic/"},"COSMIC")," (if\nit was ",(0,r.kt)("a",{parentName:"li",href:"/settings/custom-annotations#define-annotation-type"},"uploaded as a custom annotation"),")."),(0,r.kt)("li",{parentName:"ol"},"Links to the variant in embedded services: ",(0,r.kt)("img",{src:n(79321).Z,width:"30"})," is a module for visualization of variant on the genome, ",(0,r.kt)("img",{src:n(30643).Z,width:"20"})," is\na variant details page in SNV Viewer\n(",(0,r.kt)("a",{parentName:"li",href:"/results/main/snvs-indels/snv-details-page/#annotation-tab"},'"Annotation" tab'),").")),(0,r.kt)("p",null,"Below the variant row, there can be phenotypes with pathogenic or likely pathogenic clinical significance that the\nvariant is associated with according to ",(0,r.kt)("a",{parentName:"p",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," and that are listed\nin ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the ACMG SF list")," for the gene in which the variant is located.\nLinks to ",(0,r.kt)("a",{parentName:"p",href:"https://www.omim.org/"},"OMIM")," and/or ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/medgen/"},"MedGen")," are provided for each\nphenotype."),(0,r.kt)("p",null,"Next, there can be the variant interpretation text, if it was added as\ndescribed ",(0,r.kt)("a",{parentName:"p",href:"/results/main/snvs-indels/variant-interpretation#add-and-edit"},"here"),"."),(0,r.kt)("p",null,"At the end of the report, there is the version of the ACMG SF list used to generate the report."),(0,r.kt)("h3",{id:"potential-findings"},"Potential findings"),(0,r.kt)("p",null,"Variants included in the ACMG SF report can be used as ",(0,r.kt)("a",{parentName:"p",href:"/results/potential-findings"},"potential findings"),".\nTo do this, you need\nto ",(0,r.kt)("a",{parentName:"p",href:"/settings/report-templates/report-templates#block-types"},"turn on the corresponding toggle in the report template"),".\nPotential findings can include Tier I variants\n(\u201c",(0,r.kt)("em",{parentName:"p"},"Use variants in ACMG SF genes list associated with the phenotype results as potential findings"),"\u201d\noption) and/or Tier II variants\n(\u201c",(0,r.kt)("em",{parentName:"p"},"Use variants in ACMG SF genes list without established phenotype association or genes conditions as potential findings"),'"\noption).'),(0,r.kt)("h3",{id:"acmg-sf-limitations"},"Report limitations in ClinVar as a database source"),(0,r.kt)("p",null,"ACMG secondary findings report generation has limitations\nof ",(0,r.kt)("a",{parentName:"p",href:"/results/acmg-sf-report#phenotypic-data"},"phenotypic"),"\nand ",(0,r.kt)("a",{parentName:"p",href:"/results/acmg-sf-report#pathogenicity-data"},"pathogenicity")," data\ndue to variant annotation using ",(0,r.kt)("a",{parentName:"p",href:"https://www.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," database."),(0,r.kt)("p",null,"ACMG guidelines for secondary findings in sequencing data provide a list of genes and\ncorresponding phenotypes along with recommendations for reporting genetic variants\nregardless of the reason of genetic testing. The following three conditions should be satisfied:"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},"The variant affects a gene\nfrom ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/"},"the list of secondary findings"),";"),(0,r.kt)("li",{parentName:"ol"},"The variant is reported as associated with the phenotype (condition, disease) listed with\nthe gene in the list for secondary findings, or there is a possibility to establish such an association."),(0,r.kt)("li",{parentName:"ol"},"The variant must meet all additional conditions specified in the list of secondary findings, which often\ninclude pathogenic or likely pathogenic clinical significance of the phenotype.")),(0,r.kt)("p",null,"Briefly, it can be described as:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(58649).Z,width:"500"})),(0,r.kt)("h4",{id:"phenotypic-data"},"Limitations of phenotypic data"),(0,r.kt)("p",null,'For some of variants in ClinVar, it is impossible to obtain information about phenotype (or condition,\nor disease). Missense of phenotypic information in Clinvar can stem from inconsistencies in "Conditions"\nfield in reports submitted by laboratories. The field can contain values "not reported", "not specified",\nor can be filled with all the diseases associated with any of the pathogenic variants in the\ngene. In the latter case, correct conclusions can be only by a specialist. Moreover, mutations in\ndifferent portions of the gene can result in different genetic diseases, while the list for secondary\nfindings can contain only a fraction of these phenotypes. Considering these and other factors, the\nACMG Secondary Findings Report by Genomenal includes two lists of variants: those for which the phenotypes\nin the ACMG SF list match the condition reported for the variants in ClinVar, and those that are located\nin a gene in the list but have a discordant phenotype.'),(0,r.kt)("i",null,"Therefore, the interpretation of the variants in the second section of the report must be approached with caution."),(0,r.kt)("h4",{id:"pathogenicity-data"},"Limitations of pathogenicity data"),(0,r.kt)("p",null,"Usually variants are classified according to ACMG guidelines. ClinVar database is populated\nby reports of pathogenicity from laboratories assuming that the responsible submitter fully\ndescribed submitted variants according to ACMG recommendations. Similarly, Genomenal follows the\nsame assumptions and variants are included in the report if they are reported as Pathogenic or\nLikely Pathogenic without further expert evaluation of pathogenicity."),(0,r.kt)("p",null,"Additionally, new evidence might emerge in literature about functional effect or lack of it for a\nspecific variant. It can result in a revision of variant pathogenicity but will not be instantly reflected\nin ClinVar. Undoubtedly, systematical review of variants' pathogenicity status is labour-consuming\nand can lag compared to emergency of new scientific evidence. These factors can result in\noverreporting of variants, or presence of outdated data, or underreporting of variants for\nwhich new data has emerged but it is not reflected in ClinVar. Some variants can be omitted if they\npathogenicity is yet unknown and not detected by anyone."),(0,r.kt)("p",null,(0,r.kt)("em",{parentName:"p"},"Thus, the report can include the following variants:")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("em",{parentName:"li"},"with imprecise or erroneous pathogenicity status reported by the submitting laboratory;")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("em",{parentName:"li"},"with pathogenicity revised due to newly emerged data (confirmed")," ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"pathogenic"))," ",(0,r.kt)("em",{parentName:"li"},"by new data)."))),(0,r.kt)("p",null,(0,r.kt)("em",{parentName:"p"},"The following variants will not be included in the report:")),(0,r.kt)("ul",null,(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("em",{parentName:"li"},"not yet reported in ClinVar or newly discovered;")),(0,r.kt)("li",{parentName:"ul"},(0,r.kt)("em",{parentName:"li"},"with pathogenicity revised due to newly emerged data (confirmed")," ",(0,r.kt)("strong",{parentName:"li"},(0,r.kt)("em",{parentName:"strong"},"benign"))," ",(0,r.kt)("em",{parentName:"li"},"by new data)."))),(0,r.kt)("div",{className:"admonition admonition-caution alert alert--warning"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"16",height:"16",viewBox:"0 0 16 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"}))),"caution")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},"By using ACMG Secondary Findings report, you agree that you understand above mentioned limitations,\nand if clarification is needed, you are willing to discuss the result with medical specialist."))))}h.isMDXComponent=!0},17853:function(e,t,n){t.Z=n.p+"assets/images/acmg73-d892b957bb3567aa72167cdb73e077ae.png"},58649:function(e,t,n){t.Z=n.p+"assets/images/acmg73_disclaimer-e719e1595e1b27773707e84aa367aeb8.png"},62833:function(e,t,n){t.Z=n.p+"assets/images/acmg_sf_scheme-1c3033dc0cfaf3a3a96d35469566d0cb.png"},79321:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIgAAACBCAYAAAAbtfKaAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAzQSURBVHhe7ZwNjx03FYb5//8AaKEI1AIpESECykdpFaoAJYVIaZRURV2pSoigFVUSIrRJmkuf3Wut13uOfWwfz9x716/0qs29Mx6P/fj1mcltv7WZmspoAjKV1QRkKqvVAHn58uWJv/761faTNtHG8+cv3NqbOq9FAXnv/b9u3nzrV5tvf+fKBV+//v7mk0+ONk+e/Hd7tKyH//x3th1MW7du3S22NVXWIoAwqdJESmZypYnlM76Tzsl5gtKn4YC0TCpmYoNqAJNM2kxI2jQUECZZmjCrOZ9tR/quxROSeg0DpHfVj/BMknoNA0QrIt9865fnilFA4s98Lh2vWSpqaevWR3ezbaU1Dm2QVJjiF3MM/dfv4fQ7jsOcy7Vp99AAHAKItrUwmJoYWCskTGpJTLR0Lg4ASN95OIVmn7UYIEx+SRZIGHSrSBOpjaUdA7NvGgIIA5IOEpNlUW7lW9uIlWtvDQdY9iVZhgAiDYx19bB9SOdbEkiTdeta2sCS1lG7JjMgoZgLRVxuj5UGwzoI2rbQkh5BpRQBIO6J4zDX4n4x9xg7fM4xmPM4vwdC6iHa3EVQioAwKLmCjgFKb0w6ngGwSAOEfrSKa8dtBRC8JyVARNstwOwiKFlAuFHpRiSTKEHSeXxmkTawPWLA1xh4rgcwIWWk+5IcQNkFqYBwY1Lncw4TACzS96VtQksPvEurqlW1sEjpvLREQOiU1OGSIT+sGm1bAgLpphkM6fjgXVlRHqrdhtZ86hEB0RIA37t/tLl587b4HQ4TmWuDgQEIBsk6UNYtat/EeFnuf600EQFhMtIOXvvFe+d+jHN8/Pzks/S4uBbRUqTFDOJaq2gJWUFZOklFQKSOPXx48SmCNEmPg/SgXIq0uFTDHIKkxZl6yS1HBERa+RIgfJYeF+qQUk3R6ssAyWkNl0+TpbYcERBpcqk7YrHdpMfg05rCb2uJHafTZRCLIQdKWIwjZa5BMJA8fvz0xFL9UWsm3LLv4suQHJJKaTIaEhEQLjgiBQCPIiu9IQYhPPpJ5/HdZRbjVdqyR42RCAjyLDC5OesNcFxYMZw3OkL3SaRoOraxR4yVCgjySJHWreGybiklleoSb0iygCCSpBWUOcljlIPEuyYpAoKIfUA5LSpPYaFe4DNqirSTmBuYGqd4K7449n6QmADJSatVRhVNU2fKJYlX/dYNiNS5ubUsJ8ZamgPM4u1VFyAQKnZsArKocpD0JnkXIFx8RKem6qVB0luPdAGiFahT6yhXj7SqCRDAwFw47Uzr08uzZ//b3Lnz6eb37/5585cP72wePfpi+82UVSSFBklrPWIGhG1DAkJyrT78BgipnWvX/rD56qsn26OmLPLeakyAaI+ymms6kiuw8JW3f7s9csoqbUxbUiQLCBNtTY3YbD8W8as06fzU9+5/tj1jyiptq6lNkSwgtckRbC2K7itFbuqfXf3d9owpq7QUqS1YVUC0R9hgLsTrdv4p0Woh9Y8f/O3CeZK//8bPt2dM1UhLkZrXECIgTK7UMNYITH/LQedKkNz5+NNz52j+6ZXfbM+YqhHjL0FSkyIiIFp6EFs5pbHGn3OQ5ECM/ffb97dnTNVK+xFWafEGiYBItQckWpQSC625zty48dG541Ozvbx48WJ79FStelNEBIST0wZL6REkvV3Ndeb5N5P/63c+uHAOfuMH1zZHRw+3R061qidFRECkxqyASNuT5dzPP390ciM//sk7Jz+I/vjuPzavXs3/a7KH1JLB8F5EBCT8KCi29d2G1Jmp9dWaIiIgUmN8ZlFaqFrPmxoraevHpYUvAqI1VqKN79NzrMkzNV5SsVpawCIg2p7FBXKQpMljffKZWkbSzoBzcyoCgqQ6BDPpABQ3Skr0PPlMLSNt4edSXgUEADRIgqXIik2HdkH0g4qdFQTImHur9WuvX91897W3zzk9ZtSWSv/Ta6XOvU4Iqt1mVEBQ61/W4bW3FyZKi9SRZqJykd0q6VqpLQtSGpMcWFlAUOsgW2geIUvyjbZ3ilgWKvdsEX2TztegLgKCmGyp0ZyXrj+4wZZ+jrB3iliAt0JZW4cUAWndZpasP5aG44c/ui5+HtsrRbg3qf3Y1vQIqqlDsoBotMVmYmicxODfubi1/rj5p9tma7L00dPcozXyPVLEssXTnxrV1CEqICVyc1uIdWBKf5Mb/Pr3rm7POC/L6vJ2kGfsaxqRHkiqQ7R2VEC0VUI6eG0fN27cEq+RWgNkyW0Fx/e9RIpYrlGbHkgCBB8fH2+POJMISI7cnhtO1ZMgNXAwUWFrYHC4hxanGpkiXE9qL3WLtG3ZDIhGrveTSWuC1NQd2t7qoZEpMio9gqRC9csv/7P99kwiIFIRYy08a9SaIJb0YGJ6BtCqUSkitZO6BbygLkCkCdAeg3rUAog1PXoGr0YjUmR0eiBpjh88+Nf22zOJgKQn4hEvvloAsTz2te77rfJOEUt7vQvAHZARg94CSGnwRtYcmjxThGOk82PTVq+khfbZ0YPtt2cSAZEmYUyC1BWpQCp9H3uprSWVZdVbFpklIT1eM3QBIsXPiJXpDYjHymqVR4oslR5IAuTevYsAi4BoFHuvzlpASpNAv1NZBr3W0nVQb4pYIPPa6tkR0rbNgGhPCv7vQepqEA3cYG3wLANfYw2QnhRZMj2QNJbmLQZpq8Fj/wuqTZASIFrfGHxp22y1BghqTRELXBzjJWk8qgDJddgr5rwTJAev51aTA6Q1RaTjYnumB5IAMT/mBuVWAxdgQtIb5c/pZ5pqASkNfmmFWSbP4hwgqDZFLP0q3VutXABhoi03y8XiV7drAUI/cqJf0sDUugSIZcLjFLGMsbekV+3VgCBu1nIDsa11Si0gtCt9H0w/S2JSpHNrXAIEWcaMFLH0xzs9kHQd89/FpGJiaiCx1ii1gFgG0zJ5AXrNUruxrdeQzo3NtSyJFpLGS9o4PhauYwIE0ag1ni0DiGoBQaU+MOjWBNME4FLbwdb7s8BW8oj00JJYkhmQIBpnknI3z/cWtQBS2mYwfeuRFyCWFCnZOz2Q9JJMG7NqQGIxWQxWejFsUQsgDJhlZXJM6+B6AYIsfdXcC7omCRBtUXcBgrQVbZmcFkBQaQKDGeCWiPYEpCdFGNsRkp5gtHHqBgRJq4RBLqkVEGSthzDH0h/rgHsCglpSZFR6sHCl6y0OiGUQewDRbtRi+gs0mksTWgtIS4pYFliLpO0Fa3IBhAFLL8hAl9QDCCqt9FGuBQTVpMio9EA1BSpyAUSbqFId0gsI6tnjW90CSE0/tbj3UO31XABB0gop/TzAAxC0NCQtgCBLioxMD21bztVmboC0bDNegCBu3jIBHm4FxALyyPSorT+QGyAt24wnIIhrLZEmrYCgHMQj0wNJ1ywB6QYIkm4+t814AxI0GpQeQHJJN+q9B9IWcOlpyRUQaZvBWoqMAiSI6zLo9Mvy+Gp1DyCIfsX9ob2RcCBpbiyJ5QqIRqmWIk+fPjPbU0xQj/dN9FmcF0O94woIklYpr3b3cWAPRVpxapkTd0C0vb+0102NkZYelu0FuQOCZorsjrT0sNY8QwDRUiT3RDPlr970QEMAoWNSiuCZIsuJJyVpDmq2+yGAoJki60p7oqxJDzQMEDRTZB1pWwuufd8yFBAtRWbBOlZaYWp575FqKCBIS5G51YwRCSGNd+3WEjQckFzBOvr18mVTbmtpSQ80HBA0t5plpD21lH52kdMigCAtRSYkPtLqjtatJWgxQHJbDTc3IWmX9kiLe7fxxQBBbDU5SKbqpRWluLXuiLUoIEirR/CEpE45OHrqjliLA4ImJP3KwdFbd8RaBRCkbTV4QpJXCY7euiPWaoDkilY8C1dZuYIUe8KBVgMElSBhH52QnIlFI41TsDccaFVAUAmS+Z7kVNpLsOARcKDVAUElSPBl/ckiE88ikcYEe9ccqXYCkKDSKrlsW05pSxkNB9opQJD239YEs5oOPU2Y9NJiAY4ltHOAoNwb1+BDTZPSAsHc+1LaSUDQ6d6bhwQzoIcASunxNXj0lpJqZwEJsqwovK+gAEauCA1eot6QtPOAoNNBLKcJ3petp1SAxvb4S7dW7QUgiEknJWpAAaxdgoX+lIrP2By7RmrE2htAgiwVfuo1YeG61m0y9tpgBO0dIEEMoDVNYrPfx8B4QkNbAQhLXSGZ83dJewtIUEuipA7QMLHUBkwSpu3Y4fNgju+BITbt7aL2HpCgAEpLqqzlkGS7rIMBJAhQqPp7U2WkSZ1dKp5zOjhAYgVYSJW1kwUo6M++6aABSRWny2hgAILtY1+SQtOlAiQVwARomNAATilxwvccz3mcfwgwSLrUgEyVNQGZymoCMpXVBGQqo83m/+BbYUkJcn+bAAAAAElFTkSuQmCC"},30643:function(e,t){t.Z="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEwAAABhCAYAAAB8pUfDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAATvSURBVHhe7Zixjt02EEX9/38Qf0BSuDTgNu6zrYutXNlFSiP9BgdYApTeUHOHHOpJsi5wAcNPokaHl0NqP7zdCukGFtQNLKgbWFALYH98/LTpX7/+e7/yuPrzry9m7cU/fv77fmWfbmBB3cCCuoEFFQI2+rA9tCuw2Q/bQ1bdtW9gK1l11x5tKyFgr6/f3688poBh1V07Fdjnz1/NhxQD9Oi26q49qhCwK3hUC2B/f/3HfMhVTAJHdQMLagGMpm496Cp+efn2/qb9WgDj2GA96CpOB4aUneaszjhH/lbAMvQATGn8RJt+dyRbddbOaPjoAZjy8IxekCnlhJ9V8wMw5C1Lfj/Sn3qAYdVZO+s72ASmLEuSeAQp6cpajsgEpixLPqOOICVdmS3EBIaU3fLZKVPShTPrbAJTZu7ZvUytMVNNYIBQUpYZ94iekS7UBIaUGcRct6fZ8QDB5oTpp5gJric5O11oExhSUjbbQIksfYACdkaPdYHxYOslZjsKaS+5wNCeKWOCjiwJmLoBjJgedMRErSUBQ/SDWdCyPlv2kAwMzehnZ4KFQsBQFjTSejZYKAwMRZYm15ZzUjE74Kjodxjoe/a+LmCRTSCrmTMG6d56Lr8xGTPOX0VdwFBkaY5A8yC1zD3cm52+bmBoJjSu7QG1dgGXpSFgSIVG4aroS9YYI86CNgwMKdDUguk/1v0ZjkxaSynA0BY0tdAZyVqb1jCiNGCoBU1J1x6witW0W0oFhixoiqINnus5QmBSY12z5d5DczowVENTlqPatxhr64zFzqrCU9vEWlOAoQJNib+SrkjvsVJuuSdl04AhBZaSrp5GrfTEnnGnAlNED7JepnavvKT1LMunA/OWY29zRsrXQnT8qcC8TyF+t16i9qi8lClto9Y0YPQmrxivz2T8GSj7GVOAlUbuAfMafgYwL8XRxp8OrIYwmrDocrHkAXtqwtaJ8YrxgPVs+2tlpzgNmFWYV4w3+72n8VoeMH6PKAVYqygvIXvskkyaNW7x7sC2ZlBJCFCte4ujS6aWt+QxkxbREDAv7tgriMZu3VebF++RNxk9PbIbmDJ72Iu8Mg5JjSbBW4q4ZyK6gCm9p1g5GnhJwEBTxqI2dbwedQFTCirmWi8dkQngRUltnQ7u5/+U5V3sJb+lMLAIrGIlGZGXzfAuCet9KaUHqUsp0z3QZGBqk29ZWQJA4yWs+2c5Ck0CFukxLSspQ0eHJgHLWipKL0NAO2pPc4FlF66efQq0kbRxb6R+BdomsIyluDZFKUuzqICLpLyAKoq8hwdtE5g6O1wXcc8JG/HibB6c4jEQeUH+7Y2dBa0JTH0ARZ5FGdCawABhDVTbi+8RNQqtCcwaYO3epfVsRaCt39EExkXWzbXPmK5aCjQrECYwZTn2frweSVvQWqvHBGYNUPts6WJyWwAsaK1r0QMwBl8PsPYZdsZy/Cg18++WamhbsNADMHU5Hs3UDRTSb9XMmW1LQPNgoQdge/+JZU8DZVQPwKwHXcXpwKwGeCWzdEe1AMaA1oOu4q3Gr2oBTGn4Z7bX+BUtgNXb8BWdcX5cALvyDlk8qjAwZunItmquPaoQsIxdZrasumuPHi0WwLwZUk7Cz5ZVd+1UYNYDap8BmDfpN7CVZq+SELCMT4vZuoEFdQML6gYW1K7Abvm6gQV1Awvp7e1/cS4yprcXyWYAAAAASUVORK5CYII="},27922:function(e,t,n){t.Z=n.p+"assets/images/transcripts-41cae82d7eb56e98382842cd1903341a.png"}}]);