"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[2593],{3905:function(e,t,a){a.d(t,{Zo:function(){return p},kt:function(){return u}});var n=a(67294);function r(e,t,a){return t in e?Object.defineProperty(e,t,{value:a,enumerable:!0,configurable:!0,writable:!0}):e[t]=a,e}function i(e,t){var a=Object.keys(e);if(Object.getOwnPropertySymbols){var n=Object.getOwnPropertySymbols(e);t&&(n=n.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),a.push.apply(a,n)}return a}function s(e){for(var t=1;t<arguments.length;t++){var a=null!=arguments[t]?arguments[t]:{};t%2?i(Object(a),!0).forEach((function(t){r(e,t,a[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(a)):i(Object(a)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(a,t))}))}return e}function o(e,t){if(null==e)return{};var a,n,r=function(e,t){if(null==e)return{};var a,n,r={},i=Object.keys(e);for(n=0;n<i.length;n++)a=i[n],t.indexOf(a)>=0||(r[a]=e[a]);return r}(e,t);if(Object.getOwnPropertySymbols){var i=Object.getOwnPropertySymbols(e);for(n=0;n<i.length;n++)a=i[n],t.indexOf(a)>=0||Object.prototype.propertyIsEnumerable.call(e,a)&&(r[a]=e[a])}return r}var l=n.createContext({}),c=function(e){var t=n.useContext(l),a=t;return e&&(a="function"==typeof e?e(t):s(s({},t),e)),a},p=function(e){var t=c(e.components);return n.createElement(l.Provider,{value:t},e.children)},d={inlineCode:"code",wrapper:function(e){var t=e.children;return n.createElement(n.Fragment,{},t)}},m=n.forwardRef((function(e,t){var a=e.components,r=e.mdxType,i=e.originalType,l=e.parentName,p=o(e,["components","mdxType","originalType","parentName"]),m=c(a),u=r,f=m["".concat(l,".").concat(u)]||m[u]||d[u]||i;return a?n.createElement(f,s(s({ref:t},p),{},{components:a})):n.createElement(f,s({ref:t},p))}));function u(e,t){var a=arguments,r=t&&t.mdxType;if("string"==typeof e||r){var i=a.length,s=new Array(i);s[0]=m;var o={};for(var l in t)hasOwnProperty.call(t,l)&&(o[l]=t[l]);o.originalType=e,o.mdxType="string"==typeof e?e:r,s[1]=o;for(var c=2;c<i;c++)s[c]=a[c];return n.createElement.apply(null,s)}return n.createElement.apply(null,a)}m.displayName="MDXCreateElement"},51369:function(e,t,a){a.r(t),a.d(t,{frontMatter:function(){return o},contentTitle:function(){return l},metadata:function(){return c},toc:function(){return p},default:function(){return m}});var n=a(87462),r=a(63366),i=(a(67294),a(3905)),s=["components"],o={sidebar_position:5,sidebar_label:"Variant discovery preprocessing"},l="Variant discovery preprocessing",c={unversionedId:"results/workflow-details/fastq-analysis/variant-discovery-preprocessing",id:"results/workflow-details/fastq-analysis/variant-discovery-preprocessing",isDocsHomePage:!1,title:"Variant discovery preprocessing",description:'After the "Alignment" task of the stage of the same name',source:"@site/docs/results/workflow-details/fastq-analysis/variant-discovery-preprocessing.mdx",sourceDirName:"results/workflow-details/fastq-analysis",slug:"/results/workflow-details/fastq-analysis/variant-discovery-preprocessing",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing",tags:[],version:"current",sidebarPosition:5,frontMatter:{sidebar_position:5,sidebar_label:"Variant discovery preprocessing"},sidebar:"tutorialSidebar",previous:{title:"Alignment",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment"},next:{title:"Calculate coverage",permalink:"/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage"}},p=[],d={toc:p};function m(e){var t=e.components,o=(0,r.Z)(e,s);return(0,i.kt)("wrapper",(0,n.Z)({},d,o,{components:t,mdxType:"MDXLayout"}),(0,i.kt)("h1",{id:"variant-discovery-preprocessing"},"Variant discovery preprocessing"),(0,i.kt)("p",null,'After the "Alignment" task of ',(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/alignment"},"the stage of the same name"),"\nhas been successfully completed, it is recommended to further preprocess the alignment results for the most\naccurate variants discovery. This increases sensitivity to rare variants, contributes to a more accurate copy\nnumber count, which depends on the number of reads, and also speeds up sample processing. Variant discovery\npreprocessing is launched if at least one of the following stages is included in the sample analysis workflow:\n",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#somatic-snvs-discovery"},"somatic SNVs/Indels discovery"),",\n",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#germline-snvs-discovery"},"germline SNVs/Indels discovery"),",\n",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#sv-discovery"},"structural variations discovery"),",\n",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#cnv-discovery"},"copy number variations discovery"),",\n",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#enable-phenotype-prediction"},"phenotypes prediction"),",\nor ",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#polygenic-risk-scores-calculation"},"polygenic risk scores calculation"),".\nIf any of the tasks listed below fails, the sample analysis is stopped."),(0,i.kt)("p",{align:"center"},(0,i.kt)("img",{src:a(48278).Z,width:"850"})),(0,i.kt)("p",null,'The "Variant discovery preprocessing" stage of sample analysis may include the following tasks:'),(0,i.kt)("ol",null,(0,i.kt)("li",{parentName:"ol"},(0,i.kt)("p",{parentName:"li"},(0,i.kt)("strong",{parentName:"p"},"Mark Duplicates"),"\nif ",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#disable-mark-duplicates"},"the corresponding parameter")," is enabled\nfor the sample. During sequencing, the amount of original sequences of fragmented DNA increases in the polymerase\nchain reaction (PCR). For further analysis, it is very important to take into account duplicate reads of the same\nfragment. If you want to keep all reads for further analysis (for example, if you are working with amplicons),\nyou can ",(0,i.kt)("a",{parentName:"p",href:"/settings/settings/variant-discovery-settings#disable-mark-duplicates"},"disable")," the marking of\nduplicates.",(0,i.kt)("br",null),"\nMarking duplicates is performed using\nthe ",(0,i.kt)("a",{parentName:"p",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-"},"GATK MarkDuplicates")," tool,\nwhich locates and tags duplicate reads in the alignment file, where duplicate reads are defined as originating from a\nsingle fragment of DNA. Further analysis is performed only on the original reads. The resulting file in BAM format\nis sorted by coordinates and an index file is created for it. In addition, a file\nwith ",(0,i.kt)("a",{parentName:"p",href:"https://broadinstitute.github.io/picard/picard-metric-definitions.html#DuplicationMetrics"},"metrics"),' is created,\nwhich indicates the number of fragments or read pairs that were marked as duplicates. This file can be downloaded\nin the "',(0,i.kt)("em",{parentName:"p"},"Result files"),'" section in the "',(0,i.kt)("em",{parentName:"p"},"Mark Duplicates"),'" task details ("',(0,i.kt)("em",{parentName:"p"},"Download duplication metrics TXT"),'").')),(0,i.kt)("li",{parentName:"ol"},(0,i.kt)("p",{parentName:"li"},(0,i.kt)("strong",{parentName:"p"},"Base Recalibrate"),".\nThe sequencers assign a quality score to each nucleotide, which is the log likelihood of a nucleotide occurrence at a\ngiven position compared to other nucleotides. However, many sequencers tend to underestimate sequencing quality\nand make other errors. Base recalibration corrects for the systematic bias that affects the sequencer's assignment\nof quality score to nucleotides, which reduces error and improves the quality of further variant discovery."))),(0,i.kt)("p",null,"Base Quality Score Recalibration (BQSR) is performed in three steps:"),(0,i.kt)("ul",null,(0,i.kt)("li",{parentName:"ul"},(0,i.kt)("a",{parentName:"li",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360036898312-BaseRecalibrator"},"GATK BaseRecalibrator"),'\ncalculates an empirical probability of error and finds patterns in how error varies with nucleotide calling\npatterns across all nucleotides. These observations are written to a recalibration table based on such covariates\nas read group, reported quality score, machine cycle, and nucleotide context. BaseRecalibrator does a by-locus\ntraversal operating only within certain intervals at known polymorphic sites used to exclude regions around known\npolymorphisms from analysis.\nAll reference mismatches are considered errors and indicative of poor base quality. An empirical probability of\nerror (p) is then calculated given the particular covariates seen at this site, where p = num mismatches / num\nobservations. The output file is a table containing the values of the individual covariates, number of observations\nfor this combination of covariates, number of reference mismatches, and the raw empirical quality score calculated\nby phred-scaling the mismatch rate. The recalibration table can be downloaded in the "',(0,i.kt)("em",{parentName:"li"},"Result files"),'" section in\nthe "',(0,i.kt)("em",{parentName:"li"},"Base Recalibrate"),'" task details ("',(0,i.kt)("em",{parentName:"li"},"Download recalibration table TXT"),'").'),(0,i.kt)("li",{parentName:"ul"},(0,i.kt)("a",{parentName:"li",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037055712-ApplyBQSR"},"GATK ApplyBQSR")," applies base quality\nscore recalibration within certain intervals, i.e. recalibrates the base qualities of the input reads based on the\nrecalibration table produced in the first step, and outputs the recalibrated BAM files."),(0,i.kt)("li",{parentName:"ul"},(0,i.kt)("a",{parentName:"li",href:"https://gatk.broadinstitute.org/hc/en-us/articles/360037053552-MergeSamFiles-Picard-"},"GATK MergeSamFiles"),"\nmerges BAM files recalibrated within certain intervals into a single file. The resulting BAM file is sorted by\ncoordinates and an index file is created for it.")),(0,i.kt)("p",null,'The final sample alignment file after preprocessing can be downloaded in the "',(0,i.kt)("em",{parentName:"p"},"Result files"),'" section in\nthe "',(0,i.kt)("em",{parentName:"p"},"Base Recalibrate"),'" task details ("',(0,i.kt)("em",{parentName:"p"},"Download BAM"),'"). You can also open this file in the IGV browser by\nclicking on the "',(0,i.kt)("em",{parentName:"p"},"Open in IGV Browser"),'" link. The index file for the final BAM file can be downloaded there\nas well ("',(0,i.kt)("em",{parentName:"p"},"Download BAI"),'").'),(0,i.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,i.kt)("div",{parentName:"div",className:"admonition-heading"},(0,i.kt)("h5",{parentName:"div"},(0,i.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,i.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,i.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,i.kt)("div",{parentName:"div",className:"admonition-content"},(0,i.kt)("p",{parentName:"div"},"If you want to add the alignment track obtained after preprocessing your sample in Genomenal to\nyour ",(0,i.kt)("a",{parentName:"p",href:"https://igv.org/doc/desktop/"},"desktop IGV"),", you can do so via a link. To do this, follow these steps:"),(0,i.kt)("ol",{parentName:"div"},(0,i.kt)("li",{parentName:"ol"},'Right-click the alignment file link "',(0,i.kt)("em",{parentName:"li"},"Download BAM"),'" and select the "',(0,i.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,i.kt)("li",{parentName:"ol"},"Upload the track via URL to your desktop IGV, as\ndescribed ",(0,i.kt)("a",{parentName:"li",href:"https://igv.org/doc/desktop/#UserGuide/loading_data/"},"here"),"."),(0,i.kt)("li",{parentName:"ol"},'Right-click the alignment index file link "',(0,i.kt)("em",{parentName:"li"},"Download BAI"),'" and select the "',(0,i.kt)("em",{parentName:"li"},"Copy link address"),'" option.'),(0,i.kt)("li",{parentName:"ol"},"Add the index via URL to the corresponding field in IGV."),(0,i.kt)("li",{parentName:"ol"},'Click "',(0,i.kt)("em",{parentName:"li"},"OK"),'". Done! The preprocessed sample alignment track is added to IGV.')))),(0,i.kt)("p",null,'After the "Variant discovery preprocessing" stage is successfully completed, the analysis continues with\n',(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/calculate-coverage"},"coverage calculation")," and the following stages,\nif included in the\nworkflow: ",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/somatic-snvs-indels#somatic-variants-discovery"},'"Somatic SNVs/Indels discovery"'),"\n(for a tumor\nsample), ",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/germline-snvs-indels#germline-variants-discovery"},'"Germline SNVs/Indels discovery"'),"\n(for a single tumor sample or a non-tumor\nsample), ",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/structural-variations"},'"Structural variations discovery"'),",\n",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/cnv-discovery"},'"Copy number variations discovery"')," and/or\n",(0,i.kt)("a",{parentName:"p",href:"/results/workflow-details/fastq-analysis/phenotypes"},'"Phenotype prediction"')," (for a non-tumor sample)."))}m.isMDXComponent=!0},48278:function(e,t,a){t.Z=a.p+"assets/images/variant_discovery_preprocessing-593beea548bcd967506f6851888376a6.png"}}]);