"use strict";(self.webpackChunkngsw_docs_cnv_en=self.webpackChunkngsw_docs_cnv_en||[]).push([[6608],{3905:function(t,e,n){n.d(e,{Zo:function(){return c},kt:function(){return k}});var a=n(7294);function i(t,e,n){return e in t?Object.defineProperty(t,e,{value:n,enumerable:!0,configurable:!0,writable:!0}):t[e]=n,t}function r(t,e){var n=Object.keys(t);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(t);e&&(a=a.filter((function(e){return Object.getOwnPropertyDescriptor(t,e).enumerable}))),n.push.apply(n,a)}return n}function l(t){for(var e=1;e<arguments.length;e++){var n=null!=arguments[e]?arguments[e]:{};e%2?r(Object(n),!0).forEach((function(e){i(t,e,n[e])})):Object.getOwnPropertyDescriptors?Object.defineProperties(t,Object.getOwnPropertyDescriptors(n)):r(Object(n)).forEach((function(e){Object.defineProperty(t,e,Object.getOwnPropertyDescriptor(n,e))}))}return t}function o(t,e){if(null==t)return{};var n,a,i=function(t,e){if(null==t)return{};var n,a,i={},r=Object.keys(t);for(a=0;a<r.length;a++)n=r[a],e.indexOf(n)>=0||(i[n]=t[n]);return i}(t,e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(t);for(a=0;a<r.length;a++)n=r[a],e.indexOf(n)>=0||Object.prototype.propertyIsEnumerable.call(t,n)&&(i[n]=t[n])}return i}var s=a.createContext({}),u=function(t){var e=a.useContext(s),n=e;return t&&(n="function"==typeof t?t(e):l(l({},e),t)),n},c=function(t){var e=u(t.components);return a.createElement(s.Provider,{value:e},t.children)},d={inlineCode:"code",wrapper:function(t){var e=t.children;return a.createElement(a.Fragment,{},e)}},p=a.forwardRef((function(t,e){var n=t.components,i=t.mdxType,r=t.originalType,s=t.parentName,c=o(t,["components","mdxType","originalType","parentName"]),p=u(n),k=i,m=p["".concat(s,".").concat(k)]||p[k]||d[k]||r;return n?a.createElement(m,l(l({ref:e},c),{},{components:n})):a.createElement(m,l({ref:e},c))}));function k(t,e){var n=arguments,i=e&&e.mdxType;if("string"==typeof t||i){var r=n.length,l=new Array(r);l[0]=p;var o={};for(var s in e)hasOwnProperty.call(e,s)&&(o[s]=e[s]);o.originalType=t,o.mdxType="string"==typeof t?t:i,l[1]=o;for(var u=2;u<r;u++)l[u]=n[u];return a.createElement.apply(null,l)}return a.createElement.apply(null,n)}p.displayName="MDXCreateElement"},4308:function(t,e,n){n.r(e),n.d(e,{frontMatter:function(){return o},contentTitle:function(){return s},metadata:function(){return u},toc:function(){return c},default:function(){return p}});var a=n(7462),i=n(3366),r=(n(7294),n(3905)),l=["components"],o={sidebar_position:1,sidebar_label:"Variant Viewer"},s="Variant Viewer",u={unversionedId:"results/variant-viewer/variant-viewer",id:"results/variant-viewer/variant-viewer",isDocsHomePage:!1,title:"Variant Viewer",description:"Overview",source:"@site/docs/results/variant-viewer/variant-viewer.mdx",sourceDirName:"results/variant-viewer",slug:"/results/variant-viewer/variant-viewer",permalink:"/ngsw-docs-en/results/variant-viewer/variant-viewer",tags:[],version:"current",sidebarPosition:1,frontMatter:{sidebar_position:1,sidebar_label:"Variant Viewer"},sidebar:"tutorialSidebar",previous:{title:"Coverage Analysis",permalink:"/ngsw-docs-en/results/coverage-analysis"},next:{title:"Variant detailed information panel",permalink:"/ngsw-docs-en/results/variant-viewer/snv_details_panel"}},c=[{value:"Overview",id:"overview",children:[]},{value:"The main page",id:"the-main-page",children:[]},{value:"Description of table columns",id:"description-of-table-columns",children:[]}],d={toc:c};function p(t){var e=t.components,o=(0,i.Z)(t,l);return(0,r.kt)("wrapper",(0,a.Z)({},d,o,{components:e,mdxType:"MDXLayout"}),(0,r.kt)("h1",{id:"variant-viewer"},"Variant Viewer"),(0,r.kt)("h3",{id:"overview"},"Overview"),(0,r.kt)("p",null,'Variant Viewer is a built-in service for viewing and analyzing variants. Located on the main page of the sample ("',(0,r.kt)("strong",{parentName:"p"},"Main"),'" tab).'),(0,r.kt)("p",null,"On the ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"tumor"))," sample page, you will find two Variant Viewers: one with ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"somatic"))," and one with ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"germline"))," mutations discovered in this sample:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(3941).Z,width:"600"})),(0,r.kt)("p",null,"The ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"normal"))," sample page presents a Variant Viewer with ",(0,r.kt)("strong",{parentName:"p"},(0,r.kt)("em",{parentName:"strong"},"germline"))," mutations only:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(8485).Z,width:"600"})),(0,r.kt)("h3",{id:"the-main-page"},"The main page"),(0,r.kt)("p",null,"After clicking on the ",(0,r.kt)("strong",{parentName:"p"},"Variant Viewer")," button, you will see the main page of the service:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(1819).Z,width:"850"})),(0,r.kt)("p",null,"The main object of the Variant Viewer is a table with variants - each variant on a new line:"),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(5744).Z,width:"950"})),(0,r.kt)("h3",{id:"description-of-table-columns"},"Description of table columns"),(0,r.kt)("p",null,"By default, the table has the following columns (the set and order of the columns can be changed):"),(0,r.kt)("ol",null,(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"LOCUS")," is the common name of the gene in which the variant is located."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"ORIGIN")," is a mutation type (somatic or germline).")),(0,r.kt)("div",{className:"admonition admonition-info alert alert--info"},(0,r.kt)("div",{parentName:"div",className:"admonition-heading"},(0,r.kt)("h5",{parentName:"div"},(0,r.kt)("span",{parentName:"h5",className:"admonition-icon"},(0,r.kt)("svg",{parentName:"span",xmlns:"http://www.w3.org/2000/svg",width:"14",height:"16",viewBox:"0 0 14 16"},(0,r.kt)("path",{parentName:"svg",fillRule:"evenodd",d:"M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"}))),"info")),(0,r.kt)("div",{parentName:"div",className:"admonition-content"},(0,r.kt)("p",{parentName:"div"},'"Origin" column is hidden in ',(0,r.kt)("a",{parentName:"p",href:"/results/variant-viewer/variant-viewer"},"Variant Viewer with germline mutations"),'\nand in Variant Viewer with somatic mutations if "Origin Somatic" filter is on.'))),(0,r.kt)("ol",{start:3},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"POSITION")," is the coordinates of the variant in the genome (chromosome + start position)."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"HGVSP")," is an amino acid substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"the HGVS notation")," (\u201cp.\u201d prefix (protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution).\nWhen hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for and HGVSp with amino acids in the three letter code:")),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(817).Z,width:"400"})),(0,r.kt)("ol",{start:4},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"ALTERATION")," is a nucleotide substitution (reference allele > alternative allele(s))."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"HGVSC")," is a nucleotide substitution using ",(0,r.kt)("a",{parentName:"li",href:"https://varnomen.hgvs.org/"},"the HGVS notation")," (genomic position of the substituted nucleotide + reference allele > alternative allele). Starts with a prefix: \u201cc.\u201d (coding; for a substitution in the coding sequence), \u201cn.\u201d (non-coding; for a substitution in the non-coding sequence).\nWhen hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for:")),(0,r.kt)("p",{align:"center"},(0,r.kt)("img",{src:n(8237).Z,width:"500"})),(0,r.kt)("ol",{start:6},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"EXON")," is the exon number in the transcript which is affected by the variant."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"DEPTH (ALT/REF)")," is a sequencing depth; the total number of reads of the sequence overlapping the variant position for this sample. ALT is the alternative allele reads count, REF - the reference allele reads count."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"GT")," is a genotype; allelic values (0 - reference allele, 1 - first alternative allele, 2 - second alternative allele etc.), divided by \u201d/\u201d (for unphased genotype) or \u201c|\u201d (for phased genotype)."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"CONSEQUENCE")," is the effect of the variant on genes.")),(0,r.kt)("details",null,(0,r.kt)("summary",null,"Click to see a table with the consequences"),(0,r.kt)("h4",{id:"consequences-of-variants-in-the-annotation"},"Consequences of variants in the annotation"),(0,r.kt)("table",null,(0,r.kt)("tr",null,(0,r.kt)("td",null,"Consequence"),(0,r.kt)("td",null,"Description"),(0,r.kt)("td",null,"Impact (see description below)")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Coding sequence"),(0,r.kt)("td",null,"The variant hits a coding sequence"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Transcript ablation"),(0,r.kt)("td",null,"Deletion of a transcript"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Splice acceptor"),(0,r.kt)("td",null,"The variant hits a splice acceptor site"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Splice donor"),(0,r.kt)("td",null,"The variant hits a splice donor site"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Stop gained"),(0,r.kt)("td",null,"Variant causes a STOP codon"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Frameshift"),(0,r.kt)("td",null,"Insertion or deletion causes a frame shift"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Stop lost"),(0,r.kt)("td",null,"Variant causes stop codon to be mutated into a non-stop codon"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Start lost"),(0,r.kt)("td",null,"Variant causes start codon to be mutated into a non-start codon"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Transcript amplification"),(0,r.kt)("td",null,"A feature amplification of a region containing a transcript"),(0,r.kt)("td",null)),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Inframe insertion"),(0,r.kt)("td",null,"One or many codons are inserted"),(0,r.kt)("td",null,"MODERATE")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Inframe deletion"),(0,r.kt)("td",null,"One or many codons are deleted"),(0,r.kt)("td",null,"MODERATE")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Missense"),(0,r.kt)("td",null,"Variant causes a codon that produces a different amino acid"),(0,r.kt)("td",null,"MODERATE")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Protein altering"),(0,r.kt)("td",null,"A variant which is predicted to change the protein encoded in the coding sequence"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Splice region"),(0,r.kt)("td",null,"A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron"),(0,r.kt)("td",null,"LOW")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Incomplete terminal codon"),(0,r.kt)("td",null,"A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed"),(0,r.kt)("td",null)),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Start retained"),(0,r.kt)("td",null,"Variant causes start codon to be mutated into another start codon"),(0,r.kt)("td",null,"LOW")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Stop retained"),(0,r.kt)("td",null,"Variant causes stop codon to be mutated into another stop codon"),(0,r.kt)("td",null,"LOW")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Synonymous"),(0,r.kt)("td",null,"Variant causes a codon that produces the same amino acid"),(0,r.kt)("td",null,"LOW")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Mature miRNA"),(0,r.kt)("td",null,"A transcript variant located with the sequence of the mature miRNA"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"UTR5"),(0,r.kt)("td",null,"A variant hits 5'UTR region"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"UTR3"),(0,r.kt)("td",null,"A variant hits 3'UTR region"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Non coding transcript exon"),(0,r.kt)("td",null,"A sequence variant that changes non-coding exon sequence in a non-coding transcript"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Upstream"),(0,r.kt)("td",null,"Upstream of a gene"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Downstream"),(0,r.kt)("td",null,"Downstream of a gene"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"TF binding site"),(0,r.kt)("td",null,"Transcription factor binding site (TFBS)"),(0,r.kt)("td",null)),(0,r.kt)("tr",null,(0,r.kt)("td",null,"TFBS ablation"),(0,r.kt)("td",null,"Deletion of a TFBS"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"TFBS amplification"),(0,r.kt)("td",null,"Amplification of a TFBS"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Regulatory region ablation"),(0,r.kt)("td",null,"Deletion of a regulatory region"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Regulatory region amplification"),(0,r.kt)("td",null,"Amplification of regulatory region"),(0,r.kt)("td",null,"HIGH")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Feature elongation"),(0,r.kt)("td",null,"A sequence variant that causes the extension of a genomic feature"),(0,r.kt)("td",null)),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Regulatory region variant"),(0,r.kt)("td",null,"The variant hits a regulatory feature"),(0,r.kt)("td",null,"MODIFIER")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Feature truncation"),(0,r.kt)("td",null,"A sequence variant that causes the reduction of a genomic feature"),(0,r.kt)("td",null)),(0,r.kt)("tr",null,(0,r.kt)("td",null,"Intergenic variant"),(0,r.kt)("td",null,"The variant is in an intergenic region"),(0,r.kt)("td",null,"MODIFIER")))),(0,r.kt)("ol",{start:10},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("b",null,"IMPACT")," is the predicted effect of the variant on the protein.")),(0,r.kt)("details",null,(0,r.kt)("summary",null,"Click to see a table with impacts"),(0,r.kt)("h4",{id:"impacts-of-variants-in-the-annotation"},"Impacts of variants in the annotation"),(0,r.kt)("table",null,(0,r.kt)("tr",null,(0,r.kt)("td",null,"Impact"),(0,r.kt)("td",null,"Description")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"HIGH"),(0,r.kt)("td",null,"The variant has high (disruptive) impact in the protein, probably causing protein truncation, loss of function or triggering nonsense mediated decay")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"MODERATE"),(0,r.kt)("td",null,"The variant might change protein effectiveness")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"LOW"),(0,r.kt)("td",null,"The variant is mostly harmless or unlikely to change protein behavior")),(0,r.kt)("tr",null,(0,r.kt)("td",null,"MODIFIER"),(0,r.kt)("td",null,"Usually variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact")))),(0,r.kt)("ol",{start:11},(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"EXTERNAL LINKS")," are links to pages with variant information in ",(0,r.kt)("a",{parentName:"li",href:"https://www.ncbi.nlm.nih.gov/snp/"},"dbSNP"),", ",(0,r.kt)("a",{parentName:"li",href:"https://preview.ncbi.nlm.nih.gov/clinvar/"},"ClinVar")," databases."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"GNOMAD AF")," is the allele frequency according to the ",(0,r.kt)("a",{parentName:"li",href:"https://gnomad.broadinstitute.org/"},"gnomAD")," database."),(0,r.kt)("li",{parentName:"ol"},(0,r.kt)("strong",{parentName:"li"},"ACMG")," is the interpretation of the variant by ACMG recommendations. If you have not interpreted the variant, then the column value will be \u201cN/A\u201d (No interpretation).")))}p.isMDXComponent=!0},3941:function(t,e,n){e.Z=n.p+"assets/images/germline_and_somatic_vv-b30240dc3c0462f898a7338d4858557d.png"},8485:function(t,e,n){e.Z=n.p+"assets/images/germline_only-406e9c53884ef81acd63c615b2ff0bad.png"},8237:function(t,e,n){e.Z=n.p+"assets/images/hgvsc_tooltip-702b36ee2b497dad9d00bb4867fdb330.png"},817:function(t,e,n){e.Z=n.p+"assets/images/hgvsp_tooltip-5b0fd9e70e36aeec94b9469471d5ab72.png"},1819:function(t,e,n){e.Z=n.p+"assets/images/main_table-ccd33bdbc8ff217d44da7f5b431ad93c.png"},5744:function(t,e,n){e.Z=n.p+"assets/images/snv_row-7850415e0e1fa0691dffb88554b75079.png"}}]);