<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">ClinVar Phenotypes Report | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/variants-by-phenotypes"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="ClinVar Phenotypes Report | Genomenal"><meta data-react-helmet="true" name="description" content="The report includes conditions (ClinVar phenotypes) that have the risk of developing in the patient"><meta data-react-helmet="true" property="og:description" content="The report includes conditions (ClinVar phenotypes) that have the risk of developing in the patient"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/variants-by-phenotypes"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/variants-by-phenotypes" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/variants-by-phenotypes" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.72caaed9.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.d733ea5e.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">üåú</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">üåû</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/overview">Getting to Know the Service</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotypes Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg73-report">ACMG 73 Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Workflow Details</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>ClinVar Phenotypes Report</h1></header><p>The report includes <strong><em>conditions</em></strong> (ClinVar phenotypes) that have the risk of developing in the patient
due to pathogenic or likely pathogenic variants discovered in patient sample and associated with these
conditions, and <strong><em>recommendations</em></strong> that can prevent these conditions.</p><p>To obtain such a report, create <a href="/ngsw-docs-en/settings/report-templates/report-templates#add-template">a report template</a>
that includes &quot;SNVs/Indels by ClinVar phenotypes&quot; block.</p><p>The report is divided into three main
sections: <a href="/ngsw-docs-en/results/variants-by-phenotypes#report-result-and-recommendations">Report result and Recommendations</a>,
<a href="/ngsw-docs-en/results/variants-by-phenotypes#detailed-report">Detailed report</a>
and <a href="/ngsw-docs-en/results/variants-by-phenotypes#excluded-variants">Excluded variants</a>.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="report-result-and-recommendations"></a>1. Report Result and Recommendations<a class="hash-link" href="#report-result-and-recommendations" title="Direct link to heading">#</a></h3><p>The sections includes:</p><ul><li>A list of conditions (ClinVar phenotypes) that have the risk of developing in the patient or patient&#x27;s closest
relatives due to pathogenic or likely pathogenic variants discovered in patient sample and associated with these
conditions. The report may flag conditions that are characteristic of a person of the opposite sex.
This means that the variants found may not affect the risk of developing the condition in the patient
but may be transmitted from patient to a person or may have been inherited by patient from a person
who also may be at risk.</li><li>Pathogenic or likely pathogenic variants associated with the conditions listed above. Listed in a table with
the following columns:</li></ul><ol><li><strong>Gene</strong> is the common name of the gene in which the variant is located;</li><li><strong>Genetic variant</strong> is a variant ID in <a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a> database (rsId);</li><li><strong>Zygosity</strong> type of the variant: heterozygous or homozygous.</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/associated_variants-e8e25f7f6b3b8118d83bcd4579371ed3.png" width="700"></p><ul><li>Recommendations for the patient (consultation with a medical geneticist).</li><li>Information for the doctor: a brief summary of how the report is compiled.</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="detailed-report"></a>2. Detailed Report<a class="hash-link" href="#detailed-report" title="Direct link to heading">#</a></h3><p>A detailed report of the conditions listed in the first section. For each state, the following is indicated:</p><ul><li><strong>Name</strong>: corresponds to &quot;Report name&quot; field
from <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">the template</a>.</li><li><strong>Variants</strong> discovered in the sample that are associated with the condition (with MedGen ID
and/or ClinVar name in the template). <strong>Each variant meets the following conditions</strong>:</li></ul><ol><li>Pathogenic or likely pathogenic in ClinVar (&quot;Clinical significance&quot; field);</li><li>Not homozygous for reference (has at least one alternative allele);</li><li>The total allele frequency in gnomAD is above the frequency threshold for extremely rare pathogenic variants
and below the frequency threshold for too frequent pathogenic variants
(thresholds are set in <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">the template</a>).</li></ol><ul><li><strong>Recommendations</strong> that can help reduce the risk of this condition
(added in <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">the template</a>).</li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/phenotype_info-d74ddfe073d1b202dd0ff3bc5093e5c9.png" width="900"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="excluded-variants"></a>3. Excluded Variants<a class="hash-link" href="#excluded-variants" title="Direct link to heading">#</a></h3><p>Contains variants that were <strong>excluded from the detailed report</strong> according to one of the following criteria
(while these variants meet all of the above conditions for being included in the report):</p><ol><li><strong>Criterion 1</strong>: variants with <strong>too high frequency</strong> in the human population (greater than 15%).
Possible pathogenicity of these variants is inconsistent with such a high frequency.</li><li><strong>Criterion 2</strong>: variants that are <strong>homozygous</strong> and that have <strong>frequency of less than 1%</strong> in the
human population. According to <a href="https://www.bmj.com/content/372/bmj.n214" target="_blank" rel="noopener noreferrer">publication</a>, these variants
could be detected as a result of a genotyping error carried out using microarray technology.</li><li><strong>Criterion 3</strong>: variants that have a high probability of detection, expected based on the binomial
distribution and the probability of detecting a genetic variant among the identified populations.
The probability of the variant detection is below the reliability threshold, which is set
in <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">the template</a>.<br>The probability
is calculated as <em>(1 - Bin(n; m; p; 1))</em>,<br>where
<em>Bin(n; m; p; 1)</em> is the probability, expected based on the binomial distribution, that the variant
is detected <em>n</em> times among <em>m</em> representatives (samples) of the population;<br><em>n</em> is the number of representatives of the population (samples) in which this variant was detected;<br><em>m</em> is the total number of representatives (samples) in the population;<br><em>p</em> is the probability of detecting the variant in the human population (total allele frequency
in <a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD</a> database). If there is no data on the frequency in the
population, then the probability is taken as 0.001.<br>
Thus, variants are excluded from the report if they were detected more than <em>n</em> times among <em>m</em> samples
of the population, but at the same time have a low frequency in the human population, i.e. possible
pathogenicity of these variants is inconsistent with such a high frequency among representatives (samples)
of the population.<br>
The values of <em>n</em> and <em>m</em> are taken
from <a href="/ngsw-docs-en/settings/custom-annotations#define-annotation-type">the custom frequency database</a>. If the database
has not been uploaded into the system, variants will not be excluded by Criterion 3.</li></ol><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="variant"></a>Each variant is presented in the report as follows:<a class="hash-link" href="#variant" title="Direct link to heading">#</a></h4><p align="center"><img src="/ngsw-docs-en/assets/images/variant-bd379cace99d5bb1ed5f73ce236cbdf6.png" width="900"></p><ol><li><strong>Gene</strong> is the common name of the gene in which the variant is located.</li><li><strong>Variant ID</strong> is a variant ID in <a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a> database (rsId).</li><li><strong>Reference allele</strong> of the variant.</li><li><strong>Alternative allele</strong> of the variant.</li><li><strong>gnomAD AF</strong> is the total allele frequency in <a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD</a> database.</li><li><strong>Chromosome</strong> is the chromosome in which the variant is located.</li><li><strong>Start</strong> is the start position of the variant on the chromosome.</li><li><strong>End</strong> is the end position of the variant on the chromosome.</li><li><strong>Amino acid change</strong>: exon number in the transcript that the variant affected; amino acid and nucleotide
substitutions using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">the HGVS notation</a>. Amino acid substitution: ‚Äúp.‚Äù prefix
(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution.
Nucleotide substitution:  ‚Äúc.‚Äù (coding; for a substitution in the coding sequence) or ‚Äún.‚Äù (non-coding;
for a substitution in the non-coding sequence) + genomic position of the substituted nucleotide + reference
allele &gt; alternative allele.</li><li><strong>Transcript</strong> is the main transcript ID from <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a> database
(NM_xxxxxx.x).</li><li>An indication of the variant zygosity: <strong>heterozygous</strong> or <strong>homozygous</strong>.</li><li>For variants <strong>excluded</strong> from the detailed report, there is an indication of the criterion by which
they were excluded.</li></ol></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/main/alignments"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">¬´ Alignments</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/onco-genetic-testing-report"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Onco-Genetic Testing Report ¬ª</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#report-result-and-recommendations" class="table-of-contents__link">1. Report Result and Recommendations</a></li><li><a href="#detailed-report" class="table-of-contents__link">2. Detailed Report</a></li><li><a href="#excluded-variants" class="table-of-contents__link">3. Excluded Variants</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">¬© 2019-2023 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.72caaed9.js"></script>
<script src="/ngsw-docs-en/assets/js/main.d733ea5e.js"></script>
</body>
</html>