<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Germline SNVs/Indels discovery and annotation | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Germline SNVs/Indels discovery and annotation | Genomenal"><meta data-react-helmet="true" name="description" content="Germline SNVs/Indels discovery"><meta data-react-helmet="true" property="og:description" content="Germline SNVs/Indels discovery"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.7b4d41ea.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.643c0577.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Workflow Details</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/workflow-details">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Analysis of FASTQ or BAM samples</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/workflow-details">Analysis of FASTQ or BAM samples</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/upload-identify-verify">Upload, identify and verify</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">Check quality and cleanup</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">Alignment</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">Variant discovery preprocessing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">Calculate coverage</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels">Somatic SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels">Germline SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">Structural variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">Copy number variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">Phenotype prediction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">Generate reports</a></li></ul></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Germline SNVs/Indels discovery and annotation</h1></header><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="germline-variants-discovery"></a>Germline SNVs/Indels discovery<a class="hash-link" href="#germline-variants-discovery" title="Direct link to heading">#</a></h2><p>After <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">the &quot;Variant discovery preprocessing&quot; stage</a>
has been successfully completed, germline SNVs/Indels discovery can be initiated for a non-tumor or single tumor sample
if the stage is <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#germline-snvs-discovery">included</a> in the analysis
workflow.</p><p align="center"><img src="/ngsw-docs-en/assets/images/germline_snvs_indels-a0e0f11f7c79d720c1a2bd28db49910f.png" width="750"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="1-germline-variants-discovery"></a>1. Germline SNVs/Indels Discovery<a class="hash-link" href="#1-germline-variants-discovery" title="Direct link to heading">#</a></h3><p>Germline single-nucleotide variants (SNVs) and short insertions/deletions (indels) calling is performed
using <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller" target="_blank" rel="noopener noreferrer">GATK HaplotypeCaller</a>.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="intervals"></a>I. Dividing data into intervals<a class="hash-link" href="#intervals" title="Direct link to heading">#</a></h4><p>To speed up the process and save resources, the Scatter-Gather task divides the input data into 64 genomic
intervals. If the sample capture kit is defined, the data is divided
using <a href="https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html" target="_blank" rel="noopener noreferrer">bedtools intersect</a>, which
screens for overlaps between distinct genomic intervals and capture kit intervals.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="calling"></a>II. Variant calling<a class="hash-link" href="#calling" title="Direct link to heading">#</a></h4><p>Variant calling in each of the resulting intervals is performed
using <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller" target="_blank" rel="noopener noreferrer">GATK HaplotypeCaller</a>.
Call germline SNPs and indels via local re-assembly of haplotypes
Ths tool is capable of calling SNVs and indels simultaneously via local <em>de novo</em> assembly of haplotypes in an
active region. In other words, whenever the program encounters a region showing signs of variation, it discards
the existing mapping information and completely reassembles the reads in that region. This allows the HaplotypeCaller
to be more accurate when calling regions that are traditionally difficult to call, for example when they contain
different types of variants close to each other.</p><p>Reference model is emitted by homozygous-reference sites compressed into bands of similar genotype quality,
i.e. <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035531812-GVCF-Genomic-Variant-Call-Format#:~:text=The%20term%20GVCF%20is%20sometimes,UnifiedGenotyper%20with%20%2D%2Doutput_mode%20EMIT_ALL_SITES%20" target="_blank" rel="noopener noreferrer">the GVCF (Genomic VCF) format</a>.
The key difference between a regular VCF and a GVCF is that the GVCF has records for all sites, whether there
is a variant call there or not. The goal is to have every site represented in the file in order to do joint
analysis of a cohort in subsequent steps. The records in a GVCF include an accurate estimation of how confident
we are in the determination that the sites are homozygous-reference or not.</p><p>By default, soft clipped bases (i.e. bases from the ends of reads that do not align with the reference sequence)
are analyzed when calling variants. This is controlled
by <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#soft-clipped-bases">the corresponding setting</a>.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="combine-gvcfs"></a>III. Combining GVCF files<a class="hash-link" href="#combine-gvcfs" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037053272-CombineGVCFs" target="_blank" rel="noopener noreferrer">GATK CombineGVCFs</a>
combines gVCF files produced by HaplotypeCaller into a single gVCF file.
The resulting file is compressed into a GZIP archive using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>.
It can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Germline SNVs/Indels Discovery</em>&quot; task details
(&quot;<em>Download VCF_GZ</em>&quot;). You can also open this file in the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot;
link. In addition, the VCF file is indexed using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting
index file can be downloaded in the same section (&quot;<em>Download VCF_TBI</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="genotype-gvcfs"></a>2. Perform Joint Genotyping<a class="hash-link" href="#genotype-gvcfs" title="Direct link to heading">#</a></h3><p>In the next step, <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037057852-GenotypeGVCFs" target="_blank" rel="noopener noreferrer">GATK GenotypeGVCFs</a>
performs joint genotyping on samples pre-called with HaplotypeCaller. The resulting file is compressed into a GZIP
archive using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. It can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Perform Joint Genotyping</em>&quot; task details (&quot;<em>Download VCF_GZ</em>&quot;). You can also open this file in the IGV
browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF file is indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index file can be downloaded in the same
section (&quot;<em>Download VCF_TBI</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="filter-germline-variants"></a>3. Filter Germline SNVs/Indels<a class="hash-link" href="#filter-germline-variants" title="Direct link to heading">#</a></h3><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="select-variants"></a>I. Selecting SNVs/Indels<a class="hash-link" href="#select-variants" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037055952-SelectVariants" target="_blank" rel="noopener noreferrer">GATK SelectVariants</a>
selects a subset of variants (separately single nucleotide variants and separately indels) from a VCF file.
The resulting files with SNVs and with indels are compressed into GZIP
archives using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. They can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Filter Germline SNVs/Indels</em>&quot; task details (&quot;<em>Download Unfiltered SNPs VCF_GZ</em>&quot; and
&quot;<em>Download Unfiltered INDELs VCF_GZ</em>&quot;, respectively). You can also open these files in the IGV
browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF files are indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index files can be downloaded in the same
section (&quot;<em>Download Unfiltered SNPs VCF_TBI</em>&quot; and &quot;<em>Download Unfiltered INDELs VCF_TBI</em>&quot;).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="filtering"></a>II. Filtering SNVs/Indels<a class="hash-link" href="#filtering" title="Direct link to heading">#</a></h4><p>Then hard-filtering variant calls based on INFO and/or FORMAT annotations is performed
using <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037434691-VariantFiltration" target="_blank" rel="noopener noreferrer">GATK VariantFiltration</a>.
Records are hard-filtered by changing the value in the &quot;FILTER&quot; field to something other than &quot;PASS&quot;.
Filtered records are preserved in the output. Filtering is performed based
on <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#hard-filtration">hard-filtering parameters</a>
specified for the sample in the settings.</p><p>The resulting files with filtered SNVs and filtered indels are compressed into GZIP
archives using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. They can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Filter Germline SNVs/Indels</em>&quot; task details (&quot;<em>Download Filtered SNPs VCF_GZ</em>&quot; and
&quot;<em>Download Filtered INDELs VCF_GZ</em>&quot;, respectively). You can also open these files in the IGV
browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF files are indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index files can be downloaded in the same
section (&quot;<em>Download Filtered SNPs VCF_TBI</em>&quot; and &quot;<em>Download Filtered INDELs VCF_TBI</em>&quot;).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="merge-vcfs"></a>III. Merging filtered variants<a class="hash-link" href="#merge-vcfs" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360036713331-MergeVcfs-Picard" target="_blank" rel="noopener noreferrer">GATK MergeVcfs</a>
combines VCF files with filtered single nucleotide variants and filtered indels into a single variant file in VCF
format. The resulting file with all germline variants discovered in a sample is compressed into a GZIP archive
using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. It can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Filter Germline SNVs/Indels</em>&quot; task details (&quot;<em>Download Filtered SNPs/INDELs VCF_GZ</em>&quot;). You can also open
this file in the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF file is indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index file can be downloaded in the same
section (&quot;<em>Download Filtered SNPs/INDELs VCF_TBI</em>&quot;).</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="annotate-variants"></a>Germline SNVs/Indels annotation<a class="hash-link" href="#annotate-variants" title="Direct link to heading">#</a></h2><p>After <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels#germline-variants-discovery">the &quot;Germline SNVs/Indels discovery&quot; stage</a>
has been successfully completed, germline SNVs/Indels annotation is performed.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="1-annotate-variants"></a>1. Annotate Germline SNVs/Indels<a class="hash-link" href="#1-annotate-variants" title="Direct link to heading">#</a></h3><ul><li><p>Annotation of SNVs/Indels in the file with data from the <a href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank" rel="noopener noreferrer">RefSeq</a>,
<a href="https://www.internationalgenome.org/" target="_blank" rel="noopener noreferrer">1000 Genomes</a>, <a href="https://sites.google.com/site/jpopgen/dbNSFP" target="_blank" rel="noopener noreferrer">dbNSFP</a>,
<a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a>, <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/" target="_blank" rel="noopener noreferrer">gnomAD 3</a>,
<a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD 4</a>,
<a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>, <a href="https://cadd.gs.washington.edu/" target="_blank" rel="noopener noreferrer">CADD</a>,
<a href="https://spliceailookup.broadinstitute.org/" target="_blank" rel="noopener noreferrer">SpliceAI</a>, <a href="https://enigmaconsortium.org/" target="_blank" rel="noopener noreferrer">ENIGMA</a> databases.</p></li><li><p>Determination of the effect of SNVs/Indels on genes, transcripts, and protein sequence, as well as regulatory
regions, using <a href="https://www.ensembl.org/info/docs/tools/vep/index.html" target="_blank" rel="noopener noreferrer">Ensembl Variant Effect Predictor (VEP)</a>:</p><ul><li><p><a href="http://genetics.bwh.harvard.edu/pph2/" target="_blank" rel="noopener noreferrer">PolyPhen</a> predicts possible impact of an amino acid substitution on
the structure and function of a protein using straightforward physical and comparative considerations.</p></li><li><p>A flag indicating if the transcript is the canonical transcript for the gene is added.</p></li><li><p>Allele number from VCF input is identified.</p></li><li><p>Affected exon and intron numbering is added.</p></li><li><p><a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS nomenclature</a> based on <a href="https://www.ensembl.org/index.html" target="_blank" rel="noopener noreferrer">Ensembl</a> stable
identifiers is added.</p></li><li><p>Upstream variant consequences are determined if the upstream (5&#x27;) distance between a variant and a transcript
is more than 2000 bp. Downstream variant consequences are determined if the downstream (3&#x27;) distance between
a variant and a transcript is more than 1000 bp.</p><p>   The resulting file with raw annotated SNVs/Indels in TSV format can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Annotate Germline SNVs/Indels</em>&quot; task details (&quot;<em>Download Raw annotated TSV</em>&quot;).
You can also open it in Google Sheets.<br>
Files with annotated SNVs/Indels (all variants and filtered variants) without duplicates in TSV format can be
downloaded in the same section (&quot;<em>Download All variants TSV</em>&quot; and &quot;<em>Download Filtered variants TSV</em>&quot;, respectively).
You can also open them in Google Sheets. The same files in CSV format can be downloaded from the same place
(&quot;<em>Download All variants CSV</em>&quot; and &quot;<em>Download Filtered variants CSV</em>&quot;).</p></li></ul></li><li><p>Converting TSV results to VCF.</p></li><li><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037428851-FixVcfHeader-Picard-" target="_blank" rel="noopener noreferrer">GATK FixVCFHeader</a>
replaces or fixes a VCF header.</p></li><li><p>The file is compressed into a GZIP archive using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>.
The resulting files can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Annotate Germline SNVs/Indels</em>&quot; task details (file with all variants: &quot;<em>Download All variants VCF_GZ</em>&quot;,
file with filtered variants: &quot;<em>Download Filtered variants VCF_GZ</em>&quot;).
You can also open these files in the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link.</p></li><li><p>The files are indexed using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>.
The resulting index files can be downloaded in the same section (&quot;<em>Download All variants VCF_TBI</em>&quot; and
&quot;<em>Download Filtered variants VCF_TBI</em>&quot;).</p></li><li><p>Calculating variants&#x27; statistic.</p></li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="save-variants"></a>2. Store annotated variants for SNV Viewer<a class="hash-link" href="#save-variants" title="Direct link to heading">#</a></h3><ul><li>Saving results for display in <a href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer">SNV Viewer</a>, i.e. an embedded service
for viewing and analyzing variants.</li><li>Adding information about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page#occurrences">the variant occurrence</a> in
other samples of the user.</li></ul><p>After the &quot;Germline SNVs/Indels annotation&quot; stage, the analysis can proceed
with <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">report generation</a>.</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>If you want to add a track with germline SNVs/Indels discovered by the analysis of the sample you uploaded to
Genomenal to your <a href="https://igv.org/doc/desktop/" target="_blank" rel="noopener noreferrer">desktop IGV</a>, you can do so via a link.
Open the details of the necessary task (&quot;<em>Germline SNVs/Indels Discovery</em>&quot;, &quot;<em>Perform Joint Genotyping</em>&quot;,
&quot;<em>Filter Germline SNVs/Indels</em>&quot;, &quot;<em>Annotate Germline SNVs/Indels</em>&quot;) and do the following:</p><ol><li>Right-click the variant file link (depending on the selected task, this may be the link &quot;<em>Download VCF_GZ</em>&quot;,
&quot;<em>Download Unfiltered SNPs VCF_GZ</em>&quot;, &quot;<em>Download Unfiltered INDELs VCF_GZ</em>&quot;, &quot;<em>Download Filtered SNPs VCF_GZ</em>&quot;,
&quot;<em>Download Filtered INDELs VCF_GZ</em>&quot;, &quot;<em>Download Filtered SNPs/INDELs VCF_GZ</em>&quot;, &quot;<em>Download Filtered SNPs/INDELs VCF_GZ</em>&quot;,
&quot;<em>Download All variants VCF_GZ</em>&quot; or &quot;<em>Download Filtered variants VCF_GZ</em>&quot;) and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Upload the track via URL to your desktop IGV, as
described <a href="https://igv.org/doc/desktop/#UserGuide/loading_data/" target="_blank" rel="noopener noreferrer">here</a>.</li><li>Right-click on the download link for the index file corresponding to the annotation file (&quot;<em>Download VCF_TBI</em>&quot;,
&quot;<em>Download Unfiltered SNPs VCF_TBI</em>&quot;, &quot;<em>Download Unfiltered INDELs VCF_TBI</em>&quot;, &quot;<em>Download Filtered SNPs VCF_TBI</em>&quot;,
&quot;<em>Download Filtered INDELs VCF_TBI</em>&quot;, &quot;<em>Download Filtered SNPs/INDELs VCF_TBI</em>&quot;,
&quot;<em>Download Filtered SNPs/INDELs VCF_TBI</em>&quot;, &quot;<em>Download All variants VCF_TBI</em>&quot; or &quot;<em>Download Filtered variants VCF_TBI</em>&quot;)
and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Add the index via URL to the corresponding field in IGV.</li><li>Click &quot;<em>OK</em>&quot;. Done! The track with germline SNVs/Indels discovered in the sample is added to IGV.</li></ol></div></div></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Somatic SNVs/Indels discovery and annotation</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Structural variations discovery Â»</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#germline-variants-discovery" class="table-of-contents__link">Germline SNVs/Indels discovery</a><ul><li><a href="#1-germline-variants-discovery" class="table-of-contents__link">1. Germline SNVs/Indels Discovery</a></li><li><a href="#genotype-gvcfs" class="table-of-contents__link">2. Perform Joint Genotyping</a></li><li><a href="#filter-germline-variants" class="table-of-contents__link">3. Filter Germline SNVs/Indels</a></li></ul></li><li><a href="#annotate-variants" class="table-of-contents__link">Germline SNVs/Indels annotation</a><ul><li><a href="#1-annotate-variants" class="table-of-contents__link">1. Annotate Germline SNVs/Indels</a></li><li><a href="#save-variants" class="table-of-contents__link">2. Store annotated variants for SNV Viewer</a></li></ul></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2024 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.7b4d41ea.js"></script>
<script src="/ngsw-docs-en/assets/js/main.643c0577.js"></script>
</body>
</html>