---
sidebar_position: 3
sidebar_label: Filtering of Variations
---

# Filtering of Variations

The needed copy number variations can be found by filtering. Filtering options are located above the table:

<p align="center">
<img src={require('/img/vv/cnvv_filtering.png').default} width="900"/>
</p>

:::note
Table filtering is saved when you exit the CNV Viewer page.
:::

### 1. Log2FC thresholds {#log2fc-threshold}

Filter variations by the log2FC value, i.e. the logarithmic ratio of the detected copy number to the normal
copy number. Possible filtering values:
- Del: ≤-1, Dup: ≥0.58 (chrX: ≥1, chrY: ≥1)
- Del: ≤-0.9 (chrX: ≤-3.8, chrY: ≤-3.8), Dup: ≥0.55 (chrX: ≥0.94, chrY: ≥0.94)
- Del: ≤-0.8 (chrX: ≤-2.74, chrY: ≤-2.74), Dup: ≥0.51 (chrX: ≥0.88, chrY: ≥0.88)
- Del: ≤-0.7 (chrX: ≤-2.11, chrY: ≤-2.11), Dup: ≥0.46 (chrX: ≥0.82, chrY: ≥0.82)
- Del: ≤-0.6 (chrX: ≤-1.64, chrY: ≤-1.64), Dup: ≥0.42 (chrX: ≥0.74, chrY: ≥0.74)
- Del: ≤-0.5 (chrX: ≤-1.27, chrY: ≤-1.27), Dup: ≥0.37 (chrX: ≥0.66, chrY: ≥0.66)
- Del: ≤-0.4 (chrX: ≤-0.95, chrY: ≤-0.95), Dup: ≥0.31 (chrX: ≥0.56, chrY: ≥0.56)
- Del: ≤-0.3 (chrX: ≤-0.67, chrY: ≤-0.67), Dup: ≥0.24 (chrX: ≥0.45, chrY: ≥0.45)

**Filtering algorithm:**:
- Click on the filter window;
- Select the needed filtering option from the drop-down list, or type the option you are looking for in the
search box and select from the found.

Done! Filter applied. The selected filter can be seen in the window:

<p align="center">
<img src={require('/img/vv/log2fc_value.png').default} width="200"/>
</p>

Default value: *- Del: ≤-0.7 (chrX: ≤-2.11, chrY: ≤-2.11), Dup: ≥0.46 (chrX: ≥0.82, chrY: ≥0.82)*.
It will be applied when all filters are reset. It can only be changed or cleared in the filter options
(see [below](/results/main/copy-number-variations/cnv-filtering#reset-filtering)).
Using the default value as an example, let’s look at what these filtering conditions mean:
- Copy number variations on autosomes with log2FC ≤ -0.7 are considered deletions. -0.7 is the default
log2FC threshold, below which only one autosome is called instead of the normal two.
- Copy number variations on the X and Y chromosomes with log2FC ≤ -2.11 are considered deletions.
- Copy number variations on autosomes with log2FC ≥ 0.46 are considered duplications.
The threshold for duplication on an autosome is calculated using the following
formula: *log2FC = log<sub>2</sub>((2+D)/2)*, where *D* is the “number” of copy number that
is “lost” at log2FC = -0.7, and *2* is the number of sister chromatids in the autosome.
- Copy number variations on the X and Y chromosomes with log2FC ≥ 0.82 are considered duplications.
- Variations that do not satisfy the thresholds, i.e. variations on autosomes with -0.7 < log2FC < 0.46
and variations on sex chromosomes with -2.11 < log2FC < 0.82 do not pass filtering, i.e. are excluded
from the CNV Viewer table.

#### Calculation of log2FC thresholds for deletions and duplications on autosomes and sex chromosomes

The threshold of calling a single copy instead of two on autosomes is calculated as
follows: <i>T<sub>21</sub> = log<sub>2</sub>(C/2)</i>, where <i>C</i> is the number of called
copies.<br/>Delta of copy number: <i>D = 2 – C = 2 – 2 × 2<sup>T<sub>21</sub></sup></i>.<br/>
Then the corresponding threshold of calling three copies instead of two on autosomes is calculated as
follows: <i>T<sub>23</sub> = log<sub>2</sub>((2+D)/2) = log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>)</i>.<br/>
<b>Threshold of calling deletions on autosomes:</b> <i>lo = T<sub>21</sub></i>.<br/>
<b>Threshold of calling duplications on autosomes:</b> <i>hi = T<sub>23</sub></i>.<br/>
To calculate the threshold of calling three copies instead of two on the X chromosome,
we normalize to XY genotype by dividing it by copy number
of 1: <i>T<sub>23X</sub>= log<sub>2</sub>((2+D)/1) = log<sub>2</sub>(2+2-2×2<sup>T<sub>21</sub></sup>) = 1 + log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>) = 1 + T<sub>23</sub></i>.<br/>
<b>Calculating the threshold of calling duplications on the X chromosome:</b><br/>
- if there is no Y chromosome: <i>X<sub>hi</sub> = 1 + T<sub>23</sub></i>;<br/>
- if there is Y chromosome: <i>X<sub>hi</sub> = log<sub>2</sub>(3-2×2<sup>T<sub>21</sub></sup>)</i>.<br/>
The threshold of calling one copy instead of two on the X
chromosome: <i>T<sub>21X</sub> = log<sub>2</sub>((2-D)/1) = log<sub>2</sub>(2-2+2×2<sup>T<sub>21</sub></sup>) = 1 + T<sub>21</sub></i>.<br/>
Threshold of calling no copies instead of one on the X
chromosome: <i>T<sub>10X</sub> = log<sub>2</sub>((1-D)/1) = log<sub>2</sub>(1-2+2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br/>
<b>Calculating the threshold of calling deletions on the X chromosome:</b><br/>
- if there is no Y chromosome: <i>X<sub>lo</sub> = 1 + T<sub>21</sub></i>;<br/>
- if there is Y chromosome: <i>X<sub>lo</sub> = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br/>
The threshold of calling one copy instead of none on the Y chromosome in the case of genotype
XX: <i>T<sub>01</sub> = log<sub>2</sub>((0+D)/1) = log<sub>2</sub>(2-2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(2×(1-2<sup>T<sub>21</sub></sup>)) =  1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>)</i>.<br/>
<b>Calculating the threshold of calling duplication <i>Y<sub>hi</sub></i> and deletions <i>Y<sub>lo</sub></i> on the Y chromosome:</b><br/>
- if there is no Y chromosome: <i>Y<sub>hi</sub> = 1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>))</i>; <i>Y<sub>lo</sub> = -∞</i>;<br/>
- if there is Y chromosome: <i>Y<sub>hi</sub> = X<sub>hi</sub></i>; <i>Y<sub>lo</sub> = X<sub>lo</sub></i>.

### 2. Region chr:start-end {#region}

Filter variations by position. To filter, enter the position of the needed genomic interval, i.e. the
corresponding chromosome and the start and end positions. Possible options of writing a position:
- `chr1:161,190,824-161,193,921`
- `chr1:161190824-161193921`
- `chr1:161190824_161193921`
- `1:161190824-161193921`

You can also filter by more than one interval at once. For example,
by `chr1:161190824-161193921, chrX:151110769-151113866`.

The filtering result will be rows with variations whose positions intersect with the entered interval.

### 3. Gene panel {#gene-panel}

Filtering variations by localization in certain genes from the panel.
You can add a gene panel <a href="/settings/gene-panels">on "Gene Panels" page</a> or import a panel
from the Library of the most common clinical panels. To import, click on "<i>Import from Library</i>":

<p align="center">
<img src={require('/img/vv/gene_panel.png').default} width="300"/>
</p>

Select the required panels by ticking them in the window that opens:

<p align="center">
<img src={require('/img/gene_panels/checked_panels.png').default} width="600"/>
</p>

and click on <img src={require('/img/gene_panels/import_btn.png').default} width="150"/> to add panel.

### 4. Gene {#gene}

Filtering variations by localization in the gene transcript. The transcript is given in brackets next to
the official gene name. You can filter variations by several genes at once. To do this, tick the required genes
and click outside the value area:

<p align="center">
<img src={require('/img/vv/filter_by_gene.png').default} width="200"/>
</p>

If several transcripts are known for a gene, the list presents all of them:

<p align="center">
<img src={require('/img/vv/many_transcripts.png').default} height="400"/>
</p>

### 5. HPO {#hpo}

Filtering variations by localization in genes associated with
terms from <a href="https://hpo.jax.org/app/">Human Phenotype Ontology (HPO)</a>.
Click on the HPO filter container to open the phenotype search form.
The form presents the main groups of phenotypes in the drop-down list:

<p align="center">
<img src={require('/img/vv/hpo_search.png').default} width="800"/>
</p>

**Ways to search and select a term for filtering:**
- Search for a term in the drop-down list. You can interact with term groups
using <img src={require('/img/vv/show.png').default} height="15"/> and <img src={require('/img/vv/hide.png').default} height="10"/> to
expand and collapse the list of group phenotypes, respectively:

<p align="center">
<img src={require('/img/vv/list_of_phenotypes.png').default} width="270"/>
</p>

In addition, you can find a phenotype by searching for the name or ID from HPO.
For example, when searching for the disease “glioma”, several terms are found:

<p align="center">
<img src={require('/img/vv/glioma_search.png').default} width="800"/>
</p>

To select a term to filter, click on its row. You can choose both the whole group and a specific
phenotype. By selecting one of the terms (e.g. "Glioma"), you will see this disease card:

<p align="center">
<img src={require('/img/vv/glioma_page.png').default} width="800"/>
</p>

The phenotypes selected for filtering are marked above the disease card. Number in parentheses
indicates the number of genes associated with the phenotype. By clicking on a term, you will see
the phenotype card. You can remove the term by clicking on
the cross <img src={require('/img/vv/delete_term_btn.png').default} width="15"/>:

<p align="center">
<img src={require('/img/vv/delete_term.png').default} width="500"/>
</p>

On the left side of the phenotype card, there are its definition and comments. In the middle,
there is the hierarchy of the disease: the terms above the selected Glioma
disease are its ancestor phenotypes (i.e. the phenotype groups that include Glioma),
and the terms below Glioma are its child phenotypes (i.e. the diseases that are part of the Glioma group).
On the right, there are the genes associated with this disease (in this case, 45 genes were found).

- Search for a term by hierarchy in the phenotype card. By clicking on the name of the ancestor or child
phenotype, you will see this phenotype page. If the phenotype has not yet been selected for filtering and
genes associated with it have been found, then you can select it for filtering variations by
clicking on <img src={require('/img/vv/add_term.png').default} width="150"/>.

- Search for a term in multiselect mode. Switch the search mode by
clicking on <img src={require('/img/vv/switch_mode.png').default} width="120"/>. You will see a list of main
groups of phenotypes with checkboxes. You can find a phenotype by searching for the name or ID from HPO.
For example, when searching for the disease “glioma”, several terms are found:

<p align="center">
<img src={require('/img/vv/multiselect_hpo.png').default} width="800"/>
</p>

In multiselect mode, you can select several terms at once in the drop-down list.
To select a term to filter, click on its line:

<p align="center">
<img src={require('/img/vv/glioma_checked.png').default} width="400"/>
</p>

To complete adding terms for filtering, click outside the drop-down box within the phenotype search form.
At the top of the form, you will see the phenotypes selected for filtering. Number in parentheses
indicates the number of genes associated with the phenotype. By clicking on a term, you will see
the phenotype card. You can remove the term by clicking on
the cross <img src={require('/img/vv/delete_term_btn.png').default} width="15"/>.

<p align="center">
<img src={require('/img/vv/multiselect_mode.png').default} width="800"/>
</p>

To ***filter variations*** by genes associated with the selected phenotypes,
click on <img src={require('/img/custom-annot/ok_btn.png').default} width="130"/>.
You can see the selected phenotypes in the tooltip by hovering over the HPO filter field:

<p align="center">
<img src={require('/img/vv/chosen_phenotypes.png').default} width="300"/>
</p>

To ***cancel filtering*** by all phenotypes, hover over the filter window and click on the cross:

<p align="center">
<img src={require('/img/vv/cancel_hpo.png').default} width="150"/>
</p>

or click on the field and edit your selection in the phenotype search form.

### 6. Type {#cnv-type}

Filter variations by the copy number variation type: LOSS - deletion, GAIN - duplication.

### 7. CDS Overlap {#cds-overlap}

Filter variations by overlapping with CoDing Sequence (CDS):
- Overlapped CDS - the variation affects the CDS (percentage of overlapped CDS is greater than 0);
- Not overlapped CDS - the variation does not affect the CDS (percentage of overlapped CDS equals 0).

### 8. ACMG Class {#acmg-class}

Filter variations by pathogenicity assigned according to the ranking score defined
by [AnnotSV](https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf):
* Pathogenic;
* Likely pathogenic;
* Uncertain significance conflict - conflicting interpretations of pathogenicity;
* Uncertain significance;
* Likely benign;
* Benign;
* Undefined.

You can filter variations by several pathogenicity values at once.
To do this, tick the required values and click outside the value area.

### 9. Cohort 180 AF {#cohort-af}

Filter variations by frequency in a cohort of 188 normal samples. AF values are only
available for germline CNVs discovered in WGS or low-pass WGS samples normalized to the "*BGI WGS*" reference
samples panel with bin size 3000 or 5000 or the "*BGI WGS v2*" reference samples panel with bin size 3000.
The filter is only displayed in the CNV Viewer with copy number variations discovered in the bins.

Possible values:
- <0.05 or empty;
- <0.02 or empty;
- <0.01 or empty;
- <0.005 or empty;
- <0.0001 or empty.

## Reset Filters {#reset-filtering}

To reset ***all*** applied filters, click on <img src={require('/img/vv/clear_filters.png').default} width="43"/>.
Only filtering by default log2FC threshold
value *- Del: ≤-0.7 (chrX: ≤-2.11, chrY: ≤-2.11), Dup: ≥0.46 (chrX: ≥0.82, chrY: ≥0.82)* will be preserved.

To clear ***a separate filter***:
- Click on the filter window and then on <img src={require('/img/vv/clear_option.png').default} width="70"/>.
- To clear filtering by region, hover over the filter window and click on the cross:

<p align="center">
<img src={require('/img/vv/clear_region.png').default} width="250"/>
</p>

- For a filter by gene, you can remove one of the selected filtering options.
To do this, click on the filter window and uncheck the option.
- To change the selection of HPO phenotypes by which variations are filtered, click on the HPO filter
window and edit your selection in the phenotype search form. To clear filtering by all selected phenotypes,
hover over the filter window and click on the cross:

<p align="center">
<img src={require('/img/vv/cancel_hpo.png').default} width="150"/>
</p>
