---
sidebar_position: 1
sidebar_label: Copy Number Variations
---

# Copy Number Variations

The "***Somatic Copy Number Variations***" and "***Germline Copy Number Variations***" sections contain
the results of discovery of somatic and germline, respectively, copy number variations (CNVs)
and calculating copy number values for a given sample against other samples.

***Somatic variations*** occur after conception and are not passed on to children. They can cause cancer or other
diseases. Somatic variations discovery is carried out only for ***tumor tissue samples*** and, therefore, the results
are presented only on the tumor tissue sample page.

<p align="center">
<img src={require('/img/results/somatic_cnvs.png').default} width="800"/>
</p>

***Germline variations*** can be discovered in any cells of the body and are passed on from parents to offspring.
Accordingly, germline variations discovery is carried out both for ***tumor tissue samples*** (in the case of
single tumor tissue samples without control) and for ***non-tumor tissue samples***.
For a set of tumor/normal samples, the results of the germline variations discovery
in the normal sample are presented on the pages of both samples of the set.

<p align="center">
<img src={require('/img/results/germline_cnvs.png').default} width="800"/>
</p>

For a single tumor sample without control, the results of discovery of somatic and germline variations
are combined into one section:

<p align="center">
<img src={require('/img/results/somatic_and_germline_cnvs.png').default} width="800"/>
</p>

The detected and annotated copy number variations (CNVs) are presented in all sections as:

1. **CNV Viewer** is an embedded service for viewing and analyzing variations. Described in detail
in [the corresponding section](/results/main/copy-number-variations/cnv-viewer). By clicking on the
button <img src={require('/img/results/cnvv_btn.png').default} width="120"/>, you will see two options:
“*Segments*” and “*Bins*”, which open the CNV Viewer with the copy number variations discovered in the
segments or bins, respectively:

<p align="center">
<img src={require('/img/results/cnvv_options.png').default} width="160"/>
</p>

2. File in **AnnotSV CSV** format that is a table of annotated copy number variations. They are presented either
like a complete deletion or insertion, or like genes overlapped by a deletion/insertion. By clicking on the
button <img src={require('/img/results/table_btn.png').default} width="120"/>, you will see options for downloading.
Their number will depend on whether any variations remain after filtering. Possible options are "*Segments all*",
"*Segments filtered*", "*Bins all*" and "*Bins filtered*". To download a file with annotated
variations discovered in segments or bins, before or after filtering, select the appropriate option:

<p align="center">
<img src={require('/img/results/annotsv_options.png').default} width="200"/>
</p>

<details>
<summary>Click to see the main fields of AnnotSV CSV</summary>

#### The main fields of AnnotSV CSV {#annotsv-fields}

<table>
  <tr>
   <td>Field</td>
   <td>Description</td>
  </tr>
  <tr>
   <td>AnnotSV_ID</td>
   <td>Variation ID assigned by the annotator</td>
  </tr>
  <tr>
   <td>SV_chrom</td>
   <td>Chromosome</td>
  </tr>
  <tr>
   <td>SV_start</td>
   <td>Variation start position in the chromosome</td>
  </tr>
  <tr>
   <td>SV_end</td>
   <td>Variation end position in the chromosome</td>
  </tr>
  <tr>
   <td>SV_length</td>
   <td>Variation length</td>
  </tr>
  <tr>
   <td>SV_type</td>
   <td>Variation type: DEL - deletion, DUP - duplication</td>
  </tr>
  <tr>
   <td>Filter</td>
   <td>Filtering value: PASS (variation satisfies all filters) or empty (variation did not pass filtering).
   The value will always be "PASS" in the files after filtering.</td>
  </tr>
  <tr>
   <td>log2FC</td>
   <td>The logarithmic ratio of the discovered copy number to the normal copy number (equals 2 for
   autosomes and the X chromosome in the case of the XX genotype, or 1 for the sex chromosomes in the
   case of the XY genotype).</td>
  </tr>
  <tr>
   <td>deltaCP</td>
   <td>Relative copy number is the number of copies of a certain region per haploid genome.</td>
  </tr>
  <tr>
   <td>Annotation_mode</td>
   <td>Annotation line type: “full” - annotation of all complete variations, “split” - annotation of a variation
   split by gene. Each full annotation row can correspond to several split annotation rows.</td>
  </tr>
</table>

A description of the remaining 126 annotation fields can be found here:
[AnnotSV Manual](https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf).

</details>

3. File in **Google Spreadsheet** (only a limited number of variations can be shown). By clicking on the
button <img src={require('/img/results/google_btn.png').default} width="45"/>, you will see file options.
Their number will depend on whether any variations remain after filtering. Possible options are "*Segments all*",
"*Segments filtered*", "*Bins all*" and "*Bins filtered*". To open a file with annotated
variations discovered in segments or bins, before or after filtering, select the appropriate option:

<p align="center">
<img src={require('/img/results/google_options.png').default} width="200"/>
</p>

4. **CNV coverage file** in TSV format that is an unannotated file, convenient for analysing copy number
variations in large regions (chromosome arms, chromosomes). To download the file,
click on <img src={require('/img/results/cnv_tsv.png').default} width="45"/>.

<details>
<summary>Click to see the main fields of CNV coverage file</summary>

#### The main fields of CNV coverage file {#cnv-coverage-fields}

<table>
  <tr>
   <td>Field</td>
   <td>Description</td>
  </tr>
  <tr>
   <td>ID</td>
   <td>Region name (chromosome arm, chromosome)</td>
  </tr>
  <tr>
   <td>Call</td>
   <td>CNV discovery result. Possible values are "+" (increasing the number of copies compared to normal), "-"
   (reducing the number of copies compared to normal), "." (no deviations from the normal chromosome copy number
   were found) and "Not considered" (the locus was not considered at all in the analysis).</td>
  </tr>
  <tr>
   <td>Ampl_cov</td>
   <td>Coverage (in nucleotides) of the effective locus length with segments with a copy number above
   the amplification threshold</td>
  </tr>
  <tr>
   <td>Ampl_cov_frac</td>
   <td>Fraction (in %) of the effective locus length covered by segments with a copy number above
   the amplification threshold</td>
  </tr>
  <tr>
   <td>Arm</td>
   <td>Chromosome arm</td>
  </tr>
  <tr>
   <td>Chrom</td>
   <td>Chromosome</td>
  </tr>
  <tr>
   <td>Depl_cov</td>
   <td>Coverage (in nucleotides) of the effective locus length with segments with a copy number below
   the depletion threshold</td>
  </tr>
  <tr>
   <td>Depl_cov_frac</td>
   <td>Fraction (in %) of the effective locus length covered by segments with a copy number below
   the depletion threshold</td>
  </tr>
  <tr>
   <td>Effective_end</td>
   <td>Maximum coordinate value of effective length</td>
  </tr>
  <tr>
   <td>Effective_length</td>
   <td>Total effective length</td>
  </tr>
  <tr>
   <td>Effective_start</td>
   <td>Minimum coordinate value of effective length</td>
  </tr>
  <tr>
   <td>End</td>
   <td>End coordinate of the variation</td>
  </tr>
  <tr>
   <td>Length</td>
   <td>Variation length</td>
  </tr>
  <tr>
   <td>Region type</td>
   <td>Locus type: arm or whole chromosome</td>
  </tr>
  <tr>
   <td>Start</td>
   <td>Start coordinate of the variation</td>
  </tr>
  <tr>
   <td>is_masked</td>
   <td>If the repeats masking was carried out in the sequence: True/False</td>
  </tr>
  <tr>
   <td>seg_val</td>
   <td>Segment average value</td>
  </tr>
</table>

</details>

5. **Genome-wide CNV segments' plot**. By clicking on the
button <img src={require('/img/eng/25-plot_file.png').default} width="45"/>, you will see the following options:

<p align="center">
<img src={require('/img/results/cnv_graph_options.png').default} width="250"/>
</p>

- The "*Summary plot*" option opens a genome-wide CNV segments' plot in JPG format. Plot example:

<p align="center">
<img src={require('/img/eng/28-genome-wide-plot.png').default} width="1000"/>
</p>

We can see the chromosomes on a horizontal scale and the copy number in chromosomes on a vertical scale.
In the given example, the copy number of chromosome 21 is increased in the sample compared to normal diploid
number of copies.
- The "*Plots by chromosomes*" option opens a PDF file with CNV segments' plots for each chromosome.
Plot example for chr21:

<p align="center">
<img src={require('/img/results/chr21-plot.png').default} width="1000"/>
</p>

We can see the genomic intervals of the chromosome on a horizontal scale and the copy number in the chromosome
on a vertical scale. On the given plot, we see that the copy number of chromosome 21 is increased several
times for this sample.

6. **Karyogram-like graph** with denoted chromosome-level CNVs in JPG format. To open the graph,
click on <img src={require('/img/eng/26_karyotype_file.png').default} width="45"/>. Graph example:

<p align="center">
<img src={require('/img/eng/29-karyotype-plot-example.png').default} width="600"/>
</p>

The karyogram indicates an increased number of copies of chromosome 21 in the sample.

7. **[Integrative Genomics Viewer (IGV)](https://software.broadinstitute.org/software/igv/)** is an embedded
module for visualization of variations on the genome. To open the module,
click on <img src={require('/img/results/igv_btn.png').default} width="45"/>.
To see simultaneously two border regions of the certain region in IGV,
click on <img src={require('/img/vv/split_to_borders.png').default} width="109"/>, and to return to the
visualization of the entire region, click on <img src={require('/img/vv/merge_borders.png').default} width="110"/>.
If you would like to go to the position opened in IGV
in <a href="https://www.deciphergenomics.org/">Decipher</a> genome
browser, click on <img src={require('/img/vv/decipher_igv.png').default} width="75"/>.
