<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Variant discovery preprocessing | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Variant discovery preprocessing | Genomenal"><meta data-react-helmet="true" name="description" content="After the &quot;Alignment&quot; task of the stage of the same name"><meta data-react-helmet="true" property="og:description" content="After the &quot;Alignment&quot; task of the stage of the same name"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.ca7df538.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.7c50b07b.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Workflow Details</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/workflow-details">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Analysis of FASTQ or BAM samples</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/workflow-details">Analysis of FASTQ or BAM samples</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/upload-identify-verify">Upload, identify and verify</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">Check quality and cleanup</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">Alignment</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">Variant discovery preprocessing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">Calculate coverage</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels">Somatic SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels">Germline SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">Structural variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">Copy number variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">Phenotype prediction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">Generate reports</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Analysis of VCF or GT samples</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="theme-doc-markdown markdown"><header><h1>Variant discovery preprocessing</h1></header><p>After the &quot;Alignment&quot; task of <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">the stage of the same name</a>
has been successfully completed, it is recommended to further preprocess the alignment results for the most
accurate variants discovery. This increases sensitivity to rare variants, contributes to a more accurate copy
number count, which depends on the number of reads, and also speeds up sample processing. Variant discovery
preprocessing is launched if at least one of the following stages is included in the sample analysis workflow:
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#somatic-snvs-discovery">somatic SNVs/Indels discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#germline-snvs-discovery">germline SNVs/Indels discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#sv-discovery">structural variations discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#cnv-discovery">copy number variations discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#enable-phenotype-prediction">phenotypes prediction</a>,
or <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#polygenic-risk-scores-calculation">polygenic risk scores calculation</a>.
If any of the tasks listed below fails, the sample analysis is stopped.</p><p align="center"><img src="/ngsw-docs-en/assets/images/variant_discovery_preprocessing-593beea548bcd967506f6851888376a6.png" width="850"></p><p>The &quot;Variant discovery preprocessing&quot; stage of sample analysis may include the following tasks:</p><ol><li><p><strong>Mark Duplicates</strong>
if <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#disable-mark-duplicates">the corresponding parameter</a> is enabled
for the sample. During sequencing, the amount of original sequences of fragmented DNA increases in the polymerase
chain reaction (PCR). For further analysis, it is very important to take into account duplicate reads of the same
fragment. If you want to keep all reads for further analysis (for example, if you are working with amplicons),
you can <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#disable-mark-duplicates">disable</a> the marking of
duplicates.<br>
Marking duplicates is performed using
the <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037052812-MarkDuplicates-Picard-" target="_blank" rel="noopener noreferrer">GATK MarkDuplicates</a> tool,
which locates and tags duplicate reads in the alignment file, where duplicate reads are defined as originating from a
single fragment of DNA. Further analysis is performed only on the original reads. The resulting file in BAM format
is sorted by coordinates and an index file is created for it. In addition, a file
with <a href="https://broadinstitute.github.io/picard/picard-metric-definitions.html#DuplicationMetrics" target="_blank" rel="noopener noreferrer">metrics</a> is created,
which indicates the number of fragments or read pairs that were marked as duplicates. This file can be downloaded
in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Mark Duplicates</em>&quot; task details (&quot;<em>Download duplication metrics TXT</em>&quot;).</p></li><li><p><strong>Base Recalibrate</strong>.
The sequencers assign a quality score to each nucleotide, which is the log likelihood of a nucleotide occurrence at a
given position compared to other nucleotides. However, many sequencers tend to underestimate sequencing quality
and make other errors. Base recalibration corrects for the systematic bias that affects the sequencer&#x27;s assignment
of quality score to nucleotides, which reduces error and improves the quality of further variant discovery.</p></li></ol><p>Base Quality Score Recalibration (BQSR) is performed in three steps:</p><ul><li><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360036898312-BaseRecalibrator" target="_blank" rel="noopener noreferrer">GATK BaseRecalibrator</a>
calculates an empirical probability of error and finds patterns in how error varies with nucleotide calling
patterns across all nucleotides. These observations are written to a recalibration table based on such covariates
as read group, reported quality score, machine cycle, and nucleotide context. BaseRecalibrator does a by-locus
traversal operating only within certain intervals at known polymorphic sites used to exclude regions around known
polymorphisms from analysis.
All reference mismatches are considered errors and indicative of poor base quality. An empirical probability of
error (p) is then calculated given the particular covariates seen at this site, where p = num mismatches / num
observations. The output file is a table containing the values of the individual covariates, number of observations
for this combination of covariates, number of reference mismatches, and the raw empirical quality score calculated
by phred-scaling the mismatch rate. The recalibration table can be downloaded in the &quot;<em>Result files</em>&quot; section in
the &quot;<em>Base Recalibrate</em>&quot; task details (&quot;<em>Download recalibration table TXT</em>&quot;).</li><li><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037055712-ApplyBQSR" target="_blank" rel="noopener noreferrer">GATK ApplyBQSR</a> applies base quality
score recalibration within certain intervals, i.e. recalibrates the base qualities of the input reads based on the
recalibration table produced in the first step, and outputs the recalibrated BAM files.</li><li><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037053552-MergeSamFiles-Picard-" target="_blank" rel="noopener noreferrer">GATK MergeSamFiles</a>
merges BAM files recalibrated within certain intervals into a single file. The resulting BAM file is sorted by
coordinates and an index file is created for it.</li></ul><p>The final sample alignment file after preprocessing can be downloaded in the &quot;<em>Result files</em>&quot; section in
the &quot;<em>Base Recalibrate</em>&quot; task details (&quot;<em>Download BAM</em>&quot;). You can also open this file in the IGV browser by
clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. The index file for the final BAM file can be downloaded there
as well (&quot;<em>Download BAI</em>&quot;).</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>If you want to add the alignment track obtained after preprocessing your sample in Genomenal to
your <a href="https://igv.org/doc/desktop/" target="_blank" rel="noopener noreferrer">desktop IGV</a>, you can do so via a link. To do this, follow these steps:</p><ol><li>Right-click the alignment file link &quot;<em>Download BAM</em>&quot; and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Upload the track via URL to your desktop IGV, as
described <a href="https://igv.org/doc/desktop/#UserGuide/loading_data/" target="_blank" rel="noopener noreferrer">here</a>.</li><li>Right-click the alignment index file link &quot;<em>Download BAI</em>&quot; and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Add the index via URL to the corresponding field in IGV.</li><li>Click &quot;<em>OK</em>&quot;. Done! The preprocessed sample alignment track is added to IGV.</li></ol></div></div><p>After the &quot;Variant discovery preprocessing&quot; stage is successfully completed, the analysis continues with
<a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">coverage calculation</a> and the following stages,
if included in the
workflow: <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#somatic-variants-discovery">&quot;Somatic SNVs/Indels discovery&quot;</a>
(for a tumor
sample), <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels#germline-variants-discovery">&quot;Germline SNVs/Indels discovery&quot;</a>
(for a single tumor sample or a non-tumor
sample), <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">&quot;Structural variations discovery&quot;</a>,
<a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">&quot;Copy number variations discovery&quot;</a> and/or
<a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">&quot;Phenotype prediction&quot;</a> (for a non-tumor sample).</p></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Alignment</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Calculate coverage Â»</div></a></div></nav></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2025 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.ca7df538.js"></script>
<script src="/ngsw-docs-en/assets/js/main.7c50b07b.js"></script>
</body>
</html>