---
sidebar_position: 5
sidebar_label: Advanced Filtering of Variants
---

# Advanced Filtering of Variants

To enter the Advanced Variant Filtering mode, click
on the <img src={require('/img/vv/advanced_filteing_btn.png').default} width="120"/> button at the top of the table.
You will see the Query Builder:

<p align="center">
<img src={require('/img/vv/query_wizard.png').default} width="700"/>
</p>

:::note
If you are on the page of [SNV Viewer with somatic mutations](/results/main/snvs-indels/variant-viewer),
then the somatic mutation filtering condition ("Origin in Somatic") will be added by default:
<p align="center">
<img src={require('/img/vv/somatic_origin.png').default} width="550"/>
</p>
In such a case, <a href="/results/main/snvs-indels/advanced-filtering#remove-condition">remove this condition</a> or
add a new one, as described <a href="/results/main/snvs-indels/advanced-filtering#add-another-condition">here</a>.
:::

:::info
On ["Profile Settings" page](/settings/profile#filtering-mode), you can adjust which filtering mode you will
see when you first open sample SNV Viewer page: [basic](/results/main/snvs-indels/basic-filtering) or advanced.
:::

:::note
Table filtering is saved when you exit the SNV Viewer page.
:::

## Filtering Columns {#filtering-columns}

The columns are divided into groups:

1. [Common](/results/main/snvs-indels/advanced-filtering#common)
2. [Per-sample data](/results/main/snvs-indels/advanced-filtering#per-sample) (if there is data on sequencing samples)
3. [Gene](/results/main/snvs-indels/advanced-filtering#per-sample) (if you have access
to [OMIM](https://www.omim.org/) database data inside Genomenal, which is provided if the
user has a license to access the database)
4. [gnomAD 3](/results/main/snvs-indels/advanced-filtering#gnomad)
5. [gnomAD 4](/results/main/snvs-indels/advanced-filtering#gnomad-4)
6. [ClinVar](/results/main/snvs-indels/advanced-filtering#clinvar)
7. [Conservation](/results/main/snvs-indels/advanced-filtering#conservation)
8. [ExAC](/results/main/snvs-indels/advanced-filtering#exac)
9. [Protein function effect](/results/main/snvs-indels/advanced-filtering#protein-function-effect)
10. [Protein function effect (additional)](/results/main/snvs-indels/advanced-filtering#protein-function-effect-additional)
11. [Other frequencies](/results/main/snvs-indels/advanced-filtering#other-frequencies)
12. [Other](/results/main/snvs-indels/advanced-filtering#other)
13. [Custom annotation sources](/results/main/snvs-indels/advanced-filtering#custom-annotations) (if at least
one custom annotation [was added](/settings/custom-annotations#add-custom-annotation) to the system before
the variants in the sample were annotated)

You can interact with column groups by
using <img src={require('/img/vv/show.png').default} height="15"/> and <img src={require('/img/vv/hide.png').default} height="10"/> to
expand and collapse the column list of the group, respectively.

Use the search box to find the filtering column. The search is case-insensitive. Some columns can be found
by search synonyms. For example, the following columns can be found by the word "change":

<p align="center">
<img src={require('/img/vv/search_example.png').default} width="240"/>
</p>

### Common {#common}

1. **Sample Status** (basic filter ["Sample Status"](/runs#sample-interpretation-status-filter)) -
filter variants by run [sample interpretation status](/runs#sample-interpretation) (completed or
uncompleted). [Single value filter](/results/main/snvs-indels/advanced-filtering#single-value-filters).
Only available in [run](/runs) SNV Viewer.
2. **Interpretation** - filter variants by interpretation (comment)
added by the user on the variant details [page](/results/main/snvs-indels/snv-details-page#add-interpretation)
or [panel](/results/main/snvs-indels/snv-details-panel#variant-interpretation).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
3. **HPO Phenotype** - filter variants by terms from [Human Phenotype Ontology (HPO)](https://hpo.jax.org/app/).
To open the phenotype search window, click on the filter value
field <img src={require('/img/vv/hpo_advanced.png').default} width="260"/>. How to find and select the
phenotypes in this form is described [here](/results/main/snvs-indels/basic-filtering#hpo).
4. **Gene panel** (basic filter ["Gene panel"](/results/main/snvs-indels/basic-filtering#gene-panel)) - filter
variants by localization in certain genes from the panel.
[Single value filter](/results/main/snvs-indels/advanced-filtering#single-value-filters).
You can add a gene panel on <a href="/settings/gene-panels">on "Gene Panels" page</a> or import a panel from
the Library of the most common clinical panels (such option is available
for [basic filter "Gene panel"](/results/main/snvs-indels/basic-filtering#gene-panel) in
basic filtering mode. If you don't have any gene panels added, the list of filter values will be empty.
5. **Region** (basic filter ["Region"](/settings/profile#basic-mode-filters)) - filter variants
by position on the genome (chromosome, start and end positions). To filter by region, click on the input
window <img src={require('/img/vv/region_filter.png').default} width="260"/> and enter the position of the
needed genomic interval, i.e. chromosome, start and end positions. Possible options of writing the interval
position:
- `chr1:161,190,824-161,193,921`
- `chr1:161190824-161193921`
- `chr1:161190824_161193921`
- `1:161190824-161193921`

You can also filter by more than one interval at once. For example,
by `chr1:161190824-161193921, chrX:151110769-151113866`.<br></br>
To complete adding the condition, press Enter or click outside edit value container.<br></br>
The filtering result will be variants whose positions intersect with the entered interval.

6. **Gene** (column ["Locus"](/results/main/snvs-indels/variant-viewer#columns-description),
basic filter ["Gene"](/results/main/snvs-indels/basic-filtering#gene)) - filter variants by localization in
gene transcript. The filter values depend on the selected filter operator:
- [any value](/results/main/snvs-indels/advanced-filtering#any-value-filters) if the operators "*starts with*",
"*contains*", "*not contains*" are selected. For example, this is useful if you want to filter variants
by genes, the name of which contains a certain letter sequence (e.g., "*HLA-*").
- [multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) if the operators “*in*” and
“*not in*” are selected. Used to filter variants by localization in certain genes. In the value options,
the gene transcript is given in brackets next to the official gene name.
- no value if the operators "*not empty*", "*empty*" are selected.
7. **Impact** - filter by the predicted effect of the variant on the
protein: <i>High</i>, <i>Moderate</i>, <i>Low</i>, <i>Modifier</i>.
Column values are described [here](/results/main/snvs-indels/variant-viewer#variant-impacts).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
8. **Genome position** - filter by variant position in the genome: in exon (<i>Exonic</i>), in intron
(<i>Intronic</i>), in the intergenic region (<i>Intergenic</i>).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
9. **Consequence** (basic filter ["Consequence"](/settings/profile#basic-mode-filters)) - filter by the
effect of the variant on genes. Column values are described
[here](/results/main/snvs-indels/variant-viewer#variant-consequences).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
10. **Pathogenicity** (column ["Pathogenicity"](/results/main/snvs-indels/variant-viewer#columns-description),
basic filter ["Pathogenicity"](/settings/profile#basic-mode-filters)) -
[multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) filter by variant pathogenicity,
which have been determined either [manually](/results/main/snvs-indels/pathogenicity#manually-determined-pathogenicity),
[by ACMG criteria](/results/main/snvs-indels/pathogenicity#define-pathogenicity-on-acmg-tab), or
[according to the pathogenicity base](/results/main/snvs-indels/pathogenicity#define-pathogenicity-by-base).
11. **Label** (column ["Label"](/results/main/snvs-indels/variant-viewer#label), basic
filter ["Label"](/settings/profile#basic-mode-filters)) -
[multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) filter
by [variant significance defined by the user](/results/main/snvs-indels/variant-viewer#label): <i>High significance</i>, <i>Moderate significance</i>, <i>Low significance</i>, <i>Sequencing error</i>, <i>Undefined</i>.
12. **Phasing group** (column ["PG"](/results/main/snvs-indels/variant-viewer#added-columns)) -
[multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) filter
by [phasing group](/results/main/snvs-indels/variant-viewer#added-columns) status:
- <i>Phasing group</i> - merging of homozygous variants located within a specific genomic interval;
- <i>Part of group consequence equal</i> - part of phasing group with concordant amino acid substitutions;
- <i>Part of group consequence unequal</i> - part of phasing group with discordant amino acid substitutions.
13. **Compound heterozygote (Genotype)** - filter variants by whether they are compound heterozygotes (i.e.
heterozygosity of two or more variants
of the same gene; all cases are described in detail [here](/results/main/snvs-indels/advanced-filtering#compound)).
Unlike [the compound heterozygote filtering mode](/results/main/snvs-indels/advanced-filtering#compound), this
filter is applied simultaneously with other filtering conditions. As a result, the variants included
in the compound heterozygote will be found, but the remaining variants from this compound heterozygote may
be filtered out by other conditions.

In [family analysis](/group-analysis) SNV Viewer and in SNV Viewer with one sequencing sample,
to filter variants by compound heterozygote, select the filter in the list of filtering columns:

<p align="center">
<img src={require('/img/vv/compound_filter_for_one.png').default} width="700"/>
</p>

In [germline cohort analysis](/group-analysis) SNV Viewer, in SNV Viewer with several sequencing samples
and in [run](/runs) SNV Viewer (if there is data on sequencing samples), to filter variants by compound
heterozygote, select a sample in which all cases will be found where at least two heterozygotes (0/1, 1/0, 0|1, etc.)
are located in the same gene:

<p align="center">
<img src={require('/img/vv/sample_compound.png').default} width="700"/>
</p>

14. **Trio GT (Genotype)** - [multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) filter by
variant inheritance: <i>De novo variants</i>, <i>Newly formed homozygotes</i> (homozygotes in proband that are
heterozygous in parents) or <i>Single parent-inherited</i>. Only available
in [family group analysis](/group-analysis#group-analysis-variant-viewer) for Trio case
(Proband, Father and Mother).
15. **Chromosome** (column ["Position"](/results/main/snvs-indels/variant-viewer#columns-description))- filter
variants by localization in chromosomes: chr1-chr22,
chrX, chrY, chrM, chromosomes like chrN\_random (contigs located in the N chromosome but could not be reliably
ordered within the sequence, or sequences for alternative haplotypes), chromosomes like chrUn_ (contigs that
cannot be confidently placed on a specific chromosome), chrEBV.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
16. **rsId** (column ["External links"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
variants by ID from [dbSNP](https://www.ncbi.nlm.nih.gov/snp/) database.
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters). You can use as a value either
the full ID (e.g. rs1126497) or the ID digits only (e.g. 1126497).
17. **ClinVar ID** (column ["External links"](/results/main/snvs-indels/variant-viewer#columns-description)) -
filter variants by Variation ID from [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) database (e.g. 183700).
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
18. **Origin** (basic filter ["Origin"](/results/main/snvs-indels/basic-filtering#origin)) - filter variants
by mutation type: <i>Somatic</i>, <i>Germline</i>.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
19. **Start** (column ["Position"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
by the start position of the variant on the chromosome.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
20. **End** - filter by the end position of the variant on the chromosome.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
21. **Reference allele** (column ["Alteration"](/results/main/snvs-indels/variant-viewer#columns-description)) -
filter by reference allele of the variant.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
22. **Alternative allele** (column ["Alteration"](/results/main/snvs-indels/variant-viewer#columns-description)) -
filter by alternative allele of the variant.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
23. **HGVSc** - filter variants by nucleotide substitution using [the HGVS notation](https://varnomen.hgvs.org/):
“c.” (coding; for a substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding
sequence) prefix + genomic position of the substituted nucleotide + reference allele > alternative allele.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
24. **HGVSp (short)** (column ["HGVSp"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
variants by amino acid substitution using [the HGVS notation](https://varnomen.hgvs.org/) in the single-letter
amino acid code (“p.” prefix (protein) + reference amino acid + amino acid position in protein + new amino acid
resulting from the substitution). [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
e.g.: p.P72R.
25. **HGVSp (long)** (column ["HGVSp"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
variants by amino acid substitution using [the HGVS notation](https://varnomen.hgvs.org/) in the three-letter
amino acid code. Written with transcript: Ensembl transcript ID + “p.” prefix (protein) + reference amino acid +
amino acid position in protein + new amino acid resulting from the substitution.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters). e.g.: ENSP00000269305.4:p.Pro72Arg.
26. **Exon** - filter by exon number in the transcript that the variant affected.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
27. **Onco Relevance** - filter by the variant significance for any oncological
disease: <i>Relevant</i>, <i>Irrelevant</i>.
[Single value filter](/results/main/snvs-indels/advanced-filtering#single-value-filters).
28. **Caller Filters** - [multiselect](/results/main/snvs-indels/advanced-filtering#multiselect-filters) filter
by variant quality (GATK filters).
29. **Occurrence in Run** - filter variants by [occurrence](/runs#occurrence-in-run) in run samples.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
Only available in [run](/runs) SNV Viewer and run sample SNV Viewer.
30. **Cosmic ID** - filter by variant ID in [COSMIC](https://cancer.sanger.ac.uk/cosmic/) database
(e.g. COSV52665487). Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
Only available if COSMIC
database [has been uploaded as a custom annotation](/settings/custom-annotations#define-annotation-type) before
the sample variant annotation is complete.
31. **RUSeq AF** - filter variants by alternative allele frequency from [RUSeq](http://ruseq.ru/#/) database.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
Only available if RUSeq
database [has been uploaded as a custom annotation](/settings/custom-annotations#define-annotation-type) before
the sample variant annotation is complete.

### Per-sample data {#per-sample}

- The filtering column group is only present if there is sequencing sample data.

1. **Genotype** (column ["GT"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter variants
by genotype, i.e. reference and alternative allele values for the sample. Genotypes are filtered ignoring phasing
information. To add a condition for filtering variants by genotype, do the following:<br></br>1. Select a
sample to filter:
- "<i>Some</i>": any sample has the required genotype;
- "<i>Every</i>": each sample has the required genotype;
- Sample name.

<p align="center">
<img src={require('/img/vv/gt_samples.png').default} width="650"/>
</p>

2. Select a filter operator:

<p align="center">
<img src={require('/img/vv/gt_opertators.png').default} width="650"/>
</p>

3. Select the required filter value by clicking on the value row:
- "<i>REF</i>": only reference allele at the site, (e.g., `0/0`, `0|0`, `0/0/0`);
- "<i>ALT</i>": there is at least one alternative allele at the site (e.g., `1/1`, `1|1`, `0/1`, `1/0`,
`0|1`, `1|0`, `1/.`, `1|.`, `./1`, `.|1`, `1`, `0/0/1`, `0/2`, `1/2`);
- "<i>HOM</i>": homozygous site, can be as in reference or alternative (e.g., `1|1`, `1/1`, `1/1/1`, `0/0`, `0|0`);
- "<i>HET</i>": heterozygous site, containing at least two different alleles (e.g., `0/1`, `1/0`, `0|1`,
`1|0`, `2/0`, `0/1/2`);
- "<i>HOM_ALT</i>": homozygous site in alternative (e.g., `1/1`, `1|1|1`);
- "<i>MIS</i>": information is missing for the site (e.g., `./.`).

1. **Read depth** (column ["Depth (Alt/Ref)"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
variants by sequencing depth, i.e. the total number of reads of the sequence overlapping the variant position for
this sample.
2. **Ref count** - filter variants by the number of times the reference nucleotide was read
in the sequence for this sample.
3. **Alt count** - filter variants by the number of times the alternative nucleotide was read
in the sequence for this sample.
4. **AF** - filter variants by alternative allele frequency for this sample.

To add a condition for filtering variants by depth, ref, alt or AF, do the following:

1. Select a filter function:
- "<i>Max</i>": maximum value;
- "<i>Min</i>": minimum value;
- "<i>Avg</i>": mean;
- "<i>Sum</i>": total value.

<p align="center">
<img src={require('/img/vv/function.png').default} width="650"/>
</p>

2. Select a filter operator:

<p align="center">
<img src={require('/img/vv/sample_data_operator.png').default} width="650"/>
</p>

3. If a mathematical operator was selected as the filter operator, click on input
box <img src={require('/img/vv/sample_data_value.png').default} width="150"/> and enter the filter threshold
(natural number for filters by depth, ref and alt count, and positive rational number for AF).
To complete adding the condition, press Enter or click outside edit value container.

### Gene {#gene}

1. **RefSeq Transcript** - filter variants by transcript ID in [RefSeq](https://www.ncbi.nlm.nih.gov/nuccore/)
database (e.g. NM_002354.3). [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
2. **OMIM ID** - filter variants by gene ID in [OMIM](https://www.omim.org/) database (e.g. 185535).
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
3. **OMIM Phenotypes** - filter by the column values
of [the table with OMIM phenotypes associated with the gene](/results/main/snvs-indels/snv-details-panel#omim-phenotypes)
in which the variant is located. [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
Only available if you have access to [OMIM](https://www.omim.org/) database data inside Genomenal, which is provided
if the user has a license to access the database.
4. **NCBI ID** - filter variants by gene ID in [NCBI Gene](https://www.ncbi.nlm.nih.gov/gene/) database
(e.g. 4072). Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
5. **Swiss-Prot ID** - filter variants by gene ID
in [UniProtKB Swiss-Prot](https://www.uniprot.org/uniprotkb/?facets=reviewed%3Atrue&query=%2A) database (e.g. P16422).
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
6. **TrEMBL ID** - filter variants by gene ID
in [UniProtKB TrEMBL](https://www.uniprot.org/uniprotkb?query=reviewed:false) database (e.g. B5MCA4).
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
7. **Ensembl ID** - filter variants by gene ID in [Ensembl](https://www.ensembl.org/Homo_sapiens/Info/Index)
database (e.g. ENSG00000119888). Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
8. **pLI** - filter variants by pLI (Probability of Loss-of-Function Intolerance) - the probability that a transcript
is intolerant to loss-of-function (LoF) variants. Range: 0 to 1. Genes with pLI ≥ 0.9 are considered LoF-intolerant.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
9. **LOEUF** - filter variants by LOEUF (Loss-of-function Observed/Expected Upper bound Fraction) - the upper bound
of the 90% confidence interval for the observed/expected ratio of high-confidence pLoF (predicted loss-of-function)
variants. Lower values (LOEUF < 0.35) indicate a gene is more constrained against LoF variants.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
10. **Missense O/E ratio** - filter variants by the upper bound of the 90% confidence interval for the
observed/expected ratio of missense variants. Lower values suggest a gene is more constrained against missense
variation. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
11. **Missense Z-score** - filter variants by a Z-score representing the deviation of the observed number of missense
variants in a transcript from the expected number under a model of neutral mutation distribution.
Greater positive values (Z-score ≥ 3.09) indicate that the transcript is more intolerant of missense variation.
Lower or negative Z-scores indicate that the gene tolerates more missense mutations than expected.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (rational number).

### gnomAD 3 {#gnomad}

Filter columns with the following data
from [gnomAD v3](https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/) database:
1. **AF** (column ["gnomAD 3 AF"](/results/main/snvs-indels/variant-viewer#columns-description)) - filter
variants by the total allele frequency
(AF). [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **AC** - filter variants by the total alternative allele count in high quality genotypes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
3. **AN** - filter variants by the total number of called high quality genotypes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
4. **XY AF** - filter variants by allele frequency (AF) in the XY subpopulation.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
5. **XX AF** - filter variants by allele frequency (AF) in the XX subpopulation.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
6. **Number of Homozygotes** - filter variants by the number of homozygotes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
7. **[Population] AF** - filter variants by total allele frequency (AF) in different populations,
where **[Population]** is Ashkenazi Jewish, Amish, African and Afr. Am., Finnish, South Asian, Latino,
Non-finnish european, East Asian, Middle Eastern, Other.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
8. **[Population] XY/XX samples AF** - filter variants by allele frequencies (AF) in XY/XX subpopulations in different
populations, where **[Population]** is Ashkenazi Jewish, Amish, African and African American, Finnish, South Asian,
Latino, Non-finnish european, East Asian, Middle Eastern, Other.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
9. **[Population] Number of Homozygotes** - filter variants by the number of homozygotes in different
populations, where **[Population]** is Ashkenazi Jewish, Amish, African and African American, Finnish, South Asian,
Latino, Non-finnish european, East Asian, Middle Eastern, Other.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
10. **Coverage** - filter variants by average coverage depth by bases: <i>&lt;10</i>, <i>10-100</i>, <i>⩾100</i>.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).

### gnomAD 4 {#gnomad-4}

Filter columns with the following data from [gnomAD v4](https://gnomad.broadinstitute.org/) database:
1. **AF** - filter variants by the total alternative allele frequency (AF) in high quality genotypes in exomes
and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **AC** - filter variants by the total alternative allele count in exomes and genomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
3. **AN** - filter variants by the total number of called high quality genotypes in exomes and genomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
4. **Number of Homozygotes** - filter variants by the total number of individuals homozygous for alternative allele
in exomes and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
5. **XY Number of Homozygotes** - filter variants by the total number of individuals homozygous for alternative allele
in the XY subpopulation in exomes and
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
6. **XX Number of Homozygotes** - filter variants by the total number of individuals homozygous for alternative allele
in the XX subpopulation in exomes and
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
7. **XY AF** - filter variants by the total alternative allele frequency in the XY subpopulation in exomes
and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
8. **XY AC** - filter variants by the total alternative allele count in high quality genotypes in the XY
subpopulation in exomes and
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
9. **XY AN** - filter variants by the total number of called high quality genotypes in the XY subpopulation
in exomes and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
10. **XX AF** - filter variants by the total alternative allele frequency in the XX subpopulation in exomes
and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
11. **XX AC** - filter variants by the total alternative allele count in high quality genotypes in the XX
subpopulation in exomes and
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
12. **XX AN** - filter variants by the total number of called high quality genotypes in the XX subpopulation
in exomes and genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
13. **Exomes AF** - filter variants by the total alternative allele frequency (AF) in high quality genotypes in exomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
14. **Exomes AC** - filter variants by the total alternative allele count in exomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
15. **Exomes AN** - filter variants by the total number of called high quality genotypes in exomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
16. **Exomes Number of Homozygotes** - filter variants by the total number of individuals homozygous for alternative
allele in exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
17. **Exomes XY Number of Homozygotes** - filter variants by the total number of individuals homozygous for
alternative allele in the XY subpopulation
in exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
18. **Exomes XX Number of Homozygotes** - filter variants by the total number of individuals homozygous for
alternative allele in the XX subpopulation in
exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
19. **Exomes XY AF** - filter variants by the total alternative allele frequency in the XY subpopulation in
exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
20. **Exomes XY AC** - filter variants by the total alternative allele count in high quality genotypes in the XY
subpopulation in
exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
21. **Exomes XY AN** - filter variants by the total number of called high quality genotypes in the XY subpopulation
in exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
22. **Exomes XX AF** - filter variants by the total alternative allele frequency in the XX subpopulation in
exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
23. **Exomes XX AC** - filter variants by the total alternative allele count in high quality genotypes in the XX
subpopulation in
exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
24. **Exomes XX AN** - filter variants by the total number of called high quality genotypes in the XX subpopulation
in exomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
25. **Genomes AF** - filter variants by the total alternative allele frequency (AF) in high quality genotypes in
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
26. **Genomes AC** - filter variants by the total alternative allele count in genomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
27. **Genomes AN** - filter variants by the total number of called high quality genotypes in genomes.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
28. **Genomes Number of Homozygotes** - filter variants by the total number of individuals homozygous for alternative
allele in genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
29. **Genomes XY Number of Homozygotes** - filter variants by the total number of individuals homozygous for
alternative allele in the XY subpopulation
in genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
30. **Genomes XX Number of Homozygotes** - filter variants by the total number of individuals homozygous for
alternative allele in the XX subpopulation
in genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
31. **Genomes XY AF** - filter variants by the total alternative allele count in high quality genotypes in the XY
subpopulation in
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
32. **Genomes XY AC** - filter variants by the total alternative allele count in high quality genotypes in the XY
subpopulation in
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
33. **Genomes XY AN** - filter variants by the total number of called high quality genotypes in the XY subpopulation
in genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
34. **Genomes XX AF** - filter variants by the total alternative allele count in high quality genotypes in the XX
subpopulation in
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
35. **Genomes XX AC** - filter variants by the total alternative allele count in high quality genotypes in the XX
subpopulation in
genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
36. **Genomes XX AN** - filter variants by the total number of called high quality genotypes in the XX subpopulation
in genomes. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).

### ClinVar {#clinvar}

Filter columns with the following data from [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) database:
1. **Clinical significance** (column ["ClinVar"](/results/main/snvs-indels/variant-viewer#added-columns),
basic filter ["ClinVar"](/settings/profile#basic-mode-filters)) - filter variants by the clinical
significance (you can find the description of
values [here](https://preview.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv)).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
2. **Allele ID** - filter variants by allelic substitution ID (e.g. 405185).
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).
3. **Phenotypes** - filter variants by the phenotypic significance.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
4. **Phenotypes cross references** - filter variants by phenotype references in various databases
([Orphanet](https://www.orpha.net/consor/cgi-bin/index.php),
[MedGen](https://www.ncbi.nlm.nih.gov/medgen/), [MONDO](https://monarchinitiative.org/),
[OMIM](https://www.omim.org/), etc.)
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
5. **Haplotype phenotypes** - filter by the phenotypic significance of the variant that is part
of the haplotype. [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
6. **Haplotype phenotypes cross references** - filter by references to the phenotype of a variant
that is part of the haplotype in various databases ([Orphanet](https://www.orpha.net/consor/cgi-bin/index.php),
[MedGen](https://www.ncbi.nlm.nih.gov/medgen/), [MONDO](https://monarchinitiative.org/),
[OMIM](https://www.omim.org/), etc.) [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
7. **Review status** - filter variants by the summary score of the data source, in which the clinical
significance of the phenotype is
declared. [Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
Possible value options and their descriptions can be
found [here](https://preview.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status).
8. **Cross references** - filter variants by references in various databases
([ClinGen](https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/landing),
[UniProtKB](https://www.uniprot.org/uniprotkb/), [OMIM](https://www.omim.org/),
[GTR](https://www.ncbi.nlm.nih.gov/gtr/), etc.)
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
9. **Allele origin** - filter variants by allele origin.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
10. **Variant Haplotype ID** - filter by ID of the variant (Variation ID) that is part of the haplotype.
Filter [with ID input](/results/main/snvs-indels/advanced-filtering#id-filters).

### Conservation {#conservation}

1. **Ancestral allele** - filter variants by ancestral allele predicted by
[EPO](https://www.ensembl.org/info/genome/compara/epo_anchors_info.html).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
2. **Altai Neanderthal** - filter variants by Altai Neanderthal genotype.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
3. **Denisova** - filter variants by Denisova genotype.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
4. **Vindija Neanderthal** - filter variants by Vindija Neanderthal genotype.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
5. **\[Program] \[Score]** - filter variants by conservation scores obtained using prediction
by various programs, where **[Program]** is a prediction program: [FitCons](http://compgen.cshl.edu/fitCons/),
[GERP++](http://mendel.stanford.edu/sidowlab/downloads/gerp/index.html),
[PhyloP](https://ccg.epfl.ch/mga/hg19/phylop/phylop.html),
[PhastCons](http://compgen.cshl.edu/phast/phastCons-HOWTO.html),
[SiPhy](http://portals.broadinstitute.org/genome_bio/siphy/documentation.html), BStatistic,
and **[Score]** is a conservation score: rankscore, score, neutral rate.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
6. **[Score] confidence value** - filter variants by confidence value of various conservation
scores: Highly significant; Significant; Informative; Other,
where **[Score]** is a conservation score: Integrated, GM12878, H1-hESC, HUVEC.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
7. **[Distribution]** - filter variants by stationary allele distribution in
the site, estimated using [SiPhy](http://portals.broadinstitute.org/genome_bio/siphy/documentation.html)
algorithm, where **[Distribution]** is a distribution of alleles A, G, C or T: pA, pG, pC, pT.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).

### ExAC {#exac}

Filter columns with the following data from [ExAC](http://exac.broadinstitute.org/) database:
1. **AF** - filter variants by the total allele frequency (AF).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **Allele count** - filter variants by alternative allele count.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
3. **Adjusted AF** - filter variants by allele frequency (AF) in a population with genotype quality ≥20 and
depth ≥10. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
4. **Adjusted allele count** - filter variants by alternative allele count in a population with genotype
quality ≥20 and depth ≥10. [Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters)
(natural number).
5. **[Population] AF** - filter variants by total allele frequency (AF)
in different populations, where **[Population]** is African & African American, American, East Asian,
Finnish, Non-Finnish European, South Asian.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
6. **[Population] allele count** - filter variants by alternative allele count in different populations,
where **[Population]** is African & African American,
American, East Asian, Finnish, Non-Finnish European, South Asian.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
7. **[Cohort] AF** - filter variants by total allele frequency (AF) in different cohorts,
where **[Cohort]** is Non-TCGA, Non-Psych.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
8. **[Cohort] Allele count** - filter variants by alternative allele count in different cohorts,
where **[Cohort]** is Non-TCGA, Non-Psych.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
9. **[Cohort] adjusted AF** - filter variants by allele frequency (AF) in
a population with genotype quality ≥20 and depth ≥10 in different cohorts in
[ExAC](http://exac.broadinstitute.org/) database, where **[Cohort]** is Non-TCGA, Non-Psych.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
10. **[Cohort] adjusted allele count** - filter variants by alternative allele count in
a population with genotype quality ≥20 and depth ≥10 in different cohorts, where **[Cohort]** is Non-TCGA, Non-Psych.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
11. **\[Cohort] \[Population] AF** - filter variants by total allele frequency (AF) in different populations in
different cohorts, where **[Cohort]** is Non-TCGA, Non-Psych, and **[Population]** is African & African American,
American, East Asian, Finnish, Non-Finnish European, South Asian.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
12. **\[Cohort] \[Population] allele count** - filter variants by alternative allele count in different populations in
different cohorts,where **[Cohort]** is Non-TCGA, Non-Psych, and **[Population]** is
African & African American, American, East Asian, Finnish, Non-Finnish European, South Asian.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).

### Protein function effect {#protein-function-effect}

1. **\[Algorithm] \[Score]** - filter variants by prediction scores of the effect of amino acid
substitution on protein function, obtained using various algorithms and programs,
where **[Algorithm]** is a prediction algorithm: [SIFT](https://sift.bii.a-star.edu.sg/),
[Polyphen](http://genetics.bwh.harvard.edu/pph/pph_help.html),
[SIFT4G](https://sift.bii.a-star.edu.sg/sift4g/), LRT,
[FAtHMM](http://fathmm.biocompute.org.uk/), [Mutation Accessor](http://mutationassessor.org/r3/),
and **[Score]** is a prediction score: rankscore, score, omega, p-value.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **[Algorithm] prediction** - filter variants by the results of predicting
the effect of amino acid substitution on protein function, obtained using various algorithms and programs,
where **[Algorithm]** is a prediction algorithm: [Polyphen](http://genetics.bwh.harvard.edu/pph/pph_help.html),
[SIFT4G](https://sift.bii.a-star.edu.sg/sift4g/), LRT, [FAtHMM](http://fathmm.biocompute.org.uk/),
[Mutation Accessor](http://mutationassessor.org/r3/).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
3. **SpliceAI Delta score** - filter by the probability of the variant being splice-altering at any position
within a given interval (+/-50 bp between the variant and gained/lost splice site by default), predicted
[SpliceAI](https://spliceailookup.broadinstitute.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
4. **SpliceAI Delta score (acceptor gain)** - filter by the probability of the variant being splice acceptor
at any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default),
predicted [SpliceAI](https://spliceailookup.broadinstitute.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
5. **SpliceAI Delta score (acceptor loss)** - filter by the probability of the variant not to be splice acceptor
at any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default),
predicted [SpliceAI](https://spliceailookup.broadinstitute.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
6. **SpliceAI Delta score (donor gain)** - filter by the probability of the variant being splice donor
at any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default),
predicted [SpliceAI](https://spliceailookup.broadinstitute.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
7. **SpliceAI Delta score (donor loss)** - filter by the probability of the variant not to be splice donor
at any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default),
predicted [SpliceAI](https://spliceailookup.broadinstitute.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
8. **SpliceAI Delta position** - filter by the location (in bp) where splicing changes relative to the variant
position (positive values are downstream of the variant, negative values are upstream).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
9. **SpliceAI Delta position (acceptor gain)** - filter by the location (in bp) of splice acceptor relative to
the variant position (positive values are downstream of the variant, negative values are upstream).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
10. **SpliceAI Delta position (acceptor loss)** - filter by the location (in bp) which is not a splice acceptor
relative to the variant position (positive values are downstream of the variant, negative values are upstream).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
11. **SpliceAI Delta position (donor gain)** - filter by the location (in bp) of splice donor relative to
the variant position (positive values are downstream of the variant, negative values are upstream).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
12. **SpliceAI Delta position (donor loss)** - filter by the location (in bp) which is not a splice donor
relative to the variant position (positive values are downstream of the variant, negative values are upstream).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (integer number).
13. **CADD raw** - filter by the extent to which the annotation profile for a variant suggests that the variant
is likely to be "observed" (negative values) vs "simulated" (positive values),
calculated by [CADD](https://cadd.gs.washington.edu/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
14. **CADD phred** - filter by Phred-scaled score of the deleteriousness of a variant,
calculated by [CADD](https://cadd.gs.washington.edu/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
15. **MaxEntScan Alt score** - filter by score for alternative allele predicting the loss of a
native splice site for a variant, calculated
by [MaxEntScan](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
16. **MaxEntScan Ref score** - filter by score for reference allele predicting the loss of a
native splice site for a variant, calculated
by [MaxEntScan](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).
17. **MaxEntScan diff** - filter by the difference between the scores for alternative and reference
alleles predicting the loss of a native splice site for a variant, calculated
by [MaxEntScan](http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (any number).

### Protein function effect (additional) {#protein-function-effect-additional}

1. **\[Algorithm] \[Score]** - filter variants by prediction scores of the effect of amino acid
substitution on protein function, obtained using various algorithms and programs,
where **[Algorithm]** is a prediction algorithm: [BayesDel](http://bjfenglab.org/BayesDel) addAF,
[BayesDel](http://bjfenglab.org/BayesDel) noAF, [Provean](http://provean.jcvi.org/index.php),
[Meta SVM](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5270233/), Meta LR,
[M-CAP](http://bejerano.stanford.edu/mcap/), [MutPred](http://mutpred.mutdb.org/), MVP, MPC,
[Primate AI](https://illumina.github.io/NirvanaDocumentation/3.14/data-sources/primate-ai/), DEOGEN2,
[DANN](https://cbcl.ics.uci.edu/public_data/DANN/),
[FAtHMM-MKL](http://fathmm.biocompute.org.uk/), [FAtHMM-XF](http://fathmm.biocompute.org.uk/), Eigen,
and **[Score]** is a prediction score: rankscore, score, p-value.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **[Algorithm] prediction** - filter variants by the results of predicting
the effect of amino acid substitution on protein function, obtained using various algorithms and programs,
where **[Algorithm]** is a prediction algorithm: [BayesDel](http://bjfenglab.org/BayesDel) addAF,
[BayesDel](http://bjfenglab.org/BayesDel) noAF, [Provean](http://provean.jcvi.org/index.php),
[Meta SVM](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5270233/),
Meta LR, [M-CAP](http://bejerano.stanford.edu/mcap/),
[Primate AI](https://illumina.github.io/NirvanaDocumentation/3.14/data-sources/primate-ai/),
[FAtHMM-MK](http://fathmm.biocompute.org.uk/), [FAtHMM-XF](http://fathmm.biocompute.org.uk/).
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
3. **Transcript ID**: filter variants by transcript ID in [UniProtKB](https://www.ncbi.nlm.nih.gov/nuccore/) database
(e.g. P16422) for which the effect of amino acid
substitution on protein function has been predicted by [MutPred](http://mutpred.mutdb.org/).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
4. **Amino acid change** - filter variants by amino acid substitution whose
effect on protein function was predicted by [MutPred](http://mutpred.mutdb.org/).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters). e.g.: G35r.
5. **Top 5 features** - filter variants by the top five substitution effects
on protein function predicted by [MutPred](http://mutpred.mutdb.org/).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).

### Other frequencies {#other-frequencies}

1. **1000G Allele frequency** - filter variants by total allele frequency in
[the 1000 Genomes Project](https://www.internationalgenome.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
2. **1000G [Population] AF** - filter variants by total allele frequency (AF) in different populations
in [the 1000 Genomes Project](https://www.internationalgenome.org/),
where **[Population]** is East Asian, European, African, American, South Asian,
African American, European American.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
3. **UK10K allele frequency** - filter variants by total allele frequency in combined genotypes
in the UK10K cohort from [the UK10K project](https://www.uk10k.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
4. **UK10K allele count** - filter variants by alternative allele count in combined genotypes
in the UK10K cohort from [the UK10K project](https://www.uk10k.org/).
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
5. **UK10K [Cohort] AF** - filter variants by total allele frequency (AF) in combined genotypes
in different cohorts from [the UK10K project](https://www.uk10k.org/),
where **[Cohort]** is TwinsUK, ALSPAC.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
6. **UK10K [Cohort] allele count** - filter variants by alternative allele count in combined genotypes
in different cohorts from [the UK10K project](https://www.uk10k.org/),
where **[Cohort]** is TwinsUK, ALSPAC.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
7. **ESP [Population] AF** - filter variants by total allele frequency (AF) in different populations
in [NHLBI GO Exome Sequencing Project (ESP)](https://esp.gs.washington.edu/drupal/),
where **[Population]** is African American, European American.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (positive rational number).
8. **[Population] allele count** - filter variants by alternative allele count in different
populations from [NHLBI GO Exome Sequencing Project (ESP)](https://esp.gs.washington.edu/drupal/), where
**[Population]** is African American, European American.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).

### Other {#other}

1. **Coding sequence position** - filter variants by coding sequence position.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
2. **Protein position** - filter variants by protein position.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
3. **Amino acids** - filter variants by reference amino acid.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
4. **Codons** - filter variants by nucleotide substitution written as codons.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters). e.g.: GAA/GAG.
5. **Amino acid position** - filter variants by amino acid position.
[Numeric filter](/results/main/snvs-indels/advanced-filtering#numeric-filters) (natural number).
6. **APPRIS annotation** - filter variants by human splicing isoforms
([APPRIS](http://appris.bioinfo.cnio.es) annotation): Principal 1-5; Alternative 1-2.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters).
7. **Geuvadis EQTL target gene** - filter variants by gene from eQTL data analysis in
[GEUVADIS](http://www.geuvadis.org/).
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).
8. **ENIGMA clinical significance** - filter by clinical significance of the variant located
in the known or suspected breast and/or ovarian predisposition genes (such as *BRCA1*, *BRCA2*)
determined by [ENIGMA](https://enigmaconsortium.org/) consortium.
[Multiselect filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters): <i>Pathogenic</i>, <i>Likely pathogenic</i>, <i>Benign</i>, <i>Likely benign</i>, <i>Not provided</i>.
9. **ENIGMA clinical significance comment** - filter by comment on clinical significance
of the variant located in the known or suspected breast and/or ovarian predisposition genes (such
as *BRCA1*, *BRCA2*) determined by [ENIGMA](https://enigmaconsortium.org/) consortium.
[Any value filter](/results/main/snvs-indels/advanced-filtering#any-value-filters).

### Custom annotation sources {#custom-annotations}

A filtering column group is present if at least one custom
annotation [was added](/settings/custom-annotations#add-custom-annotation) to the system before the sample
variants were annotated. Filtering column names look like this: [custom annotation name]: [annotation column name].
To add a condition for filtering variants by column from a custom annotation, do the following:

1. Select a filter operator. If the annotation column type is text, then the operators will be the following:

<p align="center">
<img src={require('/img/vv/custom_filter_text.png').default} width="750"/>
</p>

If the annotation column type is numeric, then the operators will be the following:

<p align="center">
<img src={require('/img/vv/custom_filter.png').default} width="750"/>
</p>

2. If operators other than "<i>not empty</i>" and "<i>empty</i>" have been selected, click on the input
box <img src={require('/img/vv/sample_data_value.png').default} width="150"/> and enter the filter column value.
To complete adding the condition, press Enter or click outside edit value container.

## Compound Heterozygote Filtering Mode {#compound}

A compound heterozygote is a genetic condition in which two or more alleles of the same gene carry different
mutations in a patient. This condition is observed if the patient inherits a mutant allele from each parent
or inherits one mutation from one of the parents, and the second occurs *de novo*. Compound heterozygotes are
common in autosomal recessive diseases.

If there is data about sequencing samples, a button for filtering by compound heterozygote appears in the query
builder:

<p align="center">
<img src={require('/img/vv/compound.png').default} width="800"/>
</p>

**Filtering by compound heterozygote depends on SNV Viewer type:**

**SNV Viewer of [family trio analysis](/group-analysis) (Proband, Father, Mother):** the proband has at
least two heterozygotes in one gene (0/1, 1/0, 0|1, etc.) with the following inheritance options:
- at least one of the heterozygotes is inherited from the father, and at least one is inherited from the mother:

<p align="center">
<img src={require('/img/vv/compound_trio.png').default} width="1000"/>
</p>

- at least one of the heterozygotes occurred *de novo* (i.e the father and mother have homozygous reference 0/0),
and the second is inherited from the father or from the mother:

<p align="center">
<img src={require('/img/vv/from_dad_and_de_novo.png').default} width="1000"/>
</p>

To apply a filter by compound heterozygote, click the
button <img src={require('/img/vv/compound_btn.png').default} width="230"/>. As a result, there will be found all
cases where at least two heterozygotes are located in the same gene in the proband, and in the parents they are
either heterozygous, or homozygous reference, or there is no information about the site (./.). To turn off the
filter in the query builder, click the button <img src={require('/img/vv/compound_btn_on.png').default} width="230"/>.

**SNV Viewer of [family duo analysis](/group-analysis) (Father and Mother):** cases when in one gene at
least one heterozygote (0/1, 1/0, 0|1, etc.) is in the mother and at least one is in the father:

<p align="center">
<img src={require('/img/vv/compound_duo_example.png').default} width="1000"/>
</p>

To apply a filter by compound heterozygote, click the
button <img src={require('/img/vv/compound_btn.png').default} width="230"/>, and to turn off the
filter in the query builder, click the button <img src={require('/img/vv/compound_btn_on.png').default} width="230"/>.

**SNV Viewer of [family duo analysis](/group-analysis) (Proband, Parent (Father or Mother)):** the proband
has at least two heterozygotes in one gene (0/1, 1/0, 0|1, etc.) with the following inheritance options:
- at least one heterozygote is inherited, and the second is not inherited from the parent (i.e. the parent
has homozygous reference 0/0):

<p align="center">
<img src={require('/img/vv/from_mom_plus_de_novo.png').default} width="1000"/>
</p>

- both are not inherited from the parent (i.e. the parent has homozygous reference 0/0):

<p align="center">
<img src={require('/img/vv/de_novo.png').default} width="1000"/>
</p>

To apply a filter by compound heterozygote, click the
button <img src={require('/img/vv/compound_btn.png').default} width="230"/>, and to turn off the
filter in the query builder, click the button <img src={require('/img/vv/compound_btn_on.png').default} width="230"/>.

**SNV Viewer with one sequencing sample (e.g. for a single sample of tumor or healthy tissue):** there will
be found all cases where at least two heterozygotes (0/1, 1/0, 0|1, etc.) are located in the same gene.
To apply a filter by compound heterozygote, click the
button <img src={require('/img/vv/compound_btn.png').default} width="230"/>, and to turn off the
filter in the query builder, click the button <img src={require('/img/vv/compound_btn_on.png').default} width="230"/>.

**SNV Viewer of [germinal cohort analysis](/group-analysis) or SNV Viewer with several sequencing samples (e.g. for a tumor/control sample pair), or [run](/runs) SNV Viewer (if there is data on sequencing samples):** by
clicking on the compound heterozygote filter button, you will see a drop-down sample list:

<p align="center">
<img src={require('/img/vv/compound_options.png').default} width="300"/>
</p>

When one of the samples is selected, the variants will be filtered in such a way that in this sample there
will be at least two heterozygotes in one gene (0/1, 1/0, 0|1, etc.) To turn off such a filter in the query
builder, you need to click on the compound heterozygote filter button and select "*Turn off*" option.

**Applying of the compound heterozygote filtering mode with other filtering conditions:**

When applying the compound heterozygote filtering mode together with other filtering conditions, the condition
for compound heterozygote is already applied to those variants that fit the other conditions:

<p align="center">
<img src={require('/img/vv/compond_and_other_filter.png').default} width="1000"/>
</p>

This fundamentally differs the filtering mode from "*Compound heterozygote (Genotype)*" filter, which is applied
simultaneously with other filtering conditions. As a result of applying the filter, there will be found
variants that are included in the compound heterozygote, but the remaining variants from this compound
heterozygote can be filtered out by other conditions:

<p align="center">
<img src={require('/img/vv/compound_filter.png').default} width="1000"/>
</p>

## Add the First Filter Condition {#add-condition}

To add a new condition, click on the highlighted window:

<p align="center">
<img src={require('/img/vv/add_new_condition.png').default} width="600"/>
</p>

You will see a list of **[annotation columns](/results/main/snvs-indels/advanced-filtering#filtering-columns)**
available for filtering variants:

<p align="center">
<img src={require('/img/vv/fields.png').default} height="400"/>
</p>

The way of adding a filter condition depends on **the filter type**:

#### 1. Multiselect Filters {#multiselect-filters}

- Ability to select multiple column values at once.
- The way of adding a filter condition:

1. Select ***a filter column*** from the list of all columns;
2. Select ***a filter operator***:
    * "<i>in</i>";
    * "<i>not in</i>";
    * "<i>not empty</i>": the selected column has a non-empty value in the annotation for the searched variants (there
    is no such option for the "Chromosome" and "Origin" columns);
    * "<i>empty</i>": the selected column has no value in the annotation for the searched variants (there
    is no such option for the "Chromosome" and "Origin" columns).

<p align="center">
<img src={require('/img/vv/operators.png').default} width="650"/>
</p>

3. If the "<i>in</i>" or "<i>not in</i>" operators have been selected, select the required ***filter column values***
by clicking on the value rows:

<p align="center">
<img src={require('/img/vv/multiselect_options.png').default} width="650"/>
</p>

4. To complete adding the condition, click outside edit value container.

#### 2. Single Value Filters {#single-value-filters}

- Ability to select only one filter value.
- The way of adding a filter condition:

1. Select ***filter*** from the list of all filters. The filter operator "<i>is</i>" is automatically selected for
filters of this type;
2. Select the required ***filter value*** by clicking on the value row:

<p align="center">
<img src={require('/img/vv/one_option.png').default} width="700"/>
</p>

#### 3. Filters with ID input {#id-filters}

- Ability to enter the required column value.
- The way of adding a filter condition:

1. Select ***a filter column*** from the list of all columns;
2. Select ***a filter operator***:

<p align="center">
<img src={require('/img/vv/operator_id.png').default} width="650"/>
</p>

3. If the operators "<i>is</i>" or "<i>is not</i>" were selected, click on the input
box <img src={require('/img/vv/sample_data_value_text.png').default} width="200"/> and enter ***filter column value***:

<p align="center">
<img src={require('/img/vv/rsid.png').default} width="650"/>
</p>

You can enter several IDs at once in a list separated by a comma (e.g. `rs1234, rs2799066, rs4648562`), a semicolon
or a space (`rs1234 rs2799066 rs4648562`).

4. To complete adding the condition, press Enter or click outside edit value container.

#### 4. Numeric Filters {#numeric-filters}

- Ability to enter the required column value.
- The way of adding a filter condition:

1. Select ***a filter column*** from the list of all columns;
2. Select ***a filter operator***:
    * "<i>></i>";
    * "<i><</i>";
    * "<i>≥</i>";
    * "<i>≤</i>";
    * "<i>=</i>";
    * "<i>≠</i>";
    * "<i>not empty</i>": the selected column has a non-empty value in the annotation for the searched variants
    (there is no such option for the "Start" and "End" columns);
    * "<i>empty</i>": the selected column has no value in the annotation for the searched variants
    (there is no such option for the "Start" and "End" columns).

<p align="center">
<img src={require('/img/vv/exon_operators.png').default} width="650"/>
</p>

3. If a mathematical operator was selected as the filter operator, click on the input
box <img src={require('/img/vv/sample_data_value.png').default} width="150"/> and enter ***filter column value***:

<p align="center">
<img src={require('/img/vv/exon_value.png').default} width="650"/>
</p>

4. To complete adding the condition, press Enter or click outside edit value container.

#### 5. Any Value Filters {#any-value-filters}

- Ability to enter the required column value.
- The way of adding a filter condition:

1. Select ***a filter column*** from the list of all columns;
2. Select ***a filter operator***:
    * "<i>starts with</i>" (there is no such option for the "HGVSc" column);
    * "<i>contains</i>";
    * "<i>not contains</i>" (there is no such option for the "HGVSc" column);
    * "<i>not empty</i>": the selected column has a non-empty value in the annotation for the searched variants
    (there is no such option for the "Reference allele" column);
    * "<i>empty</i>": the selected column has no value in the annotation for the searched variants
     (there is no such option for the "Reference allele" column).

<p align="center">
<img src={require('/img/vv/hgvsp_operator.png').default} width="650"/>
</p>

3. If the operators "<i>starts with</i>", "<i>contains</i>" or "<i>not contains</i>" were selected, click on the
input<br></br>box <img src={require('/img/vv/sample_data_value_text.png').default} width="200"/> and
enter ***filter column value***:

<p align="center">
<img src={require('/img/vv/hgvsp_value.png').default} width="650"/>
</p>

4. To complete adding the condition, press Enter or click outside edit value container.

## Edit a Condition {#edit-condition}

You can change the ***operator*** and/or ***value*** in an already created condition.
To do this, you need to click on the corresponding field:

<p align="center">
<img src={require('/img/vv/edit_condition.png').default} width="700"/>
</p>

and choose another option:

<p align="center">
<img src={require('/img/vv/change_operator.png').default} width="700"/>
</p>

## Turn off a Condition {#turn-off-condition}

Turning off a condition is useful if you don’t want to lose the added condition, but at the moment you
don’t need to filter by it. To turn off a condition, hover over it and
click on <img src={require('/img/vv/turn_off_btn.png').default} width="25"/>:

<p align="center">
<img src={require('/img/vv/turn_off_condition.png').default} width="700"/>
</p>

## Remove a Condition {#remove-condition}

To remove a condition, hover over it and
click on <img src={require('/img/vv/delete_condition.png').default} width="25"/>:

<p align="center">
<img src={require('/img/vv/turn_off_condition.png').default} width="700"/>
</p>

:::tip
A condition that has not yet been filled in can also be removed
(if it's not the very first condition in the Query Builder):

<p align="center">
<img src={require('/img/vv/delete_new_condition.png').default} width="600"/>
</p>
:::

## Add the Following Condition {#add-another-condition}

If you want to add another condition to the query, first decide whether you want the variant you are looking for to
strictly ***match both conditions*** (the one already added and the new one), or whether the variant can
***match one of the conditions***.

#### The searched variant matches both conditions {#and-condition-matching}

1. After adding the first condition, click on <img src={require('/img/vv/and_btn.png').default} width="120"/>.
2. Complete the condition as described
in [the corresponding section](/results/main/snvs-indels/advanced-filtering#add-condition).
To get started, click on the highlighted window:

<p align="center">
<img src={require('/img/vv/and_cond_field.png').default} width="700"/>
</p>

#### The searched variant matches one of the conditions {#or-condition-matching}

1. After adding the first condition, hover over it and
click on <img src={require('/img/vv/or_btn.png').default} width="80"/>.
2. Complete the condition as described
in [the corresponding section](/results/main/snvs-indels/advanced-filtering#add-condition).
To get started, click on the highlighted window:

<p align="center">
<img src={require('/img/vv/or_cond_field.png').default} width="700"/>
</p>

## Apply a Query {#apply-query}

After you have completed all the conditions for the sought variants,
click on <img src={require('/img/vv/apply.png').default} width="150"/>.
The Query Builder closes and the query is applied to the variants.
Уou can see which query has been applied at the top of the table:

<p align="center">
<img src={require('/img/vv/filtered_var.png').default} width="1100"/>
</p>

You can turn the condition off by clicking on <img src={require('/img/vv/turn_off_created.png').default} width="20"/>,
or you can delete it by clicking on <img src={require('/img/vv/turn_off_cond_on_vv.png').default} width="20"/>:

<p align="center">
<img src={require('/img/vv/condition.png').default} width="250"/>
</p>

To ***remove all conditions*** from the query,
click on <img src={require('/img/vv/clear_all_filters_btn.png').default} width="45"/>.

To ***change or save your query*** or ***create a new one***, open the Query Builder by
clicking on <img src={require('/img/vv/query_builder_btn.png').default} width="120"/>.

To return to the SNV Viewer main page,
click on <img src={require('/img/vv/basic_filtering_btn.png').default} width="120"/>.

Sometimes there are a lot of conditions for the searched variants and the query can be very long:

<p align="center">
<img src={require('/img/vv/many_conditions.png').default} width="900"/>
</p>

In this case, you can ***collapse*** the query conditions by
clicking on the button <img src={require('/img/vv/hide.png').default} width="20"/> to the right of
the *Remove all conditions* button. Then only the opening conditions of the query will be shown:

<p align="center">
<img src={require('/img/vv/hidden.png').default} width="900"/>
</p>

To ***expand*** the query conditions back, click on <img src={require('/img/vv/show.png').default} height="23"/>.

:::info
On [Profile Settings page](/settings/profile#advanced-filtering-conditions), you can adjust
whether the query conditions are expanded or collapsed immediately after applying a filter query.
:::

## Save a Query {#save-query}

If you want to reuse the created query in the future, save it.
To do this, open the Query Builder by
clicking on <img src={require('/img/vv/query_builder_btn.png').default} width="120"/>, and then
click on <img src={require('/img/vv/save_to_library_btn.png').default} width="150"/>.
Enter the query name:

<p align="center">
<img src={require('/img/vv/query_name.png').default} width="600"/>
</p>

and click on <img src={require('/img/group_analysis/save_btn.png').default} width="150"/>.

The saved query will appear in the **Saved queries** list, on the left in the Query Builder.
To apply a saved query, click on its line:

<p align="center">
<img src={require('/img/vv/select_saved_query.png').default} width="200"/>
</p>

and then click on <img src={require('/img/vv/apply.png').default} width="150"/>.

In addition, the saved query will appear in the options of the basic
filter ["Saved queries"](/results/main/snvs-indels/basic-filtering#saved-queries).

If you want to ***delete*** a saved query, click on the query line and
then on <img src={require('/img/vv/delete_saved_query.png').default} width="25"/>.
