---
sidebar_position: 2
sidebar_label: Variant Viewer
---

# Variant Viewer

## Overview {#overview}

Variant Viewer is an embedded service for viewing and analyzing variants.
Located on **[Main tab](/results/main/main)** of the sample.

On the ***tumor*** sample page, you can find two Variant Viewers: one with ***somatic*** and
one with ***germline*** mutations discovered in this sample:

<p align="center">
<img src={require('/img/vv/germline_and_somatic_vv.png').default} width="600"/>
</p>

The ***normal*** sample page presents a Variant Viewer with ***germline*** mutations only:

<p align="center">
<img src={require('/img/vv/germline_only.png').default} width="600"/>
</p>

## Main Page {#main-page}

After clicking on <img src={require('/img/results/vv_btn.png').default} width="130"/>, you
will see the main page of the service:

<p align="center">
<img src={require('/img/vv/main_table.png').default} width="950"/>
</p>

The main object of Variant Viewer is a table with variants - each variant is on a new line:

<p align="center">
<img src={require('/img/vv/snv_row.png').default} width="950"/>
</p>

### Variant Info {#variant-information}

Besides of the information presented in the table (see
[Description of table columns](/results/main/snvs-indels/variant-viewer#columns-description)),
for each variant there are several embedded modules with additional information:

1. **Variant detailed information panel**. Click on the variant row to see a panel:

<p align="center">
<img src={require('/img/vv/details_panel.png').default} width="950"/>
</p>

You can read more about the panel and its tabs
in [the corresponding section](/results/main/snvs-indels/snv-details-panel).

2. **Variant details page**. To open the page,
click on <img src={require('/img/vv/details_info_btn.png').default} width="25"/> in the variant row.
On the page, you can [add a variant interpretation](/results/main/snvs-indels/snv-details-page#add-interpretation),
[determine a variant pathogenicity](/results/main/snvs-indels/pathogenicity#define-pathogenicity-on-acmg-tab), and
find more information about the variant. The page may include the following sections:
- [Detailed variant annotation](/results/main/snvs-indels/snv-details-page#annotation-tab).
- [Sample and Patient info](/results/main/snvs-indels/snv-details-page#sample-tab).
- [ACMG classification](/results/main/snvs-indels/snv-details-page#acmg-tab): includes criteria for variant
interpretation developed by [ACMG](https://www.acmg.net/). You can read more about this page
in [the corresponding section](/results/main/snvs-indels/pathogenicity#acmg-classification). To open
the page directly, click on the pathogenicity button
(<img src={require('/img/vv/acmg_var_btn.png').default} width="42"/> if the variant pathogenicity has not
been previously determined) in the variant row ("**Pathogenicity**" column) and click on "**ACMG Classification**"
option in the dialog:

<p align="center">
<img src={require('/img/vv/acmg_on_dialog.png').default} width="250"/>
</p>

- [Scientific publications associated with the variant](/results/main/snvs-indels/snv-details-page#literature-tab).
- [Clinical guidelines](/results/main/snvs-indels/snv-details-page#clinical-guidelines-tab).
The page is available if the variant is [onco relevant](/results/main/snvs-indels/onco-relevance).
To open page directly, click on <img src={require('/img/vv/onco_relevance_btn.png').default} width="30"/> in
the variant row ("**Onco Relevance**" column - can be added
in [column edit mode](/results/main/snvs-indels/edit-table-columns#add-columns)).
- [Variant report](/results/main/snvs-indels/snv-details-page#reporting-tab).

3. **Integrative Genomics Viewer (IGV)** is a module for visualization of variant on the genome.
Click on <img src={require('/img/vv/igv_var.png').default} width="33"/> in the variant row to open the module.

### Pin Variants {#pin-variant}

The variants you want can be pinned. Pinned variants are not
affected by [filtering and sorting](/results/main/snvs-indels/variant-viewer#sort-and-filter) the
table and always remain at the top of the table. The visibility of pinned variants in the table can
only be affected by filtering by origin (for example, pinned germline variants will not be presented
in a table filtered by somatic origin). Pinned variants are sorted by the variant start position
("Position" column).

Click on <img src={require('/img/vv/pin_variant.png').default} width="28"/>
in the variant line to pin it. The pinned variant will appear at the top of the table (the row highlighted in green):

<p align="center">
<img src={require('/img/vv/pinned_variant.png').default} width="950"/>
</p>

To unpin a variant, click on the same button <img src={require('/img/vv/unpin.png').default} width="28"/> in the
variant row.

You can also pin or unpin a variant using keyboard shortcuts. To do this, select the variant row (the selected row
is highlighted in blue) and press **P** on the keyboard.

:::note
Variant remains pinned when you exit the Variant Viewer page.
:::

### Hide Variants {#hide-variants}

Variants can be hidden from Variant Viewer table. To do this,
click on <img src={require('/img/vv/hide_btn.png').default} width="25"/> in the variant row.
Hidden variants will only be available when a specific filter condition is applied.

**Variant Visibility Filter**: in the upper right corner of Variant Viewer page (regardless of variant filtering mode),
there is a variant visibility filter button: <img src={require('/img/vv/without_hidden.png').default} width="150"/>
(default state):

<p align="center">
<img src={require('/img/vv/filter_hide_options.png').default} width="200"/>
</p>

- By default, only visible (not hidden) variants are shown in Variant Viewer table (filter
option "***Without hidden***").
- To see in the table all variants discovered and annotated in the sample, click on the filter button
and select "***Show all***" option. Then the table will display all variants, including hidden ones.
The visible variants in the row have a button to hide: <img src={require('/img/vv/hide_btn.png').default} width="25"/>,
while the hidden variants have a button to show: <img src={require('/img/vv/show_btn.png').default} width="25"/>.
- To see only hidden variants, click on the filter button and select "***Hidden only***" option.

:::note
[Pinned variants](/results/main/snvs-indels/variant-viewer#pin-variant) are always visible.
At the same time, they are displayed in the table under any condition of variant visibility filter.
:::

To make a hidden variant visible, click on <img src={require('/img/vv/show_btn.png').default} width="25"/> in its row.

You can also hide or show a variant using keyboard shortcuts. To do this, select the variant row (the selected row
is highlighted in blue) and press **H** on the keyboard.

:::note
When you pin a hidden variant, it not only becomes pinned in Variant Viewer table, but also becomes visible.
:::

### Include Variants to Report {#include-to-report}

Variants included to report will be included in report blocks
of ["SNVs/Indels selected by user for reports"](/settings/report-templates/report-templates#block-types) type.
To include a variant to the report,
click on <img src={require('/img/vv/include_to_report.png').default} width="25"/> in the variant row.
To remove a variant from the report, click the same
button <img src={require('/img/vv/remove_from_report.png').default} width="25"/> in the variant row.
Variants included to report can also be removed from the report on the page of the report itself.

You can also include a variant to report or remove a variant from report using keyboard shortcuts.
To do this, select the variant row (the selected row is highlighted in blue) and press **R** on the keyboard.

:::info
When a variant is [determined](/results/main/snvs-indels/pathogenicity) as pathogenic or likely pathogenic,
it is automatically included to report.
:::

### Variant Significance Label {#label}

You can interpret a variant by its significance (clinical or for your study, etc.) To do this,
add "**Label**" column in [column edit mode](/results/main/snvs-indels/edit-table-columns#add-columns) (not
added by default). The column contains a label
button <img src={require('/img/vv/label_non_defined.png').default} width="25"/> ("*Undefined significance*"
if the label value has not been defined).

To interpret a variant by significance, click on the label
button <img src={require('/img/vv/label_non_defined.png').default} width="25"/> and select the required value
in the drop-down list:

<p align="center">
<img src={require('/img/vv/label_drop_down.png').default} width="250"/>
</p>

You can later redefine the significance in the same way or clear it by clicking on the label button
and selecting "*Clear*" option:

<p align="center">
<img src={require('/img/vv/clear_label.png').default} width="250"/>
</p>

As a result of the variant interpretation by significance, the label button will change color:
- High significance: <img src={require('/img/vv/high_label.png').default} width="25"/>
- Moderate significance: <img src={require('/img/vv/medium_label.png').default} width="25"/>
- Low significance: <img src={require('/img/vv/low_label.png').default} width="25"/>
- Sequencing error: <img src={require('/img/vv/seq_error.png').default} width="25"/>

Variants can be sorted by significance: from high significance to sequencing error by clicking on "Label" column
name once, and vice versa by clicking on the name twice. In addition, variants can be filtered by significance:
- ["Label" filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters) in advanced filtering mode;
- an additional filter "Label" in basic filtering mode
(can be added on ["Profile" page](/settings/profile#basic-mode-filters)).

### Sort and Filter Variants {#sort-and-filter}

By default, the table is sorted by the variant start position ("Position" column).
You can ***sort*** the table in ascending or descending order of the values of the "Locus", "Origin", "Position",
"Exon", "Depth (Alt/Ref)", "Consequence", "gnomAD AF", "Pathogenicity". To sort by the column, click on its name
(one click to sort in ascending order, two clicks - in descending order).

***Filtering of variants*** in Variant Viewer is described in the
[Basic Filtering](/results/main/snvs-indels/basic-filtering) and
[Advanced Filtering](/results/main/snvs-indels/advanced-filtering) sections.

### Keyboard Shortcuts in Variant Viewer {#hotkeys}

In Variant Viewer, you can interact with variants using keyboard shortcuts.
To use keyboard shortcuts, click on a variant row, or press one of keys: **S**, **W**, ⬇️, or ⬆️ to select
the first variant in the table.

**Available keyboard shortcuts**

<table>
  <tr>
   <td>Press this key</td>
   <td>To do this</td>
  </tr>
  <tr>
   <td><b>S</b> or ⬇️</td>
   <td>Move to the row of the next variant (or the first variant in the table if no variant was selected)</td>
  </tr>
  <tr>
   <td><b>W</b> or ⬆️</td>
   <td>Move to the row of the previous variant (or the first variant in the table if no variant was selected)</td>
  </tr>
  <tr>
   <td><b>P</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#pin-variant">Pin/unpin</a> variant</td>
  </tr>
  <tr>
   <td><b>H</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#hide-variants">Hide/show</a> variant</td>
  </tr>
  <tr>
   <td><b>R</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#include-to-report">Include variant to report</a> or
   remove it from report</td>
  </tr>
</table>

### Export Variants {#export}

Variant Viewer allows to export variants in CSV format:

- ***Export of all variants***: click on <img src={require('/img/vv/export_btn.png').default} width="90"/> and select
"*All to CSV*" option:

<p align="center">
<img src={require('/img/vv/all_to_csv.png').default} width="200"/>
</p>

- ***Export of only filtered variants***: filter the needed variants by
[Basic](/results/main/snvs-indels/basic-filtering) or [Advanced](/results/main/snvs-indels/advanced-filtering)
filtering, then click on <img src={require('/img/vv/export_btn.png').default} width="90"/> and select
"*All to CSV*" option:

<p align="center">
<img src={require('/img/vv/all_to_csv.png').default} width="200"/>
</p>

- ***Export of only pinned variants***: [pin](/results/main/snvs-indels/variant-viewer#pin-variant)
the needed variants, then click on <img src={require('/img/vv/export_btn.png').default} width="90"/> and select
"*Pinned only to CSV*" option:

<p align="center">
<img src={require('/img/vv/only_pinned.png').default} width="200"/>
</p>

:::note
If no variant has been pinned in Variant Viewer, then "*Pinned only to CSV*" option will not be in the
drop-down list of export options.
:::

- ***Export of only modified variants***: click
on <img src={require('/img/vv/export_btn.png').default} width="100"/> and select "*Modified only to CSV*" option:

<p align="center">
<img src={require('/img/vv/modified_only.png').default} width="230"/>
</p>

**Modified variants** are variants for which any of the following has been done:
- a variant is [pinned](/results/main/snvs-indels/variant-viewer#pin-variant) in Variant Viewer table;
- a variant is [included to report](/results/main/snvs-indels/variant-viewer#include-to-report);
- [interpretation was added](/results/main/snvs-indels/snv-details-page#add-interpretation) for a variant;
- [pathogenicity was determined](/results/main/snvs-indels/pathogenicity) for a variant;
- a variant is [interpreted by significance](/results/main/snvs-indels/variant-viewer#label).

:::note
If Variant Viewer table is filtered by some condition, then the table of exported modified variants
will include only those modified variants that match this condition. An exception is pinned variants,
whose visibility in the table can only be affected by filtering by origin.
:::

## Table Columns' Description {#columns-description}

By default, the table has the following columns (the set and order of the columns can be changed - how to do it is
described [here](/results/main/snvs-indels/edit-table-columns)):

1. **Locus** is the common name of the gene in which the variant is located. If the gene is unknown,
the locus is designated as <img src={require('/img/vv/no_gene.png').default} width="110"/>.
2. **Origin** is a mutation type (somatic or germline).

:::info
"Origin" column is hidden in [Variant Viewer with germline mutations](/results/main/snvs-indels/variant-viewer)
and in Variant Viewer with somatic mutations if "Origin is Somatic" filter is on.
:::

3. **Position** is a coordinate of the variant in the genome (chromosome + start position).
4. **HGVSp** is an amino acid substitution using [the HGVS notation](https://varnomen.hgvs.org/) (“p.” prefix
(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution).
When hovering over the value of the "HGVSp" column for a variant, you can see the transcript which the substitution
was predicted for and HGVSp with amino acids in the three letter code:

<p align="center">
<img src={require('/img/vv/hgvsp_tooltip.png').default} width="350"/>
</p>

4. **Alteration** is a nucleotide substitution (reference allele > alternative allele).
5. **HGVSc** is a nucleotide substitution using [the HGVS notation](https://varnomen.hgvs.org/) (genomic position of
the substituted nucleotide + reference allele > alternative allele). Starts with a prefix: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence). When hovering
over the value of the "HGVSc" column for a variant, you can see the transcript which the substitution was predicted
for (the transcript IDs from Ensembl (ENSTxxxxxxxxxxx) and RefSeq (NM_) are given):

<p align="center">
<img src={require('/img/vv/hgvsc_tooltip.png').default} width="300"/>
</p>

6. **Exon** is the exon number in the transcript which is affected by the variant.
7. **Depth (Alt/Ref)** is a sequencing depth; the total number of reads of the sequence overlapping the variant
position for this sample. **Alt** is the alternative allele reads count, **Ref** is the reference allele reads count.
Also the percentage of the share of the alternative allele reads count among all reads (Alt/Depth) is shown.

The column cells are colored depending on the depth value:

- green if depth > 60;
- yellow if 20 ≤ depth ≤ 60;
- red if depth < 20.

8. **GT** is a genotype; allelic values (0 - reference allele, 1 - first alternative allele, 2 - second alternative
allele etc.), divided by ”/” (for unphased genotype) or “|” (for phased genotype).
9. **Consequence** is the effect of the variant on genes.

#### Consequences of variants in the annotation {#variant-consequences}

<table>
  <tr>
   <td>Consequence</td>
   <td>Description</td>
   <td>Impact*</td>
  </tr>
  <tr>
   <td>Coding sequence</td>
   <td>The variant hits a coding sequence</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Downstream</td>
   <td>Downstream of a gene</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Feature elongation</td>
   <td>A sequence variant that causes the extension of a genomic feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Feature truncation</td>
   <td>A sequence variant that causes the reduction of a genomic feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Frameshift</td>
   <td>Insertion or deletion causes a frame shift</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Incomplete terminal codon</td>
   <td>A sequence variant where at least one base of the final codon of an incompletely annotated transcript
   is changed</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Inframe deletion</td>
   <td>One or many codons are deleted</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>Inframe insertion</td>
   <td>One or many codons are inserted</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>Intergenic variant</td>
   <td>The variant is in an intergenic region</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Intron</td>
   <td>The variant is located in the intron</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Mature miRNA</td>
   <td>A transcript variant located with the sequence of the mature miRNA</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Missense</td>
   <td>Variant causes a codon that produces a different amino acid</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>NMD transcript</td>
   <td>The variant is located in the transcript susceptible to nonsense-mediated decay</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Non coding transcript</td>
   <td>The variant is located in the non-coding transcript</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Non coding transcript exon</td>
   <td>A sequence variant that changes non-coding exon sequence in a non-coding transcript</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Protein altering</td>
   <td>A variant which is predicted to change the protein encoded in the coding sequence</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Regulatory region ablation</td>
   <td>Deletion of a regulatory region</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Regulatory region amplification</td>
   <td>Amplification of regulatory region</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Regulatory region variant</td>
   <td>The variant hits a regulatory feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Splice acceptor</td>
   <td>The variant hits a splice acceptor site</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Splice donor</td>
   <td>The variant hits a splice donor site</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Splice region</td>
   <td>A sequence variant in which a change has occurred within the region of the splice site, either within 1-3
   bases of the exon or 3-8 bases of the intron</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Start lost</td>
   <td>Variant causes start codon to be mutated into a non-start codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Start retained</td>
   <td>Variant causes start codon to be mutated into another start codon</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Stop gained</td>
   <td>Variant causes a STOP codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Stop lost</td>
   <td>Variant causes stop codon to be mutated into a non-stop codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Stop retained</td>
   <td>Variant causes stop codon to be mutated into another stop codon</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Synonymous</td>
   <td>Variant causes a codon that produces the same amino acid</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>TF binding site</td>
   <td>Transcription factor binding site (TFBS)</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>TFBS ablation</td>
   <td>Deletion of a TFBS</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>TFBS amplification</td>
   <td>Amplification of a TFBS</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Transcript ablation</td>
   <td>Deletion of a transcript</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Transcript amplification</td>
   <td>A feature amplification of a region containing a transcript</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Upstream</td>
   <td>Upstream of a gene</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>UTR3</td>
   <td>A variant hits 3'UTR region</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>UTR5</td>
   <td>A variant hits 5'UTR region</td>
   <td>MODIFIER</td>
  </tr>
</table>

***Impact** is the predicted effect of the variant on the protein. The corresponding column can be added in
[the Edit Table Columns window](/results/main/snvs-indels/edit-table-columns).

#### Impacts of variants in the annotation {#variant-impacts}

<table>
  <tr>
   <td>Impact
   </td>
   <td>Description
   </td>
  </tr>
  <tr>
   <td>HIGH
   </td>
   <td>The variant has high (disruptive) impact in the protein, probably causing protein truncation, loss of
   function or triggering nonsense mediated decay
   </td>
  </tr>
  <tr>
   <td>MODERATE
   </td>
   <td>The variant might change protein effectiveness
   </td>
  </tr>
  <tr>
   <td>LOW
   </td>
   <td>The variant is mostly harmless or unlikely to change protein behavior
   </td>
  </tr>
  <tr>
   <td>MODIFIER
   </td>
   <td>Usually variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact
   </td>
  </tr>
</table>

The "Consequence" column cells are colored depending on the impact:

- red if the impact is HIGH;
- yellow if the impact is MODERATE;
- no color if impact is LOW or MODIFIER.

10. **External links** are links to pages with variant information
in <a href="https://www.ncbi.nlm.nih.gov/snp/">dbSNP</a>: <img src={require('/img/vv/rsId_btn.png').default} width="35"/>, <a href="https://varsome.com/">VarSome</a>: <img src={require('/img/vv/varsome.png').default} width="35"/>, <a href="https://preview.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>: <img src={require('/img/vv/clinvar.png').default} width="35"/>.
There may also be a link to the variant in <a href="https://cancer.sanger.ac.uk/cosmic/">COSMIC</a>: <img src={require('/img/vv/cosmic.png').default} width="35"/> (if it was [uploaded as a custom annotation](/settings/custom-annotations#define-annotation-type)).

11. **gnomAD AF** is the allele frequency according to the [gnomAD](https://gnomad.broadinstitute.org/) database.

The column cells are colored depending on the allele frequency value:

- green if AF ≥ 0.015;
- yellow if 0.01 ≤ AF < 0.015;
- red if AF < 0.01.

12. **Pathogenicity** is the variant pathogenicity, which can be determined either manually, by ACMG criteria, or
according to the pathogenicity base. If you have not [uploaded the pathogenicity database](/settings/custom-annotations#define-annotation-type)
or have not determined the variant pathogenicity, then the column
value will be <img src={require('/img/vv/acmg_var_btn.png').default} width="40"/> (No interpretation).
