---
sidebar_position: 2
sidebar_label: SNV Viewer
---

# SNV Viewer

## Overview {#overview}

SNV Viewer is an embedded service for viewing and analyzing single-nucleotide
variants (SNVs) and short insertions/deletions (Indels) discovered in the sample.
Located on **[Main tab](/results/main/main)** of the sample.

On the ***tumor*** sample page, you can find two SNV Viewers: one with ***somatic SNVs/Indels*** discovered
in this sample and one with ***germline SNVs/Indels*** discovered in the control sample in the case of the
tumor sample from a pair of tumor/normal samples
or in the tumor sample itself, if it was analyzed without a control sample.
In SNV Viewer with somatic mutations, you can also see SNVs/Indels from SNV Viewer with germline mutations
if you change the value of the filter by origin from Somatic to Germline or clear filtering by it.

<p align="center">
<img src={require('/img/vv/germline_and_somatic_vv.png').default} width="700"/>
</p>

The ***non-tumor*** sample page presents SNV Viewer with ***germline SNVs/Indels*** only discovered in this sample:

<p align="center">
<img src={require('/img/vv/germline_only.png').default} width="700"/>
</p>

## Main Page {#main-page}

After clicking on <img src={require('/img/results/vv_btn.png').default} width="110"/>, you
will see the main page of the service:

<p align="center">
<img src={require('/img/vv/main_table.png').default} width="950"/>
</p>

The main object of SNV Viewer is a table with variants - each variant is on a new line:

<p align="center">
<img src={require('/img/vv/snv_row.png').default} width="950"/>
</p>

Below the table, above the variant details panel, there are:
- Information about the amount of variants in SNV Viewer: the order of variants displayed on the page (e.g.,
if the maximum amount of variants shown on the page is 50, then the order will be: 1-50, 51-100, etc.) and
the total amount of variants (pinned variants are not taken into account).
The amount of variants changes when filtering variants and displays the amount of variants that
match the specified conditions.
- A component to choose the amount of variants displayed on the page
(excluding pinned variants): <img src={require('/img/vv/records_on_page.png').default} width="170"/>.
By default, 50 variants are displayed per page. To change the amount of variants displayed on the page to
20, 100 or 200, click on the corresponding option. Your choice will be remembered and will be valid for SNV Viewer
and CNV Viewer of all samples. Please note that the more variants shown on a page, the longer it will take to
load the page.
- Pagination component: <img src={require('/img/eng/pagination.png').default} width="370"/>, which allows you
to navigate between pages with variants. The number of the current page is highlighted in a blue circle.
If you click on this number, the page will reload. To move to a page with a specific number, either click on
that number, or click on <img src={require('/img/eng/three_points.png').default} width="30"/>, enter the
number and press the enter key or click outside the value field. To move to the page with previous number,
click on <img src={require('/img/eng/back.png').default} width="30"/>, and to open the next page,
click on <img src={require('/img/eng/next.png').default} width="30"/>.

### Variant Info {#variant-information}

Besides of the information presented in the table (see
[Table Columns' Description](/results/main/snvs-indels/variant-viewer#columns-description)),
for each variant there are several embedded modules with additional information:

1. **Variant detailed information panel**. Click on the variant row to see a panel:

<p align="center">
<img src={require('/img/vv/details_panel.png').default} width="950"/>
</p>

You can read more about the panel and its tabs
in [the corresponding section](/results/main/snvs-indels/snv-details-panel).

2. **Variant details page**. To open the page,
click on <img src={require('/img/vv/details_info_btn.png').default} width="25"/> in the variant row.
On the page, you can [add a variant interpretation](/results/main/snvs-indels/snv-details-page#add-interpretation),
[determine a variant pathogenicity](/results/main/snvs-indels/pathogenicity#define-pathogenicity-on-acmg-tab), and
find more information about the variant. The page may include the following sections:
- [Detailed variant annotation](/results/main/snvs-indels/snv-details-page#annotation-tab).
- [Sample and Patient info](/results/main/snvs-indels/snv-details-page#sample-tab).
- [Information about occurrences of the variant in other samples](/results/main/snvs-indels/snv-details-page#occurrences).
- [ACMG classification](/results/main/snvs-indels/snv-details-page#acmg-tab): includes criteria for variant
interpretation developed by [ACMG](https://www.acmg.net/). You can read more about this page
in [the corresponding section](/results/main/snvs-indels/pathogenicity#acmg-classification). To open
the page directly, click on the pathogenicity button
(<img src={require('/img/vv/acmg_var_btn.png').default} width="42"/> if the variant pathogenicity has not
been previously determined) in the variant row ("**Pathogenicity**" column) and click on "**ACMG Classification**"
option in the dialog:

<p align="center">
<img src={require('/img/vv/acmg_on_dialog.png').default} width="250"/>
</p>

- [Scientific publications associated with the variant](/results/main/snvs-indels/snv-details-page#literature-tab).
- [Clinical guidelines](/results/main/snvs-indels/snv-details-page#clinical-guidelines-tab).
The page is available if the variant is [onco relevant](/results/main/snvs-indels/onco-relevance).
To open page directly, click on <img src={require('/img/vv/onco_relevance_btn.png').default} width="30"/> in
the variant row ("**Onco Relevance**" column - can be added
in [column edit mode](/results/main/snvs-indels/edit-table-columns#add-columns)).
- [Variant report](/results/main/snvs-indels/snv-details-page#reporting-tab).

3. **Integrative Genomics Viewer (IGV)** is a module for visualization of variant on the genome.
Click on <img src={require('/img/vv/igv.png').default} width="30"/> in the variant row to open the module.

### Pin Variants {#pin-variant}

The variants you want can be pinned. Pinned variants are not
affected by [filtering and sorting](/results/main/snvs-indels/variant-viewer#sort-and-filter) the
table and always remain at the top of the table. The visibility of pinned variants in the table can
only be affected by filtering by origin (for example, pinned germline variants will not be presented
in a table filtered by somatic origin). Pinned variants are sorted by the variant start position
("Position" column).

Click on <img src={require('/img/vv/pin_variant.png').default} width="28"/>
in the variant line to pin it. The pinned variant will appear at the top of the table (the row highlighted in green):

<p align="center">
<img src={require('/img/vv/pinned_variant.png').default} width="950"/>
</p>

To unpin a variant, click on the same button <img src={require('/img/vv/unpin.png').default} width="28"/> in the
variant row.

You can also pin or unpin a variant using keyboard shortcuts. To do this, select the variant row (the selected row
is highlighted in blue) and press **P** on the keyboard.

:::note
Variant remains pinned when you exit the SNV Viewer page.
:::

### Hide Variants {#hide-variants}

Variants can be hidden from SNV Viewer table. To do this,
click on <img src={require('/img/vv/hide_btn.png').default} width="25"/> in the variant row.
Hidden variants will only be available when a specific filter condition is applied.

**Variant Visibility Filter**: in the upper right corner of SNV Viewer page (regardless of variant filtering mode),
there is a variant visibility filter button: <img src={require('/img/vv/without_hidden.png').default} width="150"/>
(default state):

<p align="center">
<img src={require('/img/vv/filter_hide_options.png').default} width="200"/>
</p>

- By default, only visible (not hidden) variants are shown in SNV Viewer table (filter
option "***Without hidden***").
- To see in the table all variants discovered and annotated in the sample, click on the filter button
and select "***Show all***" option. Then the table will display all variants, including hidden ones.
The visible variants in the row have a button to hide: <img src={require('/img/vv/hide_btn.png').default} width="25"/>,
while the hidden variants have a button to show: <img src={require('/img/vv/show_btn.png').default} width="25"/>.
- To see only hidden variants, click on the filter button and select "***Hidden only***" option.

:::note
[Pinned variants](/results/main/snvs-indels/variant-viewer#pin-variant) are always visible.
At the same time, they are displayed in the table under any condition of variant visibility filter.
:::

To make a hidden variant visible, click on <img src={require('/img/vv/show_btn.png').default} width="25"/> in its row.

You can also hide or show a variant using keyboard shortcuts. To do this, select the variant row (the selected row
is highlighted in blue) and press **H** on the keyboard.

:::note
When you pin a hidden variant, it not only becomes pinned in SNV Viewer table, but also becomes visible.
:::

### Include Variants to Report {#include-to-report}

Variants included to report will be included in report blocks
of ["SNVs/Indels selected by user for reports"](/settings/report-templates/report-templates#block-types) type.
To include a variant to the report,
click on <img src={require('/img/vv/include_to_report.png').default} width="25"/> in the variant row.
To remove a variant from the report, click the same
button <img src={require('/img/vv/remove_from_report.png').default} width="25"/> in the variant row.
Variants included to report can also be removed from the report on the page of the report itself.

You can also include a variant to report or remove a variant from report using keyboard shortcuts.
To do this, select the variant row (the selected row is highlighted in blue) and press **R** on the keyboard.

:::info
When a variant is [determined](/results/main/snvs-indels/pathogenicity) as pathogenic or likely pathogenic,
it is automatically included to report.
:::

### Variant Significance Label {#label}

You can interpret a variant by its significance (clinical or for your study, etc.) To do this,
add "**Label**" column in [column edit mode](/results/main/snvs-indels/edit-table-columns#add-columns) (not
added by default). The column contains a label
button <img src={require('/img/vv/label_non_defined.png').default} width="25"/> ("*Undefined significance*"
if the label value has not been defined).

To interpret a variant by significance, click on the label
button <img src={require('/img/vv/label_non_defined.png').default} width="25"/> and select the required value
in the drop-down list:

<p align="center">
<img src={require('/img/vv/label_drop_down.png').default} width="250"/>
</p>

You can later redefine the significance in the same way or clear it by clicking on the label button
and selecting "*Clear*" option:

<p align="center">
<img src={require('/img/vv/clear_label.png').default} width="250"/>
</p>

As a result of the variant interpretation by significance, the label button will change color:
- High significance: <img src={require('/img/vv/high_label.png').default} width="25"/>
- Moderate significance: <img src={require('/img/vv/medium_label.png').default} width="25"/>
- Low significance: <img src={require('/img/vv/low_label.png').default} width="25"/>
- Sequencing error: <img src={require('/img/vv/seq_error.png').default} width="25"/>

Variants can be sorted by significance: from high significance to sequencing error by clicking on "Label" column
name once, and vice versa by clicking on the name twice. In addition, variants can be filtered by significance:
- ["Label" filter](/results/main/snvs-indels/advanced-filtering#multiselect-filters) in advanced filtering mode;
- an additional filter "Label" in basic filtering mode
(can be added on ["Profile" page](/settings/profile#basic-mode-filters)).

### Filter and Sort Variants {#sort-and-filter}

***Filtering of variants*** in SNV Viewer is described in
[Basic Filtering](/results/main/snvs-indels/basic-filtering) and
[Advanced Filtering](/results/main/snvs-indels/advanced-filtering) sections.

By default, the table is sorted by the variant start position ("Position" column).
You can ***sort*** the table by the values of the following columns:
- [default columns](/results/main/snvs-indels/variant-viewer#columns-description): "Locus", "Origin",
"Position", "Exon", "Depth (Alt/Ref)", "Consequence", "gnomAD 3 AF", "Pathogenicity";
- all columns [not added by default](/results/main/snvs-indels/edit-table-columns#add-columns).

To sort a table by column values, click on the column name (click once to sort in ascending order,
or twice to sort in descending order). After that, you will see an arrow next to the column
name: <img src={require('/img/vv/sort_asc.png').default} width="17"/> (when sorting in ascending order)
or <img src={require('/img/vv/sort_desc.png').default} width="17"/> (when sorting in descending order).

You can sort a table by 1-3 columns at the same time to group data by the same value in one column, and
then sort another column within that group of equal values. The sort order will depend on the order in which
you click on the corresponding column names. The sort arrow will be presented for the column that was sorted last.

:::note
Table sorting and filtering are saved when you exit the SNV Viewer page.
:::

To reset sorting, use the clear all filters
button: <img src={require('/img/vv/clear_filters.png').default} width="35"/>.
Please note that this will also reset the filtering of variants.

The basic sorting orders are alphabetical for columns with text values (e.g. "Locus" column),
or smallest to largest and vice versa for columns with numeric values (e.g. "gnomAD 3 AF" column).
However, the sorting order is special for some columns:
- "**[Consequence](/results/main/snvs-indels/variant-viewer#variant-consequences)**" column:
sorting from most significant consequence to least significant: Transcript ablation, Splice acceptor, Splice donor,
Stop gained, Frameshift, Stop lost, Start lost, Transcript amplification, Inframe insertion,
Inframe deletion, Missense, Protein altering, Splice region, Incomplete terminal codon, Start
retained, Stop retained, Synonymous, Coding sequence, Mature miRNA, UTR5, UTR3, Non coding transcript
exon, Intron, NMD transcript, Non coding transcript, Upstream, Downstream, TFBS ablation,
TFBS amplification, TF binding site, Regulatory region ablation, Regulatory region amplification,
Feature elongation, Regulatory region variant, Feature truncation, Intergenic variant,
Splice donor 5th base variant, Splice donor region variant, Splice polypyrimidine tract variant, Unknown.
Composite values (containing multiple consequences) are sorted by the least significant value in the list.
- "**[Pathogenicity](/results/main/snvs-indels/pathogenicity)**" column: sorting from pathogenic
to benign variant: Pathogenic, Likely pathogenic, Conflicting interpretations of pathogenicity,
Uncertain significance, Likely benign, Benign.
- "**Impact**" column (not added by default,
values are described [here](/results/main/snvs-indels/variant-viewer#variant-impacts)):
sorting from high to low impact: HIGH, MODERATE, LOW, MODIFIER.
- "**Genome position**" column (not added by default): sorting by the significance of the variant
location in the genome: Exonic, Intronic, Intergenic.
- "**Label**" column (not added by default, described [here](/results/main/snvs-indels/variant-viewer#label)):
sorting from significant to insignificant
variants: <img src={require('/img/vv/high_label.png').default} width="25"/> (high
significance), <img src={require('/img/vv/medium_label.png').default} width="25"/> (moderate
significance), <img src={require('/img/vv/low_label.png').default} width="25"/> (low
significance), <img src={require('/img/vv/seq_error.png').default} width="25"/> (sequencing
error).
- "**Caller Filters**" column (not added by default,
described [here](/results/main/snvs-indels/variant-viewer#added-columns)):
sorting from variants which didn't pass filtration to variants satisfying all filters: FAIL, N/A, PASS.
- "**ClinVar**" column (not added by default,
described [here](/results/main/snvs-indels/variant-viewer#added-columns)):
sorting from unconditionally significant variants to insignificant ones:
1. Pathogenic;
2. Pathogenic, low penetrance;
3. Likely pathogenic;
4. Likely pathogenic, low penetrance;
5. Composite values including "Pathogenic" or "Pathogenic, low penetrance";
6. Composite values including "Likely pathogenic" or "Likely pathogenic, low penetrance";
7. Uncertain significance;
8. Conflicting interpretations of patogenicity / Conflicting data from submitters;
9. Established risk allele;
10. Likely risk allele;
11. Uncertain risk allele;
12. Risk factor;
13. Affects;
14. Drug response;
15. Association;
16. Confers sensitivity;
17. Protective;
18. Likely benign;
19. Association not found;
20. Benign;
21. Other;
22. Not provided.

Composite values (other than those that include the values "Pathogenic", "Pathogenic, low penetrance",
"Likely pathogenic", or "Likely pathogenic, low penetrance") are sorted by the most significant value from the list.
- "**ENIGMA**" column (not added by default,
described [here](/results/main/snvs-indels/variant-viewer#added-columns)): sorting from significant to
insignificant variants: Pathogenic, Likely pathogenic, Likely benign, Benign, Not provided.
Composite values are sorted by the most significant value from the list.

### Keyboard Shortcuts in SNV Viewer {#hotkeys}

In SNV Viewer, you can interact with variants using keyboard shortcuts.
To use keyboard shortcuts, click on a variant row, or press one of keys: **S**, **W**, ⬇️, or ⬆️ to select
the first variant in the table.

**Available keyboard shortcuts**

<table>
  <tr>
   <td>Press this key</td>
   <td>To do this</td>
  </tr>
  <tr>
   <td><b>S</b> or ⬇️</td>
   <td>Move to the row of the next variant (or the first variant in the table if no variant was selected)</td>
  </tr>
  <tr>
   <td><b>W</b> or ⬆️</td>
   <td>Move to the row of the previous variant (or the first variant in the table if no variant was selected)</td>
  </tr>
  <tr>
   <td><b>P</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#pin-variant">Pin/unpin</a> variant</td>
  </tr>
  <tr>
   <td><b>H</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#hide-variants">Hide/show</a> variant</td>
  </tr>
  <tr>
   <td><b>R</b></td>
   <td><a href="/results/main/snvs-indels/variant-viewer#include-to-report">Include variant to report</a> or
   remove it from report</td>
  </tr>
</table>

### Export Variants {#export}

SNV Viewer allows to export variants in CSV format:

- ***Export of all variants***: click on <img src={require('/img/vv/export_btn.png').default} width="120"/> and select
"*All*" option:

<p align="center">
<img src={require('/img/vv/all_to_csv.png').default} width="220"/>
</p>

- ***Export of only filtered variants***: filter the needed variants by
[Basic](/results/main/snvs-indels/basic-filtering) or [Advanced](/results/main/snvs-indels/advanced-filtering)
filtering, then click on <img src={require('/img/vv/export_btn.png').default} width="120"/> and select
"*All*" option:

<p align="center">
<img src={require('/img/vv/all_to_csv.png').default} width="220"/>
</p>

- ***Export of only pinned variants***: [pin](/results/main/snvs-indels/variant-viewer#pin-variant)
the needed variants, then click on <img src={require('/img/vv/export_btn.png').default} width="120"/> and select
"*Pinned only*" option:

<p align="center">
<img src={require('/img/vv/only_pinned.png').default} width="220"/>
</p>

:::note
If no variant has been pinned in SNV Viewer, then "*Pinned only*" option will not be in the
drop-down list of export options.
:::

- ***Export of only modified variants***: click
on <img src={require('/img/vv/export_btn.png').default} width="120"/> and select "*Modified only*" option:

<p align="center">
<img src={require('/img/vv/modified_only.png').default} width="220"/>
</p>

**Modified variants** are variants for which any of the following has been done:
- a variant is [pinned](/results/main/snvs-indels/variant-viewer#pin-variant) in SNV Viewer table;
- a variant is [included to report](/results/main/snvs-indels/variant-viewer#include-to-report);
- [interpretation was added](/results/main/snvs-indels/snv-details-page#add-interpretation) for a variant;
- [pathogenicity was determined](/results/main/snvs-indels/pathogenicity) for a variant;
- a variant is [interpreted by significance](/results/main/snvs-indels/variant-viewer#label).

:::note
If SNV Viewer table is filtered by some condition, then the table of exported modified variants
will include only those modified variants that match this condition. An exception is pinned variants,
whose visibility in the table can only be affected by filtering by origin.
:::

To ***export all modified variants without filtering***,
click on <img src={require('/img/vv/export_btn.png').default} width="120"/> and select
"*Modified only without filtering*" option:

<p align="center">
<img src={require('/img/vv/modified_without_filters.png').default} width="220"/>
</p>

## Table Columns' Description {#columns-description}

By default, the table has the following columns (the set and order of the columns can be changed - how to do it is
described [here](/results/main/snvs-indels/edit-table-columns)):

1. **Locus** is the common name of the gene in which the variant is located. If the gene is unknown,
the locus is designated as <img src={require('/img/vv/no_gene.png').default} width="110"/>.
You can copy the gene name to the clipboard by moving the cursor over the column value and
clicking on <img src={require('/img/vv/copy.png').default} width="20"/>.
2. **Origin** is a mutation type (somatic or germline).

:::info
"Origin" column is hidden in [SNV Viewer with germline mutations](/results/main/snvs-indels/variant-viewer)
and in SNV Viewer with somatic mutations if "Origin is Somatic" filter is on.
:::

3. **Position** is a coordinate of the variant in the genome (chromosome + start position).
You can copy the variant position to the clipboard by moving the cursor over the column value and
clicking on <img src={require('/img/vv/copy.png').default} width="20"/>.
4. **HGVSp** is an amino acid substitution using [the HGVS notation](https://varnomen.hgvs.org/) (“p.” prefix
(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution).
Amino acids are written in one letter code.
When hovering over the value of the "HGVSp" column for a variant, you can see the transcript which the substitution
was predicted for and HGVSp with amino acids in the three letter code:

<p align="center">
<img src={require('/img/vv/hgvsp_tooltip.png').default} width="350"/>
</p>

:::note
If you prefer to use amino acid three letter coding, [add](/results/main/snvs-indels/edit-table-columns#add-columns)
the "*HGVSp (long)*" column in [the Edit Table Columns window](/results/main/snvs-indels/edit-table-columns)
and [hide](/results/main/snvs-indels/edit-table-columns#remove-columns) the "*HGVSp (short)*" column.
:::

4. **Alteration** is a nucleotide substitution (reference allele > alternative allele).
5. **HGVSc** is a nucleotide substitution using [the HGVS notation](https://varnomen.hgvs.org/) (genomic position of
the substituted nucleotide + reference allele > alternative allele). Starts with a prefix: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence). When hovering
over the value of the "HGVSc" column for a variant, you can see the transcript which the substitution was predicted
for (the transcript IDs from Ensembl (ENSTxxxxxxxxxxx) and RefSeq (NM_) are given):

<p align="center">
<img src={require('/img/vv/hgvsc_tooltip.png').default} width="300"/>
</p>

6. **Exon** is the exon number in the transcript which is affected by the variant.
7. **Depth (Alt/Ref)** is a sequencing depth; the total number of reads of the sequence overlapping the variant
position for this sample. **Alt** is the alternative allele reads count, **Ref** is the reference allele reads count.
Also the percentage of the share of the alternative allele reads count among all reads (Alt/Depth) is shown.

The column cells are colored depending on the depth value:

import Highlight from '@site/src/components/HighlightBlackText';

- <Highlight color="#DEF1DB">green</Highlight> if depth > 60;
- <Highlight color="#FFFAEB">yellow</Highlight> if 20 ≤ depth ≤ 60;
- <Highlight color="#F9DAE0">red</Highlight> if depth &lt; 20.

8. **GT** is a genotype; allelic values (0 - reference allele, 1 - first alternative allele, 2 - second alternative
allele etc.), divided by ”/” (for unphased genotype) or “|” (for phased genotype).
9. **Consequence** is the effect of the variant on genes.

#### Consequences of variants in the annotation {#variant-consequences}

<table>
  <tr>
   <td>Consequence</td>
   <td>Description</td>
   <td>Impact*</td>
  </tr>
  <tr>
   <td>Coding sequence</td>
   <td>The variant hits a coding sequence</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Downstream</td>
   <td>Downstream of a gene</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Feature elongation</td>
   <td>A sequence variant that causes the extension of a genomic feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Feature truncation</td>
   <td>A sequence variant that causes the reduction of a genomic feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Frameshift</td>
   <td>Insertion or deletion causes a frame shift</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Incomplete terminal codon</td>
   <td>A sequence variant where at least one base of the final codon of an incompletely annotated transcript
   is changed</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Inframe deletion</td>
   <td>One or many codons are deleted</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>Inframe insertion</td>
   <td>One or many codons are inserted</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>Intergenic variant</td>
   <td>The variant is in an intergenic region</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Intron</td>
   <td>The variant is located in the intron</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Mature miRNA</td>
   <td>A transcript variant located with the sequence of the mature miRNA</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Missense</td>
   <td>Variant causes a codon that produces a different amino acid</td>
   <td>MODERATE</td>
  </tr>
  <tr>
   <td>NMD transcript</td>
   <td>The variant is located in the transcript susceptible to nonsense-mediated decay</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Non coding transcript</td>
   <td>The variant is located in the non-coding transcript</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Non coding transcript exon</td>
   <td>A sequence variant that changes non-coding exon sequence in a non-coding transcript</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Protein altering</td>
   <td>A variant which is predicted to change the protein encoded in the coding sequence</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Regulatory region ablation</td>
   <td>Deletion of a regulatory region</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Regulatory region amplification</td>
   <td>Amplification of regulatory region</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Regulatory region variant</td>
   <td>The variant hits a regulatory feature</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>Splice acceptor</td>
   <td>The variant hits a splice acceptor site</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Splice donor</td>
   <td>The variant hits a splice donor site</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Splice donor 5th base variant</td>
   <td>A variant causes a change at the 5th base pair after the start of the intron in the orientation of
   the transcript</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Splice donor region variant</td>
   <td>A variant falls in the region between the 3rd and 6th base after splice junction (5' end of intron)</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Splice polypyrimidine tract variant</td>
   <td>A variant falls in the polypyrimidine tract at 3' end of intron between 17 and 3 bases from the end
   (acceptor -3 to acceptor -17)</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Splice region</td>
   <td>A sequence variant in which a change has occurred within the region of the splice site, either within 1-3
   bases of the exon or 3-8 bases of the intron</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Start lost</td>
   <td>Variant causes start codon to be mutated into a non-start codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Start retained</td>
   <td>Variant causes start codon to be mutated into another start codon</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Stop gained</td>
   <td>Variant causes a STOP codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Stop lost</td>
   <td>Variant causes stop codon to be mutated into a non-stop codon</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Stop retained</td>
   <td>Variant causes stop codon to be mutated into another stop codon</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>Synonymous</td>
   <td>Variant causes a codon that produces the same amino acid</td>
   <td>LOW</td>
  </tr>
  <tr>
   <td>TF binding site</td>
   <td>Transcription factor binding site (TFBS)</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>TFBS ablation</td>
   <td>Deletion of a TFBS</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>TFBS amplification</td>
   <td>Amplification of a TFBS</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Transcript ablation</td>
   <td>Deletion of a transcript</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Transcript amplification</td>
   <td>A feature amplification of a region containing a transcript</td>
   <td>HIGH</td>
  </tr>
  <tr>
   <td>Upstream</td>
   <td>Upstream of a gene</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>UTR3</td>
   <td>A variant hits 3'UTR region</td>
   <td>MODIFIER</td>
  </tr>
  <tr>
   <td>UTR5</td>
   <td>A variant hits 5'UTR region</td>
   <td>MODIFIER</td>
  </tr>
</table>

Cells of "Consequence" column are colored depending on the impact*:
- <Highlight color="#F9DAE0">red</Highlight> if the impact is HIGH;
- <Highlight color="#FFFAEB">yellow</Highlight> if the impact is MODERATE;
- no color if the impact is LOW or MODIFIER.

Values that include multiple consequences are indicated with an asterisk.
You can see the full list of consequences by hovering over a cell:

<p align="center">
<img src={require('/img/vv/consequences.png').default} width="250"/>
</p>

***Impact** is the predicted effect of the variant on the protein. The column is not added
to the table by default, it can be added in [Edit Table Columns window](/results/main/snvs-indels/edit-table-columns).

#### Impacts of variants in the annotation {#variant-impacts}

<table>
  <tr>
   <td>Impact
   </td>
   <td>Description
   </td>
  </tr>
  <tr>
   <td>HIGH
   </td>
   <td>The variant has high (disruptive) impact in the protein, probably causing protein truncation, loss of
   function or triggering nonsense mediated decay
   </td>
  </tr>
  <tr>
   <td>MODERATE
   </td>
   <td>The variant might change protein effectiveness
   </td>
  </tr>
  <tr>
   <td>LOW
   </td>
   <td>The variant is mostly harmless or unlikely to change protein behavior
   </td>
  </tr>
  <tr>
   <td>MODIFIER
   </td>
   <td>Usually variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact
   </td>
  </tr>
</table>

The "Impact" column cells are colored:
- <Highlight color="#F9DAE0">red</Highlight> if the impact is HIGH;
- <Highlight color="#FFFAEB">yellow</Highlight> if the impact is MODERATE;
- no color if impact is LOW or MODIFIER.

10. **External links** are links to pages with variant information
in <a href="https://www.ncbi.nlm.nih.gov/snp/">dbSNP</a>: <img src={require('/img/vv/rsId_btn.png').default} width="35"/>, <a href="https://varsome.com/">VarSome</a>: <img src={require('/img/vv/varsome.png').default} width="35"/>, <a href="https://preview.ncbi.nlm.nih.gov/clinvar/">ClinVar</a>: <img src={require('/img/vv/clinvar.png').default} width="35"/>, <a href="https://cancer.sanger.ac.uk/cosmic/">COSMIC</a>: <img src={require('/img/vv/cosmic.png').default} width="35"/> (if
COSMIC database was <a href="/settings/custom-annotations#define-annotation-type">uploaded as a custom annotation</a>), <a href="https://franklin.genoox.com/clinical-db/home">Franklin</a>: <img src={require('/img/vv/franklin.png').default} width="35"/>, <a href="https://genome-euro.ucsc.edu/">UCSC</a>: <img src={require('/img/vv/ucsc.png').default} width="35"/>, <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/">gnomAD 3</a>: <img src={require('/img/vv/gnomad.png').default} width="35"/>, <a href="http://ruseq.ru/#/">RUSeq</a>: <img src={require('/img/vv/ruseq.png').default} width="35"/> (if
RUSeq database was <a href="/settings/custom-annotations#define-annotation-type">uploaded as a custom annotation</a>).
There may be up to three buttons maximum in a cell. If the variant is found in more than three databases,
then there will be three dots <img src={require('/img/vv/three_points.png').default} width="15"/> in the cell.
By clicking on these dots, you can see all the other links:

<p align="center">
<img src={require('/img/vv/added_external_links.png').default} width="260"/>
</p>

11. **gnomAD 3 AF** is the allele frequency according to
the [gnomAD v3](https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/) database.

The column cells are colored depending on the allele frequency value:

- <Highlight color="#DEF1DB">green</Highlight> if AF ≥ 0.015;
- <Highlight color="#FFFAEB">yellow</Highlight> if 0.01 ≤ AF &lt; 0.015;
- <Highlight color="#F9DAE0">red</Highlight> if AF &lt; 0.01.

12. **Pathogenicity** is the variant pathogenicity, which can be determined either manually, by ACMG criteria, or
according to the pathogenicity base. If you have not [uploaded the pathogenicity database](/settings/custom-annotations#define-annotation-type)
or have not determined the variant pathogenicity, then the column
value will be <img src={require('/img/vv/acmg_var_btn.png').default} width="40"/> (No interpretation).

### Description of Additional Table Columns {#added-columns}

In addition to the columns described [above](/results/main/snvs-indels/variant-viewer#columns-description),
other annotation columns can be added to SNV Viewer table as
described [here](/results/main/snvs-indels/edit-table-columns#add-columns).
The columns with special features are described below:
- "**Label**" column, in which the variant significance can be determined, is
described [above](/results/main/snvs-indels/variant-viewer#label).
- "**PG**" column contains information about the phasing group of the variant.
It is called "Phasing group status" ("Common" section)
in [Edit Table Columns window](/results/main/snvs-indels/edit-table-columns#add-columns).
The phasing group (i.e. a merging of homozygous variants located within a specific genomic interval)
is indicated by <img src={require('/img/vv/pg_icon.png').default} width="35"/>.
The variants that are part of phasing group with discordant amino acid substitutions indicated
by <img src={require('/img/vv/variant_same_change.png').default} width="35"/>
and the variants that are part of phasing group with concordant amino acid substitution
indicated by <img src={require('/img/vv/variant_different_change.png').default} width="35"/>.
- "**Onco Relevance**" column contains a
button <img src={require('/img/vv/onco_relevance_btn.png').default} width="30"/> if the variant
is [onco-relevant](/results/main/snvs-indels/onco-relevance), i.e. associated with the development of cancer.
The button opens ["Clinical Guidelines" tab](/results/main/snvs-indels/onco-relevance#clinical-guidelines) of the
variant details page.
- "**Caller Filters**" column contains information about the variant quality:<br></br>1.
PASS - site contains at least one allele that satisfies all filters.
The cell is colored <Highlight color="#DEF1DB">green</Highlight>.<br></br>2.
N/A - the column value is unknown (dot in VCF file). The cell is
colored <Highlight color="#FFFAEB">yellow</Highlight>.<br></br>3.
FAIL - the variant does not satisfy one or more GATK filters. By hovering over a cell,
you can see these filters. The cell is colored <Highlight color="#F9DAE0">red</Highlight>.
- "**ClinVar**" column contains information about the clinical significance of the variant
in [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) database.
It is called "Clinical significance" ("ClinVar" section)
in [Edit Table Columns window](/results/main/snvs-indels/edit-table-columns#add-columns).
Column values are presented
as icons:<br></br><img src={require('/img/vv/affects.png').default} width="45"/> -
Affects;<br></br><img src={require('/img/vv/assoc_not_found.png').default} width="45"/> - Association
not found;<br></br><img src={require('/img/vv/association.png').default} width="45"/> -
Association;<br></br><img src={require('/img/vv/benign.png').default} width="45"/> -
Benign;<br></br><img src={require('/img/vv/conflicting_pathogenicity.png').default} width="45"/> -
Conflicting interpretations of patogenicity or Conflicting data from
submitters;<br></br><img src={require('/img/vv/drug_response.png').default} width="45"/> - Drug
response;<br></br><img src={require('/img/vv/likely_benign.png').default} width="45"/> - Likely
benign;<br></br><img src={require('/img/vv/probably_pathogenic.png').default} width="45"/> - Likely
pathogenic or Likely pathogenic, low
penetrance;<br></br><img src={require('/img/vv/likely_risk_allele.png').default} width="45"/> - Likely
risk allele;<br></br><img src={require('/img/vv/not_provided.png').default} width="45"/> - Not
provided;<br></br><img src={require('/img/vv/other.png').default} width="45"/> -
Other;<br></br><img src={require('/img/vv/pathogenic.png').default} width="45"/> - Pathogenic or
Pathogenic, low penetrance;<br></br><img src={require('/img/vv/protective.png').default} width="45"/> -
Protective;<br></br><img src={require('/img/vv/established_ra.png').default} width="45"/> - Established
risk allele;<br></br><img src={require('/img/vv/risk_factor.png').default} width="45"/> - Risk
factor;<br></br><img src={require('/img/vv/confers_sensitivity.png').default} width="45"/> -
Confers sensitivity;<br></br><img src={require('/img/vv/uncertain_risk_allele.png').default} width="45"/> - Uncertain
risk allele;<br></br><img src={require('/img/vv/uncertain_significance.png').default} width="44"/> -
Uncertain significance.<br></br>
You can see the full value by hovering over the icon.
Values that include multiple significances are indicated with an asterisk,
and the icon in this case indicates the first value from the list:

<p align="center">
<img src={require('/img/vv/conflict.png').default} width="360"/>
</p>

- "**ENIGMA**" column contains information about clinical significance of the variant if it's located in the known or
suspected breast and/or ovarian predisposition genes (such as *BRCA1*, *BRCA2*) determined
by [ENIGMA](https://enigmaconsortium.org/) consortium.
It is called "ENIGMA clinical significance" ("Other" section)
in [Edit Table Columns window](/results/main/snvs-indels/edit-table-columns#add-columns).
Column values are presented
as icons:<br></br><img src={require('/img/vv/pathogenic.png').default} width="45"/> -
Pathogenic;<br></br><img src={require('/img/vv/probably_pathogenic.png').default} width="45"/> -
Likely pathogenic;<br></br><img src={require('/img/vv/likely_benign.png').default} width="45"/> - Likely
pathogenic;<br></br><img src={require('/img/vv/benign.png').default} width="45"/> -
Benign;<br></br><img src={require('/img/vv/not_provided.png').default} width="45"/> -
Not provided.<br></br>
You can see the full value by hovering over the icon.
Values that include multiple significances are indicated with an asterisk,
and the icon in this case indicates the first value from the list:

<p align="center">
<img src={require('/img/vv/conflict_enigma.png').default} width="170"/>
</p>
