<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Alignment | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Alignment | Genomenal"><meta data-react-helmet="true" name="description" content="After the &quot;Check quality and cleanup&quot; stage"><meta data-react-helmet="true" property="og:description" content="After the &quot;Check quality and cleanup&quot; stage"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.f1704bbd.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.7c50b07b.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Workflow Details</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/workflow-details">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Analysis of FASTQ or BAM samples</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/workflow-details">Analysis of FASTQ or BAM samples</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/upload-identify-verify">Upload, identify and verify</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">Check quality and cleanup</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">Alignment</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">Variant discovery preprocessing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">Calculate coverage</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels">Somatic SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels">Germline SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">Structural variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">Copy number variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">Phenotype prediction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">Generate reports</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Analysis of VCF or GT samples</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="theme-doc-markdown markdown"><header><h1>Alignment</h1></header><p>After the <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">&quot;Check quality and cleanup&quot;</a> stage
has been successfully completed, further sample analysis (SNVs/Indels, structural variations, or copy number
variations discovery, as well as phenotype prediction) requires alignment (mapping) of reads to the reference
genome. <a href="https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_000001405.40/" target="_blank" rel="noopener noreferrer">GRCh38.p14</a>, the most recent version
of the human genome is used for alignment. If alignment
is <a href="/ngsw-docs-en/settings/settings/alignment-settings#disable-alignment">excluded</a> from the workflow, the analysis is
completed after the &quot;Upload, identify and verify&quot; and &quot;Check quality and cleanup&quot; stages have been successfully
completed. If any of the tasks listed below fails, the sample analysis is stopped.</p><p align="center"><img src="/ngsw-docs-en/assets/images/alignment-84b2df0e13e9d1760c38cd38c6991837.png" width="650"></p><p>The &quot;Alignment&quot; stage of sample analysis may include the following tasks:</p><ol><li><strong>Re-pair</strong> is launched if the following conditions are met:<ul><li>A paired-end sequencing sample is being analyzed (i.e. sequencing in which both ends of a DNA fragment are
sequenced);</li><li><a href="/ngsw-docs-en/settings/settings/alignment-settings#disable-repair">&quot;Ensure 1st and 2nd read file consistency&quot; parameter</a> is
enabled and/or the number of reads in the paired files of the sample does not match.</li></ul></li></ol><p><a href="https://github.com/BioInfoTools/BBMap/blob/master/sh/repair.sh" target="_blank" rel="noopener noreferrer">BBMap Repair</a> is used to re-pair reads.
It includes the following:</p><ul><li>Sorting lines in files.</li><li>Restoring the order in paired files: the first read in the first file must match the first read in the
second file, etc. The order is restored by analyzing the read names, which must either be in Illumina format
(an identical prefix followed by &quot;1:&quot; and &quot;2:&quot; or &quot;/1&quot; and &quot;/2&quot;) or completely identical for both reads in a pair.</li><li>Restoring paired reads, which is necessary for alignment, so that each read from one file contains a pair
in the other. Paired reads can be broken during cleanup, which leads to the forming of singletons (a singleton
is a read with a sequence that is present exactly once, i.e. is unique among the reads) that have lost their
pair in the paired file. By default, singletons are removed from the analysis, but they can be used for genome
alignment if <a href="/ngsw-docs-en/settings/settings/alignment-settings#use-singleton-reads">the corresponding parameter</a> is enabled.</li></ul><ol start="2"><li><strong>Alignment</strong> is the process of matching sequence reads to a reference sequence (the version hg38 of the reference
genome is used). By default, the alignment is performed using the community-recommended
tool <a href="https://github.com/bwa-mem2/bwa-mem2" target="_blank" rel="noopener noreferrer">BWA2 Burrows-Wheeler Aligner</a> (BWA-MEM2), which is the latest and
faster version of the BWA-MEM algorithm, which is based on backward search with Burrowsâ€“Wheeler Transform (BWT)
to efficiently align short sequencing reads against a human genome, allowing mismatches and gaps<sup id="fnref-1"><a href="#fn-1" class="footnote-ref">1</a></sup>.
The aligner can be <a href="/ngsw-docs-en/settings/settings/alignment-settings#change-alignment-tool">changed</a> in the parameters
to <em>BWA Burrows-Wheeler Aligner</em> (i.e. an earlier version of the BWA-MEM2 algorithm<sup id="fnref-2"><a href="#fn-2" class="footnote-ref">2</a></sup>), <em>Bowtie2</em><sup id="fnref-3"><a href="#fn-3" class="footnote-ref">3</a></sup>, <em>HISAT2</em><sup id="fnref-4"><a href="#fn-4" class="footnote-ref">4</a></sup>
or <em>STAR</em><sup id="fnref-5"><a href="#fn-5" class="footnote-ref">5</a></sup>. The alignment is written to a file in SAM format (i.e. a text-based format for storing biological
sequences aligned to a reference sequence).</li></ol><p>After alignment, the mapped reads are sorted by leftmost coordinates
using <a href="http://www.htslib.org/doc/samtools-sort.html" target="_blank" rel="noopener noreferrer">samtools sort</a> and written to a file in BAM format
(i.e. a binary form of the SAM format, which uses less disk space and is faster to process than SAM).
You can download the resulting file in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Alignment</em>&quot; task
details (&quot;<em>Download BAM</em>&quot;). You can also open this file in the IGV browser by clicking on the
&quot;<em>Open in IGV Browser</em>&quot; link. The BAM file is indexed
using <a href="http://www.htslib.org/doc/samtools-index.html" target="_blank" rel="noopener noreferrer">samtools index</a>. The resulting index file can be
downloaded in the same section (&quot;<em>Download BAI</em>&quot;).</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>If you want to add the alignment track obtained from your sample analysis in Genomenal to
your <a href="https://igv.org/doc/desktop/" target="_blank" rel="noopener noreferrer">desktop IGV</a>, you can do so via a link. To do this, follow these steps:</p><ol><li>Right-click the alignment file link &quot;<em>Download BAM</em>&quot; and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Upload the track via URL to your desktop IGV, as
described <a href="https://igv.org/doc/desktop/#UserGuide/loading_data/" target="_blank" rel="noopener noreferrer">here</a>.</li><li>Right-click the alignment index file link &quot;<em>Download BAI</em>&quot; and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Add the index via URL to the corresponding field in IGV.</li><li>Click &quot;<em>OK</em>&quot;. Done! The sample alignment track is added to IGV.</li></ol></div></div><p>The &quot;<em>Metrics</em>&quot; section in the &quot;<em>Alignment</em>&quot; task details provides metrics for assessing the quality of alignment:</p><p align="center"><img src="/ngsw-docs-en/assets/images/alignment_metrics-8cf2e606d7ac505043ce2db7bf72edf2.png" width="900"></p><p>For each metric, the following are provided:</p><ul><li>metric name;</li><li>description of the result of checking the compliance of a certain sample alignment indicator and the threshold
of this metric: the indicator value for the sample (e.g. Mapped reads, i.e. the percent of mapped reads in the
alignment file) and the used metric threshold, at which the alignment file is considered high-quality
(can be changed in the <a href="/ngsw-docs-en/settings/settings/alignment-settings#change-metric-thresholds">parameters</a>), are provided;</li><li>result of assessing the quality of alignment by
metric: <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVMAAABCCAYAAADqtjX+AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAA/wSURBVHhe7Z1psxRFFkDnr8zvmB9huI6Mgoqyiiju+664IMOiICgwoCIqIoq4oIIiiMATZPWJgCA8FlnGcJuYmBhyOD1RYdHv5lpZzeuu++GEIa8zs7q76nRm3sybfzr92xmjKIqiVENlqiiKkgGVqaIoSgYaK9Odx4+bj/btMAu3rW79V3qNoihKKI2U6eT1S8yfZ118Dn9ZNK7179LrFUVRfDROppcuu3+QSMtMXDVTLKcoiuKiUTL1ibSAob9UXlEUxUZjZBoqUuC1Uh2Koig2GiHTGJEC86dSPYqiKDZ6XqaxIi2Q6lIURbHR0zJNFakGoRRFiaVnZZoqUtB1p4qixNKTMq0iUspKdSrN4Ojp382BI6fPgX+TXttJTv36X3Po+M/nXNfBoz+ZEz//R3y90nl6Tqa9LNIff/q3eeOtd82zs1+ozNsrPzKbtu42Ayd/FdvKzUeffiFeB8yeO998s29ALFcnCGrvDz+a15etMJNuuc1cetll5oILLhC56KKLzaSbbzULX37N9O8/0ior1ZkLJLll53dm1px5ZtTosebCCy8UrwsuHzbMPPDQo2blqk/NoR9/FuuLgc9k7ryF4neVG9qhPek6yvDepPIxzF+02Hz82Qaz67tDrWdJaqcKPSXTXu+R0kO674GHxQeqCjysCCXHgyiBsO+8+16x7YKXlywVy9YBIvzsi6/MxJsmidcSwvUTJppP1n6ZXapHTv1mFr/+lrniiivFdn0g/Ucee8Ls3ntYrD+Ebbv3myuuHC7WnxvaoT3pOspMmzlLLJ8KP04Tb5zUuu/3HTohthlLz8i010UKdcm0gJ7ZgkWvmuOZf7X7tvWbv15+udhmwe133G0GTtTfS6ZXefudd4vXkAJ1UafUVgxIGTmPuOoqsZ1YkOqcFxeaY6f/JbbnogkyLYNYH3z4MbNzz8FKP449IdMmiBTqlmkBPbZcw25uzudfWCC2U+aSSy41G76qL/BX9EbrkATD7E/Wbkx+EPnxenHBS86hfCpMTew5cExs10bTZFrA5//01Onm8IlfxGvw0fUybYpIoVMyhZEjrwu6yX0QJAkdTiPdKj0DF2vWbXLOiVaFumlDatsFc6PM50l15uKGiTdFDWWbKtOCMWPHm6937xOvw0VXy7RJIoVOyhRiH0IJeoOhPS6ki3yleqqwdddec9XV14htlrlu1GgzZeoM89KrS83qzze2ghUELSY/OcUMHz5CLFOGNmhLugYJfjiYs/N9PkXwa8Zzc1qBQ66NgAyBuzvvujfoR4IAVWiwsekyBUYba9ZtFq/FRtfKtGkiBZdMudmkMu0QxUSQH65e15rv8z3I0599Pnn5zcmz5Z6ZNlOsV4JrQb5SXakQVHPNkSIqhnbfHTwuli9AfDv6D7Tm1lyfGX/ne5LqaGfztn4zbNjfxHqAIBSy9c178p2yWmLsuPFiPcA1L12+Uizfjkumazfk/X5Cscn0rnvua01jlJeM2WD1Cp8n8/N871J9ZWKF2pUybaJIIYdM22HS3TUM52HnoZfK+kBQY8aOE+u1gXyRsFRfCu+897FVfvQk12/eFjW1wGvpFdp6g8z9hjyAzJM+9sTTYh1w7/0Pmv2HT4plbbASYOr0Z8X6YMLEG4NGGt0kU56H0B+vMvxALVvxgTfgxz2yvf97sY52uk6mTRUp1CFTYML9/oceEeuFVMGtOtv7lerjBmXIKv0N+fp6iaG4Pq/UOc4CIu+IU6o75PNyCavK9AqjCEYTUr38qNCDlcqVaYJMC/gBou4co42ukmmTRQp1yRR27z3UCjpJdTOXGCs4hp6PPv6kWB//zjpI6pX+vuL9T8Q6Y2FFwjUjrxXbmDlrbqUeMNJiekCqe/yEG7xzv7xHqWxoz9YFw95Ro8eI9fO+pTJlmiRTKOaubUN/RMsIRypbpmtk2nSRQp0y5YayLWHiZoqdy2SXiS3oQ48VkT01ZZr494cfnZxlreu6jV+LPY6U9yNhC64hIoQklSlAau3lYNz4Ceb7gdNimVD4Lm290xD5NE2mUAhV+j4hJDjaFTJVkf6fOmUKfY7F9UveWC6WsbF85SqxnnIvlyGndPMiYWTcXmcstmmGENmFsHPPD61tqCylKcPc5Ffb94hlwPVDkksO9KTarwv4ofKto2yiTME3j809LZUrcMp0KJzgqSL9g7plun/g1NkHTg4YxdTv2j6KRIrhtStA9dqbKwbVG8ubb78n1p1LpqkQ/GCpknRtOeWQSlNlCq737mvLKtOhcIKnivRc6pbp0VO/m3vue0Csn4c/dGvilh17xCU/9ELLARCkals6hYyrJmHhwZfqzjXMr4JNDjmG+VVpskxd012+H2FRpj6JdeJYDxXpYGqXqaP+mJuWzEpSHVKk3jbUR8ZIufzaWPoc0xYM53LnIIjB9hnxWYQEO+qkyTIF133DaEcqA4NkynBeElQ7dQpVRSpTu0wz9EzJuXnTzbeIdUhLhuiF0RuTXo9wyq+NxVU30pr3j1eSNyRUxRa8AkT2+ZdbxXKdoOkydd3D5WmqdgbJlCM7JElJ1CFUFamdumWaY86URfDS+kvb0NoVeeaG5sZuLxOKa8hW8PjZHmrs4vgc+HIWsA72tTffOS+956bLNPWeHCRTBCmJykZOoapI3dQt076K0XyXvFxzgTYB82/8TSoTimv9bAHi4vMjOs97kOqpA9furIKR117XWrJTR84CG02XKdjWAbNu2ZZRrbJMIYdQVaR+6pSpS4S2XmU7rmE1ddtE5eql0UOoKrgQaRWwLIvs9qQDrCMbexnfUpwyXD/JjJcsfbv1MNcpfZWpfY0y+RJsGaUGyTRValWEqiINo06ZunpwoVs8bcEkXw/TJfLQtl1QP0k+QpJblCkExtwtgqljfpUVC2TGl9p3QW+aLcD0oDj2I6dcXTJla2X7cSCpkONAal+i0zJN+UEZJFPWlErSCiFFqCrScOqSKQ+0bd0jhOw1dy1zCtk9Qk/QNtcasp/cB7JZu2FLpUz2COyhRx5vPUw5xUoPdf7CV6JlX6boUbNNt6pYXSLJScw925UyhZggVDsxQlWRxlGHTHn4br3tDrFOCM0a5doHz/EZUpkyqRHUWAg2sae+iriA4R5HvOQ6NwsBbtyyyxmUCoXzqcjFmip8lemZVso+W0rDKJmy86mK6EKEqiKNJ5dMqYdh91PPTPNKhd5OiMhs20fpbYYeR4J0pTpck/6p9O8faH1m9DalNkMh5yUnxubqqSJVlkWF5Jr1gVTJ4Sm140JlmlGmUKdQVaRpuGRaB6Gp4BimsudbqiPmoLw+x2oC377oVFg7y/lNDN+riJVpkgNH/ym2kQqfPZF85m1TxcqPJaedxsheZXrG7Pj2gBk+Qp4SipYp1CFUFWk6nZRpzBlQrocvZuE90kW+Uj25Mkm5oFdID5hrThEYR4uk5iH1wcoCRhPTZ862ZuOywftghBEqVJWp+zNIkinkFKqKtBqdkinzdjHDas68l+qhl0lvUypjw1YXN3ao3HNRCGzK32e0hvPSdbVDD7VqToEQCOgRDQ89ggOh0ssNCU6liKRuOi1TW0DUdU97ZQo5hKoirU7dMmWYO3f+ouCEJuDKEJWSrMSWJAUQrVSmE9Cr69v+bevMIVePlb+FnrWUCz5jxMoCf+maCujRhhzBoTK1p29k6M8UgFQmSKZQVaipqEj/oC6ZFrtsUiLT/Erb5jlzEzP/WhdF1J3coNI1Qo61sSkwDcIWVNfcb9UjVZoiU1sw1HWKQrBModNCVZGei0um46+/QVwYbQN5frF5e6W972C76eogZmVA3TA3yhypdJ1Az0Yq1wk49sQ2LRFyBE3TZep6zjhyx7YzLkqm0CmhqkgH4/qSudmkMnXi2gZaF65tqZ3GdVRzyFlLdcHns2jx6+J1MQ3BVkmpXEHTZep6/66AarRMoW6hqkhlhppMXWnk6iJkN1WncH0fdTzkMbiE4DvFoOkytQVBfSOjJJlCXUJVkdoZSjJ1bR+tE+R9vrPkl+nkQx6Da9G5ytQO0zec4SW15UsJmSxTyC1UFamboSRT5t1suU8JFLGdcfXnG5NZ/u6H1gc6JIhCykCpbMqx1S5iH3JXAu6c22ZVpvG4pkfAt5qkkkwhl1BVpH6GkkxtS0cgx24l166qkGi561z6XEGslJMJCF4QxJDKVE2GXYY0ceQPkNpxHb0BTZXpmnWbrCsh2MRCZjWpXEFlmUJVoapIwxgqMnUJIec+epsQwRct73Ms2SI4lKMHmBqAIoghlUH0ROKlMjFoACoejuZ27Szj8/QFPrPIFFKFqiINZ6jIlDPtbTeea+lILK5MVL7tpa6VBvQ+NvRV65360ha6ZM+uKml3DbDc6odj1QJsW3fttX4/ujTqXBAkU1K29wvcRxzpI5Uvk02mECtUFWkcQ0WmtgxRkHN9Jb1H5hGldpAFUpfKAQ+JLeE0UJ7eiFTWx+ETv5gnnp4q1gu+aQjfkjKSPtOGVNYHw3vXhgJdtP8HZA7jDDDXihR+eEMDnlllCqFCVZHGMxRk6rqG3MEdcM3N+gIpvvOfWNjOjqHQ7bMImh4tqe2k+gpChoS+o1Rog7Z89RTwHngvrhwCoblpe1WmfJbks2XrLTl8fcv6+HtoPgPILlNAqJPXLxElCirSNIaCTF1750N6PbEgZ9uqAd/efx4CHgbfQ0PvY8rUGS1RMJwrpikoz5CbYTOJoH173yF0SOibJiigTdrmGriW4sHmGmmHa+baXVtIC0Jz07pkmvPYkoId/fJe9zI2mfK+uT/ojbu4+pqRYnkb3DNE72NSF9Yi0wKOQCFrP4lOimQnnMsvvVbxMxRkaguecPPlOF6kHR5+23pWpI7cpXIFodLKAVMHSE+6DgnX3GZuYrJZuWRaByG9XZtM6wBBL1vxQXCPtKBWmSp5Od8ydR0tErJcKRXbQX0Qki8ViaQcWhcDw+uUSDzi8h1FXZXYPKtNlmnq6QSgMu0izrdMXVHoOob4Ba6hfujaTOYUZ7+wwDvkT4EH0Hb8bwj9+484z+FKhffKeVexwawmypQ1uRw/UyUBucq0izifMmXIU/Vc/VRom/PzpbaRO5KXyrVDPZu+/iabuOiNMieb4wQA6mDXV5XTU8sgeFIFxg5VoSky5b7lRIX3P14blcPXhsq0izifMv1+4LQZN36C2Db/zt+lcrlwJVWJzSTFa7fu3BuVQb8g9wPYDlJdtWZ9cAb9Msz1sc6XJV8pEi3oRZnyWY4aPaYVtJy/aHFr40LoHHIoKlOl0SAdlst8un5z6yEjujz5ySmtCPAtZ3uwM56bY2bPnd9aTkPAKNeGhBBoi7W0tM01cG3IgGsjoMT/c83kMthz4FglgSrVUZkqiqJkQGWqKIqSAZWpoihKBlSmiqIoGVCZKoqiZEBlqiiKkgGVqaIoSgZUpoqiKJU5Y/4H+6yctwgIkxcAAAAASUVORK5CYII=" width="90"> if the sample alignment satisfies the metric threshold, or <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAATwAAABXCAYAAACZQfkrAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAwHSURBVHhe7Z39s1dFHcf7DxnwIW5FJmDJFUms8THHYignigbTVHzAISy1sCx8GAkfIISUS4YUXZXgKl6exDsMQzWNk5vvO506c76f3bO737O7n/M97x9eA3Pv3Yfv9+59fT9n97O7n1v4+6eGEEKGAIVHCBkMFB4hZDBQeISQwUDhEUIGQ2+E99sX9phbb7/TrFp9nVmyZMki+D++hu+9O3dWLEcIIRXqhfeDH937P8G1se2nP6f4CCFW1AoP4kL0JonNBcpQeoQQCZXCg7AkmfmCR11KjxDSRKXwYiK7JpQeIaSJOuFhHk4SWAyoS2qDEDJMVAlv3EdZCUZ5hJAKVcILWZH1BXVKbRFChoca4aWI7ioY5RFCgBrh1ROKuwaLIFKbhJBhoUJ4XS5U2MBuDKltQshwUCE8SVBdwzQVQkhx4eWI7iqYpkLIsCkqvJQLFTYY5REyXIoKL2d0V8Eoj5DhUkx4JaK7CkZ5hAyTYsKLPQmlOvsOHDj0dlSyMtNUCBkmRYQHUUkicoEyUl0AEgzN42OaCiHDo4jwQuXkM+8WOh/INBVChkd24cWISapHIoVICSGTQ1bhxSxUhMy3xcwLTlqUd/7iv8wvf/2cuf/BR5Lz8GPbzfEPzov98OF3rx0Q6wUPPrzN/PVv82K5cXl+92tim3jf8P5JZeocnZ0Ty4/7foSCttCm1Jeu8XltXY29HU/uNHtfnzHvnDxjPrr0idhWLFmFFxrdgZAoLGYBY9KivDML/zB33b1BfK1d88UvrTBHjp0Q+9HG/IXL5rY7viXWW/HEUzvFsuOy5b4HxPbwvuH9k8rU2XfwsFh+nPcjBrSFNqW+dI3Pa0sx9pYtu8Lccusdix9Sp85fEtsNIZvwYqI7ECKkGKGCSYry+iK8mSOz5vPLl4v1VmCgz390WSw/DhReOKWEVwfyu/f+h8x7758T2/chm/Biz7rLIbxJOjOvD8L7+PK/zUOPPC7WWefKK68yBw8fFesYBwovHA3Cq4D4tm3/mTnnMf3QJIvwYqM7kEN4YFKivD4I7/0zF826m9aLdTaBGCFIqZ5YKLxwNAmvAk8AoXOmWYQXunpaJ5fwJiUZuQ/C23vgsFm6dKlYZxOIEYKU6omFwgtHo/DAypWrg97z5MIbR0Igl/DAJCQjuwYd/tClMjm5cOkTs3nLfWL/JCBGCFKqK5YhCA99lMqkxDX2Nm3eYuZOL7RyYv7jxU0G25942qz7+nqvD8YQ6SUXntTBEHIKbxKSkbULDxPOX7t+jdg/GxAkRCnVFwOFl4YUYw8rs5ivWz41JdZbgScByFKqo05S4Y0rIJBTeCCkPY1oF97uV/aLfbvmK9eaH/9kq/g9CHKclbkmFF4aUo69Ex9eMHd/Z6NYd8UDWx9r/WBMJrxxFirq5BYe6HOUp1l4SEzd+L3vi33D148d/9Csvu6r4vd3vfiyWGcMFF4aUo89rMpCarbH3KuuvtocnHGv6icTXlfyKSG8kDa1oVl4f3nv1GIkJ/UNkR8+nTf9cIv4/Q0b74lKQ5Cg8NKQY+xh5wV2Y0htgLZxkkR4XUV3oITwQF+jPM3Ce/b53WK/ENVVj6zYbiZ9gkOUEGazzhgovDTkGnsfnL1o1t/8TbEd5G6+fuiIWA4kEV7MnlYbpYTX1zQVrcJzbSVDVFfNvbgWNZ5+ZtdIvTFQeGnIOfZe3X9oUW5SW9iNYcvd7Fx4MWfduSglPNDHNBWtwvvj0XfN1NQXRvqEaA5RXfVzrrQVCBPirNcbA4WXhpxjD4+teHyV2lozvdacnF8Qy3UuvHGSjCVKCq+PaSpahYe8KqlP0gqs7bEWwoQ46z8bA4WXhtxjz7bij7Gz/40/iWU6FV7XwgElhQdC2teARuGdOnfJ3HTzN8Q+STl2+HTGp7T08xBn/WdjoPDSkHvs4fiwa1euEtuzjZPOhNflQkWd0sIDfYryNAoPk8jSfAs+iaVdFJh/wTxM8+cBxAmBNsuEQOGlIffYc7WHNCfpbMPOhJdKNhqEF9KH0mgTnutkFNdci02SbatwPlB4aSgx9rY+ul1sz/bB2InwUkV3QIPwQF+iPG3Ccz2euk5CcZ2o4lqF84HCS0OJsffMb14Q28PcsLTVrBPhxZ5154MW4fXlzDzXoJu+4caR47RjCDnK3LYA0RapuSLDcbeaDUF4397wXfF3FwNOG5bab1JCeKG/i7GFlzK6A1qEB/oQ5bkGXVf4/mG7Ukx8jn3C4Z+2ub96KksoQxBel/jKahDC6zoNpYkm4fUhGVmT8FyraJh7kcrUca3u1pOVQ6HwwtAsPNt7kER4qQUDNAkPaE9G1iQ821YyRG2+R7fbJqUh0thbzSi8MCi8/9JsJAUhc2c5hKc9GVmL8FyZ8CGX88w4LvuBUKUybVB4YWgWHhKMpTnizoWXQy4g5DGyyz28LkKiztzkEB7AH73UfoXrjzEkeRhihCClemJPUKHwwtAsvNDfRZTwUi9UNPGNqKSyqdAa5ZUYdBK4U1bqA6K1mc+iNqmMDVtdsYIZgvDaPpBSUGLs4ZxEqb36CTx1ooSXK7qr8ImoNPapBBqE5zoZJeYAANvBAyDmsm4KLw0lxp5tj/b02hvFLIBg4eWO7ipciwVdn9Dii8YoT4PwZjwu2e6KmMu6Kbw05B57rhO0cRz82YV/jpQJFl6ueTIJLGBAbpVoIMGS/dGYpqJBeLaV1RSErPhWUHhpyD32XLt4bGlPQcIrFUlpRluaSmnhhVyy3RWhl3VTeGnIPfZc9xvj6CipTJDwUicZ9xFtaSqlhRdyyXZXhF7WTeGlIefYc+3iceVoegsv96JAn9C0gFFSeK5BmBIINuSybgovDTnHnuu1246GAl7CK7VQ0Se0RHklhee6jwKLC3v2HjSv/P6NaJ597iXrIJcOErVB4aUh19jDzWW2A0va9ll7CY/RXTtaorySwrMduQ1id0XUce3eCDlBhcJLQ46xh7naX/xql3XaBLeZ4VYzqSxoFZ626A4ro7A7/sX8maZ5RQ1RXinhuVIExtn32sSWaApsE9VNKLw0pB57kN3L+/6weOG21AYk+FLLGGgVni10zA0Eh1XiZv/wNS3SC9n3m4pSwnNdsu2aUwnFdQKL71YzCi8NKcceHmMR2dlkB3AGYNvvzyk8LdEdhCb1r44W6ZWO8koJz3YyCvCNvHzAPB2OhpLa8b2sm8JLQ4qxh6jurT8fN7fcJu+nrsDv/u3Zk2IddZzC0yIRKbJrAtFo6G/pZOQSwnO1advTOA6uuUKfy7opvDR0NfYQzb1z8sxiRHfD2nVifXUQ9b26/02xriZW4WlZqPCJ7iq0CLpkMnIJ4bn2uoasnvriWg322aubSnjY9XHPps3iMekxPL7jKedrcQmvyyPeK47Ozon9qHCNveVTU5/9zqZbWbHiy2J5G5Ddi3v2eSeeW4UnVV6CkNVPLfONEG+pR9sSwrNt4G5LEYjFle/nc1l3KuF1zfVrps3caflWN+ASXgraokbX2EsBXjvyL0N22YjC05SGErIQoKnfpaK83MJzHcM+7mU7LmyXA4G28/YovDg0CQ9zerMnTov9cCEKT8ujIQiZE8PPSnWUoNRcXm7h2e6PBSkeZytcj7Vtl3VTeHFoEN7KVasXP+xCoro6I8LTsjJbx+fxsK/97pqcwsOgs12liOgrZLtXKGgb99NKbUPArisgKbw4Sglv2bIrzO133rXYPhY0pLZ9GRGepsfCCp85MU3RXUWJx9qcwnMdz4Ov4/tSua5wHVTgOkGFwosjh/CwCIHIHefZ7Xhyp3nzrWOd5XCCXggPQGiS9JCyolF2QEMiMiHk/4wIT8tKpw3IDX0EmuYaJSg8QnTRO+H1iZCUGkJIekaEh3kn6Y+XhEPhEaKLEeHxGPfu8NkSRwjJx4jwgPa5sT6A91B6bwkh5RCFp3Wltk/wcZYQfYjCA4zy4mF0R4hOrMJDzhulF0eJHRaEkHaswgOaThPuC1yoIEQvTuEBRnp+2HaCEEL00Cq8Cs1buEqCRG1GdYT0A2/hVSCKwR84EpSxEjlE8NrxHjCiI6RfBAuPEEL6CoVHCBkMFB4hZDBQeISQwUDhEUIGwqfmPxU1Q8Zc6H67AAAAAElFTkSuQmCC" width="80"> if it does not.</li></ul><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="alignment-metrics"></a>The alignment quality metrics<a class="hash-link" href="#alignment-metrics" title="Direct link to heading">#</a></h4><table><tr><td>Metric</td><td>The metric threshold value at which the alignment file is considered high-quality (this is the default value, can be changed in <a href="/ngsw-docs-en/settings/settings/alignment-settings#change-metric-thresholds">parameters</a>)</td></tr><tr><td><b>Mapped reads</b></td><td>The alignment file contains at least 85% mapped reads.</td></tr><tr><td><b>Multiple alignments</b></td><td>The alignment file contains no more than 15% multiple alignments of the same read per genome.</td></tr><tr><td><b>Forward/reverse balance</b></td><td>The difference between forward and reverse strand reads is no more than 10% in the alignment file.</td></tr><tr><td><b>Paired mapped reads</b></td><td>The alignment file contains at least 80% paired mapped reads.</td></tr><tr><td><b>Paired properly mapped reads</b></td><td>The alignment file contains at least 75% paired properly mapped reads.</td></tr><tr><td><b>Coverage per nucleotide</b></td><td>The minimum acceptable coverage per nucleotide is 0.1.</td></tr></table><p>If the metric value does not meet the threshold in the sample alignment file, the file is marked as not meeting
the metric criteria, i.e. the alignment is completed successfully, but the stage status indicates the number of
metrics that do not meet the criteria. If no sample reads are aligned to the genome, the &quot;Alignment&quot; stage completes
with an error and the analysis is stopped.</p><p>If at least one of the following stages is included in the sample
analysis: <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#somatic-snvs-discovery">somatic SNVs/Indels discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#germline-snvs-discovery">germline SNVs/Indels discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#sv-discovery">structural variations discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#cnv-discovery">copy number variations discovery</a>,
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#enable-phenotype-prediction">phenotypes prediction</a> or
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#polygenic-risk-scores-calculation">polygenic risk scores calculation</a>,
then after the &quot;Alignment&quot; task is successfully completed,
<a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">the &quot;Variant discovery preprocessing&quot; stage</a>
will start.</p><ol start="3"><li><strong>Determine Kit</strong></li></ol><p>If the user has defined the sample sequencing type as WGS (whole-genome sequencing), then the capture kit
determination is not performed, and the sequencing type can remain WGS or be redefined as Low-pass WGS (whole genome
sequencing with low coverage). WGS and Low-pass WGS are differentiated by the average sample coverage: the
sequencing type of samples with an average coverage less than 4x is defined as Low-pass WGS, and above or
equal to 4x as WGS.</p><p>If the user has defined the sample sequencing type as Targeted selection, the sequencing type can be defined as
Panel (sequencing using a targeted panel) or WES (whole-exome sequencing). Next, the most suitable capture kit
(i.e. the targeted panel used in targeted selection) for the sample is determined or the coverage statistics
of the sample by the capture kit selected by the user are calculated:</p><ul><li><a href="https://bedtools.readthedocs.io/en/latest/content/tools/genomecov.html" target="_blank" rel="noopener noreferrer">bedtools genomecov</a>
computes coverage summaries of aligned sequences for a reference genome and reports depth
in <a href="http://genome.ucsc.edu/goldenPath/help/bedgraph.html" target="_blank" rel="noopener noreferrer">bedGraph format</a> (File A).</li><li><a href="https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html" target="_blank" rel="noopener noreferrer">bedtools intersect</a>
screens for overlaps between File A and either the capture kit selected for the sample by the user,
or each <a href="/ngsw-docs-en/settings/capture-kits">capture kit that is built into or uploaded into the system</a> if the user has
not selected a capture kit. For a capture kit, the coverage depth, kit length, intersection length, complement
coverage depth, and complement intersection length of aligned sample sequences with capture kit intervals
are calculated.</li><li>For a capture kit selected by the user, or for the best matched capture
kit, <a href="https://bedtools.readthedocs.io/en/latest/content/tools/complement.html" target="_blank" rel="noopener noreferrer">bedtools complement</a> returns a file
with all intervals in a genome that are not covered by at least one interval in the capture kit file (File B).</li><li><a href="https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html" target="_blank" rel="noopener noreferrer">bedtools intersect</a>
screens for overlaps between File A and File B by computing the coverage index, depth, and intersection length
of these files.</li></ul><p>As a result, if the user has not selected a capture kit and if the best matched capture kit for the sample was
found, the sample is further analyzed with this kit. If a suitable capture kit could not be found, SNVs/Indels
filtration (variants discovery in the same genomic intervals in which targeted sequencing was performed)
and copy number variation discovery will not be performed for the sample. The file with the statistics of the
capture kit selected by the user or determined by the system can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Determine Kit</em>&quot; task details (&quot;<em>Download Capture kit stats JSON</em>&quot;).</p><div class="footnotes"><hr><ol><li id="fn-1"><a href="https://doi.org/10.1109/IPDPS.2019.00041" target="_blank" rel="noopener noreferrer">Vasimuddin M., Sanchit M., Heng L., Srinivas A. Efficient Architecture-Aware Acceleration of BWA-MEM for Multicore Systems. IEEE Parallel and Distributed Processing Symposium (IPDPS) (2019)</a><a href="#fnref-1" class="footnote-backref">â†©</a></li><li id="fn-2"><a href="https://doi.org/10.48550/arXiv.1303.3997" target="_blank" rel="noopener noreferrer">Li H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2 (2013)</a><a href="#fnref-2" class="footnote-backref">â†©</a></li><li id="fn-3"><a href="https://doi.org/10.1038/nmeth.1923" target="_blank" rel="noopener noreferrer">Langmead B., Salzberg S. Fast gapped-read alignment with Bowtie 2. Nature Methods 9, 357:359 (2012)</a><a href="#fnref-3" class="footnote-backref">â†©</a></li><li id="fn-4"><a href="https://doi.org/10.1038/s41587-019-0201-4" target="_blank" rel="noopener noreferrer">Kim D., Paggi J.M., Park C. et al. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol 37, 907:915 (2019)</a><a href="#fnref-4" class="footnote-backref">â†©</a></li><li id="fn-5"><a href="https://doi.org/10.1093/bioinformatics/bts635" target="_blank" rel="noopener noreferrer">Dobin A., Davis C.A., Schlesinger F. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29(1), 15:21 (2013)</a><a href="#fnref-5" class="footnote-backref">â†©</a></li></ol></div></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Check quality and cleanup</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Variant discovery preprocessing Â»</div></a></div></nav></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2025 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.f1704bbd.js"></script>
<script src="/ngsw-docs-en/assets/js/main.7c50b07b.js"></script>
</body>
</html>