<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Variant Details Panel | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/main/snvs-indels/snv-details-panel"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Variant Details Panel | Genomenal"><meta data-react-helmet="true" name="description" content="Click on the variant row to see a variant details panel under the table:"><meta data-react-helmet="true" property="og:description" content="Click on the variant row to see a variant details panel under the table:"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/main/snvs-indels/snv-details-panel"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/main/snvs-indels/snv-details-panel" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/main/snvs-indels/snv-details-panel" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.e7631c00.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.9097e47c.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">🌜</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">🌞</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/overview">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Main</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/main">Main</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">SNVs/Indels</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/snvs-indels">SNVs/Indels</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer">SNV Viewer</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel">Variant Details Panel</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/basic-filtering">Basic Filtering of Variants</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/advanced-filtering">Advanced Filtering of Variants</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/edit-table-columns">Edit Table Columns</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page">Variant Details Page</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/pathogenicity">Variant Pathogenicity</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/snvs-indels/onco-relevance">Variant Onco Relevance</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/structural-variations">Structural Variations</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Copy Number Variations</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/main/alignments">Alignments</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotypes Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Workflow Details</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Variant Details Panel</h1></header><p>Click on the variant row to see a variant details panel under the table:</p><p align="center"><img src="/ngsw-docs-en/assets/images/details_panel-d47880aa5aa579358ed63f6078272270.png" width="900"></p><p>The panel can be resized by hovering over the top border of the panel and dragging down or up.
You can also collapse or expand the panel by
clicking on <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACwAAAASCAYAAAAg9DzcAAABLUlEQVRIDdWVsQ3CMBBFMwQ1PS0lNT0lJRNQ0bEBDRU7IFEwBRJiDWAPoxfpR050sZ3ggHISuojg/1++z6FwI6tiZLxu3MDP19sdjqe/hY43DKGqJbxab9xkOnPzxfKn4EDKmx6qCvh2f5SwAOuz3e2jTxwST7lHqvJTD6VcASOukdBC+lBp+6nKD68QLIw1YCVyvlxLUAkJPCam9bFupZp6dkxgDIdIuy1VxjG1WoElgAlb1Uy7iwla7JqvwXVqqmKhR4H5kZW2DLkXqhyp+vpJwFqAOW8OP6nQobTePH1SlT+9E7AWphzK5sHiwbqOkfz83gsYAWtMgGIHmjMf+zPwgWLXvYElDHgTUCOTK1V50b8GRsRKO2eq2YElCDig3x4s6Vk9S8KW8FDffQB0pjRcbUu4nQAAAABJRU5ErkJggg==" width="30">or <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACkAAAARCAYAAABASYU2AAAA+UlEQVRIDeWUvQ3CQAxGMwQ1PS0lNT0lJRNQ0bEBDRUjISHWAPY49IqLLo4d7reIsBQlciJ/z5/P6dwMopsBo/tPyNf7446ns7tcb9WGVNVJwBbLVX/t9gcHdGlUgQRkvdn2cCEo+VJXiyCBk+4BSE5CcwRyIxvy/niOnAvHqzWQ62oyJOLAxI6UZqSrqeNPgtRGGzNGy1UaiIkoSMu9WBEPIpuMHf9PSFnYL4YXTr1brpK3woTUFoPOp4pZIlpeNj/l6gjSGi1Fa4elJY0YQPIy3Fqe2eSWgabmanjeB5DA+N8L9ocftgSlNrBoasaMIH1nraG0+pb2F3Unu19fQerdAAAAAElFTkSuQmCC" width="30">, respectively.</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>On <a href="/ngsw-docs-en/settings/profile#variant-details-bottom-panel">&quot;Profile settings&quot; page</a>, you can adjust whether
the variant details panel is expanded or collapsed in SNV Viewer by default.</p></div></div><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="panel-tabs"></a>Details Panel Tabs<a class="hash-link" href="#panel-tabs" title="Direct link to heading">#</a></h2><p>The panel is divided into 11 tabs (or 10 if there are no <a href="/ngsw-docs-en/settings/custom-annotations">custom annotations</a> in
the system):</p><ol><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#common-tab">Common</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#gene">Gene</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#clinvar-tab">ClinVar</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#gnomad-tab">GnomAD</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#exac-tab">ExAC</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#other-frequencies-tab">Other frequencies</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#conservation-tab">Conservation</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#protein-function-effect-tab">Protein function effect</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#protein-function-effect-tab">Protein function effect (additional)</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#other-tab">Other</a></li><li><a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#custom-annotation-sources-tab">Custom annotation sources</a> (if
at least one custom annotation was <a href="/ngsw-docs-en/settings/custom-annotations#add-custom-annotation">added</a> to the system
before the variants in the sample were annotated)</li></ol><p>On the panel, only those tabs are visually highlighted that include at least one non-empty variant annotation
field. For example, &quot;ClinVar&quot; tab filled with data looks
like this: <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAiUAAAC2CAYAAAAGAP0vAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAABXVSURBVHhe7d0JVxRXGodxv/83mIlZzMyYmbjjgoCCRokbuOKGIZqIGEBNzBeo4d85JJ3y7XrvW8vt6uapc37nTM5gdQNN19P3Vt06VLCxsbGxsbGx9WAjStjY2NjY2Nh6sRElbGxsbGxsbL3YDr36ebsAAAAYN6IEAAD0AlECAAB6gSgBAAC9cOj3338vyspfZH0NAABAhNcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkAAMjC6wuiBAAAZOH1BVECAACy8PqCKAEAAFl4fUGUAACALLy+IEoAAEAWXl8QJQAAIAuvL4gSAACQhdcXRAkwJltvt4vVu4+Lufnl4vry3WJ9/WXx8eNH82sBYBp4fUGUAJltb+8Wp88sFv/87Hjxj38e+5uv/zVTPH78wvx3ADDpvL4gSjAWGhF4+3a72Nj4qVhZXfvEw4fPis03b4t3796b/35S6Xs6+s2FT2Jk2GeHjxerez8D698DwCTz+oIoQRa//vpbsba2XsxevFYc+XrGHCUY5fDnJ4pv/nuhuPrdneLnn9+Y+2/bzs67Yn5+uTh3/qrr/oOn5j7Kfvvtt+LcuSvm91j25Veni40ffzb3M8mePPnB/BmOMs5Ro62tX4qLc9fN52W5fn1173fM9BtQxesLogSd+nHvwDpzdmnw6d86+NahA/aVq7eL7e0d8zHboAPSka/PmI9fduXKbXMfZfpZfPnVKXMflkuXb5j7mWSKyq+OnDa/X8upU5cGMWftq2u3bj80n9MoihJrPwD+4vUFUYJOPHu2MRjdiIyIRCl0ZmevdRInXUTJvXtPzH8/yrfH5or37z+Y+5pUCgyFhvX9WhQwuUbHhkWfp2JT0WntC8BfvL4gStCqzc23xcm9N/MuY6RM0zvXrq20euVKH6Lkf99enLpzauTOnUeh14dGLKz9dCk6onP8xHzx4cOv5r4A/MXrC6IErbl3/0nxxRfp0xNt08jC1lY7r9cuouTRo+ehg/HZs0vmfibdHwf8tJ+tjGMKZxLCCZhEXl8QJWhMJ7EuLt7MOjoyikLixYtX5vOM6CJKdPKsprSsfZTpZznNV+AouKzv25J7CkcBpEu2rediUWCNY4oJmEReXxAlaOT9hw/JV5Tk8sWXp4qnTzfM55uqiygRTeGknPSrqzkUe9Y+poF+Dn0diYiO5ChgxnUyLjBpvL4gSlCbDpo6eFpv1OOmMHn+/EfzeafoKkrk/v2nxedfnDT3pQP16TOXB6Mq1r+dFlrN9l//Pmv+DCw5p3AiUzf6OgWWtR8An/L6gihBLQqS8xeuhj7t5qaD3uvXW+bz93QZJaIrhnTJr1Zw1ciJIurEiYXBVUvW10+jC7PfmT9PS64pnOjUjdbcefPmrbkvAJ/y+oIoQS0636HJ2iOKGUWDFqfSJ1ONaigEZO3xenHjxr3izMziYE0S69+nOnFyYTDFZH0PVbqOEsRO/NXX6XVi7adN0akbhZW1HwA2ry+IEoS9ePFy8MneepP26Oqc6MJnusxY8aJLf619VtHBbHn5rrnfKkRJ93Z2dt0l94flOHcjOnXz4OEzcz8AbF5fECUI0aiDRh+sN+kqCgrFSJ1Ri31110DRFEl06J8oyWNh4XvzZ2rp+iqX6NTNv/9zrvjll+5WFQamkdcXRAlCbt1+EI4CHdybnHRadvPm/fDU0cKl7819jUKU5PH02UbyCJhed11O4USnbhRU1n4AjOb1BVGCZLqrrz4dWm/Qo+jr655sWuXOyqNQmEQ/ZRMleUTWbpEup3AilykfPnyiePJkfDcLBCaV1xdECZLphmPWG/QoTa5+SaHpoMioTeSGaQc9SnR1lS7b3Xzztnj48NmfJyJ3MV2hn5/1c7V0OYUTWdBNIdXVZdtarl4/a9FJ34ol/R3pvyfxUnE9Z71+VlbXBvR60utq2i97h83rC6IESaKfaDUk/7jjT5K6L8yx43Pm41uOHZ9PPqelqyjRSbfar+e//5sdjExZ+yjT1+nrrf0M8/apc3Y0zaXLXKtiTyNUei1oKq+Ne/Osr78cuW5LWVdTOLqsV9+39ZiWNu/grMi6+t2dwc80ZSpLPwOdJzV78drgCqYu7rmjE8ut11DZqJPIdSL73Pxy5Qnx03pvJ1Tz+oIoQRJdZRAZlYiew1HXw7035dRpnMidXLuKktRRAT22noO1j7LU5zpqn01uoqiD6NzeAWx3t/6nXoWigtHav6WLKZzI1I0CSiFl7SeVDsa6iWTTS95Fr/+Zs0vFy5evzceqQ/uzHqus/NrXCJtGMFPiiig5mLy+IEqQJDK0rU9xOtBZ+2lb9ICWulz5QYmSm7ceJJ9oWkUH17W19b/tOyIyNdjFFE7k9R0ZcSvTQVvfaxc3rlRUKS7b+NurEyUaTT11+rL5dRai5GDy+oIogSs6tB05d6MNy9/fNZ+HZXb2mrmPsmmPEh0cNQVRZ3RkFH1i15VR5eeXQiNYGsmy9lum59zmFE6u17dCSlNo1j7bpJGcpkvfR6NEQaK7dFtfMwpRcjB5fUGUwBVZebPrtSQskXMSjp+YT5qDn/YoiZ4knEphUufuxpqO0f1trH1a2pzCiUzd6ByJjY2fzP1U0aXPbUzVpNLvocmHg2iULF25Zf7/VYiSg8nrC6IErsgCVxoGt/bRJQ2l33/w9M+z+6voaoaUg9k0R4lOQG5jymaUujdD1NSatT9Lm/eciUzdpEbtMP0s6q6A3ETdQJRIlGwMRrniwUWUHExeXxAlqKQ3YL0RW28qZfq02XTYuC+mNUpevdocHAys/79NuioqesDRCFvq4mVtvdaiUzep5yTt06W8kbsht02xoGiwnluV1ChZ2ns9R26sOIwoOZi8viBKUElrVejEVetNpSxydUvfTWOU6ICvKz66mLYp02NEP6VrBCuyzHsbo3KRqZvonYp13k5kFGaYnpOCQr/XfXVHt3TyqZ6L9RxHSY2Sc+euJL32LETJweT1BVGCSi9+eJV8pYBO4tvdnY43mWmMEo+mGE6eXCjOnb86oBMXU8/VsdS5SiUSCW1M4USiIXoeS+RyddH3fWZmqfhh729uVEQois5fuJr8MxLFjM5psfY3SmqUNEGUHExeXxAlqKRzNVLfAPVGZu1jEh2UKNEBS+uMaETM2rdo9Es3YYwcCKXOeh6R6RQ9nyZTOPqeU6dW9FiRK36il6rr9xc5Dyd64mzqVWf7mkaJRld1MrWmC4fPwdH/1utV53bpcvTo+TmYfF5fECWoFDn5MHJw7ruDECVHv7kQmo6os6aJLte29lUlco5CkymcyIKA0VGZyI0G696OITISo9/1zs6uuR9L3SjRqOrtOw+Ljx8/mvsFvL4gSlAp9SAq0ZMA+2zao0RD51tb8b9tnScSmZI4M7No7qdKl7EwLBI/+lprH6OkXrFW90oliZyDEz3fq06U1H1N4WDx+oIoQaVIlEzLlTcyzVGiA+GLF/WWSY+evFnnPCPd9C/1btR1p3CiUzcKJWs/o+jnpFjSzec0jaGDvB6vPHrS9B46qSOZ+h603pC1D0s0SuqO9uDg8fqCKEElnV1vvQlZiBJfH6Kk6X2JIlMTke9jWNdr40RGY3TArTrnJmo/WLQs//Z2+pSKRaMshz9POxk58vcZiRKdO/QseCItDi6vL4gSVEp9c6pzUmOfTWuUaJSkzoqkw3TFhIbqrf2X6RwDXcFl7afKEy3wdjg1fOJTOJGpm+hJojlFVjPuKkr0waWt1XUx/by+IEpQiSixv99hkxQlbV2GqQO1tf+yuq8L3UtFt/K39lkWncKJTN0ojBRI1n76IBJvXURJdFoI8PqCKEGlg/rmNK1Rok+11j6iUr+PJrGq8y2sfVoiUziRqRuFkQLJ2s+4aRpI68lYz9vSRZRoQb7Nmica42Dy+oIoQaXFpfQbbUXe9PpuWqMk8lyr6Hdt7b+sSZREpiYiUziRqZumJ6J2QScCr6ysFUePnjef8yhdRAkLoCHK6wuiBJVSD6JClPiIknSRc1c08pFyhUzkyh6dzBtdCbUNGpnR6IOu3NHVNZoq00q7GpWIXI5dRpSgD7y+IEpQSYtfWW9GlrYOeH1AlFTLESWi2+9b+7WkrCWiKcbUqZuuD7iaftHN8nTJsKJDK7SmPrc6iBL0gdcXRAkqpR58pM9XKUQRJdVyRYmuFEq97X/KpbupJ+iKgsjaRxM6gN+6/WCwwmqXAWIhStAHXl8QJagUWQfh+In5qbmXBVFSLVeU6PWk15W17zJvCicyddPGpdPDdnffDe4xFF2mv01ECfrA6wuiBJU0t625bOsNqUw34WpzkalxIkqq5YoSidx/qWoKJzJ101Zg6x4wuhdM6p22u0SUoA+8viBKUElLhGupcOsNqazvazpEECXVckaJbhr41ZG0O+JWTeFEpm7q3EiwTOeMnL9wtdNpms/2/uZS90+UoA+8viBK4IosNd/HSyjrIEqq5YySyI3nRk3hRKZuFECRuydbomuIRGhq6fSZy4Orc3R1kH7G1teVESXoA68viBK4IsPn41hsSgdljebowOw5eerSYH7f2s8woqRaziiRO3ceJY8IWFM4kakbBVDTZdPvrKQ/X4v+raZNdVXO3PxysbK6Vrx6tfnJlJJ+tkQJJonXF0QJXLrluW59br0plY36pNqlyAHn1F6UpBxwiJJquaNEC6NpgTTrMcqsKZzUqRu9jhRAw/826u3eY6eOyuzT4357bG5wZc6bN2m/dyFKMGm8viBK4NKbjt58rDcly4m9T3fvP3ww99WFyLkCqQdloqRa7igRLSVvPUZZOYwjUzd1bu5XFhnV0ddp9G5zs95jEiWYNF5fECVIElnESqtORt4Am/jjJMi0A3LkRFyipNo4okShkXqwH57Cidy0LnIPHYtG4TQaZ+27TN+LFk6z9pOKKMGk8fqCKEGSyMFf9MlUw9jWvtq0cOl78/Etek761Gztp4woqTaOKInc3Xf4d72wkPYaKY+w1BEZlVEA6YRYaz+piBJMGq8viBIki9zITHT1QdM33Spa2C11tU+JXBlElFQbR5RI5NwQnWu0s7M7WD3V+pqylBVhPannX7V1+XxkFIgoQR94fUGUIFnkU5nowLB05dZgASlrf028fr2V/KlZoit0EiXVxhUlkYOwAib69dZjRqT+jWjUsY1b/kemVYkS9IHXF0QJQmYvpp9UKgoTjVC0OWLy00+byUPk+7TWSuQyT6Kk2riiJDLyoXObUm+R0NbIRWqURH6/o+jSe12Cb+3fQpSgD7y+IEoQoiCInFuyT5c71r3CYNjDR88Hd1O1HmOUOgdGoqTauKJEFLnWYzWh0FHwWI8XkRolmuLRVI+1j1TRtVCIEvSB1xdECcK0lkLkzXCfbkZ2efFmsb2ddrLpML2B61LjOo+rk2GtfVYhSqqNM0q0imnbN7ZrayXi1CXx9Tpush5KdPpSiBL0gdcXRAnCmi6hrTfk/YWiNHpinXOilSt1DsjS0q3Bjf6s/aT4Zu8TcJ0IIkqqjTNKdBDUwdB6vDoUOAod67GiIlff1H1t1pm+FKIEfeD1BVGCWup8Uqui+X9NC2lqps5oiEUnt+oKHev5e4iSauOMEkn9OaZo+8Causib6LFTpzWb3nGYKEEfeH1BlKA2Tak0GcXokiJndXXNfN4piJJq444SjaJFLgevoitYrMeoK7Kiq2ik5uLc9cEIiDVqqGXnddfipn9rRAn6wOsLogSNPF//MXziadcUJDdv3jefbyqipNq4o0S3MTh2fN58zIjopeIpIou8lSlm9Pek32fKqKFe65+xTgkmiNcXRAka69OIiT51rqzUHyHZR5RUG3eUiEYPrMeMUNh0cZ+m5eW7rU1DVtFJ3Gdm0gKCKEEfeH1BlKAVW1vbg6tjrDeuXHRgrnsOSRlRUq0PUZJ6pUsVhY2176YUOl3/Pehkca1VkhoQRAn6wOsLogSt0Xz4tWsrrV+u6dEnUt1pVWFkPa86iJJqfYiSyM3vLAoahY217zbo9aiDtvXYTWl6SCeb63GIEkwSry+IErROVxOcOn05y/C1po3W1tbN59EEUVKtD1Ei0ZNKhyloIqv81qEw0YiG9fh1KUiGF14jSjBJvL4gStAZfQo9e+7K4GQ86w2tLh2EtMbD/ftPO7mvjhAl1foSJdG7V+/Ta6jJ4mURmsq5cvV24xFEPWfFwu7uu7/tnyjBJPH6gihB5/SmpZNPjx2fqx0oekM+8vXMYDG1Npar9xAl1foSJRJZF2SfQqbLqRuLFkpbXLwZvpRZr32dkDtqWXqiBJPE6wuiBFlpZENRsXr3cTE3vzx489NwtA6qwzTkff7Cd8WNG/cGJ69qhVdrf8Ck0d+A1iTRSbYnTy4MXu+KJMWHgkX//Z+j5wdrlzx69JzXPqaK1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68viBIAAJCF1xdECQAAyMLrC6IEAABk4fUFUQIAALLw+oIoAQAAWXh9QZQAAIAsvL4gSgAAQBZeXxAlAAAgC68vDpW/AAAAYByIEgAA0AtECQAA6AWiBAAA9MKhgo2NjY2NjY2tBxtRwsbGxsbGxtaLjShhY2NjY2Nj68FWFP8H8u7IyQlLeLkAAAAASUVORK5CYII=" width="80">, and empty - like this: <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAiQAAAC3CAYAAAAinkW0AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAABOISURBVHhe7d17c9NIFsbh+f4fa4CBhUnIhZgkBHJ1CBBCCIH5U6tXVHY9yrHcp3U5sv1T1VO1UxtasqOoX/X1j4KDg4ODg4ODI/ggkHBwcHBwcHCEHwQSDg4ODg4OjvCDQMLBwcHBwcERfvxxcvapmOf65g4AAKBTVuYgkAAAgEFZmYNAAgAABmVlDgIJAAAYlJU5/ri7uyse1P/Pf/75BwAAoJV6IJnNHg8IJAAAoFcEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAjdPv9rjg9uyz2JkfF0dFZ8enTtflzALAqCCTAiNzd/Si2tvaLP59sFH/++W/PX2wXl5f8zQFYTQQSjNL373fFly9fi5PT6SMXF1fFt2/fi/v7e/PfLit9ppcvdx8FkVlPnm4Up+V3YP17AFhmBBKE+/nzVzGdfirevHlXPH++bbYOzPP06Wbx8tVucXh4UtzcfDPL79rd3X0xmRwVO7uHC52ffzTLqPv161exs3Ngfsa6Z39tlWHtxixnmV1efja/w3kiW4tub78Xb/bS7gE5Ojotf8d2WQB+I5AgjCrV7e2D6q3fqnhzqLI+ODyu7lXrnF1QZfT8+ZZ5/rqDg2OzjDp9F8/+em2WYXn79r1ZzjJToPwr8XuV16/fVkHOKqtvx8cX5jXNo0BilQPg/wgkGNzV1ZeqVcPTEuKlkKMWF92z1jW00UcgOTu7NP/9PH9v7BX39z/NspaVwoVChvV5LQovQ7WKzfJep4LmKrZoAV0jkGAwGiOxqQd5j0GkTl06796dmNeTawyB5NXfb1ZuDI2cnExd94daKqxy+uRtydnYnBQ/f65WeAT6QCDBIM7OL4tnz9K7JLqmFoXb225aS/oIJNPplasiVleXVc6yW4Zum2UITcAyIpCgVxqwur//YdBWkXkUIrpYz6OPQKKBslU3llHGI+V3ucozbRS2zM9tGLrbRuGnmpZtXIslqlsJWEYEEvTm/ufP5JkjQ1ErzcePX8zrTdVHIBF126QM8NWsDQU9q4xVUHVfjbQFwtuCo/ASNfAWWDYEEvRCFaYqTushHU2hRANrretO0VcgEU0Tfvps0yxLlfTW1tvy73D1xo7M0iq1L15s29+BYchuG1d3TflzCldWOQAeI5CgcwojuwojjrfcoanCu7m5Na9/kT4DiejvTNN6tTKrWkwUoDY3J61C1LLZfZMeZofqFvF212hNHQ3ktsoC8BiBBJ3T+IZWa4uUQUaBQQtP6Y1UFbFCgEwvPxXv358VW9v71Zoj5r9PpEpe3UrWZ2jSdyCBc5Bv+XO6T6xyuuTtrlGossoBYCOQoFMaNJo7m0b/zruomd5AFVw0vdcqs1FZkWnjOqvcJgSS/mlPn0XL6M8aYqyGt7vm/OLKLAeAjUCCzqi1Qa0O5gO6gcKEgkhOa8WD3DVO1C3ibe4nkAxjMnlvfqeWvrttvN01L/6zU+3HZJUFwEYgQWeOj8/9gaCsSLocG/Hhw7m7u2jiXIadQDKMj+V9kdzyVd53fXbbeLtrFKascgDMRyBBJ/Q2qLdC6+E8j34+d2BpE1VMnlDifbsmkAzDtTZLqc9uG89UZIWoyI3/gGVFIEEntHmY9XCep80slxTqAvK01ng2P1v3QKJZVJqaq26yi4ur/w067qOLQt+f9b1a+uy28SzWphDV19RsLUGv71o0wFtBSX9H+u9lnA6ua9b9c3I6reh+0n216lPbYSOQoDXvm+wQb5Da52VjY888v2VjI33GTV+BRANsVe4ir169Sa789XP6eaucWYvKVCWhri1NZW0KemqZ0r2g7rsu9trRIOm567LU9dRto89efW7rnIYud2JWwDo8PKm+06Tuq/I70LgobSypmUp97KGjQeTWPVQ3b8C46pG9yVHj4PdV3asJzQgkaE2zCTytEd4xG7kuygdyateNZ0fWvgJJamuAzq1rsMqoS73WeWXmDhYWVaB7ZeX140d+xaKQqLBolW/po9vG1V1Thqe22xOoItaGkG2ntYvuf7XuXF93t9twamtR/d5Xy5paLlOCFYFkPRFI0JqnOVtvb6rkrHK65q3MUpcgX5dA8uH4PO2tfAFVrNNp/rPC0x3YR7eN5/72tLTVqcLWZ82dNt+oDFQKll387eUEErWivt4qg63xcxYCyXoikKAVb3O2Z6xGF47en5nXYVEzt1VG3aoHElWM6nbIaRWZR2/qmgFVv74UarlSC5ZV7iPlNXfZbTPU/a0QpW4zq8wuqQWn7XL23kCiMKLdtq2fmYdAsp4IJGjFs6Jm32tFWDxjEDY2J0l97qseSLwDglMplOTsUqwuGO1XY5Vp6bLbxtNdo5aNL1++muU00fTmLrpnUun30ObFwBtIUu/rWQSS9UQgQSuexav0ILPK6JOaz7Vh3cMo/iaatZBSka1yINFg4y66aeZRpZ2z7oy606zyLF3uIePqrkkMtLP0XfTSRbNAbjgUTyD53brlD1sEkvVEIEE2PXz1ELYeKI+Ub5ltm4rHYlUDyfX1t6oisP7/Lmn2k7eyUcta8sJkHd1r3u6a1DFIDzRd17OrcdcUFFIHcs/yBBLPJomzCCTriUCCbFqLQoNUrQdKnWcWy9itYiBRZa+ZHX101TxSnsP7dq6WK8/S7V20xnm6a7zdkRqn42l9+ZfymhQm9Ht9kNuqpYGmuhbrGudJve6dnYOke89CIFlPBBJk+/Q5fSM9DdhrM/1zTFYxkCyi37P2KdrZPaxokGLy+iCGnNkonoDQRbeNJzB4x614pqRXys+9VV7P5/Jvbl6AUCDaLX83qd+RKMhoDItV3jzZQcqBQLKeCCTIprEZqQ+/Lt5Yx2JdAokqK60jopYwq2xRq1e1oaKjEpSc9TpcXSjl9bTpttFnTu5OKc/lmdnjnY6u359n3I13kGzq7LIHbQOJWlU1cFpdhLNjbvS/db9qLJemnHvH42D5EUiQzTPQ0FMxj906BBJt/e/pgshZs0RTsq2ymnjGJLQJwZ7F/rytMZ5NA3O3WPC0wOh3fXf3wyzHkhtI1Mp2fOIbZ4P1QiBBttQKVLwD/sZs1QOJmstvb/370mhciKcbYmt73yynSZ9BYZYn+OhnrTLmSZ2ZljsjSTxjbrzju3ICSe49hfVCIEE2TyBZlRk2ssqBRJVg7tLn3oGaOeOKXLtKZ3bbeLtrFJKscubR96SgpI3k1HWh70znq7eatN0TJ7kFs/wMWk/IKsPiDSS5rTxYPwQSZNMoeusBZCGQLDaGQNJ2nyFPd4Tnc8zqe+0bTyuMKtumMTZeD2FFS+17ulEsal1J/V14/j49gURjhXJbebB+CCTIlvpgyhnAOGarGkjUOpKz0ugszYxQ87xVfl3VGvPZf194Fm/L6bbxdNd4B4QOybNKcV+BRC8tXa2ai9VHIEE2Aon9eWctUyDpaqqlKmmr/Lrc+0J7o2g7fqvMR5zdNp7uGoUihSOrnDHwBLdeAomzKwggkCDbuj6YVjWQ6G3WKsMr9XO0CarV5n9GmRbdp1YZFk93jUKRwpFVTjR1/Wi9GOu6LX0EEi0WlzuoGOuJQIJs+4kVj3geeGO3qoHEc61N9Lu2yq9rE0g83RGebhtPd03bQad90KBfrYmiqbzWNc/TRyBhcTN4EUiQLbUCFQLJYgSSdJ6xKmrxSJkJ45nBo64Q7wqnXVCLzMMMHc2iUfeYFqZTa4Rr5dcaAgnGgECCbFrYynoQWbqq7MaAQNJsiEAi2kLfKteSslaIuhVTu2v6rmzV5aK1QTQtWIGjWnk18dpyEEgwBgQSZEuteGTMsxG8CCTNhgokmhGkmTpW2XUp03NTB+OKwpBVRhuqvI+Pz393t/QYPiwEEowBgQTZPOscbJRveauyNwWBpNlQgUT3k+4rq+xHFnTbeLprupgePUuLw2nPoNS/pT4QSDAGBBJkU1+2+q6th1GdNtTqcgGpSASSZkMFEvHsp9TUbePprukyXGtvl9RWnj4RSDAGBBJk05udlv+2HkZ1Y1+zwYNA0mzIQKINAFNDcVO3jau7JmNTwDqNEdnVtNweu2aeqMUlsXwCCcaAQIJWPMvHj3GaZA4CSbMhA4lnE7l53Tae7hqFH88uyBbvGiEeam3Z2npbzcKplvEvv2Pr5+oIJBgDAgla8TSZRywkpQpZrTiqlBfZfP02abM3AkmzIQOJaN2N1JYAq9vG012j8NN2KXTP9ZrKf6tgpNk3e5Oj4uR0Wlxff3vUjaTvlkCCZUIgQSuamqjty60H0iNz3lD75KlsXpeBJKWyIZA0GzqQaCyTFj+zzlFnddskd9eU95HCxOy/9XLtVvygPO/fG3vVDBzPyqcEEiwbAgla0QNHDx7rgWTRW939gLNtPGMDUitkAkmzoQOJpFaS9VDsCQg5G/XVuVpHyp9Tq13uOQkkWDYEErTmWaBKq0l6Hn5teAY8egbdEkiaRQQShYzUin6228azAZ0q4tlzeqn1Ta1wVtmPlJ9Fi6JZ5aQikGDZEEjQmqfiF72R6s3UKqtLE8cGbJ5rIpA0iwgknl16Z3/Xk0niPVJrWcnhaY1Rpa/Br1Y5qQgkWDYEEnTCsymZaJZB2wduEy3a5lnfwTMDiEDSLCKQiGcsiMYW3d39SN6ELmWl10VSx1t1NUXe0/pDIMEYEEjQCc/bWEVN0h1VgHU3N7fJb8viXXmTQNIsKpB4KmCFF+/PW+f0SP0bUWtj27Eq4ulKJZBgDAgk6IxnAGmlDCVqmeiypeTr12/JzeIPtJaKZyongaRZVCDxtHhoLFNqGOmqxSI1kHh+v/Noer2m2VvlWwgkGAMCCTqjMOAZS/JAUxq7eCO8mF793hXVOMc8OZUigaRZVCARBVzrXG0o5CjsWOfzSA0k6tZR945VRirvWicEEowBgQSd0loJngfhA72Fvt3/UN1jVrlN9PDWdOKc82rgq1VmEwJJs8hAUq1OmtjykaqrFYaTB3+X93Gb9U68XZZCIMEYEEjQqdbLYpcP40WLQGlFSo35UAWqTfvMchL8fvP1ByACSbPIQKIKUBWhdb4cCjcKOda5vDyzbHLvzZwuSyGQYAwIJOhczhtaE/X3682y6o7JaAWxaCCrZuJY178IgaRZZCCR1O8xRdeVamplLjq3pyuzzc7BBBKMAYEEvVA3SpvWiz4p4Jye5jeJE0iaRQcStZ7lVsx1mqlinSOXd2yHWmje7B1VLR9WeQos2n247d8agQRjQCBBb64+fXEPMu2bwsiHD+fm9aYikDSLDiTammBjY2Ke08M7HTyFZwG3R8ogo78n/T5TWgt1rz9JHE9DIMEYEEjQqzG1lOhts+3maEIgaRYdSEStBtY5PRRq+th36eiovLaOuh6baMD2VmJ4IJBgDAgk6N3t7d3vWTDGQ2soqpRzx4zUEUiajSGQeLczsCjUWGW3pZDT99+DBoZrLZLU8EAgwRgQSDCYd+9OOp+SuVD5JqodUxWKrGvKQSBpNoZA4trIzqAwo1Bjld0F3Y+qsK1zt6UuIQ0s13kIJFgmBBIMSoPwXm+VFcUATdbqKppOu79HCSTNxhBIxDuAdJbCjGf13hwKJWrJsM6fS2FkdlE1AgmWCYEEIfT2ub1zUA28sx5m2coKSGs4nJ9/NM/bBQJJs7EEkuxum/Ie6mKsUQp13xwcHrdvOSyvWUHhx49/BwACCZYJgQSh9MDSw3+jfFPMDiflw/j58+2qQu1iCfpFCCTNxhJIJLXynNV3d41Fz9b9/Q/+6crlva/Bt/OWmieQYJkQSDAqChSn5cNxb3JUPfjUBK0KdZaauXd3D4v378+qgapaudUqC1hGWnNEA2o18FX3e9XKUwYPhRX9939e7lZrk0ynV9z7WCkEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIQjkAAAgHAEEgAAEI5AAgAAwhFIAABAOAIJAAAIRyABAADhCCQAACAcgQQAAIRLCiT1EAIAADA0AgkAAAhHIAEAAOEIJAAAINwfBQcHBwcHBwdH8EEg4eDg4ODg4Ag/CCQcHBwcHBwcwUdR/BfmwLilMgtjGQAAAABJRU5ErkJggg==" width="80">.</p><p>You can navigate through the panel tabs using keyboard shortcuts. The navigation will be carried out only
through non-empty tabs of the panel.</p><p><strong>Available keyboard shortcuts</strong></p><table><tr><td>Press this key</td><td>To do this</td></tr><tr><td><b>D</b> or ➡️</td><td>Move to the next tab that has at least one non-empty variant annotation field</td></tr><tr><td><b>A</b> or ⬅️</td><td>Move to the previous tab that has at least one non-empty variant annotation field</td></tr></table><p>As you move between variants in the table, the selected panel tab is retained.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="common-tab"></a>Common<a class="hash-link" href="#common-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/common_tab-beca0fcb2d2125b721da0d3db8dd6de9.png" width="1000"></p><p>The tab contains basic information about the
variant: <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#external-links">external links</a>,
<a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#sample-info">samples&#x27; info</a> and
<a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#variant-interpretation">variant interpretation</a>.
There may also be information
about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#normalized-variant">the normalized variant</a>
and <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel/#phasing-group">the phasing group</a>.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="external-links"></a>External Links<a class="hash-link" href="#external-links" title="Direct link to heading">#</a></h4><p>Links to pages with variant information in <a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a> (rsId),
<a href="https://varsome.com/" target="_blank" rel="noopener noreferrer">VarSome</a>, <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>,
<a href="https://cancer.sanger.ac.uk/cosmic/" target="_blank" rel="noopener noreferrer">COSMIC</a> (if the database
was <a href="/ngsw-docs-en/settings/custom-annotations">uploaded as a custom annotation</a>),
<a href="https://franklin.genoox.com/clinical-db/home" target="_blank" rel="noopener noreferrer">Franklin</a>, <a href="https://genome-euro.ucsc.edu/" target="_blank" rel="noopener noreferrer">UCSC</a>,
<a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD</a>, <a href="http://ruseq.ru/#/" target="_blank" rel="noopener noreferrer">RUSeq</a> (if the database
was <a href="/ngsw-docs-en/settings/custom-annotations">uploaded as a custom annotation</a>).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="sample-info"></a>Samples Info<a class="hash-link" href="#sample-info" title="Direct link to heading">#</a></h4><ol><li><strong>Table with samples</strong> with the following columns:</li></ol><ul><li><strong><em>Name</em></strong> is a name of the sample, for which the analysis was performed.</li><li><strong><em>Genotype</em></strong> is an allele combination for this sample (0 - reference allele, 1 - first alternative allele,
2 - second alternative allele etc.), divided by ”/” (for unphased genotype) or “|” (for phased genotype).
The number of alleles suggests ploidy of the sample.</li><li><strong><em>Read depth</em></strong> is a total number of reads of the sequence overlapping the variant position for this sample.</li><li><strong><em>Ref count</em></strong> is a reference allele reads count for this sample.</li><li><strong><em>Alt count</em></strong> is an alternative allele reads count for this sample.</li><li><strong><em>AF</em></strong> is an allelic frequency for this sample.</li></ul><ol start="2"><li>Information about <strong>the variant occurrences in other samples</strong> uploaded into the system:
number of variant/number of samples.
Here is the number of variants that completely coincide with the variant in terms of origin (somatic or germline),
chromosome, start position, reference and alternative allele and gene. These variants contain at least one
alternative allele (i.e. have genotype 1/1, 1|1, 0/1, 1/0, 0|1, 1|0, 1/., 1|., ./1, .|1, 1, 0/0/1, 0 /2, 1/2, etc.)
These variants are searched among all samples uploaded into the system (including deleted ones), for which the
stage of variant discovery (or annotation for samples uploaded in VCF format) has been successfully completed.
To calculate the occurrences of the somatic variant, those samples are taken into account in which the stage
&quot;Somatic SNVs/Indels discovery&quot; (or &quot;Somatic SNVs/Indels annotation&quot; stage for VCF samples) has been
successfully completed. To calculate the occurrences of the germline variant, samples with a successfully completed
stage &quot;Germline SNVs/Indels discovery&quot; (or &quot;Germline SNVs/Indels annotation&quot; for VCF samples) are taken into account.
By clicking on the link, you will see <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page#occurrences">&quot;Occurrences&quot; tab</a>
of the variant details page.</li><li>Information about <strong>the variant occurrence in run samples</strong>.
Only presented on the variant panel in <a href="/ngsw-docs-en/runs#run-variant-viewer">run SNV Viewer</a>
and run sample SNV Viewer. For example, in the following case, a variant was discovered in two of the
three run samples:</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/occurrence-17afdb45dcb88ade1120442a02490bb0.png" width="450"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="variant-interpretation"></a>Variant Interpretation / Comment<a class="hash-link" href="#variant-interpretation" title="Direct link to heading">#</a></h4><p>There may be displayed the variant interpretation if it has been added by the user.
Interpretation can be added and edited:</p><ol><li>On the variant details panel: to do this, click on the text field, enter the text and click outside
the text field to save the interpretation:</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/interpretation_on_panel-00d44172e30b78fa4623242bd33a74fc.png" width="400"></p><ol start="2"><li>On the variant details page, as described <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page#add-interpretation">here</a>.</li></ol><p>No matter where an interpretation has been added, the text is automatically updated everywhere.</p><p>After adding an interpretation, the information about the interpretation author (the name of the user who
interpreted the variant) will appear in parentheses next to the section name:</p><p align="center"><img src="/ngsw-docs-en/assets/images/interpretation_author_on_panel-ec170ae49a4579698ff17a398fd4f2fe.png" width="500"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="normalized-variant"></a>Normalized Variant<a class="hash-link" href="#normalized-variant" title="Direct link to heading">#</a></h4><p>Start position, reference allele, or alternative allele of some variants in the uploaded VCF file may
not be left-normalized. In this case, the variant will be written in SNV Viewer table in the same form
as in the original VCF file, and the normalized data will be shown on the details panel:</p><p align="center"><img src="/ngsw-docs-en/assets/images/normalized_variant-4db06c6817f816413f654e0463a87f5f.png" width="800"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="phasing-group"></a>Phasing Group Info<a class="hash-link" href="#phasing-group" title="Direct link to heading">#</a></h4><p>Phasing group is a merging of homozygous variants located within a specific genomic interval
(window of size N, which can be changed as
described <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#pg-merge-distance">here</a>).
In addition, the phasing group is merged in such a way that at least one variant from the group
has an effect on the protein-coding part of at least one gene.</p><p>The phasing group details panel shows the variants that the group contains
(you can open the variant details pages by clicking on the links):</p><p align="center"><img src="/ngsw-docs-en/assets/images/pg_includes-4f770691367ef4545b358eb9100ee1d4.png" width="280"></p><p>The details panel of a variant included in the phasing groups shows these groups
(you can open the phasing group details pages by clicking on the links):</p><p align="center"><img src="/ngsw-docs-en/assets/images/in_pg-d31470c9f988df25fc30e40a821e914b.png" width="300"></p><p>If a variant is a part of phasing group with discordant amino acid substitutions, then that variant is
marked with <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFYAAABQCAYAAACDD4LqAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAOKSURBVHhe7ZpBjhUxDEQ5H7fgFpyCPXvEBWbJgguwYscBkFiPGIEQqFENtKYlnMROXLa/SEm1GI1+T/J+dexY8+zYomiDJWmDJWmDJWmDJak82J+fPx4/Pr07vn9482j8/Ov+y9/f1lVZsID59e2L4/71c9EPdy8fIVdVObBII6BJMCV/e/+qZIJLgUUCJXgjI9nV4JYBCzASNK2rJbcMWMvr3zKKWxWVAItCJYGacZXUlgArAZp1ldSmgwUICdCKK6Q2FexqwWoZ53W2UsEy0noa53am0sCy0no6u7dNA+vRXo2cWchSwM7esGacldoUsL3hirezUhsO1lqwcGSgECF5SDo+b/1iMlIbCtZasHD/bwm/kz4jufcclkLBWtKq6UUtyY2e3YaBtaZVA8LyRUW3X2FgLa8urJG1u4gsZCFgrWlFujSaeW5UakPAWi8DLLBwVGrpYGdmrUywcERq6WCljY3MBhuRWipYS9W+mg0WZqeWBnZl0xFg2TNbGtjZtMIRYGHmzJYCdnXDUWDxd/AMhihgV2etUWBhViFzB+sxa40ECzNS6w4WUKTFWxwNlpFaV7ArBevqaLCwd2rdwHpuMgOs98zWDaxXWuEMsLDnzNYFrPcGs8Di7+KZHnIBa521jpwFFvYqZMtgGZvLBOuV2mWwq5cByZlgYY/ULoH1/L/Wq7PBwqupXQIrLcjDFcCupnYarGd7JVkjJlh4JbVTYNkbgjWbYn+5KzPbKbDsDcGaVxFHhvRZT8/ObM1gI9J6urcp79655dn2ywyW0V71DIC4ap6bA+zoNcwUMhNYj1nrrdqaWhPYiDOtqq2pVYONKFjVbUmtCmxkwapsy8xWBXan9cname0QbHZa0QGgE8A6sCk486zXtl9DsFH9ouReOjLXpSlkXbCZadVcJ7OSq0ltF2x0I3615ixjjS01HqW2CTZz0bBGgC99Nsq91DbBZl8GNMo8quBeakWw2QuGR2cYVKENbK1TBFthwZrKm/1Wwa0JnAg2s2hd3Vo0lNluXd26jYlgpQdkGQtHkTpfOcCu8sXDJrAVXrFbcavf3mAX3aoFItgq59ct2AQ2+3JwS25JBAvt42DsXkvYBFuhl61sBK+nJlhow217NCTqgoU23H+tmbwNwUIoZvvM/dOzaqBCKrCn8FA8HJD/F9DY73n7s8gEdkuvDZakDZakDZakDZai4/gNwz9pnVGlZ00AAAAASUVORK5CYII=" width="28">. If the variant is a
part of phasing group with concordant amino acid substitution, then the variant is not marked with such icon.</p><p><strong>Example of phasing group representation in IGV:</strong> phasing groups are located in &quot;phasing groups&quot; track.
There are three phasing groups on the left in the track: chr15:48505102 CA&gt;TT (contains 2 variants),
chr15:48505102 CAA&gt;TTG (contains 3 variants), chr15:48505103 AA&gt;TG (contains 2 variants).
There are one phasing group in the middle of the track: chr15:48505109 CA&gt;TG (contains 2 variants),
and one phasing group on the right in the track: chr15:48505117 CA&gt;AG (contains 2 variants).</p><p align="center"><img src="/ngsw-docs-en/assets/images/pg_example_igv-9f49937d5b6131c32d576bdbd299b23e.png" width="1000"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="gene"></a>Gene<a class="hash-link" href="#gene" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/gene_tab-af086ee0bc91c267d836c8ff9c4e7e56.png" width="900"></p><p>The tab contains basic information about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#gene-info">the gene</a>
and <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#transcripts">the transcripts</a> in which the variant is located.
If you are granted access to OMIM database, there may also be information
on <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#omim-phenotypes">gene-phenotype relations</a>.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="gene-info"></a>1. Gene<a class="hash-link" href="#gene-info" title="Direct link to heading">#</a></h4><ul><li><strong><em>Official full name</em></strong>. When hovering over the gene name, you can see from which databases it was taken:</li></ul><p align="center"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAYQAAABQCAYAAAAZZQJeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAACNWSURBVHhe7Z2LcxTHncf3T0iuqKtKUpW6yuPu0L04VOW6yp1VRy6q2GUHGRyCbGKfijgofoJjgeKA5RyKkkAwBhuFJcg4ZwsfmMUGFoyRbAvJBoRsSUgGGWzJCGTQC8FKSGhBtn/X35numZ6ZntXualcSUnfVx5a6e3p6enq+3+6eHhEYGRmhGzdu0PXr1ykajVoMDw/TtWvXDIaGhgwGBwfp6tWrFgMDA0r6+/vjJhKJaDQajYah0kg/VNoLZI2GZgv9FnoObZe1HtoPD4AXBPCLbAbiIIDChAnIJ3RXDBdy5coVi8uXL4+Zvr4+jUajmbKodC8RZM1VmYms2cIcgMoYhA8YhuA2A7iJcBhhACrRFxd26dIlJb29vUnT09Oj0Wg0UxKV5sWLSmuBbDSyVguzEMYAxIzBbQoB/CCWh+RlIRyMgo6/X0/rn9tMjz7xJP384V8aPPDQ43HxsweXaTQaTUr4zx/erYyfbqi01sHDJoZeP/KEod3QcGg5NF2eMUDzof/CGAL4AZFiVgAngascOXacFvz0AZrxzZkajUajmQJA06HtwhiEKYjZQkDMCoAwg//dvlNZmEaj0WhufqDxYilJ6D9MIYAfxMwAa097w28oC9BoNBrN1AFajwmAbAoB2QzwwuKWrNuUB2s0Go1m6gCth+ZD++EB8IKAbAalW15QHqjRaDSaqQc0X5gC3ikEMGXAVqWuri7K+cn9yoM0Go1GM/WA5kP74QHwggCcobu7my5cuEDf/ed/Ux6k0Wg0mqkHNB/aDw+AFwQwXejs7KRz584pD9BoNBrN1AXa3/7xx9T26QUKwBk+++wz+vTTT5WZU86d+ZS/rMBk8Xx1Ho2XrDzKG692S8W59H2eXtxs/XPMzKfcuOuQSN6ZNGcxz/tgHs1WpKeaf/z6d+ir3/gu/dU3/pYCWD86f/48tba2KjOnjKxiCp0eIE/ob6PQymz1MVOF1bUUYZcaObpGnc4oOoq2aaNyd1pWAZUe7aCo0VhSGOygmq0Fng5T3momt+/Od8TbrKGafpahv5aK3Gl3+t+jyqBfeS4m232Oo+2TwWhnVRsmRDaVnTJah7qrVinSx4lp1T9R315eX8X1MOYUhekM6sCCt02yKf+lFoqMmOkiRDtqqXSxnC/RvDNp9kr7vFZg7VhZkt7n5qtf/zYzhO+YhnDx4kVqb2+n06dPKzOnhnzWEXALotReF6IS7pYlrzRSu3FnOiisaKApQ7IPXBZ7OHrQPuzY1loKbSpm7VZM63fXUjvvON01axwPnXjg/NvU54FbvJ2ajTKj1N1UTWVrcI82UFlVC3Ub9yhKZ3aO9tBNwvs8mQ0hi7W5EIsxm8sYmCb90xZ6/O0e/N99PZJZmBk8bTJ7Y6OZjgGOUQf0b1YH3Mc+Z50TyTsjaxs1I3OUGYDRjiz/Gl7fKKvnIilvinlyZ4PhAfCCQEdHh7Fc9NFHHykzpwTe4ZSjoIJq6mZpqX5gJxVJPnBmHOvou70jLeNh7GPJ1EuVBXa89cCh0/VU01L5GAPVA5dH5afRGweoTjXSEueKtlBplitNZjLe50lsCPP2dKBRqa6uzfz/OnW+tDMV+ydmqqxda14SdePl9jRS6SPZvB4uQ9hp3of2g8U0R9kmeRQ6iyze50CIf/uevCTyMl4xz90cdM0GxDNVWeCMTyH7mPbDA/DqwDCEtrY2amlpUWZOCUZDE3NwRZpYX3Otrc1+ZBtVtg5QlI+gon1sWuiegrLpWBRTqiIpDhRVU/sga9yXRFwxVXawUUFTyJzC4W6wzlWzmqdjKtrUy+NZ6O+l5t2sU8hlKvJFlfmyqTAYplCw2K5rUg8cGzHg2jsqKFfK54CXG20IWnGio4dZm6OaETZCcxyjeuCWmWKNcjwPNmf2iy0omN3DGNPXBO5z3h5Wv8FeqlGIYFEVG6Wx+xouxO/i3m2n3KA98kR/MJag+BKV6CeRjmoquVMqT2r7wt1tnnvsvd75LB8bxfXzjCMDbEQapkK5TIbXEBT3PSZcMFBGVojOsB+jdRsU+UB8dYovXz4bpYapfJ0krFOxfxoCy8JIG5UZeQpo/Vb7WVUawupthlmYP6vahN8n6XpseL1Ph/jvieRlg4P9GBxImiTIClM7S4l1b8bKDx5dSzuqPqCWNmYIeH/wySef0MmTJ5WZU8KyCuOGUk+t82H1YzFrTPSWaC+dOVpBod3V1NxjdnI0jNUpDAFSNCK/mbYwiU42QJERNuU8XUuVR2upfBlLs0Yy5lQ0tLuCapgRITjOJfKxh6y9Lka+IvPcxsgI5SMumQdOjCJ2xxBgBXZHt0dVNY41SNEW9gNndcYSu5ykSOQ+L6owOrpXBDdQ3SDKEKNHXt+eXupm/aG5ioluFRN2iBGbSp9hVY+craXw7jBVMrM2esnZMM0T5QlR6oNpDMTuT0zUzfvA4jsaqZKVGT5qn6tcWuLwGMLP+bWjHd0DFBVc5MzZFH+XMMJGk568dp2iPS3OOmHZwRp9xpnPEkrpXFOxf7LnNXya9Y/9ziUrgdIQZGIYgrqd3PVOJC9jndme3ZXO2XXubnMW2Ry041LNV77+Les9QgBbjj7++GP68MMPlZlTAxs9sZtqPIJckMMvbqBcpWjwkdNIB4Uca4zz2UODEqQpaKKG4Ol8M6mkTtUps2l9Az8XF3XzgfDmM+OlOrGOWNnBjIeNnKybLTpXwzZzbVBBaYPrgfO7tlFwdHSxLukQDm9n9DzsSZPEfR5spBI5nj8Y9nRa3DvnmvPsIJsdIpqJvz1CzabSJsRK18Lb3rtmLd539FJYGDc/t9Mk2LlW8hHqqW1WvMcQUB6bqWCGYre1P0vZLEjuN2KE2/yiS2D96sQF2VpKiDcf1uL7okZfdJvm1O+fNskZQgFV4p2J476bzC7h/cxKSyQvYFpSgz7BztmKAQ4bcJ42Bzjdddv8Z2EpIOf7/8QN4dsUwMuEM2fOUHNzszJzKplTsJ0q2UXKb92NN+5imgb4yFHprHwd2urciRoCE2nrITAImqPR1pB3FGFsm3yM5hkd1cynnP7x+sZc47NEabRgd1BTMBTXtixEdRj5yNSFKJ+nuzu6EARbKLwPXNlp5zFjJa77zJht3L8o1W2040yDZoMB6yWaz71TPrCsTGNEJbWbaHsm5nI+OU3cO3MQwNrBI+jeEbzXEBKBXxObBYn7Zi3BnNru6Iv+dTKX4PIWmm0abz4l06x/guQMgc1W2AwL9Y2ereBLcdk0r4RpACJx/6R6J5IXaflba/nmCylgVqxc2kwdmAx88tFJqq/ZQ4GzZ8+OmyHYsIYpDFJITGkVD7B6HZp3FrH2lqghMOF35Is5rZMx82HUGx30ghCzDNG5UjEC4+9H5HNjhFXI070dXcxiILwQBe8Dl44RmEmM+wz4LhvbaLlBO0TR5975PLCeduP51IbtvP+xRN4yGr4cNCZDEMsDVWscfaDcmAHLZhj/ecZUn2nYP731cOHXv1DfSrFt1Q4R1mfDMC7HPYg/rzCPyGnpnc+dq6jMaPd4dvglD7QfHoDJgWEI2HLa1NSkzJx2FoepHWIhRJ7fiJiGIMRhvA2hr8M58pGoe2X0GUKs83g6PZ9Wtu+XdiJ48F6XsqM7dmF4HzhT7Jwj9ZTjvs+cUnnkzUeLzmUTn3vn16Y+hqBuR/O+ivcYMUXVVe5YBNicBfkHefdJvOcZS32mY/9U1kNmlDaZvXANle0OU2h3iNYXYKNEHoVRzbNhz2h+9Lx8ecm9fGqQT+GLLC3ZexsHz1eYhgAvGBdDWLoJjRFUbDEDrgsWL9tUIzr3G3fxkLpfNvGbObohiGm6YjnBARcO5Y6BOEjmgfO8XFXAl9DkF7N+HV2sW0ZP11Kd64GbschsV3mN3E08u4wSus8CPlpu3srNgT0U6+X0FBmC8t7xvhapKTZ+d5iTC/P+sNE7X5JJXoDt3TmFnpE4mzHxey6WWGLVSSbefEqmYv8sqDD2+0fZAES1/p6cIah3RBrwnWK2biWS10+fTMwBRIy6jpGvfO1b9NWv8XcI4huEdBqC2Zmi7KFXiMlicw3eFuVV9ssY13roUjb9QjnWfm3eiZz73sUUNB5DEGvD7hfYMynX2BYptj7yfcuKPcUzCkJUV1fNnN+Om73wMeeabVIPnLheHLfBuwU2q4BC/KWod5+3uvMU1ZjtYgSHmGUzQUFZPnvKxa4viLVnjdomsfss4MJyqtFcPvLsOkqNIajunaiv6E9CVDxf0YqXn9LupWQNYTRjNevkfdnsqRObcZ0ZxJ55s+/Hmw/gTyM4XvRPxf4pNh0oR93JGoLY8NJCZY6+JDRHXu5LJK89Q/A8X4oZU6pRGsKJEyeUmVPCIn7D2IPaflR8wSp/ZSg9wIzZW82bGb3YSOXGF37FtL7K3L0SZcJgOz4XE5bSXofthGGqxKftWL9ksaMbAkMIRn8bhfkXgiXYr444eY+1Il/hpgr+5aO0g4WPZgyTETc8yQfO3gnDQl8b1exnU85N2ym0v5HajetmZqj8EtSvo3OzRXB3MDFtZ8Hc5bCdyl6Uti4aMzG1iFkkeJ8F5gtKv/TUGEKkj7VvT4vVn8r4F6kYQdovq0X7DNCZg0FzBL8mRM08Tr5+lSGUNbF+1+ee4chIH0S5H3yB2DhhDXLEfZHrJL64lXdOxZmPz8hQd0sop2T/zKb8jUEq8nmZnpwhMG3ig1BLB4qCFOZ/TgN55WtNKC/XPOStfHGDqUMvVltfVqfzHUJT7TH64IN6av7wpGkIWDJKqyEw8KFZTQfvPHLAB0ZF3mlVbrDR/MRbCsYLF9eDNHslE2Xe+RCM3SxB80bEZQiMOUXOMhCi7g+cGMq/NcJuoONv9AhRlEekST9wAB8biU4vhZh/KyZGRxcjddWIQ/xdGte5ohdb4v47RIneZwMugurlh9QYAh6osibzYRRB1Z/woVtYiJwIaGvXF7JeQ+D1dOwcciEGC66dRE64KMoj2zvVdapc52rPePLx9nC09TTqn4JkDQEYH0jKzTzCBqRVillSMnldOuTXjqkEq0N4hwAvGDdDsJD+CmbMrXAG2TTvQeQV2z/9wTKNer97/Ii/MjhaOdZfI/T7y4VZ82nOKPVNBlxjPPUbO/G3uy+J3Od4tu6miLjbUNTf7x57CBnrwrFEdczEW6fR8t3J+qcqfozcVP0zBZg6kB9XWyaSV7Tj6PqYGqD98ADHS+VxMwSNxoFqTfUmxPhCPc07tTSaNIAZgscQJmzbqWbagr3t8ncc6ZwWpx38SYhkd/loNBOIryH89d9kKA/QaFIP/lAgvuFooZpXFH9I8CbDmObHvbyk0UwOoPniHYJhCOJPVyAyI/M/lAdpNBqNZuoBzReGgL9rZ/xxO2EIC376M+VBGo1Go5l6QPPFn66wDEH8tdPS4FblQRqNRqOZepQGy4x/+gD/hLJlCOLfQ2hoaKDbchYqD9RoNBrN1AFaD80/deqUYQj4t3EC+GfT8C+m4WvlxsZG2hc+QDNn/7uyAI1Go9Hc/EDjofXQfGg/PMD4JzQvXLhgfKEmdhq9//77tGPnLrol64fKgjQajUZz8wJth8ZD68X7A+wwghcE8C/tu5eNjh07RtXV1fTI4yvo7//1e8pCNRPD3/3Dv6QF1bk0Gs3UAVoOTa+pqaHa2lpD6+X3B52dnRTo7u4m+R/axyyhrq6Ojhw5YpjCW2+9RVu2vkBPF/+efrXqN1S48mkq/PXTtPzJpywKfrVKyROFKzUpZukThfT9//oBZWZmpgSUhTJV59JoJhO33ZmrjJ9OFBQybQUurZX1GECjodXQbGg3NBxaDk0/evSoMTuA1mNlCCtE8AB4QaCnp4cwS8D3CGKWgD9jIZvCO++8Q5WVlXTo0CF688036Y033qADBw7Q/v37DcLhsIN9+/Z52Lt376js2bNHEwc7d+6ku+66SynwiYAyUJbqHBrNZOOhh1co46cCKj2UUWmqW3eFHkObodHQamg2tBsaDi2HpkPbofGYAGB2AO2HB8ALApcuXaKuri7jhQKcAutJeOuMlw1wEbjJu+++axnD22+/bZxAGIQwCXDw4EFfUEFN6njttddo/vz5SqGPBxyLMlRlazSTkUcf+7Uyfqqj0lOB0F4h/sIAALRaGAE0HK8CoOnQdmg8PjeA5kP7MTvo6+ujwOXLlw1ngEOIpSNhCnCR+vp6w1FQGNwFBWMN6vDhwwZVVVUGOLEfqJgm9WAkcPfddysFPxY4BseqytRoJiuP/7JIGT+dERordFjoMjT6vffeMzQbg3poOLQcmg5th8ZD66H5eHfQ29tLV65coUAkEiGYAhwCpoC9qHANuAemFPhgDYXgBcQHH3xgFIwXEgAmgZMBnNgNKqRJL1gXXLBggVL4VSAvjlGVpdFMZpavKFbGTzdUWit0GJos9BlaDc2GdkPD8ToAmo5XA9hVBDMQS0XwAHhBYGBgwHAGTBfkmQLWlbC+BCfBPlW4CrYoARSOaQdOBOA8AlRgLGBKo0kMjAYWLlyoNAAZ5EFeVRkazWTn1yvXKONvRlTalyiy7gKhx9BnIPQa2g0NF7MCaLuYGUDzof0wA3hB4OrVq9Tf32+ZAqYOMAWsK2G2gINRCFwFBQqDgNPgRADOg5lEvIiKalLH8ePHKTc3V2kEAGnIozpWo7kZKCp6Rhk/lVHppxvoLxB6DG0WBgCg3cIIoOnQdmg8tB6aD+2HBwwODlIA/4EpyDMFZBSzBWEMANMMLCcBzB4AToblJT9EpTTpB7O2e++912MGiEOa6hiN5mah+LcblfHTBZW+ykCLhS4LnYZmQ7sxIxBGAG2XzQDaDw8wDGF4eNj4QZiCeKeA3Uc4CO8WMLXAV2wAhQJ8yCCA8whQAc3EgRHCokWLLDPAz4hT5dVobiZ+9/tSZfx0RdZdIGuy0Gmh29BwaDk0Hdou3hnIZgAvMAwBDA0NWaYgLyHJMwZsTwUoXADHAeLEiSIqrkkdGC3cd999BvhZlUejudn4w5otyviphkonR0PosNBlodXQbXlGIGYF0HhhBteuXTM8IBqNUgD/EaaABGEKsjHATcSsQcwcxIngOn6ISmnGHzFlVKVpNDcjf1y3VRk/nVDprEBosjAAoddCvzEjEEbgNgPLEK5fv278IJsCwBRCNgYUBnOQDQII1wGiAhqNRpNqnlm/TRk/3ZE1WNZmAL12GwG0Xei8bAY3btygAP4DECGMQWTGMpJ4vyAQhQJhEPHirqxGo9HEy4aNf1HGT1VUGhoPskbL2u1nBJgUCB8IjIyMWL/4zRYAChPIJ5FBBeBEGo1Gk2o2PveSMn66ohJ9GaHXGNjLWi70HXovmwG8wDAE2RRUxuA2ByBmDxqNRjMePPf8y8p4jRO3AQBZy91GAIQPBD7//HPrF5UxCHNwG4TAfWJURqPRaFLNptJyZfx0xq2/Ko0Wy0KxjAB88cUXFMB/gJwA5IOA2xxkVJXQaDSaVLKp9BVlvMbErct+JgDcei98wDIEgTsjcBcmiGUSGo1Gk0pK//R/yvjx5vzrT9CPfvSjGJTSB4rj0o2f+AtU2u7Wf48hCNxLSW5UJ3QjKqjRaDRjBYagih9vLjeEWF1KTdYXUC4zgdwVz9pxf3qbPlUclypUWutGpdkCaLtK84GvIciMZg4ajUaTbjYHdyjjJ5QLe6mAGULBns+8aUMRivRFaCjKfr6Kn4c8eYZYOvJErjrjDeRjjJ8ZQ648guiQme6TR2j5MHYmXWEMeXV+mMUHvvzySwLuxNGASbhxV0Iz3Riixr/cT5nfnEV3/eZNuqjMMwFErlDf5dgMqY4DQ13U8v4ROvyuSWO796HWjA+bgzuV8RPKxTAVzJ1LBfsueNIu7CuguXMLaPPWlXQ3yzN37mZqEOkdVfS7n93N4hAP7qYHgg0UkY5v+BPiN9JrB8TxZr6VB5znipx8iZb+xEzPycmhnB8/QL97+6JTs3veo40PLTDTDRbQw2UnaICnf/nlCdrC4i1DSNYYYqEyDZkh64EcUqZrxgfrPvSr0+OnntZlzqQZ3wRLaG+nKs94c4UOPSnq5McSerXDfVwXHX42l5mbN3/mfX+gwz3u/Jp0A0NQxU8okiG400xDmEs/XrqZ6tr7qf/yNZ7WTjseYvErdtAnl3neg7+hH8/9MW2ut49vZIYA8f7vZ6ro3OA1unapkcoeguD/kd6zRvgtVPbTHFqw+hBdvMF+vzFAjdseopz7NlLdgMhzkcIr2HGL11HVZ8MszzB1Vj9DecwUVr/Tx7WfGwKx4DYFGVngU0s3vfoL/pD94lXqVObRpJ96ekaI3YZ6RXpinD20mnKzb6UlL5yka4r08UfqZ74wQ/jMeczepbPs9Kw7aN6i+TQv5xY7LmcN1VoPnGY8CG7ZqYyfUDrDtJwJ6fJ9rhE54+K+5UzQf047zjrjLSDg4ufPXqNlrnIaN2Mkv04S/y9ouGYdi1tO4U4R10hBdty9G+toQC5P5qMX6F6W55l3hyVtv0rnm96nD9oUhiAH+4D4UVZiVLppl2UIu6hLmUeTfhokQ2hQpN/s2Nc3b3urIt3LtSOrKYMfc/uGY3RZSjsbyrfScneccxynSS/BLa8q4yeUrv2mIYQ7PWmdYRjCctrf5Yw3GO6kE+GX6bniZbTkF3m0gJWB2YBczgnDEIJ0Qj6uIcjiVjjK7AyvMI9fmEdLVq2jl8MnqHPE1ue+N1bwY+w4GTM0qQ1BDqqDU8HwhQaqPhKm3y7iQrRoDe07cozFNVDXMPIMU9cJ/A5a6YryeDk/Y/g8NRlxx6jpApywn9qP7KKtm5+lTTvC1NQtlXGplSp2sHiW9vLBU9R1Q0pzwOpx8jDt2mbmNcoxynblc5/7Rg8d37PNLL++35vfQLrGE+dpGHHdp2hfrHolcJ7hC6eogqdt2lxO+07yc0hc+QRlbaOlXOBmFG4z66Noc7TZ8YPlMctTXhNgx5rlHqPTlxAn3Zttu6j6E782kkmivcDwYSri15cf7vGmK2jfs9ycESxaTRURd7rTQJ1pmnQCQ1DFTyjMEFYwIV3BhNydBqFWCvG1ExRcjCWcp+iF8GF6v6mVOtte95RjGYJ8bMMWZZnRvlY6UbmTnvvtw3TvApS9hU5cM9OEIRzgx6hDHIYgB/nkY6VrT775QHnIp10XkKeHdj0o4tbScd/jRX7GhV2Ux8vJCx2jXcukKb/BHbS+fpiG69fS7Y54Rs4qqhDlCC4dppKF7jLALJq3OkztsvjEOHfeHj8Rkq7xwV10un4DzbPW3zlZebT1pGRAcZ2nh6rXzbdGsTIZC1fRvvPi/F/S8Y3ePCZymw/T6R2PKdfS0W5yee5r6hLxrM3FMeurT9HWxe52ZW36fIPCYGSSaC8gtdn6eik+aY5RCS8vg9VZnUeTDrb8eYoYgiHqObSlQYq78Do97jaEIAyBCbvIA9yGMBKlq5Hz1Nkn5al/3iy/kf/e/iotYb8/9dZlruYIXXT05S20paqD/56gIfgFqxIJcKV+O5UGiyn/Nv4g35ZPJcENLG47HTdGZD0UitMQQpYhhKwHf07OHZSReSvllxTTUse6Lxv1MRHJzHmMikoeo9uzeDwjY80xW5CuNtD6HJF2C+WuXMvqtoGKHrTLylh5yB5Fu8/N/p+Jc7FRZuGb/oZgXeNt91Aua4vM+5cb7VD06K22oGcupwpjVM0Y9TzDTOTvMI9DGi+vtCTPFnSpvPYKtPlyyhVpLD+uszR4iNp5PbvC9hKJVd46doxoO7l+8jU9GFIawu2s3liTz2NtWlRwh2Q0s6jkiGL2ZZFEe4GTQZpjpM2nMja7KJXyZmTfQ0V7TnlnQ36wGVnFGtG+d1DpSUUeTdrY8uddyvgJJRlDOP86PYxjtn9EndeiFO36iF59eoGx7DOqITSahiBG+1/2VdNqNiPIe7aazkeuGuZQ/Wwey/Mw7RFaT5ep+n8WUM6CFVTe2EGD/R3U9DKWmRbQM8eu8zwpMoR4g3VBFj7i4U5jhlDnSGMitTe2IczILKZKa6o/THXrsnl+JqTraiXhr6USYUqZG6iJx7fvvIfnn0VFVfJyBivLEtxsWxDkc2MmUhfPEoh8jUwoNzY6Rsjt7BqFcM3ZesqMH+08n263xD2jpNohdMMN9sxoTpCXZ9BI63n8DFYHO54xzNqHj8LRbg7hjFRTkUgT9fO7p+zcRhzIYffzKo9nXKkqtgV6k+v8DpJoLyCf2wd3WU4aqcxYPrLNKyM7n0rr4lt+0qSOyWkIB0xD2K8whP3OpRpH2sGnrPcGOQuW0PNHKw1BlssRhtDENdQIwhC6+e8sDDaX0+MLeVlg4eO0rV6eDbAw2EYH1i6RznkvrX6tjQZ58rgbgjf0OsRDuj4WpDQYAo8VoVs2hIs88qIkluwBd4Q6acmigcfxULeOH2Odp43KrXcb2+mcESeFvv3WmnuGOI98bibEUTN2lCBf42MU7uPRVmihUmFWS0JmPUY5z7lX5vPy5lP5WR5phQEKF/BjM+U2dRqCHKI1xTHKI2reyo3Waifpmm4LUrMRx4Ikynm7rWELD9L5Pf1ADkm0FwuRhiAtZYJ+e9Ysyt1UTWd60GpRinTU0npr6YoZf43fXZPqZ3AL3V6wlkINvTxdh/EKMIQpFT6/zkbrg3T9c/77GMN146+dihG/T7jO8sQ45/QwBEmQPIZgraPz80TZyNevHCNIhiHqHevcvkG6RpXxMAGvXMnTRd1inidKNSWu/K5gG4bUbjEMwc4/i7630FyacpAtBFWcz+e+yYaw1y2kSRhCvO01Whi0ZzkzVlZQhEe7Q7R/gCJsmt1cFaL11rLhLMrf6zY3HdIZppwhTMKgDcFtCKOKu6LeYzUEHyFMrG6x2wxB2W4xDME+/2iI8/nUIdWGEG97jRokE83cYM9oYoYO6RqXU2U/j9Yh7UEbQvqDNgSPiEgCpVz+YeliVLlsf2oMQSlGTHiWuNJHOY99LcWkWgFp3iRG9PkU7uGRMQxBbmc7f6wwToYQb3uxYI7ugXpJyFoyTOCYc7vzzGMY7v6kQ/qCNoT0h0lqCPLyh3u9WFoLT4shSEsPmUxY7bcuZjgldq3MpHmvtJlxYzUEvKA+xaNFkMsUyxmjnCdSuZyXp1oTl9bYHUsu/oaAdjNf1M6ikqNecYy0ttA5CKfVRuNkCPG2Fwv2O6JVVOm+l8olI9l4FPef9c3KIp7u825Fh/QEbQjpD5PDEDKZsLc6R2TnrJ0+Mylj8Qaq6WDpPS0UWi3vsU+HIbBH/qi98+X2ov10TlSrp5pKrO2ocZ7bN8gCx8hZReGz/ER9jVQqvfC0xHi080TtXUGO8kZ6qdLaLukWZUmQc9ZSTY8s8G1Ufr+iPBYip7dTPj9XhkpM02oIjHjaiwX5XmYs207NfWZatK+Fyq3vOJzHuPteZWuvOWPoaaPKTffY/e9+1bsMHdIVtCGkP0ygITgfPBNJRPoqqFCImwMmxLuFwKTHEDAKtLeXqriFlu6XXiiO2RAeo8KVQpycZCyzd8zEc54ou1bPh3cSmQXM4HheM7DZ2BrXuaVdOtQRsoQfZGTLL5MZMHOrKcbLEOJsLyOMdi/Nbae2HSB0UMjzYaMLx3XrMB5BG0L6w4QaAh7yuqD00ZRrCh5tDVGR9LUw9n+XnWKPriUw6TIEhCid2Vtsf4DFwcdQ5U0DPA8PYzYEdu5BNhovkD6iY6aTF6yl7hGeHSHO8xjtdr9cFiPrHirc2ajeScPO7Zx5ud5B9NQ6PugymUXfezRIdY7lvPEyhDjbywrsXr65lvJlI2MY3xMcdddHhAHlMTMyb6XckhCd0S+Txz1oQ0h/mGBDECFqTsmdwzQ7DDqXk8Y7WC8ZPevJYwk+4mlc6wBFlcKWYIjyesfbdiPmffA9N09PWf0SCilqL54/oXspjpnAPqiDNoTxCJPEEKZj8BE4HXyCbq/pHrQhpD9oQ5iwoAUusaDba7oHbQjpD9oQJixogUss6Paa7kEbQvqDNoQJDKN9AKWDM+j2mt5BG0L6gzYEHXTQ4aYI2hDSH7Qh6KCDDjdF0IaQ7kD0/ygCV5zT/vzbAAAAAElFTkSuQmCC" width="300"></p><ul><li><strong><em>Links</em></strong> to pages with information about the gene in various databases
(<a href="https://www.ensembl.org/Homo_sapiens/Info/Index" target="_blank" rel="noopener noreferrer">Ensembl</a>, <a href="https://www.uniprot.org/uniprot/" target="_blank" rel="noopener noreferrer">UniProt</a>,
<a href="https://www.gtexportal.org/home/" target="_blank" rel="noopener noreferrer">GTEx</a>, <a href="https://www.omim.org/" target="_blank" rel="noopener noreferrer">OMIM</a>, <a href="https://www.ncbi.nlm.nih.gov/gene/" target="_blank" rel="noopener noreferrer">NCBI</a>).</li><li><strong><em>Search for all variants in gene</em></strong>: search for all variants discovered in the sample and located in the same
gene. To search, click on &quot;<em>All variants in gene</em>&quot; link and new tab with SNV Viewer filtered by this gene
will open.</li></ul><div class="admonition admonition-note alert alert--secondary"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"></path></svg></span>note</h5></div><div class="admonition-content"><p>If you make any changes on the new tab, they will be saved in SNV Viewer. In this case, if variants
were filtered and/or sorted on the original SNV Viewer page, these filtering and sorting may be reset
or changed, and the state of SNV Viewer on the new tab will retain.</p></div></div><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="transcripts"></a>2. Transcripts<a class="hash-link" href="#transcripts" title="Direct link to heading">#</a></h4><p>The panel contains gene transcripts from two databases: <a href="https://www.ensembl.org/Homo_sapiens/Info/Index" target="_blank" rel="noopener noreferrer">Ensembl</a>
and <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a>.
Ensembl transcript models are annotated directly on the reference genome, while RefSeq annotates on mRNA sequences.
Due to sequence differences between the reference genomes and individual mRNAs, some of the RefSeq mRNAs may not
map perfectly to the reference genome.</p><p>By default, a table with transcripts from Ensembl is shown. To open the table with transcripts from RefSeq,
click on the corresponding switch button:</p><p align="center"><img src="/ngsw-docs-en/assets/images/refseq_transcripts-001899ce3fc2e8524fea5697d7cbeb57.png" width="700"></p><div class="admonition admonition-note alert alert--secondary"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"></path></svg></span>note</h5></div><div class="admonition-content"><p>On &quot;<a href="/ngsw-docs-en/settings/profile#transcripts">Profile Settings&quot; page</a>, you can choose which of the gene transcript
models will be shown by default.</p></div></div><p>The gene transcript table from <a href="https://www.ensembl.org/Homo_sapiens/Info/Index" target="_blank" rel="noopener noreferrer">Ensembl</a> has the following columns:</p><ul><li><strong><em>Transcript</em></strong> is a transcript ID from <a href="https://www.ensembl.org/Homo_sapiens/Info/Index" target="_blank" rel="noopener noreferrer">Ensembl</a> database
(ENSTxxxxxxxxxxx). For the first, main transcript, a transcript from <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a>
(NM_xxxxxx.x) is also given. You can open the transcript page by clicking on the ID.
The main gene transcript, according to <a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/#Select" target="_blank" rel="noopener noreferrer">MANE Select</a>,
<a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/#:~:text=MANE%20Plus%20Clinical%3A%20The%20MANE,variants%20available%20in%20public%20resources." target="_blank" rel="noopener noreferrer">MANE plus clinical</a>
or <a href="https://www.ensembl.org/info/genome/genebuild/canonical.html" target="_blank" rel="noopener noreferrer">Ensembl canonical</a>, is marked
with <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAF0AAABdCAYAAADHcWrDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAQDSURBVHhe7ZrBjRQxEEXJiDiIgziQCIEbV0KACAiAAyFw4ohWQgghJNTshynJ9lS7v+2ye9uuL73b2O1501Plcc+zzTM8Lv2EUNK/PvzZPn35vb39+HN7+e779vz1N+fGizcP26v3P/65gSMmh9IxkXYxRwcfAj6AXHal4+7GYG1i55ic+F3p+MS0yRweONSiSve6bQfqfZo76V7D7fnw+dfN7v/cSdcGOW2kZSaSjk9EG+S0E24nI+m+W+lHuJuJpKPoawOcdrA5kUTSfZvYj7CuR9K1Fzt2SFz6QCQufSASE+moV2jCJWjznIG2thwtfU9iIh2LKQ3GaHONZPS6JadJfwrHDez5d5hLS0fOPFgL980lubz0M48d0kMoNpeXjgcl2nwjwLVrcnnpyBklpmW9U0g/o6HWNFDJFNKR0Xd7S6aRPrKh1jZQyTTSRzbU2gYqmUY6MqLEWKxzKukjGmpraUGmko6vfe+HKBaZSjrS8xmt1Rqnk96zobY2UMl00pEeDdVyfVNK79FQLRqoZErpPUqMZS4nHa9jzj1a3lhKydqYtKxNMlx67n/bEsu7nWmgKD9TSw//dJOLRUPFHEzkwTqTS0rH65kSY3EIxjRQuc700pk70KLEMJE1TS+dLTEtb5BdU+nrW9YkOUU6YL76LXv20hK2hPSeDbVm7iWkg14NteZbtIx0ZlxNQ2X25jVrQdJxJUhOld6jxLBrwbVrxl1eOmBKTElDrS1ZS0lnfzWydzsTba6lpAPm7mSeKjHnOog2djnpVodg7OGWNnY56RYNlV1D2kBLx08jHbQ2VGZvntvzLymdaai5EsMkd/0lpbMlRmuo7PXTcSFLSgdMidDu9pYGKiwrvaahstc+2ucvKx2U/qJkvh25BiwsLZ2ZKywxTKyuizBz7SF5ctLZEoO52Ovu7c1DLN/DHpInJx0wJYZpnshRAxWWl87s2dkcNVBheemAuduZaHNruPRH2BPDXJiTScGlP8I21FyYBiq49BstJYZtoIJLv9HSUEuv5dJvtJQYbb4cLj2A+ZmfprS0AJceUHO3s3vzEJeeUNJQmcMtDZeewM6P9L5G7fxAcgnpJSWmZG8e4tIVmBJT00AFl67A7NlrGqjg0nc4utu1MSyXk446isUcUVtvBYzfQ3t9CZhDW3NKy7UkJtIdDolLH4jEpQ9EEkm3qI3OPpJIesuWy8mDJiyJpJc83nLKCB8/RtJrD4ucY8Ij6kg6/kvidb0PYSLpSMv5haOT/pK+k46g6GuDnXLCBipRpXuZsQEOtajSEYj33Uw92h0u2ZUuQY33u54Hro5OQw+lI7jrMRE+gNaTttmAC/yoRFU4ki2hpHts49JPiEsfnm37CzH4jI5dvUxJAAAAAElFTkSuQmCC" width="20">. The gene transcript chosen
as the main one for this variant is framed. Impact, HGVSp, and HGVSc values for this transcript are shown
for the variant in SNV Viewer table.</li></ul><ul><li><strong><em>IMPACT</em></strong> is the predicted effect of the variant on the protein
(see <a href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer#variant-impacts">table with variant impacts values in the annotation</a>).
HIGH impact value is colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>, MODERATE
is colored <span style="background-color:#E2CC81;border-radius:2px;color:#fff;padding:0.2rem">yellow</span>, and LOW and MODIFIER are
colored <span style="background-color:#999AAB;border-radius:2px;color:#fff;padding:0.2rem">gray</span>.</li><li><strong><em>HGVSp</em></strong> is an amino acid substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS notation</a>: “p.” prefix
(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution.</li><li><strong><em>HGVSc</em></strong> - is a nucleotide substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS notation</a>: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence) prefix +
genomic position of the substituted nucleotide + reference allele &gt; alternative allele.</li></ul><p>The gene transcript table from <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a> has the following columns:</p><ul><li><strong><em>Transcript</em></strong> is a transcript ID from <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a> database
(NM_xxxxxx.x). You can open the transcript page by clicking on the ID.
The main gene transcript, according to <a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/#Select" target="_blank" rel="noopener noreferrer">MANE Select</a>,
<a href="https://www.ncbi.nlm.nih.gov/refseq/MANE/#:~:text=MANE%20Plus%20Clinical%3A%20The%20MANE,variants%20available%20in%20public%20resources." target="_blank" rel="noopener noreferrer">MANE plus clinical</a>
or <a href="https://www.ensembl.org/info/genome/genebuild/canonical.html" target="_blank" rel="noopener noreferrer">Ensembl canonical</a>, is marked
with <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAF0AAABdCAYAAADHcWrDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAQDSURBVHhe7ZrBjRQxEEXJiDiIgziQCIEbV0KACAiAAyFw4ohWQgghJNTshynJ9lS7v+2ye9uuL73b2O1501Plcc+zzTM8Lv2EUNK/PvzZPn35vb39+HN7+e779vz1N+fGizcP26v3P/65gSMmh9IxkXYxRwcfAj6AXHal4+7GYG1i55ic+F3p+MS0yRweONSiSve6bQfqfZo76V7D7fnw+dfN7v/cSdcGOW2kZSaSjk9EG+S0E24nI+m+W+lHuJuJpKPoawOcdrA5kUTSfZvYj7CuR9K1Fzt2SFz6QCQufSASE+moV2jCJWjznIG2thwtfU9iIh2LKQ3GaHONZPS6JadJfwrHDez5d5hLS0fOPFgL980lubz0M48d0kMoNpeXjgcl2nwjwLVrcnnpyBklpmW9U0g/o6HWNFDJFNKR0Xd7S6aRPrKh1jZQyTTSRzbU2gYqmUY6MqLEWKxzKukjGmpraUGmko6vfe+HKBaZSjrS8xmt1Rqnk96zobY2UMl00pEeDdVyfVNK79FQLRqoZErpPUqMZS4nHa9jzj1a3lhKydqYtKxNMlx67n/bEsu7nWmgKD9TSw//dJOLRUPFHEzkwTqTS0rH65kSY3EIxjRQuc700pk70KLEMJE1TS+dLTEtb5BdU+nrW9YkOUU6YL76LXv20hK2hPSeDbVm7iWkg14NteZbtIx0ZlxNQ2X25jVrQdJxJUhOld6jxLBrwbVrxl1eOmBKTElDrS1ZS0lnfzWydzsTba6lpAPm7mSeKjHnOog2djnpVodg7OGWNnY56RYNlV1D2kBLx08jHbQ2VGZvntvzLymdaai5EsMkd/0lpbMlRmuo7PXTcSFLSgdMidDu9pYGKiwrvaahstc+2ucvKx2U/qJkvh25BiwsLZ2ZKywxTKyuizBz7SF5ctLZEoO52Ovu7c1DLN/DHpInJx0wJYZpnshRAxWWl87s2dkcNVBheemAuduZaHNruPRH2BPDXJiTScGlP8I21FyYBiq49BstJYZtoIJLv9HSUEuv5dJvtJQYbb4cLj2A+ZmfprS0AJceUHO3s3vzEJeeUNJQmcMtDZeewM6P9L5G7fxAcgnpJSWmZG8e4tIVmBJT00AFl67A7NlrGqjg0nc4utu1MSyXk446isUcUVtvBYzfQ3t9CZhDW3NKy7UkJtIdDolLH4jEpQ9EEkm3qI3OPpJIesuWy8mDJiyJpJc83nLKCB8/RtJrD4ucY8Ij6kg6/kvidb0PYSLpSMv5haOT/pK+k46g6GuDnXLCBipRpXuZsQEOtajSEYj33Uw92h0u2ZUuQY33u54Hro5OQw+lI7jrMRE+gNaTttmAC/yoRFU4ki2hpHts49JPiEsfnm37CzH4jI5dvUxJAAAAAElFTkSuQmCC" width="20">. The gene transcript chosen
as the main one for this variant is framed.</li><li><strong><em>HGVSp</em></strong> is an amino acid substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS notation</a>: “p.” prefix
(protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution.</li><li><strong><em>HGVSc</em></strong> - is a nucleotide substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS notation</a>: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence) prefix +
genomic position of the substituted nucleotide + reference allele &gt; alternative allele.</li></ul><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="omim-phenotypes"></a>3. OMIM Gene-Phenotype Relationships<a class="hash-link" href="#omim-phenotypes" title="Direct link to heading">#</a></h4><p>Access to data from <a href="https://www.omim.org/" target="_blank" rel="noopener noreferrer">OMIM</a> database inside Genomenal is provided if the user has a
license to access the database. If you have been granted access to OMIM, there will be information on the
phenotypes associated with the gene in which the variant is located on the right side of &quot;Gene&quot; tab:</p><p align="center"><img src="/ngsw-docs-en/assets/images/omim_phenotypes-dee84bdc012e4850ec472bd00e5f5b08.png" width="500"></p><p>The information is presented in the form of a table with the following columns:</p><ol><li><strong><em>Phenotype</em></strong> is the full name of the phenotype associated with the gene. For some phenotypes,
there is a question mark before the name (e.g. &quot;<b>?</b>Bleeding disorder, platelet-type, 22&quot;), which
indicates that the relationship between the phenotype and gene is provisional. Some phenotypes are enclosed
in square brackets (e.g. &quot;<b>[</b>Urate oxidase deficiency<b>]</b>&quot;), which indicates mainly genetic variations
that lead to apparently abnormal laboratory test values. Some phenotypes are enclosed in curly brackets
(e.g. &quot;<b>{</b>Prostate cancer/brain cancer susceptibility, somatic<b>}</b>&quot;), which indicates mutations
that contribute to susceptibility to multifactorial disorders or to susceptibility to infection.</li><li><strong><em>Phenotype MIM number</em></strong> is a phenotype ID in OMIM database.
You can open the phenotype page by clicking on the link.</li><li><strong><em>Inheritance</em></strong> is a pattern of inheritance phenotype: <b>AD</b> - autosomal
dominant; <b>AR</b> - autosomal recessive; <b>DD</b> - digenic dominant; <b>DR</b> - digenic
recessive; <b>IC</b> - isolated cases; <b>Mi</b> - mitochondrial; <b>Mu</b> - multifactorial; <b>PADom</b> -
pseudoautosomal dominant; <b>PARec</b> - pseudoautosomal recessive; <b>SomMos</b> - somatic
mosaicism; <b>SMu</b> - somatic mutation; <b>XL</b> - X-linked; <b>XLD</b> - X-linked
dominant; <b>XLR</b> - X-linked recessive; <b>YL</b> - Y-linked.</li><li><strong><em>Phenotype mapping key</em></strong> is an OMIM phenotype map key (the map displays relationships of the
phenotype with associated phenotypes, phenotypic series and genes): <b>1</b> - the phenotype is placed on
the map due to its association with a gene, but the underlying defect is not known; <b>2</b> - the phenotype
has been placed on the map by statistical methods, no mutation has been found; <b>3</b> - the molecular basis of the
disorder is known; <b>4</b> - a contiguous gene duplication or deletion syndrome in which multiple genes are involved.</li></ol><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="clinvar-tab"></a>ClinVar<a class="hash-link" href="#clinvar-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/clinvar_tab-5fdca523057f4d95875e1165e4fc75e2.png" width="1000"></p><p>The tab contains information
about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#haplotype-phenotypes">haplotype phenotype significance</a>,
if the variant is part of a haplotype, and
about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#phenotypes">variant phenotype significance</a>
from <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> database. Both sections contain a table of
phenotypes that have been observed in individuals with this haplotype/variant and reported to ClinVar.
The table has the following columns:</p><ol><li><strong><em>Phenotype</em></strong> is a phenotype name (with links to various databases).</li><li><strong><em>Clinical Significance</em></strong> is a clinical significance of the phenotype (you can find the description of
significances <a href="https://preview.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv" target="_blank" rel="noopener noreferrer">here</a>).
Benign significance is colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">dark green</span>; Likely
benign - <span style="background-color:#0EB555;border-radius:2px;color:#fff;padding:0.2rem">light green</span>; Likely
pathogenic - <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>;
Pathogenic - <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>; all other values, including ones mixed from
different clinical significances, are colored <span style="background-color:#141E29;border-radius:2px;color:#fff;padding:0.2rem">black</span>.</li><li><strong><em>Review status</em></strong> is an assessment of the data source in which the clinical significance of the phenotype
was reported (you can find the description of review
statuses <a href="https://preview.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status" target="_blank" rel="noopener noreferrer">here</a>).</li></ol><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="haplotype-phenotypes"></a>Haplotype Phenotypes<a class="hash-link" href="#haplotype-phenotypes" title="Direct link to heading">#</a></h4><p>The section is present on the tab if the variant is part of a haplotype. The section includes:</p><ul><li><strong><em>Variant Haplotype ID</em></strong> is a haplotype ID in which the variant is included.
You can open the haplotype page in ClinVar by clicking on the link.</li><li>A table with phenotypes that have been observed in individuals with this haplotype and reported in
ClinVar (see table column descriptions above).</li></ul><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="phenotypes"></a>Phenotypes<a class="hash-link" href="#phenotypes" title="Direct link to heading">#</a></h4><p>The section on the variant phenotype significance includes the following information:</p><ul><li><strong><em>Clinical significance</em></strong> is the overall clinical significance of all phenotypes that were observed in
individuals with this variant and were reported in ClinVar. You can find the description of
significances <a href="https://preview.ncbi.nlm.nih.gov/clinvar/docs/clinsig/#clinsig_scv" target="_blank" rel="noopener noreferrer">here</a>.</li><li><strong><em>ClinVar ID</em></strong> is a variant ID in <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>.
You can open the variant page in ClinVar by clicking on the link.</li><li><strong><em>Review status</em></strong> is the total assessment of all data sources in which the clinical significance
of the phenotypes associated with the variant was reported. You can find the description of review
statuses <a href="https://preview.ncbi.nlm.nih.gov/clinvar/docs/variation_report/#review_status" target="_blank" rel="noopener noreferrer">here</a>.</li><li><strong><em>Allele ID</em></strong> is a variant allele ID in ClinVar.</li><li><strong><em>Allele origin</em></strong> is a total allele origin of the variant in different sources provided by ClinVar.</li><li><strong><em>Cross references</em></strong> are links to a variant in various databases.</li><li>A table with phenotypes that have been observed in individuals with this variant and reported in
ClinVar (see table column descriptions above).</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="gnomad-tab"></a>GnomAD<a class="hash-link" href="#gnomad-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/gnomad_tab-aff90c341183da2ae959ebc864dfeddc.png" width="1000"></p><p>The tab contains the information about allele frequencies of the variant
from the <a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD</a> database.
On the left, there is a general information about the variant:</p><ul><li><strong><em>Allele frequency</em></strong> is a total alternative allele frequency of the variant in high quality genotypes;</li><li><strong><em>Coverage</em></strong> is a median per-base depth of coverage binned to rages &lt;10, 10-100, ⩾100;</li><li><strong><em>Number of Homozygotes</em></strong> is a number of individuals homozygous for alternative allele;</li><li><strong><em>XY samples AF</em></strong> is a total alternative allele frequency in XY subpopulation;</li><li><strong><em>XX samples AF</em></strong> is a total alternative allele frequency in XX subpopulation.</li></ul><p>On the right, there are variant frequencies in different populations: Ashkenazi Jewish, Amish,
African/African American, European (Finnish), South Asian, Latino/Admixed American, European (non-Finnish),
East Asian, Middle Eastern and Other (includes the samples that were not uniquely assigned to other
ancestries by the random forest model). For each population, there are total alternative allele frequency (AF)
in high quality genotypes in the population, AFs in XY and XX subpopulations, and the number of homozygotes
in the population:</p><p align="center"><img src="/ngsw-docs-en/assets/images/af_example-ae9a0de8dacb18db22aa1a667c6b5c9f.png" width="260"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="exac-tab"></a>ExAC<a class="hash-link" href="#exac-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/exac_tab-8ed3b9b70083152a04163ccab90980fe.png" width="1000"></p><p>The tab contains information about the allele frequencies of the variant
in <a href="http://exac.broadinstitute.org/" target="_blank" rel="noopener noreferrer">ExAC</a> database:</p><ul><li><strong><em>Allele frequency</em></strong> is a total alternative allele frequency of the variant.</li><li><strong><em>Adjusted AF</em></strong> is an alternative allele frequency of the variant in the population with
genotype quality ≥20 and depth of coverage ≥10.</li><li>Alternative allele frequencies in African/African American, American, East Asian, Finnish, European
(non-Finnish), South Asian populations.</li><li>Alternative allele frequencies in non-TCGA (cohort from which normal tissue samples from cancer patients
were excluded) and non-Psych (cohort from which normal tissue samples from patients with mental disorders
were excluded) cohorts: a total alternative allele frequency (AF) in cohort, AF in cohort of the population with
genotype quality ≥20 and depth of coverage ≥10 (adjusted AF), and AFs in cohort of different populations:
African/African American, American, East Asian, Finnish, European (non-Finnish), South Asian.</li></ul><p>For each frequency, there is an alternative allele count in high quality genotypes (<strong><em>Allele count</em></strong>):</p><p align="center"><img src="/ngsw-docs-en/assets/images/allele_count-c3e86c5bde0ebd24644505b0f45612b1.png" width="170"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="other-frequencies-tab"></a>Other Frequencies<a class="hash-link" href="#other-frequencies-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/other_freq-f1a753105fba61a2c6f79bd123b2019e.png" width="1000"></p><p>The tab contains information about the allele frequencies of the variant
from <a href="https://www.internationalgenome.org/" target="_blank" rel="noopener noreferrer">1000 Genomes</a> (1000G), <a href="https://www.uk10k.org/" target="_blank" rel="noopener noreferrer">UK10K</a>
and <a href="https://esp.gs.washington.edu/drupal/" target="_blank" rel="noopener noreferrer">NHLBI GO Exome Sequencing Project</a> (ESP) projects.</p><p>From 1000 Genomes project, there are:</p><ul><li><strong><em>Allele frequency</em></strong> is a total alternative allele frequency of the variant;</li><li>Alternative allele frequencies in East Asian, European, African, American, South Asian populations.</li></ul><p>From UK10K project, there are:</p><ul><li><strong><em>Allele frequency</em></strong> is a total alternative allele frequency of the variant in combined genotypes
in UK10K cohort (TwinsUK + ALSPAC);</li><li><strong><em>TWINSUK AF</em></strong> is an alternative allele frequency in TwinsUK, that is a cohort of adult twins living in the UK;</li><li><strong><em>ALSPAC AF</em></strong> is an alternative allele frequency in ALSPAC, that is a cohort of fetuses.</li></ul><p>From ESP project, there are alternative allele frequencies in African American and European American populations.</p><p>For frequencies from UK10K and ESP projects, there is an alternative allele count in high quality
genotypes (<strong><em>Allele count</em></strong>):</p><p align="center"><img src="/ngsw-docs-en/assets/images/allele_count_uk10k-bed4a3c4a7d84f08014666024fb8c65b.png" width="170"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="conservation-tab"></a>Conservation<a class="hash-link" href="#conservation-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/conservation_tab-47bbd193349fe4fee65e01f8def04f63.png" width="1000"></p><p>The tab contains the information on the variant conservation:</p><ul><li><strong><em>Ancestral allele</em></strong> inferred from
the <a href="https://www.ensembl.org/info/genome/compara/epo_anchors_info.html" target="_blank" rel="noopener noreferrer">Enredo-Pecan-Ortheus (EPO)</a> multiple
alignments using a probabilistic method Ortheus. <strong>ACTG</strong> - high-confidence alleles, ancestral state supported
by the other two sequences; <strong>actg</strong> - low-confidence alleles, ancestral state supported by one sequence
only; <strong>N</strong> - failure, the ancestral state is not supported by any other sequence; <strong>-</strong> - the extant species
contains an insertion at this position; <strong>.</strong> - no coverage in the alignment.</li><li><strong><em>Altai Neanderthal</em></strong> is a genotype of a deep-sequenced Altai Neanderthal.</li><li><strong><em>Denisova</em></strong> is a genotype of a deep-sequenced Denisova.</li><li><strong><em>Vindija Neanderthal</em></strong> is a genotype of a deep-sequenced Vindija Neanderthal.</li><li>Conservation scores for the variant position in the genome estimated
using <a href="http://compgen.cshl.edu/fitCons/" target="_blank" rel="noopener noreferrer">fitCons</a> method. fitCons is a computational method that estimates
the likelihood of a mutation at any genomic position to affect fitness. It integrates functional assays
(such as ChIP-Seq) with selective pressure inferred using the INSIGHT method. The result of the prediction
is a score ρ in the range [0-1] that indicates the fraction of genomic positions evincing a particular pattern
of functional assay results, that are under selective pressure.
From fitCons results, there are <strong><em>Integrated FitCons rankscore</em></strong>, that is an integrated score obtained by
combining scores from three cell lines (HUVEC, H1 hESC and GM12878), and <strong><em>GM12878 FitCons rankscore</em></strong>,
<strong><em>H1-hESC FitCons rankscore</em></strong>, <strong><em>HUVEC fitCons rankscore</em></strong>, that are conservation scores in GM12878, H1 hESC
and HUVEC cell lines, respectively. For each rankscore, there are also score and confidence value.</li><li>Conservation scores for the variant position in the genome estimated
using <a href="http://mendel.stanford.edu/sidowlab/downloads/gerp/index.html" target="_blank" rel="noopener noreferrer">GERP++</a> program. GERP (Genomic Evolutionary
Rate Profiling) identifies constrained elements in multiple alignments by quantifying substitution deficits.
These deficits represent substitutions that would have occurred if the element were neutral DNA, but did not
occur because the element has been under functional constraint. GERP authors refer to these deficits as &quot;Rejected
Substitutions&quot; (RS). RS score is defined as the number of substitutions expected under neutrality minus the
number of substitutions &quot;observed&quot; at the position. From GERP++ results, there are <strong><em>GERP++ RS rankscore</em></strong>,
neutral rate and score.</li><li><strong><em>PhyloP vertebrate conservation rankscore</em></strong>, <strong><em>PhyloP mammalian conservation rankscore</em></strong>
and <strong><em>PhyloP primate conservation rankscore</em></strong>, and corresponding scores are conservation scores
derived from alignment of vertebrate, mammalian, and primate species, respectively, by
<a href="https://ccg.epfl.ch/mga/hg19/phylop/phylop.html" target="_blank" rel="noopener noreferrer">PhyloP</a> (phylogenetic P-values) program.
Interpretations of the scores are compared to the evolution that is expected under neutral drift.</li><li><strong><em>PhastCons vertebrate conservation rankscore</em></strong>, <strong><em>PhastCons mammalian conservation rankscore</em></strong>
and <strong><em>PhastCons primate conservation rankscore</em></strong>, and corresponding scores are conservation scores
derived from alignment of vertebrate, mammalian, and primate species, respectively, by
<a href="http://compgen.cshl.edu/phast/phastCons-HOWTO.html" target="_blank" rel="noopener noreferrer">phastCons</a> program.
PhastCons is a program for identifying evolutionarily conserved elements in a multiple alignment,
given a phylogenetic tree. It is based on a phylogenetic hidden Markov model.</li><li><strong><em>SiPhy rankscore</em></strong> and score are conservation scores for the variant position in the genome estimated
by <a href="http://portals.broadinstitute.org/genome_bio/siphy/documentation.html" target="_blank" rel="noopener noreferrer">SiPhy</a> using a simple generalization
of the substitution pattern estimation.</li><li><strong><em>BStatistic rankscore</em></strong> is a conservation score for the variant position in the genome,
derived from analysis of the genomic distributions of human polymorphisms and sequence differences
among five primate species relative to the locations of conserved sequence features
(you can read more <a href="//doi.org/10.1371%2Fjournal.pgen.1000471" target="_blank" rel="noopener noreferrer">here</a>).</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="protein-function-effect-tab"></a>Protein function effect<a class="hash-link" href="#protein-function-effect-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/protein_func_eff-019ffc3d57ac6ade2de5f8b4c26c12cc.png" width="1000"></p><p>The tab contains the results of the prediction of amino acid substitution effect on protein function,
obtained using the following algorithms and programs:</p><ul><li><strong><a href="https://sift.bii.a-star.edu.sg/" target="_blank" rel="noopener noreferrer">SIFT</a></strong> (Sorting Intolerant From Tolerant) predicts whether
an amino acid substitution affects protein function basing on sequence homology and the physical properties
of amino acids. The result of the prediction is <strong>rankscore</strong> and <strong>score</strong>, the values of which
vary from 0 to 1. An amino acid substitution is considered damaging if score ≤ 0.05 and tolerated if score &gt; 0.05.</li><li><strong><a href="http://genetics.bwh.harvard.edu/pph2/" target="_blank" rel="noopener noreferrer">PolyPhen</a></strong> (Polymorphism Phenotyping) predicts possible
impact of an amino acid substitution on the structure and function of a protein using
sequence homology and different databases and tools. The result is:<br>1. <strong>prediction</strong>:
<strong><em>Benign</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>); <strong><em>Possibly damaging</em></strong>
(colored <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>); <strong><em>Probably damaging</em></strong>
(colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>);<br>2. <strong>score</strong> whose values vary from 0
to 1: the lower, the more likely that the variant is benign, the higher - damaging.</li><li><strong><a href="https://sift.bii.a-star.edu.sg/sift4g/AboutSIFT4G.html" target="_blank" rel="noopener noreferrer">SIFT4G</a></strong> (SIFT Databases for Genomes) is a
faster version of SIFT. The result of SIFT4G is <strong>prediction</strong>: <strong><em>Tolerated</em></strong>
(colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>), or <strong><em>Damaging</em></strong>
(colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>.</li><li><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752137/" target="_blank" rel="noopener noreferrer">LRT</a></strong> (Likelihood Ratio Test)
identifies deleterious mutations that disrupt highly conserved amino acids within protein-coding
sequences basing on the evolutionary models of the DNA sequence. The result is <strong>prediction</strong>:
<strong><em>Neutral</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>), or
<strong><em>Deleterious</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>; <strong>omega</strong>.</li><li><strong><a href="http://fathmm.biocompute.org.uk/" target="_blank" rel="noopener noreferrer">FAtHMM</a></strong> (Functional Analysis through Hidden Markov Models)
predicts the functional consequences of both coding variants and non-coding variants
basing on sequence homology. The result is <strong>prediction</strong>: <strong><em>Tolerated</em></strong>
(colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>), or <strong><em>Damaging</em></strong>
(colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>.</li><li><strong><a href="https://spliceailookup.broadinstitute.org/" target="_blank" rel="noopener noreferrer">SpliceAI</a></strong> is a deep neural network that predicts
the probability of the position to be a splice acceptor or donor from a pre-mRNA sequence.
The result is: <br>1. <strong>Delta score</strong> (DS) is the probability of the variant being splice-altering
at any position within a given interval (+/-50 bp between the variant and gained/lost splice site by default).
Delta scores range from 0 to 1:<br>- <strong><em>Acceptor gain</em></strong> is the probability of a position within a
given interval to be splice acceptor; <br>- <strong><em>Acceptor loss</em></strong> is the probability of a position within a
given interval not to be splice acceptor; <br>- <strong><em>Donor gain</em></strong> is the probability of a position within a
given interval to be splice donor; <br>- <strong><em>Donor loss</em></strong> is the probability of a position within a
given interval not to be splice donor. <br>DS is defined as the maximum of these four scores.
Values 0 ≤ DS &lt; 0.2 are colored <span style="background-color:#141E29;border-radius:2px;color:#fff;padding:0.2rem">black</span>,
0.2 ≤ DS &lt; 0.5 - <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>,<br>0.5 ≤ DS &lt;
0.8 - <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>, 0.8 ≤ DS ≤
1 - <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>. <br>2. <strong>Delta position</strong> conveys information about
the location where splicing changes relative to the variant position (positive values are downstream of
the variant, negative values are upstream). Delta positions corresponds to the DS
scores.<br>3. Link to <a href="https://spliceailookup.broadinstitute.org/" target="_blank" rel="noopener noreferrer">SpliceAI Lookup</a> website for
more detailed information about the scores for the variant.</li><li><strong><a href="https://cadd.gs.washington.edu/" target="_blank" rel="noopener noreferrer">CADD</a></strong> (Combined Annotation-Dependent Depletion) scores the
deleteriousness of a variant by integrating multiple annotations into one metric by contrasting variants
that survived natural selection with simulated mutations. The result is: <br>- <strong>Raw</strong> score has
relative meaning and is interpretable as the extent to which the annotation profile for a variant
suggests that the variant is likely to be &quot;observed&quot; (negative values) vs &quot;simulated&quot; (positive values).
Higher values indicate that a variant is more likely to have deleterious
effect. <br>- <strong>Phred</strong> is a Phred-scaled score. Phred &lt; 10 is
colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>, 10 ≤ Phred &lt;
20 - <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>, Phred ≥
20 - <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>.</li><li><strong><a href="http://hollywood.mit.edu/burgelab/maxent/Xmaxentscan_scoreseq.html" target="_blank" rel="noopener noreferrer">MaxEntScan</a></strong> predicts
the fitness of a variant position as either a donor or an acceptor splice site based on a maximum entropy model.
The result of prediction is scores for alternative (<strong>Alt score</strong>) and reference (<strong>Ref score</strong>) alleles necessary
to predict the loss of a native splice site for a variant. The difference between these scores (<strong>Diff</strong>) is also
given.<br>Alt score &lt; -19 is colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>, -19 ≤
Alt score &lt; -3 - <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>, Alt
score ≥ -3 - <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>.</li><li><strong><a href="http://mutationassessor.org/r3/" target="_blank" rel="noopener noreferrer">MutationAssessor</a></strong> predicts the functional impact of variant amino acid
substitution in proteins basing on evolutionary conservation of the affected amino acid in protein homologs.
The result is:<br>1. <strong>Prediction</strong>: predicted functional, i.e. <strong><em>High</em></strong>
(colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>) or <strong><em>Medium</em></strong>
(colored <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>), or predicted non-functional, i.e.
<strong><em>Low</em></strong> or <strong><em>Neutral</em></strong> (both are colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>).<br>2.
<strong>Score</strong>. Score ≥ 3,5 corresponds to the high impact, 3,5 &lt; Score ≤ 1,935 - to the medium impact,
1,935 &lt; Score ≤ 0,8 - to the low impact, 0,8 &lt; Score - to the neutral impact.</li></ul><p>Additional results of the prediction of amino acid substitution effect on protein function are located
on <strong><em>Protein function effect (additional)</em></strong> tab:</p><p align="center"><img src="/ngsw-docs-en/assets/images/protein_func_add-82cf3cb2a2c4bb5efe6bbb6a7dee0715.png" width="1000"></p><ul><li><strong><a href="http://provean.jcvi.org/index.php" target="_blank" rel="noopener noreferrer">PROVEAN</a></strong> (Protein Variation Effect Analyzer) predicts whether
an amino acid substitution has an impact on the biological function of a protein. The result
is <strong>prediction</strong>: <strong><em>Neutral</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>.</li><li><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5270233/" target="_blank" rel="noopener noreferrer">Meta SVM</a></strong>
is a meta-analytic support vector machine (SVM) that accommodates 10 component scores (SIFT, PolyPhen-2 HDIV,
PolyPhen-2 HVAR, GERP++, MutationTaster, MutationAssessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum
frequency observed in the 1000 genomes populations. The result is <strong>prediction</strong>:
<strong><em>Tolerated</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>; <strong>reliability index</strong>.</li><li><strong><a href="https://academic.oup.com/hmg/article/24/8/2125/651446" target="_blank" rel="noopener noreferrer">Meta LR</a></strong> is a meta-analytic
logistic regression (LR) that accommodates 10 component scores (SIFT, PolyPhen-2 HDIV,
PolyPhen-2 HVAR, GERP++, MutationTaster, MutationAssessor, FATHMM, LRT, SiPhy, PhyloP) and the maximum
frequency observed in the 1000 genomes populations. The result is <strong>prediction</strong>:
<strong><em>Tolerated</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>; <strong>reliability index</strong>.</li><li><strong><a href="http://bejerano.stanford.edu/mcap/" target="_blank" rel="noopener noreferrer">M-CAP</a></strong> (Mendelian Clinically Applicable Pathogenicity) is a
pathogenicity classifier for rare missense variants in the human genome that is tuned to the high sensitivity
required in the clinic. It combines various pathogenicity scores (including SIFT, Polyphen-2 and CADD)
with novel features and a powerful model. The result is <strong>prediction</strong>: <strong><em>Tolerated</em></strong>
(colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or <strong><em>Damaging</em></strong>
(colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>.</li><li><strong><a href="https://academic.oup.com/bioinformatics/article/25/21/2744/227925" target="_blank" rel="noopener noreferrer">MutPred</a></strong> classifies an
amino acid substitution as disease-associated or neutral and predicts molecular cause of disease.
The result is <strong>rankscore</strong>, <strong>score</strong>, <strong>transcript ID</strong> (from <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">UniProtKB</a>),
<strong>amino acid change</strong>, <strong>top 5 features</strong>. Higher score values indicate that a variant is more
likely to have deleterious effect.</li><li><strong><a href="https://www.nature.com/articles/s41467-020-20847-0" target="_blank" rel="noopener noreferrer">MVP</a></strong> (Missense Variant Pathogenicity)
is a prediction method that uses deep residual network to leverage large training data sets and
many correlated predictors. The result is <strong>rankscore</strong> and <strong>score</strong>.</li><li><strong><a href="https://www.biorxiv.org/content/10.1101/148353v1" target="_blank" rel="noopener noreferrer">MPC</a></strong> (Missense badness, PolyPhen-2, and Constraint)
is a <em>de novo</em> missense deleteriousness metric based on identifying sub-genic regions that are depleted
of missense variation. The result is <strong>rankscore</strong> and <strong>score</strong>. Higher score values indicate that
a variant is more likely to be pathogenic.</li><li><strong><a href="https://illumina.github.io/NirvanaDocumentation/3.14/data-sources/primate-ai/" target="_blank" rel="noopener noreferrer">Primate AI</a></strong>
is a deep residual neural network for classifying the pathogenicity of missense mutations
based on the common missense variants in non-human primate species. The result is <strong>prediction</strong>:
<strong><em>Tolerated</em></strong> (colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>score</strong>; <strong>rankscore</strong>.</li><li><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570203/" target="_blank" rel="noopener noreferrer">DEOGEN2</a></strong>
is a method that predicts whether a protein-coding variant likely affects the carrier individual&#x27;s health.
The method incorporates heterogeneous information about the molecular effects of the variants,
the domains involved, the relevance of the gene and the interactions in which it participates.
The result of the prediction is <strong>rankscore</strong>.</li><li><strong><a href="https://cbcl.ics.uci.edu/public_data/DANN/" target="_blank" rel="noopener noreferrer">DANN</a></strong> (Domain-Adversarial Neural Network)
is a deep neural network trained on the same feature set and training data as CADD.
The result is <strong>rankscore</strong> and <strong>score</strong>. Higher score values indicate that a variant is more
likely to have deleterious effect.</li><li><strong><a href="https://fathmm.biocompute.org.uk/fathmmMKL.htm" target="_blank" rel="noopener noreferrer">FAtHMM-MKL</a></strong> (Multiple Kernel Learning)
is a machine learning approach that integrates functional annotations from ENCODE with nucleotide-based
sequence conservation measures. The result is <strong>prediction</strong>: <strong><em>Neutral</em></strong>
(colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>p-value</strong>; <strong>rankscore</strong>; <strong>group of features</strong>.</li><li><strong><a href="http://fathmm.biocompute.org.uk/fathmm-xf/" target="_blank" rel="noopener noreferrer">FAtHMM-XF</a></strong> (eXtended Feature)
is a method for predicting pathogenic point mutations in the human genome based on an extensive feature set.
It outperforms competitors particularly in non-coding regions. The result is <strong>prediction</strong>: <strong><em>Neutral</em></strong>
(colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">green</span>) or
<strong><em>Damaging</em></strong> (colored <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>); <strong>p-value</strong> (lower p-value indicate
that a variant is more likely to have deleterious effect); <strong>rankscore</strong>.</li><li><strong><a href="http://www.columbia.edu/~ii2135/eigen.html" target="_blank" rel="noopener noreferrer">Eigen</a></strong> is a spectral approach integrating functional
genomic annotations for coding and non-coding variants. The result is <strong>rankscore</strong> and <strong>score</strong>.</li><li><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4731313/" target="_blank" rel="noopener noreferrer">Eigen-PC</a></strong> is a conceptually simpler meta-score,
which is based on the decomposition of the annotation covariance matrix, and using the lead vector
to weight the individual annotations. The result is <strong>rankscore</strong>, <strong>score</strong> and <strong>score in phred scale</strong>.</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="other-tab"></a>Other<a class="hash-link" href="#other-tab" title="Direct link to heading">#</a></h3><p align="center"><img src="/ngsw-docs-en/assets/images/other_tab-9deab340d4df7a9011d80ac507aeb97e.png" width="1000"></p><p>The tab contains additional information about the variant:</p><ul><li><strong>Coding sequence position</strong> is a genomic position of a variant reference allele.</li><li><strong>Protein position</strong> is a reference amino acid position in protein.</li><li><strong>Amino acids</strong>: reference amino acid / new amino acid resulting from the substitution.</li><li><strong>Codons</strong> is an amino acid substitution written as codons (e.g. GAG/CAG = E/Q (Glutamic acid -&gt; Glutamine)).</li><li><strong>Amino acid position</strong> is a reference amino acid position in protein.</li><li><strong><a href="https://appris.bioinfo.cnio.es/#/" target="_blank" rel="noopener noreferrer">APPRIS</a> annotation</strong> is an annotation of principal splice isoforms of human.
The isoform for each gene is selected as the principal one by combining protein structural information,
functionally important residues and evidence from cross-species alignments.
Possible annotation values: PRINCIPAL1-5 (principal isoforms, with 1 being the most reliable),
ALTERNATIVE1 (candidate transcript model conserved in at least three tested non-primate species),
ALTERNATIVE2 (candidate transcript model conserved in fewer than three tested non-primate species).</li><li><strong>Geuvadis EQTL target gene</strong> is target gene IDs from <a href="https://www.ensembl.org/index.html" target="_blank" rel="noopener noreferrer">Ensembl</a>
identified by the analysis of expression quantitative trait loci (eQTL)
in <a href="https://www.biorxiv.org/content/10.1101/008797v1" target="_blank" rel="noopener noreferrer">GEUVADIS</a> (Genetic European Variation in Disease) project.</li><li><strong>ENIGMA clinical significance</strong> is a clinical significance of the variant if it&#x27;s located in the
known or suspected breast and/or ovarian predisposition genes (such as <em>BRCA1</em>, <em>BRCA2</em>)
determined by <a href="https://enigmaconsortium.org/" target="_blank" rel="noopener noreferrer">ENIGMA</a> consortium.
Benign significance is colored <span style="background-color:#219653;border-radius:2px;color:#fff;padding:0.2rem">dark green</span>; Likely
benign - <span style="background-color:#0EB555;border-radius:2px;color:#fff;padding:0.2rem">light green</span>; Likely
pathogenic - <span style="background-color:#F2994A;border-radius:2px;color:#fff;padding:0.2rem">orange</span>;
Pathogenic - <span style="background-color:#E95726;border-radius:2px;color:#fff;padding:0.2rem">red</span>; all other values, including ones mixed from
different clinical significances, are colored <span style="background-color:#141E29;border-radius:2px;color:#fff;padding:0.2rem">black</span>.
When hovering over a field value, you will see a comment on clinical significance.</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="custom-annotation-sources-tab"></a>Custom annotation sources<a class="hash-link" href="#custom-annotation-sources-tab" title="Direct link to heading">#</a></h3><p>The tab is present in the details panel if at least one custom annotation
was <a href="/ngsw-docs-en/settings/custom-annotations#add-custom-annotation">added</a> to the system
before the variants in the sample were annotated. The tab contains the column values from the
custom annotations that the variant was annotated with:</p><p align="center"><img src="/ngsw-docs-en/assets/images/custom_annot_tab-cbef0e8fd9809c5ef6793d92455ffb45.png" width="1000"></p><p>Deleted annotation data are marked &quot;<em>deleted</em>&quot; for variants that were annotated with
it before it was deleted.</p></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">« SNV Viewer</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/main/snvs-indels/basic-filtering"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Basic Filtering of Variants »</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#panel-tabs" class="table-of-contents__link">Details Panel Tabs</a><ul><li><a href="#common-tab" class="table-of-contents__link">Common</a></li><li><a href="#gene" class="table-of-contents__link">Gene</a></li><li><a href="#clinvar-tab" class="table-of-contents__link">ClinVar</a></li><li><a href="#gnomad-tab" class="table-of-contents__link">GnomAD</a></li><li><a href="#exac-tab" class="table-of-contents__link">ExAC</a></li><li><a href="#other-frequencies-tab" class="table-of-contents__link">Other Frequencies</a></li><li><a href="#conservation-tab" class="table-of-contents__link">Conservation</a></li><li><a href="#protein-function-effect-tab" class="table-of-contents__link">Protein function effect</a></li><li><a href="#other-tab" class="table-of-contents__link">Other</a></li><li><a href="#custom-annotation-sources-tab" class="table-of-contents__link">Custom annotation sources</a></li></ul></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">© 2019-2023 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.e7631c00.js"></script>
<script src="/ngsw-docs-en/assets/js/main.9097e47c.js"></script>
</body>
</html>