<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Quality Control Settings | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/settings/settings/quality-control-settings"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Quality Control Settings | Genomenal"><meta data-react-helmet="true" name="description" content="Quality control is an essential first step in sequencing data analysis."><meta data-react-helmet="true" property="og:description" content="Quality control is an essential first step in sequencing data analysis."><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/settings/settings/quality-control-settings"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/settings/settings/quality-control-settings" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/settings/settings/quality-control-settings" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.bbcfd9a8.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.6362fd06.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/overview">Getting to Know the Service</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Analysis Results</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Settings</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Settings Presets</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/settings/settings">Settings Presets</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/settings/settings/quality-control-settings">Quality Control Settings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/settings/alignment-settings">Alignment Settings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/settings/variant-discovery-settings">Variant Discovery Settings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/settings/main-presets">Main Settings Presets</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Report Templates</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/gene-panels">Gene Panels</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/capture-kits">Capture Kits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/custom-annotations">Custom Annotations</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/settings/profile">Profile Settings</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Quality Control Settings</h1></header><p>Quality control is an essential first step in sequencing data analysis.
It includes checking quality of sample reads and, if some of the quality metrics were not met,
cleaning the data up (trimming and/or filtering). Reads quality is checked using the
<strong><em>Falco</em></strong> tool, which runs on average three times faster than the equivalent FastQC tool<sup id="fnref-1"><a href="#fn-1" class="footnote-ref">1</a></sup>.
Reads are trimmed using the <strong><em>Cutadapt</em></strong> tool, which finds and removes adapter sequences, primers,
poly(A) tails, and other types of unwanted sequence from reads<sup id="fnref-2"><a href="#fn-2" class="footnote-ref">2</a></sup>. Reads are filtered using the
<strong><em>FASTX-Toolkit Quality Filter</em></strong> tool from <a href="http://hannonlab.cshl.edu/fastx_toolkit/" target="_blank" rel="noopener noreferrer">FASTX-Toolkit</a>,
which filters sequences based on quality.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="disable-qc"></a>Disable quality control<a class="hash-link" href="#disable-qc" title="Direct link to heading">#</a></h3><p>The primary quality check is carried out for all FASTQ or BAM files uploaded to the system and
cannot be disabled. You can disable the following cleaning the data and re-checking the quality.
To do this, turn off the toggle:</p><p align="center"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAiYAAAArCAYAAABMz0OxAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAA2USURBVHhe7Z3tT1vnGYf3H+3D+n1Sp33mkxXNEhPLFlQJRSrL5EWaoi5Wt2UZaF1ZpxAprFPJutEshTYkWZyF2CFAHAjYvNmAeTMhTgokJE4JCe1v5z7nPOb4+DF+wQZX+l3SrTjPeV4OTqRzcT8v53sghBBCCKkRKCaEEEIIqRkoJoQQQgipGSgmhBBCCKkZKCaEEEIIqRkoJoQQQkgN8+rVazx9lsbjJ0+xsrqGpeQTLC4/rnosLKWsWExhfvFRJqRseeUJHqXWsb7xHFtb2/j222/tu90/FBNCCCGkBtl8/hIPDRHRSUM1Q8RDBCSxsIq5+YeYmUvasYzYzJIZ8lnKZo2QOsvJx4Y8vaiIoFBMCCGEkBoi/XLLzIzopKGakRESW0Zi8UVMTiUQnZjBWDSOB2PTRkzZMW2WybXJ6YQhK4tmm8XllCFU6X0JCsWEEEIIqRHWNp7nCEO1Q03ZSIZE5GIqtoDI+AyCQ9P403/mcPTcEn589hHe8q/h+79dN0M+S5lckzqhe9NmG2krfTz+6qkpJ+UICsWEEEIIOWTkAS5rSHTiUM0wpWQxhbnEipkhiU7MInB3GscvLGQkpNiQNoGBaYwbfUgGZeXhE7x586ZkOaGYkIJsbW3h2bNnWFxcRCqVsksJIYRUisOSkvmFR5g1pGTakJLRSAxnP5vBW/6vtOJRTEhb6UP6kj5FTnZ2dkqSE4oJyYuISCgU0kY4HDavE0II2R+HNX0j60kkUyLTL7Ju5MTf57WyUU5IX7IORfp+9Hgd33zzTdFysqeYpK74UFdXlxseL3wfBZHcsSt+FxhtN+/dd4W/8RdCMiSRSCRHQiTi8XjWNfks9QkhhJSOLHTViUM1Q60pkd000/EFjBhS8suOhFYw9hPSp/QtYzzbtHbsFCMnRYlJ0/vtaD9vx0d+NNV7zHLPyV6kvityQjEpCpEMp5Dkkw4pl+uF6hFCCMnPYe2+kYWusu03Eo3hj/+Oa8WiEiF9R6JxQ4JWil5vUpSYtI/aBYqdFHpPWnLS2r9tF9Y4FJOiUNkQyYwoRDokWyLX5E8lIapcZU4IIYQUj5xTohOHaoaawpHzRyam5nDtdnRfa0oKhfR9PRg1x/pq7WlRUzrliYnBdn+rec1z3vFAmupE4zuNOHNzwy6wWQvgjFHeeDaAzBVV9kkM6dletJ5ogEf6aziFtlDSrlQcqXAn/Kp9fSNOfRRAYtO+qNhDTLLbN8F/MYhk2r7oZCeNxM02nDrm3XuszPeQwsZIh13fA+9xP7qi7o43EDhrfA/vdCJmlyhin0h5Izqn7AIDq+wMAmtpxC7Z2SuPF03vd2L4sV2pTGSBq1syRD6kzBmSIZG66rqSGS6MJYSQ4jmsw9OsKZxFjIxOoel85adw3CFjyFhziWRRWZOyxQQjbbliku/h/7gXPqO8zteLzBVV9p4ffo8HDSIp71hyUGc8xP19LrnRkkbkfKPV5kgDTrW0wX/ckoa6o+2IOB1Ae2+77T31PrSeb0fryTztjTtXWSJPkx9tRt220/bYUtc5k2GP1XzajwZDGkQuGm2ZqatrQEeWgRj9+qTc6MMuUUTOSXn292+VNaPtnA8e98/s8aF31a5YBmpqRkmH4MygqJ058nedvEh7QgghhZFj5nXiUM3IZEsSK+aW3huhUa1IVCNuhMYwPjmHFy9eFsyaVHYqpxwxqfOh25EgSRvC43XXzUesAw1S93gHYg6JkPuWB7X3Y4cB6O7Nbu9eK5MOW/fgveBonwzgzIlGNJ0bNnRml9gFb97voa6+DcOObEqqx/o+PeecClKOmOT/mT2ng7tZqRJRAuJEyiQUIiEiIG4JUQJDCCGkMPLuG508VDOUmMRnlzEamcbv/jmplYhqhIwlYz5KrRXcPlz64teWU2g4Yj0ccxa/liMmxoM0KzFhPJ7bpVzzoHYT+1ikwIPWQdc6l60UIuFhDEdTyFzR3JvVvhFdS3ZBhiS63jHuod54+Nsl+Uj1NOf+zPZYWWIk6L6HMsUkd22Pfc91fgTdU0tFINMwbjERCZFwozIrTlQmxZltIYQQouewzi1JzK9icnoe9+5HUd9W/WkcFT81xpIx5xcKL4ItSkx0IZmDnB055YhJVvZAKFZMNtB7Uur50FvM2oqce1PtPfAes9ZyOMPr0dzDzgYS4V50nm/FKanXYGWNcn7mUr6HssRE/zMPf5j/WiGUmBRaJ6ITGEGtTxFBIYQQsjeHtRtHpnEmphIYGHqAH515qJWIaoSMJWPKibCvX782p3PyUfJUTvJyk1nmNYTC/Tv7wYqJeqCXKyaqvRe+Dx0ZoawIIjPLtDGA1qNSvw6ehkacajGuXwogeD5/xuSgxcS61pQ1NVYsxWQ89tqBU6zYEEIIAZaST7TyUM2w1pckER2fQag/XNXdOO6QsWRMWWeyvb1dWTHBliEO9dYDMGcKxH4gN/e4Hk6r3Wh2P5D3LSYqQ+BFh2PXimJ7M410Onfdh1MWrPbN6C5iwaiaNvK7dhyp72j/YtKGYVcGyro/nZh4sv9NTFLofleuGdKyZheVQDEZD7kuoUMyKBQTQggpDp04VDtETOQFe/JW4NuhQa1AVDNuh4bMM00qLyYGG31+/ULLWIe1cPV4l+NU2LTxMG0w+6m0mGwPtlr30TKQvU5lyrqPrEWmGllQ7RuMelnt0xF0trSh/fJuuSUE7vUoaQy0WNM55YtJGsHT0rcHrf2Ou1g16prTSToxyf3u08aY1kLg7t0sTwmonTW6bIgg4iJrS9xTOAolJro1KYQQQrI56IyJc+HrWCSGW7cHDzxj0hccNN9ALM+JvXbmlCUm2Emi67g8IL3GNUdWYieGDjXdcUy21LbB3+SB1+dDozw0Kywmzt1BnhNt6A0NI3DRj0bzge7K6OhkIdPeg8bTnQjIgtlQN84cs8p81xz6EDpjSdC77Va9cBDdcv7IUUu6yheTXdGTMZtliqjFB6/HB5+ZSdGJSSN8Jxs0P3ODUTd7KXEpiJSIfOimc+Q/kpTrxENlW/JlUwghhGRzGGtMMmISjZuS8PbvV7QSUY14+w8r5phy0JoSk3yUJyYG6X4r25CdHTFYDdoPdutBK4eqJZOaB3IlxEQwJCnQ0mQ/2K3w1PvRPet6QOeTBU37uiNNaL2SyM6iGH+LXWx21DOE63QvkmFNvyWKSU7fR5rRGU1j2M6O5IqJD72zCXS/p75nq03HSLkbhS3UOhH3VmBhrzUkaquwTmgIIYTkchi7csypnNkkIuNxBO8M4Sd/rt5R9O6QsWTM+MwCXr16Vb6Y7IutNLbdu3aqyc420u51JaWg2m8WaK/qVWPGQvoucP8ZMVGLX43vueA9l4CSDPdaE5EOueaWDzWFk28KiBBCSC6HcY6Jtfh1BdGJWfQP3MdvLtzXSkQ1QsbqHxjG0vLq/taYkNojR0wqjKTY1Dkl8qdu6kaQciUxe9UjhBCSy2Gd/DpnbxceujeKrp4+rURUI7p6bmMoPIq1tXVzu3DZ55iQ2qPaYiKIZKitwSIdEpIZkTIJJSTMlBBCSPkc9LtyzAPWFlYxHV/A8INx3LgZxM/+Mq4ViUqGjCFjjUWm8PLly/2d/Epqj2SfnK/SjUgZp7uWikiIyp64Q8rd0z2EEEKK56DfLqx25szMLiMSjSF45x7+8VkAPzj9RCsUlQjpW8aQsRaXVsxffPfakSNQTEhB5D+SLHqV4AJXQgipHAe9O0dN50xOz2EoHMH1G334dXv1zjSRvmUMydA8f/684KmvAsWEEEIIOSTSL7e0AlGtsLImq/a2YStr0vPldfzig8q/aVj6lL5DxhjLyYfmL7mFpnEEigkhhBByiKxtPNdKRDVCxGQ3azKP+8PjuNV3F5e7e/HzDx5oBaOckL6kT+l7KjaLdDqdyZZQTAghhJAa5yDPNcmsNZlLYnxiDoP3xhD4X8gUiV/9rX9fa06krfQhfUmfY5FJPH36NHN2SSEpESgmhBBCyCEjD+yDlpPEwkPEZpbMl/oNDo0aInEHPV9cR/snX+KnraVP7UgbaSt9SF+jYxNYX183p3DevHlTlJQIFBNCCCGkRjioaZ3MlM68JSfyDpuh8Bhu9Q3gytWbuHT5S1y4+AVO/DUEz9kJ/PD9paxMinyWMrkmdaSutJG20sf4ZAwbGxtZUkIxIYQQQr6DyILYg9ito+REMidxQ07GJ+fMNSd3+sMI3LyDXkMyPu+5iq5LX+BfXZ/j008v4+Knl8yQz1Im16SO1JU2A0MjSMwvmDs4y5ESgWJCCCGE1CByzkm1D2HblZNVzMwtmwtiZbfOvfsR9N+9j1u37+JGIIRr/72Fq9du4crVgBnyWcrkmtQZHHqAWHzOnLp58eKFuaZE7cApRUoEigkhhBBSw8jx9fJuHVmDIpmUpeQTrWTsJ5ScyLt0ZCvx5FTCfNnfyOgUwoakyBqU/oER8x07dweHzWkfWUMyNT2LZPKhOW0jQvL111+bu2+K2RacD4oJIYQQQjLZDZEKkQvJeohoiHBsbm6a0zOyw0ZCPkuZkhGp6xSScqVEoJgQQgghJINTUJSkSMhbgZ2hylW9/QqJgmJCCCGEEC1KNlTIWSTqPBJnVBKKCSGEEEJqBooJIYQQQmoE4P+UCDho3xffyAAAAABJRU5ErkJggg==" width="400"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="qc-metrics"></a>Quality control metrics<a class="hash-link" href="#qc-metrics" title="Direct link to heading">#</a></h3><table><tr><td>Metric</td><td>Metric threshold</td><td>Threshold default value</td></tr><tr><td>Number of reads</td><td><b>Min Total Sequences</b> is the minimal meaningful number of reads per file. If the sample file contains fewer reads before or after cleaning, then automatic sample processing stops.</td><td>10,000</td></tr><tr><td rowspan="2">Reads length distribution</td><td><b>Short Sequence Bound</b> is the maximal length of reads per file. Reads shorter than this value are discarded in processing.</td><td>20</td></tr><tr><td><b>Max Short Sequences Percent</b> is the maximal percent of short reads per file. If this threshold is exceeded, the file is considered faulty by length distribution.</td><td>25</td></tr><tr><td rowspan="2">Tiles reads quality</td><td><b>Max Tile Quality Deviation</b> is the maximal acceptable tile quality deviation per file. Tiles that exceed mean quality across all tiles by this threshold are considered faulty.</td><td>7</td></tr><tr><td><b>Max Tile Bad Quality Cells</b> is the maximal acceptable number of bad quality cells in a tile per file. If this threshold is exceeded, the cleaning by tiles is activated.</td><td>10</td></tr><tr><td rowspan="3">Per base reads quality</td><td><b>Base Start Count</b> is the number of nucleotides that represent the start of a read when checking quality distribution across all reads.</td><td>3</td></tr><tr><td><b>Base End Count</b> is the number of nucleotides that represent the end of a read when checking quality distribution across all reads.</td><td>3</td></tr><tr><td><b>Min Base Percentile Quality</b> is the minimal acceptable quality of a position in a read. When evaluating, the first (in the amount equal to Base Start Count), last (in the amount equal to Base End Count) and middle bases are considered separately. If this threshold is exceeded, the trimming/filtering of reads is performed.</td><td>20</td></tr><tr><td>Overrepresented sequences</td><td><b>Max Overrepresented Sequences Percent</b> is the maximal percent of overrepresented sequences per file. If this threshold is exceeded, the file is considered faulty by overrepresented sequences.</td><td>1</td></tr><tr><td>Adapter contamination</td><td><b>Max Adapter Contaminated Percent</b> is the maximal percent of reads contaminated with adapter sequences per file. If this threshold is exceeded, adapter clipping is activated.</td><td>1</td></tr><tr><td>Base N content</td><td><b>Max N Content Percent</b> is the maximal percent of unknown nucleotides (N) in any position among all nucleotides per file. If this threshold is exceeded, the file is considered faulty by base N content.</td><td>20</td></tr><tr><td rowspan="3">GC content</td><td><b>Min Derivative Threshold</b> is the minimal threshold for detecting the local extremes of GC per file. If secondary extremes are detected, the file is considered faulty by GC content.</td><td>0.085</td></tr><tr><td><b>Max Wave Sloping Threshold</b> is the maximal threshold for the slope of the local extreme of GC. Exceeding this threshold indicates GC-content disturbances, and in this case the file is considered faulty by GC content.</td><td>0.035</td></tr><tr><td><b>Min Wave Length</b> is the minimal acceptable length of GC-content disturbances (distance between two local extremes of GC). Exceeding this threshold indicates GC-content disturbances, and in this case the file is considered faulty by GC content.</td><td>4</td></tr><tr><td rowspan="5">Base sequence content (AT/GC ratio)</td><td><b>Max Cutoff at End</b> is the number of nucleotides from 3&#x27;-end that are skipped when evaluating GC-content.</td><td>3</td></tr><tr><td><b>Max Cutoff at Start</b> is the number of nucleotides from 5&#x27;-end that are skipped when evaluating GC-content.</td><td>15</td></tr><tr><td><b>Cutoff Difference Percent</b> is the difference threshold between base pairs (in percent). If the difference between A and T, or G and C exceeds this threshold, then the end nucleotides are skipped.</td><td>1</td></tr><tr><td><b>Mean Difference Threshold</b> is the threshold of the mean of difference between the base pairs (in percent). If the mean of difference between A and T, or G and C exceeds this threshold, then the file is considered faulty by base sequence content.</td><td>1</td></tr><tr><td><b>Max Difference Threshold</b> is the threshold of the maximum difference between base pairs (in percent). If in any position the difference from all reads maximum between A and T, or G and C exceeds this threshold, then the file is considered faulty by base sequence content.</td><td>20</td></tr></table><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="change-metric-thresholds"></a>Change metric thresholds<a class="hash-link" href="#change-metric-thresholds" title="Direct link to heading">#</a></h3><p>To change the metric threshold value, click on the current value and edit.
To save changes, press Enter or click outside edit value container.</p><div class="footnotes"><hr><ol><li id="fn-1"><a href="https://doi.org/10.12688/f1000research.21142.2" target="_blank" rel="noopener noreferrer">de Sena Brandine G. and Smith A.D. Falco: high-speed FastQC emulation for quality control of sequencing data. F1000Research 2021, 8:1874 (2021)</a><a href="#fnref-1" class="footnote-backref">â†©</a></li><li id="fn-2"><a href="https://doi.org/10.14806/ej.17.1.200" target="_blank" rel="noopener noreferrer">Martin M. Cutadapt Removes Adapter Sequences From High-Throughput Sequencing Reads. EMBnet.journal 17, 10:12 (2011)</a><a href="#fnref-2" class="footnote-backref">â†©</a></li></ol></div></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/settings/settings/settings"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Settings Presets</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/settings/settings/alignment-settings"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Alignment Settings Â»</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#disable-qc" class="table-of-contents__link">Disable quality control</a></li><li><a href="#qc-metrics" class="table-of-contents__link">Quality control metrics</a></li><li><a href="#change-metric-thresholds" class="table-of-contents__link">Change metric thresholds</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2023 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.bbcfd9a8.js"></script>
<script src="/ngsw-docs-en/assets/js/main.6362fd06.js"></script>
</body>
</html>