<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">ACMG Secondary Findings Report | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/acmg-sf-report"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="ACMG Secondary Findings Report | Genomenal"><meta data-react-helmet="true" name="description" content="The report includes the variants discovered in the sample, which are located in the genes"><meta data-react-helmet="true" property="og:description" content="The report includes the variants discovered in the sample, which are located in the genes"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/acmg-sf-report"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/acmg-sf-report" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/acmg-sf-report" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.b81db154.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.660889f1.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">🌜</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">🌞</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotypes Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Workflow Details</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>ACMG Secondary Findings Report</h1></header><p>The report includes the variants discovered in the sample, which are located in the genes
from <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the Secondary Findings (SF) list</a> of
American College of Medical Genetics and Genomics (ACMG). For these genes, specific mutations are known
to be causative of disorders with defined phenotypes that are
clinically actionable by an accepted intervention. The ACMG recommends that pathogenic or likely pathogenic
variants detected in any of these genes should be reported as they are of medical relevance and could be used
in the future to inform clinical treatment.</p><div class="admonition admonition-caution alert alert--warning"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg></span>caution</h5></div><div class="admonition-content"><p>ACMG secondary findings report generation has its limitations due to variant annotation
using <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>.
These limitations are described <a href="/ngsw-docs-en/results/acmg-sf-report#acmg-sf-limitations">below</a>.</p></div></div><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="acmg-sf-conditions"></a>Conditions for the report generation<a class="hash-link" href="#acmg-sf-conditions" title="Direct link to heading">#</a></h3><p>The report is generated for the sample if the following conditions are met:</p><ol><li>The sample is uploaded in FASTQ or BAM format or as a VCF file with a single sample.</li><li>The sample analysis was successfully completed (that is, all stages of the analysis have the status &quot;Complete&quot;).</li><li>For a sample, the variant annotation stage was successfully completed:
&quot;Germline SNVs/Indels annotation&quot; stage if Germline origin was selected when creating a report template,
or &quot;Somatic SNVs/Indels annotation&quot;, if Somatic origin.</li><li>The report template including &quot;SNVs/Indels in ACMG SF genes&quot; block
was <a href="/ngsw-docs-en/settings/report-templates/report-templates#add-template">created</a> before the sample was processed.</li><li>The report template including &quot;SNVs/Indels in ACMG SF genes&quot; block is active.</li></ol><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="acmg-sf-content"></a>Report content<a class="hash-link" href="#acmg-sf-content" title="Direct link to heading">#</a></h3><p>The report includes two sections with lists of variants discovered in the sample, which are filled in according
to the following scheme:</p><p align="center"><img src="/ngsw-docs-en/assets/images/acmg_sf_scheme-1c3033dc0cfaf3a3a96d35469566d0cb.png" width="560"></p><p><strong>TIER I</strong>: The most important variants for interpretation for which the following conditions are met:</p><ul><li>The variant is located in a gene from <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a>;</li><li>The variant is associated according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> with a
phenotype only of pathogenic or likely pathogenic clinical significance that is
listed in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a> for
the gene in which the variant is located;</li><li>The allele frequency in <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/" target="_blank" rel="noopener noreferrer">gnomAD 3</a> is empty or
below the frequency threshold for too frequent pathogenic variants, specified in the report template block
&quot;SNVs/Indels in ACMG SF genes&quot; (recommended default value = 0.03);</li><li>The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)
in the case of a VCF file with a single sample;</li><li>The variant meets all the conditions specified in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a>
for the gene in which the variant is located. These conditions may be associated with:<ul><li>the clinical significance of the phenotype (as noted above, the report primarily includes variants
associated with a phenotype only with pathogenic or likely pathogenic clinical significance
according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>; this condition is not required only
for variants in <em>HFE</em> gene);</li><li>type of inheritance;</li><li>the number of variants discovered in the sample and suitable for the conditions
that are located in one gene from the ACMG SF list (for example, for some variants with an autosomal
recessive inheritance to be included in the report, at least two such variants must be discovered in the sample);</li><li>the variant origin;</li><li>the genotype (for example, some variants are reported only if they are homozygous in alternative allele);</li><li>specific nucleotide and amino acid substitution of the variant.</li></ul></li></ul><p><strong>TIER II</strong>: Variants that require further analysis for which the following conditions are met:</p><ul><li>The variant is located in a gene from <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a>;</li><li>The variant is associated with at least one phenotype of pathogenic and/or likely pathogenic clinical
significance according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>;</li><li>The allele frequency in <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/" target="_blank" rel="noopener noreferrer">gnomAD 3</a> is empty or
below the frequency threshold for too frequent pathogenic variants, specified in the report template block
&quot;SNVs/Indels in ACMG SF genes&quot; (recommended default value = 0.03);</li><li>The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)
in the case of a VCF file with a single sample;</li><li>The variant does not meet at least one of the conditions in Tier I:<ul><li>the variant is not associated according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> with a
phenotype that is listed in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a> for the gene
in which the variant is located;</li><li>according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>, the variant is associated with at
least one phenotype of benign, likely benign, or uncertain clinical significance, which is listed
in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a> for the gene in which the variant is located;</li><li>the variant does not meet at least one of the conditions specified
in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a> for the gene in which the variant is located.</li></ul></li></ul><p align="center"><img src="/ngsw-docs-en/assets/images/acmg73-d892b957bb3567aa72167cdb73e077ae.png" width="1000"></p><p>Each section contains a table of variants with the following columns:</p><ol><li><strong>Gene</strong> is the common name of the gene in which the variant is located.
If you click on a gene, you will see a window with all the gene transcripts:</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/transcripts-41cae82d7eb56e98382842cd1903341a.png" width="600"></p><p>You can find the description of the transcripts&#x27; table
columns <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-panel#transcripts">here</a>.</p><ol start="2"><li><strong>Position</strong> is a coordinate of the variant in the genome (chromosome + start position).</li><li><strong>Genetic variant</strong> is the nucleotide and amino acid substitutions
using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">the HGVS notation</a>. Nucleotide substitution: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence) prefix +
genomic position of the substituted nucleotide + reference allele &gt; alternative allele.
Amino acid substitution: “p.” prefix (protein) + reference amino acid + amino acid position in protein + new
amino acid resulting from the substitution.</li><li><strong>Transcript</strong> is the main transcript ID from the <a href="https://www.ncbi.nlm.nih.gov/nuccore/" target="_blank" rel="noopener noreferrer">RefSeq</a> database
(NM_xxxxxx.x). If you click on a transcript, you will see the same window with all the gene transcripts, as when
you click on the &quot;Gene&quot; field.</li><li><strong>Consequence</strong> is the effect of the variant on genes. A detailed description of the possible values
can be found <a href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer#variant-consequences">here</a>.</li><li><strong>Allele frequency</strong> is an alternative allele frequency for the sample (in percentages).</li><li><strong>Coverage depth</strong> is a sequencing depth; the total number of reads of the sequence overlapping the variant
position for the sample.</li><li><strong>External links</strong> are links to pages with variant information in <a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a>,
<a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> and <a href="https://cancer.sanger.ac.uk/cosmic/" target="_blank" rel="noopener noreferrer">COSMIC</a> (if
it was <a href="/ngsw-docs-en/settings/custom-annotations#define-annotation-type">uploaded as a custom annotation</a>).</li><li>Links to the variant in embedded services: <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIgAAACBCAYAAAAbtfKaAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAEnQAABJ0Ad5mH3gAAAzQSURBVHhe7ZwNjx03FYb5//8AaKEI1AIpESECykdpFaoAJYVIaZRURV2pSoigFVUSIrRJmkuf3Wut13uOfWwfz9x716/0qs29Mx6P/fj1mcltv7WZmspoAjKV1QRkKqvVAHn58uWJv/761faTNtHG8+cv3NqbOq9FAXnv/b9u3nzrV5tvf+fKBV+//v7mk0+ONk+e/Hd7tKyH//x3th1MW7du3S22NVXWIoAwqdJESmZypYnlM76Tzsl5gtKn4YC0TCpmYoNqAJNM2kxI2jQUECZZmjCrOZ9tR/quxROSeg0DpHfVj/BMknoNA0QrIt9865fnilFA4s98Lh2vWSpqaevWR3ezbaU1Dm2QVJjiF3MM/dfv4fQ7jsOcy7Vp99AAHAKItrUwmJoYWCskTGpJTLR0Lg4ASN95OIVmn7UYIEx+SRZIGHSrSBOpjaUdA7NvGgIIA5IOEpNlUW7lW9uIlWtvDQdY9iVZhgAiDYx19bB9SOdbEkiTdeta2sCS1lG7JjMgoZgLRVxuj5UGwzoI2rbQkh5BpRQBIO6J4zDX4n4x9xg7fM4xmPM4vwdC6iHa3EVQioAwKLmCjgFKb0w6ngGwSAOEfrSKa8dtBRC8JyVARNstwOwiKFlAuFHpRiSTKEHSeXxmkTawPWLA1xh4rgcwIWWk+5IcQNkFqYBwY1Lncw4TACzS96VtQksPvEurqlW1sEjpvLREQOiU1OGSIT+sGm1bAgLpphkM6fjgXVlRHqrdhtZ86hEB0RIA37t/tLl587b4HQ4TmWuDgQEIBsk6UNYtat/EeFnuf600EQFhMtIOXvvFe+d+jHN8/Pzks/S4uBbRUqTFDOJaq2gJWUFZOklFQKSOPXx48SmCNEmPg/SgXIq0uFTDHIKkxZl6yS1HBERa+RIgfJYeF+qQUk3R6ssAyWkNl0+TpbYcERBpcqk7YrHdpMfg05rCb2uJHafTZRCLIQdKWIwjZa5BMJA8fvz0xFL9UWsm3LLv4suQHJJKaTIaEhEQLjgiBQCPIiu9IQYhPPpJ5/HdZRbjVdqyR42RCAjyLDC5OesNcFxYMZw3OkL3SaRoOraxR4yVCgjySJHWreGybiklleoSb0iygCCSpBWUOcljlIPEuyYpAoKIfUA5LSpPYaFe4DNqirSTmBuYGqd4K7449n6QmADJSatVRhVNU2fKJYlX/dYNiNS5ubUsJ8ZamgPM4u1VFyAQKnZsArKocpD0JnkXIFx8RKem6qVB0luPdAGiFahT6yhXj7SqCRDAwFw47Uzr08uzZ//b3Lnz6eb37/5585cP72wePfpi+82UVSSFBklrPWIGhG1DAkJyrT78BgipnWvX/rD56qsn26OmLPLeakyAaI+ymms6kiuw8JW3f7s9csoqbUxbUiQLCBNtTY3YbD8W8as06fzU9+5/tj1jyiptq6lNkSwgtckRbC2K7itFbuqfXf3d9owpq7QUqS1YVUC0R9hgLsTrdv4p0Woh9Y8f/O3CeZK//8bPt2dM1UhLkZrXECIgTK7UMNYITH/LQedKkNz5+NNz52j+6ZXfbM+YqhHjL0FSkyIiIFp6EFs5pbHGn3OQ5ECM/ffb97dnTNVK+xFWafEGiYBItQckWpQSC625zty48dG541Ozvbx48WJ79FStelNEBIST0wZL6REkvV3Ndeb5N5P/63c+uHAOfuMH1zZHRw+3R061qidFRECkxqyASNuT5dzPP390ciM//sk7Jz+I/vjuPzavXs3/a7KH1JLB8F5EBCT8KCi29d2G1Jmp9dWaIiIgUmN8ZlFaqFrPmxoraevHpYUvAqI1VqKN79NzrMkzNV5SsVpawCIg2p7FBXKQpMljffKZWkbSzoBzcyoCgqQ6BDPpABQ3Skr0PPlMLSNt4edSXgUEADRIgqXIik2HdkH0g4qdFQTImHur9WuvX91897W3zzk9ZtSWSv/Ta6XOvU4Iqt1mVEBQ61/W4bW3FyZKi9SRZqJykd0q6VqpLQtSGpMcWFlAUOsgW2geIUvyjbZ3ilgWKvdsEX2TztegLgKCmGyp0ZyXrj+4wZZ+jrB3iliAt0JZW4cUAWndZpasP5aG44c/ui5+HtsrRbg3qf3Y1vQIqqlDsoBotMVmYmicxODfubi1/rj5p9tma7L00dPcozXyPVLEssXTnxrV1CEqICVyc1uIdWBKf5Mb/Pr3rm7POC/L6vJ2kGfsaxqRHkiqQ7R2VEC0VUI6eG0fN27cEq+RWgNkyW0Fx/e9RIpYrlGbHkgCBB8fH2+POJMISI7cnhtO1ZMgNXAwUWFrYHC4hxanGpkiXE9qL3WLtG3ZDIhGrveTSWuC1NQd2t7qoZEpMio9gqRC9csv/7P99kwiIFIRYy08a9SaIJb0YGJ6BtCqUSkitZO6BbygLkCkCdAeg3rUAog1PXoGr0YjUmR0eiBpjh88+Nf22zOJgKQn4hEvvloAsTz2te77rfJOEUt7vQvAHZARg94CSGnwRtYcmjxThGOk82PTVq+khfbZ0YPtt2cSAZEmYUyC1BWpQCp9H3uprSWVZdVbFpklIT1eM3QBIsXPiJXpDYjHymqVR4oslR5IAuTevYsAi4BoFHuvzlpASpNAv1NZBr3W0nVQb4pYIPPa6tkR0rbNgGhPCv7vQepqEA3cYG3wLANfYw2QnhRZMj2QNJbmLQZpq8Fj/wuqTZASIFrfGHxp22y1BghqTRELXBzjJWk8qgDJddgr5rwTJAev51aTA6Q1RaTjYnumB5IAMT/mBuVWAxdgQtIb5c/pZ5pqASkNfmmFWSbP4hwgqDZFLP0q3VutXABhoi03y8XiV7drAUI/cqJf0sDUugSIZcLjFLGMsbekV+3VgCBu1nIDsa11Si0gtCt9H0w/S2JSpHNrXAIEWcaMFLH0xzs9kHQd89/FpGJiaiCx1ii1gFgG0zJ5AXrNUruxrdeQzo3NtSyJFpLGS9o4PhauYwIE0ag1ni0DiGoBQaU+MOjWBNME4FLbwdb7s8BW8oj00JJYkhmQIBpnknI3z/cWtQBS2mYwfeuRFyCWFCnZOz2Q9JJMG7NqQGIxWQxWejFsUQsgDJhlZXJM6+B6AYIsfdXcC7omCRBtUXcBgrQVbZmcFkBQaQKDGeCWiPYEpCdFGNsRkp5gtHHqBgRJq4RBLqkVEGSthzDH0h/rgHsCglpSZFR6sHCl6y0OiGUQewDRbtRi+gs0mksTWgtIS4pYFliLpO0Fa3IBhAFLL8hAl9QDCCqt9FGuBQTVpMio9EA1BSpyAUSbqFId0gsI6tnjW90CSE0/tbj3UO31XABB0gop/TzAAxC0NCQtgCBLioxMD21bztVmboC0bDNegCBu3jIBHm4FxALyyPSorT+QGyAt24wnIIhrLZEmrYCgHMQj0wNJ1ywB6QYIkm4+t814AxI0GpQeQHJJN+q9B9IWcOlpyRUQaZvBWoqMAiSI6zLo9Mvy+Gp1DyCIfsX9ob2RcCBpbiyJ5QqIRqmWIk+fPjPbU0xQj/dN9FmcF0O94woIklYpr3b3cWAPRVpxapkTd0C0vb+0102NkZYelu0FuQOCZorsjrT0sNY8QwDRUiT3RDPlr970QEMAoWNSiuCZIsuJJyVpDmq2+yGAoJki60p7oqxJDzQMEDRTZB1pWwuufd8yFBAtRWbBOlZaYWp575FqKCBIS5G51YwRCSGNd+3WEjQckFzBOvr18mVTbmtpSQ80HBA0t5plpD21lH52kdMigCAtRSYkPtLqjtatJWgxQHJbDTc3IWmX9kiLe7fxxQBBbDU5SKbqpRWluLXuiLUoIEirR/CEpE45OHrqjliLA4ImJP3KwdFbd8RaBRCkbTV4QpJXCY7euiPWaoDkilY8C1dZuYIUe8KBVgMElSBhH52QnIlFI41TsDccaFVAUAmS+Z7kVNpLsOARcKDVAUElSPBl/ckiE88ikcYEe9ccqXYCkKDSKrlsW05pSxkNB9opQJD239YEs5oOPU2Y9NJiAY4ltHOAoNwb1+BDTZPSAsHc+1LaSUDQ6d6bhwQzoIcASunxNXj0lpJqZwEJsqwovK+gAEauCA1eot6QtPOAoNNBLKcJ3petp1SAxvb4S7dW7QUgiEknJWpAAaxdgoX+lIrP2By7RmrE2htAgiwVfuo1YeG61m0y9tpgBO0dIEEMoDVNYrPfx8B4QkNbAQhLXSGZ83dJewtIUEuipA7QMLHUBkwSpu3Y4fNgju+BITbt7aL2HpCgAEpLqqzlkGS7rIMBJAhQqPp7U2WkSZ1dKp5zOjhAYgVYSJW1kwUo6M++6aABSRWny2hgAILtY1+SQtOlAiQVwARomNAATilxwvccz3mcfwgwSLrUgEyVNQGZymoCMpXVBGQqo83m/+BbYUkJcn+bAAAAAElFTkSuQmCC" width="30"> is a module for visualization of variant on the genome, <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEwAAABhCAYAAAB8pUfDAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAATvSURBVHhe7Zixjt02EEX9/38Qf0BSuDTgNu6zrYutXNlFSiP9BgdYApTeUHOHHOpJsi5wAcNPokaHl0NqP7zdCukGFtQNLKgbWFALYH98/LTpX7/+e7/yuPrzry9m7cU/fv77fmWfbmBB3cCCuoEFFQI2+rA9tCuw2Q/bQ1bdtW9gK1l11x5tKyFgr6/f3688poBh1V07Fdjnz1/NhxQD9Oi26q49qhCwK3hUC2B/f/3HfMhVTAJHdQMLagGMpm496Cp+efn2/qb9WgDj2GA96CpOB4aUneaszjhH/lbAMvQATGn8RJt+dyRbddbOaPjoAZjy8IxekCnlhJ9V8wMw5C1Lfj/Sn3qAYdVZO+s72ASmLEuSeAQp6cpajsgEpixLPqOOICVdmS3EBIaU3fLZKVPShTPrbAJTZu7ZvUytMVNNYIBQUpYZ94iekS7UBIaUGcRct6fZ8QDB5oTpp5gJric5O11oExhSUjbbQIksfYACdkaPdYHxYOslZjsKaS+5wNCeKWOCjiwJmLoBjJgedMRErSUBQ/SDWdCyPlv2kAwMzehnZ4KFQsBQFjTSejZYKAwMRZYm15ZzUjE74Kjodxjoe/a+LmCRTSCrmTMG6d56Lr8xGTPOX0VdwFBkaY5A8yC1zD3cm52+bmBoJjSu7QG1dgGXpSFgSIVG4aroS9YYI86CNgwMKdDUguk/1v0ZjkxaSynA0BY0tdAZyVqb1jCiNGCoBU1J1x6witW0W0oFhixoiqINnus5QmBSY12z5d5DczowVENTlqPatxhr64zFzqrCU9vEWlOAoQJNib+SrkjvsVJuuSdl04AhBZaSrp5GrfTEnnGnAlNED7JepnavvKT1LMunA/OWY29zRsrXQnT8qcC8TyF+t16i9qi8lClto9Y0YPQmrxivz2T8GSj7GVOAlUbuAfMafgYwL8XRxp8OrIYwmrDocrHkAXtqwtaJ8YrxgPVs+2tlpzgNmFWYV4w3+72n8VoeMH6PKAVYqygvIXvskkyaNW7x7sC2ZlBJCFCte4ujS6aWt+QxkxbREDAv7tgriMZu3VebF++RNxk9PbIbmDJ72Iu8Mg5JjSbBW4q4ZyK6gCm9p1g5GnhJwEBTxqI2dbwedQFTCirmWi8dkQngRUltnQ7u5/+U5V3sJb+lMLAIrGIlGZGXzfAuCet9KaUHqUsp0z3QZGBqk29ZWQJA4yWs+2c5Ck0CFukxLSspQ0eHJgHLWipKL0NAO2pPc4FlF66efQq0kbRxb6R+BdomsIyluDZFKUuzqICLpLyAKoq8hwdtE5g6O1wXcc8JG/HibB6c4jEQeUH+7Y2dBa0JTH0ARZ5FGdCawABhDVTbi+8RNQqtCcwaYO3epfVsRaCt39EExkXWzbXPmK5aCjQrECYwZTn2frweSVvQWqvHBGYNUPts6WJyWwAsaK1r0QMwBl8PsPYZdsZy/Cg18++WamhbsNADMHU5Hs3UDRTSb9XMmW1LQPNgoQdge/+JZU8DZVQPwKwHXcXpwKwGeCWzdEe1AMaA1oOu4q3Gr2oBTGn4Z7bX+BUtgNXb8BWdcX5cALvyDlk8qjAwZunItmquPaoQsIxdZrasumuPHi0WwLwZUk7Cz5ZVd+1UYNYDap8BmDfpN7CVZq+SELCMT4vZuoEFdQML6gYW1K7Abvm6gQV1Awvp7e1/cS4yprcXyWYAAAAASUVORK5CYII=" width="20"> is
a variant details page in SNV Viewer
(<a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page/#annotation-tab">&quot;Annotation&quot; tab</a>).</li></ol><p>Below the variant row, there can be phenotypes with pathogenic or likely pathogenic clinical significance that the
variant is associated with according to <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> and that are listed
in <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the ACMG SF list</a> for the gene in which the variant is located.
Links to <a href="https://www.omim.org/" target="_blank" rel="noopener noreferrer">OMIM</a> and/or <a href="https://www.ncbi.nlm.nih.gov/medgen/" target="_blank" rel="noopener noreferrer">MedGen</a> are provided for each
phenotype.</p><p>Next, there can be the variant interpretation text, if it was added as
described <a href="/ngsw-docs-en/results/main/snvs-indels/variant-interpretation#add-and-edit">here</a>.</p><p>At the end of the report, there is the version of the ACMG SF list used to generate the report.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="potential-findings"></a>Potential findings<a class="hash-link" href="#potential-findings" title="Direct link to heading">#</a></h3><p>Variants included in the ACMG SF report can be used as <a href="/ngsw-docs-en/results/potential-findings">potential findings</a>.
To do this, you need
to <a href="/ngsw-docs-en/settings/report-templates/report-templates#block-types">turn on the corresponding toggle in the report template</a>.
Potential findings can include Tier I variants
(“<em>Use variants in ACMG SF genes list associated with the phenotype results as potential findings</em>”
option) and/or Tier II variants
(“<em>Use variants in ACMG SF genes list without established phenotype association or genes conditions as potential findings</em>&quot;
option).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="acmg-sf-limitations"></a>Report limitations in ClinVar as a database source<a class="hash-link" href="#acmg-sf-limitations" title="Direct link to heading">#</a></h3><p>ACMG secondary findings report generation has limitations
of <a href="/ngsw-docs-en/results/acmg-sf-report#phenotypic-data">phenotypic</a>
and <a href="/ngsw-docs-en/results/acmg-sf-report#pathogenicity-data">pathogenicity</a> data
due to variant annotation using <a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> database.</p><p>ACMG guidelines for secondary findings in sequencing data provide a list of genes and
corresponding phenotypes along with recommendations for reporting genetic variants
regardless of the reason of genetic testing. The following three conditions should be satisfied:</p><ol><li>The variant affects a gene
from <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/" target="_blank" rel="noopener noreferrer">the list of secondary findings</a>;</li><li>The variant is reported as associated with the phenotype (condition, disease) listed with
the gene in the list for secondary findings, or there is a possibility to establish such an association.</li><li>The variant must meet all additional conditions specified in the list of secondary findings, which often
include pathogenic or likely pathogenic clinical significance of the phenotype.</li></ol><p>Briefly, it can be described as:</p><p align="center"><img src="/ngsw-docs-en/assets/images/acmg73_disclaimer-e719e1595e1b27773707e84aa367aeb8.png" width="500"></p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="phenotypic-data"></a>Limitations of phenotypic data<a class="hash-link" href="#phenotypic-data" title="Direct link to heading">#</a></h4><p>For some of variants in ClinVar, it is impossible to obtain information about phenotype (or condition,
or disease). Missense of phenotypic information in Clinvar can stem from inconsistencies in &quot;Conditions&quot;
field in reports submitted by laboratories. The field can contain values &quot;not reported&quot;, &quot;not specified&quot;,
or can be filled with all the diseases associated with any of the pathogenic variants in the
gene. In the latter case, correct conclusions can be only by a specialist. Moreover, mutations in
different portions of the gene can result in different genetic diseases, while the list for secondary
findings can contain only a fraction of these phenotypes. Considering these and other factors, the
ACMG Secondary Findings Report by Genomenal includes two lists of variants: those for which the phenotypes
in the ACMG SF list match the condition reported for the variants in ClinVar, and those that are located
in a gene in the list but have a discordant phenotype.</p><i>Therefore, the interpretation of the variants in the second section of the report must be approached with caution.</i><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="pathogenicity-data"></a>Limitations of pathogenicity data<a class="hash-link" href="#pathogenicity-data" title="Direct link to heading">#</a></h4><p>Usually variants are classified according to ACMG guidelines. ClinVar database is populated
by reports of pathogenicity from laboratories assuming that the responsible submitter fully
described submitted variants according to ACMG recommendations. Similarly, Genomenal follows the
same assumptions and variants are included in the report if they are reported as Pathogenic or
Likely Pathogenic without further expert evaluation of pathogenicity.</p><p>Additionally, new evidence might emerge in literature about functional effect or lack of it for a
specific variant. It can result in a revision of variant pathogenicity but will not be instantly reflected
in ClinVar. Undoubtedly, systematical review of variants&#x27; pathogenicity status is labour-consuming
and can lag compared to emergency of new scientific evidence. These factors can result in
overreporting of variants, or presence of outdated data, or underreporting of variants for
which new data has emerged but it is not reflected in ClinVar. Some variants can be omitted if they
pathogenicity is yet unknown and not detected by anyone.</p><p><em>Thus, the report can include the following variants:</em></p><ul><li><em>with imprecise or erroneous pathogenicity status reported by the submitting laboratory;</em></li><li><em>with pathogenicity revised due to newly emerged data (confirmed</em> <strong><em>pathogenic</em></strong> <em>by new data).</em></li></ul><p><em>The following variants will not be included in the report:</em></p><ul><li><em>not yet reported in ClinVar or newly discovered;</em></li><li><em>with pathogenicity revised due to newly emerged data (confirmed</em> <strong><em>benign</em></strong> <em>by new data).</em></li></ul><div class="admonition admonition-caution alert alert--warning"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg></span>caution</h5></div><div class="admonition-content"><p>By using ACMG Secondary Findings report, you agree that you understand above mentioned limitations,
and if clarification is needed, you are willing to discuss the result with medical specialist.</p></div></div></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/sample-interpretation"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">« Sample Interpretation</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/variants-by-phenotypes"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">ClinVar Phenotypes Report »</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#acmg-sf-conditions" class="table-of-contents__link">Conditions for the report generation</a></li><li><a href="#acmg-sf-content" class="table-of-contents__link">Report content</a></li><li><a href="#potential-findings" class="table-of-contents__link">Potential findings</a></li><li><a href="#acmg-sf-limitations" class="table-of-contents__link">Report limitations in ClinVar as a database source</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">© 2019-2024 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.b81db154.js"></script>
<script src="/ngsw-docs-en/assets/js/main.660889f1.js"></script>
</body>
</html>