---
sidebar_position: 2
sidebar_label: Main report templates
---

# Main report templates

This section describes the main report templates that are used to present analysis results.
Some of them may have been added to your profile by an administrator when registering in the system.
If you need other report template described here, you can contact your administrator or create them yourself,
as described below.

## 1. Glioma report

- a clinical report on the characteristics of gliomas, based on tumor markers identified in the sample,
and a knowledge base created on the basis of expert knowledge and published research on this topic;
- suitable for presenting the results of the analysis of glioma samples in any file format.

### Conditions for the report generation

Glioma report is generated for the sample if the following conditions are met:
1. The sample is uploaded as a tumor tissue sample.
2. The sample analysis was successfully completed (that is, all stages of the analysis have the status "Complete").
3. For a single tumor sample (tumor-only), at least one of the following stages was successfully completed:
"Somatic SNV/Indels annotation", "Germline SNV/Indels annotation", or "Copy number variations discovery".
If you have a set of tumor and corresponding non-tumor samples (tumor-normal), then it is necessary that the
"Somatic SNV/Indels annotation" stage was successfully completed for tumor sample, and "Germline SNV/Indels
annotation" stage was successfully completed for normal sample.
3. The report template is active.
4. The report template was created before the sample was processed.

### Add Glioma Report template

To add Glioma Report template to the system, you can either import it or create it yourself.

#### Import Glioma Report template

Download the template file (<a target="_blank" href="/reports/Glioma report.json" download="Glioma report.json">Glioma report</a>)
and import it as described [here](/settings/report-templates/report-templates#import-report-template).

#### Create Glioma Report template

To create a template yourself, follow these steps:

1. On the Report templates page, click on <img src={require('/img/eng/add_template.png').default} width="170"/>
to open the page for a report template adding.

2. On the page that opens, fill in the following fields:

- Name: Glioma report
- Applicable to samples: Tumor
- Active: yes

3. Click on <img src={require('/img/report_templates/save_temaplate.png').default} width="140"/> to save the template.

4. Add all the blocks listed below to the template. The list of blocks is given in the order in which they appear in
the template. If there are no clarifications after the block name, then the block generated by default is used.

#### The template blocks

1. "Patient info".
2. "Section".

Name:
```
SOMATIC MUTATIONS REPORT
```
Description:
```
Tumor specimen
```
3. "Sample info".
4. "Glioma: Tier I variants".
5. "Glioma: Tier II variants".
6. "Immunotherapy markers".
7. "Glioma: Suggested diagnosis".
8. "Glioma: Suggested drugs".
9. "Glioma: Mutations details".
10. "Glioma: Tier III variants".
11. "Clinical trials".

Description:
```
List of ongoing clinical trials from <https://clinicaltrials.gov> found for Glioma
```
Phenotypes:
```
Glioma;Astrocytoma;Glioblastoma;Diffuse Intrinsic Pontine Glioma;Ependymoma;Glioma, Subependymal;Ganglioglioma;Gliosarcoma;Medulloblastoma;Oligodendroglioma;Optic Nerve Glioma
```
12. "Glioma: Glioma genes panel details".
13. "Section".

Name:
```
GERMLINE VARIANTS REPORT
```
Description:
```
Normal specimen
```
14. "Control (normal) sample info".
15. "Glioma: Germline pathogenic for gliomas".
16. "SNV/Indels in ACMG 73 genes".
17. "Glioma: Germline pathogenic for any other diseases".
18. "Glioma: Germline glioma associated".
19. "Section". Name:
```
REFERENCE LIST
```
20. "Glioma: Literature".
21. "Section". Name:
```
TECHNICAL LIMITATIONS
```
22. "Disclaimer". Text:
```
# Limit of detection

The NGS method may be not sensitive enough for detection of SNVs and Indels with less than 5% and 10% variant allele frequency respectively.

# Analytical sensitivity

Analytical sensitivity for all variant classes is available upon request or through our website.

# Clinical disclaimer

 All content found in this report was provided for informational purposes only. It is not intended to substitute a professional medical advice, diagnosis, treatment or care. The report should be used with all available clinical and laboratory data and should not be used separately to diagnose a malignancy. Please contact your doctor or any other qualified professional to seek a medical advice. Never use this report to disregard a medical advice or delay contacting a qualified professional.
```

## 2. CNV Report

- a report on the results of the analysis of the patient sample or samples in order to detect changes in the
number of copies of chromosomes;
- affects [Sample CNV Report](/results/cnv-report/sample-cnv-report) and
[Patient CNV Report (for all patient samples)](/results/cnv-report/patient-cnv-report) generation;
- suitable for presenting the results of the analysis of non-tumor samples in FASTQ or BAM format.

### Conditions for the report generation

CNV Report is generated for the sample if the following conditions are met:
1. The sample is uploaded as a non-tumor tissue sample.
2. The sample analysis was successfully completed (that is, all stages of the analysis have the status "Complete").
If the analysis failed, then there will be a corresponding message for the sample in the report
for the patient, and the report for the sample will not be built.
3. The "Copy number variations discovery" stage has successfully completed. If the stage failed,
then there will be a corresponding message for the sample in the report for the patient,
and the report for the sample will not be built. If the stage is turned off, then the report
for the sample will not be built, and the sample will not be mentioned in the report for the patient.
If the stage is turned off for all patient samples, then the report for the patient will not be built.
3. The report template is active.
4. The report template was created before the sample was processed.

### Add CNV Report template

To add CNV Report template to the system, you can either import it or create it yourself.

#### Import CNV Report template

Download the template file (<a target="_blank" href="/reports/CNV Report.json" download="CNV Report.json">CNV Report</a>)
and import it as described [here](/settings/report-templates/report-templates#import-report-template).

#### Create CNV Report template

To create a template yourself, follow these steps:

1. On the Report templates page, click on <img src={require('/img/eng/add_template.png').default} width="170"/>
to open the page for a report template adding.

2. On the page that opens, fill in the following fields:

- Name: CNV Report
- Applicable to samples: Normal
- Active: yes
- Report title:
```
REPORT ON THE RESULTS OF THE ANALYSIS OF SEQUENCING DATA IN ORDER TO DETECT CHANGES IN THE NUMBER OF COPIES OF CHROMOSOMES
```

3. Click on <img src={require('/img/report_templates/save_temaplate.png').default} width="140"/> to save the template.

4. Add all the blocks listed below to the template. The list of blocks is given in the order in which they appear in
the template. If there are no clarifications after the block name, then the block generated by default is used.

#### The template blocks

1. "Patient info". Display fields: Patient id, First name, Last name, Date of birth, Sex.
2. "Sample info". Display fields: Sequencing type, Sample collection date.
3. "Germline CNV: results".
4. "Germline CNV: interpretation".
5. "Disclaimer". Text:
```
# Technical constraints

Whole genome sequencing may be insufficiently sensitive for detecting aneuploidies and determining sex of a sample at a genome reading depth of less than 0.01 per one base pair (bp). Analytical sensitivity for all types of aneuploidies is available upon request or in the software documentation.

# Restrictions on the use of the Report

The information provided in this document is intended solely for internal professional use. The information cannot be used as a substitute for professional medical advice, diagnosis, treatment or care. The report can only be used in conjunction with all available clinical and laboratory data and should not be used in isolation from this data. The studies were carried out to the extent specified by the customer.

Additional information on this analysis can be obtained at {url} in the section "Bioinformatics report" or upon request from the company that provided this report.
```
6. "Sign place".
