<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Variant Viewer | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/variant-viewer/variant-viewer"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Variant Viewer | Genomenal"><meta data-react-helmet="true" name="description" content="Overview"><meta data-react-helmet="true" property="og:description" content="Overview"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.ico"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/variant-viewer/variant-viewer"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/variant-viewer/variant-viewer" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/variant-viewer/variant-viewer" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://7Q65UNCM90-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.4b843568.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.ebe2c139.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">🌜</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">🌞</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick start for prenatal genetic testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/overview">Getting to know the service</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add_patient">Adding patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Analysis results</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Variant Viewer</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/variant-viewer/variant-viewer">Variant Viewer</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variant-viewer/snv_details_panel">Variant detailed information panel</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variant-viewer/basic_filtering">Basic filtering</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variant-viewer/advanced_filtering">Advanced filtering</a></li></ul></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Variant Viewer</h1></header><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="overview"></a>Overview<a class="hash-link" href="#overview" title="Direct link to heading">#</a></h3><p>Variant Viewer is a built-in service for viewing and analyzing variants. Located on the main page of the sample (&quot;<strong>Main</strong>&quot; tab).</p><p>On the <strong><em>tumor</em></strong> sample page, you will find two Variant Viewers: one with <strong><em>somatic</em></strong> and one with <strong><em>germline</em></strong> mutations discovered in this sample:</p><p align="center"><img src="/ngsw-docs-en/assets/images/germline_and_somatic_vv-b30240dc3c0462f898a7338d4858557d.png" width="600"></p><p>The <strong><em>normal</em></strong> sample page presents a Variant Viewer with <strong><em>germline</em></strong> mutations only:</p><p align="center"><img src="/ngsw-docs-en/assets/images/germline_only-406e9c53884ef81acd63c615b2ff0bad.png" width="600"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="the-main-page"></a>The main page<a class="hash-link" href="#the-main-page" title="Direct link to heading">#</a></h3><p>After clicking on the <strong>Variant Viewer</strong> button, you will see the main page of the service:</p><p align="center"><img src="/ngsw-docs-en/assets/images/main_table-ccd33bdbc8ff217d44da7f5b431ad93c.png" width="850"></p><p>The main object of the Variant Viewer is a table with variants - each variant on a new line:</p><p align="center"><img src="/ngsw-docs-en/assets/images/snv_row-7850415e0e1fa0691dffb88554b75079.png" width="950"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="description-of-table-columns"></a>Description of table columns<a class="hash-link" href="#description-of-table-columns" title="Direct link to heading">#</a></h3><p>By default, the table has the following columns (the set and order of the columns can be changed):</p><ol><li><strong>LOCUS</strong> is the common name of the gene in which the variant is located.</li><li><strong>ORIGIN</strong> is a mutation type (somatic or germline).</li></ol><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>&quot;Origin&quot; column is hidden in <a href="/ngsw-docs-en/results/variant-viewer/variant-viewer">Variant Viewer with germline mutations</a>
and in Variant Viewer with somatic mutations if &quot;Origin Somatic&quot; filter is on.</p></div></div><ol start="3"><li><strong>POSITION</strong> is the coordinates of the variant in the genome (chromosome + start position).</li><li><strong>HGVSP</strong> is an amino acid substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">the HGVS notation</a> (“p.” prefix (protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution).
When hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for and HGVSp with amino acids in the three letter code:</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/hgvsp_tooltip-5b0fd9e70e36aeec94b9469471d5ab72.png" width="400"></p><ol start="4"><li><strong>ALTERATION</strong> is a nucleotide substitution (reference allele &gt; alternative allele(s)).</li><li><strong>HGVSC</strong> is a nucleotide substitution using <a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">the HGVS notation</a> (genomic position of the substituted nucleotide + reference allele &gt; alternative allele). Starts with a prefix: “c.” (coding; for a substitution in the coding sequence), “n.” (non-coding; for a substitution in the non-coding sequence).
When hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for:</li></ol><p align="center"><img src="/ngsw-docs-en/assets/images/hgvsc_tooltip-702b36ee2b497dad9d00bb4867fdb330.png" width="500"></p><ol start="6"><li><strong>EXON</strong> is the exon number in the transcript which is affected by the variant.</li><li><strong>DEPTH (ALT/REF)</strong> is a sequencing depth; the total number of reads of the sequence overlapping the variant position for this sample. ALT is the alternative allele reads count, REF - the reference allele reads count.</li><li><strong>GT</strong> is a genotype; allelic values (0 - reference allele, 1 - first alternative allele, 2 - second alternative allele etc.), divided by ”/” (for unphased genotype) or “|” (for phased genotype).</li><li><strong>CONSEQUENCE</strong> is the effect of the variant on genes.</li></ol><details class="details_2Ziz alert alert--info details_1VDD" data-collapsed="true"><summary>Click to see a table with the consequences</summary><div><div class="collapsibleContent_3OHp"><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="consequences-of-variants-in-the-annotation"></a>Consequences of variants in the annotation<a class="hash-link" href="#consequences-of-variants-in-the-annotation" title="Direct link to heading">#</a></h4><table><tr><td>Consequence</td><td>Description</td><td>Impact (see description below)</td></tr><tr><td>Coding sequence</td><td>The variant hits a coding sequence</td><td>MODIFIER</td></tr><tr><td>Transcript ablation</td><td>Deletion of a transcript</td><td>HIGH</td></tr><tr><td>Splice acceptor</td><td>The variant hits a splice acceptor site</td><td>HIGH</td></tr><tr><td>Splice donor</td><td>The variant hits a splice donor site</td><td>HIGH</td></tr><tr><td>Stop gained</td><td>Variant causes a STOP codon</td><td>HIGH</td></tr><tr><td>Frameshift</td><td>Insertion or deletion causes a frame shift</td><td>HIGH</td></tr><tr><td>Stop lost</td><td>Variant causes stop codon to be mutated into a non-stop codon</td><td>HIGH</td></tr><tr><td>Start lost</td><td>Variant causes start codon to be mutated into a non-start codon</td><td>HIGH</td></tr><tr><td>Transcript amplification</td><td>A feature amplification of a region containing a transcript</td><td></td></tr><tr><td>Inframe insertion</td><td>One or many codons are inserted</td><td>MODERATE</td></tr><tr><td>Inframe deletion</td><td>One or many codons are deleted</td><td>MODERATE</td></tr><tr><td>Missense</td><td>Variant causes a codon that produces a different amino acid</td><td>MODERATE</td></tr><tr><td>Protein altering</td><td>A variant which is predicted to change the protein encoded in the coding sequence</td><td>MODIFIER</td></tr><tr><td>Splice region</td><td>A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron</td><td>LOW</td></tr><tr><td>Incomplete terminal codon</td><td>A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed</td><td></td></tr><tr><td>Start retained</td><td>Variant causes start codon to be mutated into another start codon</td><td>LOW</td></tr><tr><td>Stop retained</td><td>Variant causes stop codon to be mutated into another stop codon</td><td>LOW</td></tr><tr><td>Synonymous</td><td>Variant causes a codon that produces the same amino acid</td><td>LOW</td></tr><tr><td>Mature miRNA</td><td>A transcript variant located with the sequence of the mature miRNA</td><td>MODIFIER</td></tr><tr><td>UTR5</td><td>A variant hits 5&#x27;UTR region</td><td>MODIFIER</td></tr><tr><td>UTR3</td><td>A variant hits 3&#x27;UTR region</td><td>MODIFIER</td></tr><tr><td>Non coding transcript exon</td><td>A sequence variant that changes non-coding exon sequence in a non-coding transcript</td><td>MODIFIER</td></tr><tr><td>Upstream</td><td>Upstream of a gene</td><td>MODIFIER</td></tr><tr><td>Downstream</td><td>Downstream of a gene</td><td>MODIFIER</td></tr><tr><td>TF binding site</td><td>Transcription factor binding site (TFBS)</td><td></td></tr><tr><td>TFBS ablation</td><td>Deletion of a TFBS</td><td>HIGH</td></tr><tr><td>TFBS amplification</td><td>Amplification of a TFBS</td><td>HIGH</td></tr><tr><td>Regulatory region ablation</td><td>Deletion of a regulatory region</td><td>HIGH</td></tr><tr><td>Regulatory region amplification</td><td>Amplification of regulatory region</td><td>HIGH</td></tr><tr><td>Feature elongation</td><td>A sequence variant that causes the extension of a genomic feature</td><td></td></tr><tr><td>Regulatory region variant</td><td>The variant hits a regulatory feature</td><td>MODIFIER</td></tr><tr><td>Feature truncation</td><td>A sequence variant that causes the reduction of a genomic feature</td><td></td></tr><tr><td>Intergenic variant</td><td>The variant is in an intergenic region</td><td>MODIFIER</td></tr></table></div></div></details><ol start="10"><li><b>IMPACT</b> is the predicted effect of the variant on the protein.</li></ol><details class="details_2Ziz alert alert--info details_1VDD" data-collapsed="true"><summary>Click to see a table with impacts</summary><div><div class="collapsibleContent_3OHp"><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="impacts-of-variants-in-the-annotation"></a>Impacts of variants in the annotation<a class="hash-link" href="#impacts-of-variants-in-the-annotation" title="Direct link to heading">#</a></h4><table><tr><td>Impact</td><td>Description</td></tr><tr><td>HIGH</td><td>The variant has high (disruptive) impact in the protein, probably causing protein truncation, loss of function or triggering nonsense mediated decay</td></tr><tr><td>MODERATE</td><td>The variant might change protein effectiveness</td></tr><tr><td>LOW</td><td>The variant is mostly harmless or unlikely to change protein behavior</td></tr><tr><td>MODIFIER</td><td>Usually variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact</td></tr></table></div></div></details><ol start="11"><li><strong>EXTERNAL LINKS</strong> are links to pages with variant information in <a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a>, <a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a> databases.</li><li><strong>GNOMAD AF</strong> is the allele frequency according to the <a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD</a> database.</li><li><strong>ACMG</strong> is the interpretation of the variant by ACMG recommendations. If you have not interpreted the variant, then the column value will be “N/A” (No interpretation).</li></ol></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/coverage-analysis"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">« Coverage Analysis</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/variant-viewer/snv_details_panel"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Variant detailed information panel »</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#overview" class="table-of-contents__link">Overview</a></li><li><a href="#the-main-page" class="table-of-contents__link">The main page</a></li><li><a href="#description-of-table-columns" class="table-of-contents__link">Description of table columns</a></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">© 2019-2021 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.4b843568.js"></script>
<script src="/ngsw-docs-en/assets/js/main.ebe2c139.js"></script>
</body>
</html>