<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Somatic SNVs/Indels discovery and annotation | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Somatic SNVs/Indels discovery and annotation | Genomenal"><meta data-react-helmet="true" name="description" content="Somatic SNVs/Indels discovery"><meta data-react-helmet="true" property="og:description" content="Somatic SNVs/Indels discovery"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.e577558a.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.7c50b07b.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">ðŸŒœ</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">ðŸŒž</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Workflow Details</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/workflow-details">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Analysis of FASTQ or BAM samples</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/workflow-details">Analysis of FASTQ or BAM samples</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/upload-identify-verify">Upload, identify and verify</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">Check quality and cleanup</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">Alignment</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">Variant discovery preprocessing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">Calculate coverage</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels">Somatic SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels">Germline SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">Structural variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">Copy number variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">Phenotype prediction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">Generate reports</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Analysis of VCF or GT samples</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Somatic SNVs/Indels discovery and annotation</h1></header><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="somatic-variants-discovery"></a>Somatic SNVs/Indels discovery<a class="hash-link" href="#somatic-variants-discovery" title="Direct link to heading">#</a></h2><p>After <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">the &quot;Variant discovery preprocessing&quot; stage</a>
has been successfully completed, somatic SNVs/Indels discovery can be initiated for tumor samples if the stage is
<a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#somatic-snvs-discovery">included</a> in the analysis workflow.</p><p align="center"><img src="/ngsw-docs-en/assets/images/somatic_pipeline-d72433b6d0204050bbb650b1dc1953a6.png" width="750"></p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="1-somatic-variants-discovery"></a>1. Somatic SNVs/Indels Discovery<a class="hash-link" href="#1-somatic-variants-discovery" title="Direct link to heading">#</a></h3><p>Somatic single-nucleotide variants (SNVs) and short insertions/deletions (indels) calling is performed
using <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2" target="_blank" rel="noopener noreferrer">GATK Mutect2</a>.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="intervals"></a>I. Dividing data into intervals<a class="hash-link" href="#intervals" title="Direct link to heading">#</a></h4><p>To speed up the process and save resources, the Scatter-Gather task divides the input data into 64 genomic
intervals. If the sample capture kit is defined, the data is divided
using <a href="https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html" target="_blank" rel="noopener noreferrer">bedtools intersect</a>, which
screens for overlaps between distinct genomic intervals and capture kit intervals.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="calling"></a>II. Variant calling<a class="hash-link" href="#calling" title="Direct link to heading">#</a></h4><p>Variant calling in each of the resulting intervals is performed
using <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2" target="_blank" rel="noopener noreferrer">GATK Mutect2</a>.
Mutect2 works primarily by matching variants in tumor and control samples (tumor-normal mode).
In this case, it includes logic to skip emitting variants that are clearly present in the germline based on
provided evidence in the matched normal. This is done at an early stage to avoid spending computational
resources on germline events. If the variant&#x27;s germline status is borderline, then Mutect2 will emit the
variant to the callset for
subsequent <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#filter-mutect-calls">filtering by FilterMutectCalls and review</a>.
Mutect2 is also suitable for calling variants in a tumor sample without a control sample (tumor-only mode),
but this calling has a high number of false positives.</p><p>It is recommended to use a panel of normals (PON), i.e. a panel made from from normal samples (i.e.
samples derived from healthy tissue that is believed to not have any somatic alterations), to capture recurrent
technical artifacts in order to improve the results of the variant calling analysis. Normals that are as
technically similar as possible to the tumor (same exome or genome preparation methods, sequencing technology and
so on) should be used to include in a PON. As a result, the most important selection criteria for choosing normals
to include in a PON are the technical properties of how the data was generated. The more accurate the panel,
the more specific the analysis becomes and more variants can be discovered. By default, the analysis is performed
with &quot;<em>Basic panel of normals</em>&quot;. To select a different PON or to perform the analysis without any panel,
select <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#panel-of-normal">the corresponding option</a> in the parameters.</p><p><a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/" target="_blank" rel="noopener noreferrer">The gnomAD database</a> is used as a resource containing
population allele frequencies of common and rare variants.</p><p>By default, soft clipped bases (i.e. bases from the ends of reads that do not align with the reference sequence)
are analyzed when calling variants. This is controlled
by <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#soft-clipped-bases">the corresponding setting</a>.</p><p>To eliminate possible substitution errors that occur in one strand before sequencing, an orientation bias filter is
applied. The filter consists of three stages. Mutect2 includes the first stage of filtering, such as collecting
F1R2 counts (F1R2 is a read pair if the sequence of bases in Read 1 maps to the forward strand of the reference,
and the sequence of Read 2 to the reverse strand of the reference).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="merge-vcfs"></a>III. Combining the discovered variants<a class="hash-link" href="#merge-vcfs" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360036713331-MergeVcfs-Picard" target="_blank" rel="noopener noreferrer">GATK MergeVcfs</a> combines
files with variants discovered in distinct genomic intervals into a single variant file in VCF format.
The resulting file with all somatic variants discovered in a sample is compressed into a GZIP archive
using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. It can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Somatic SNVs/Indels Discovery</em>&quot; task details (&quot;<em>Download VCF_GZ</em>&quot;). You can also open this file in
the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF file is indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index file can be downloaded in the same
section (&quot;<em>Download VCF_TBI</em>&quot;).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="merge-mutect-stats"></a>IV. Merging stats<a class="hash-link" href="#merge-mutect-stats" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035531132--How-to-Call-somatic-mutations-using-GATK4-Mutect2" target="_blank" rel="noopener noreferrer">GATK MergeMutectStats</a>
merges the stats files obtained from previously split intervals. The resulting file in TSV format can be downloaded
in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Somatic SNVs/Indels Discovery</em>&quot; task details (&quot;<em>Download Mutect2 stats TSV</em>&quot;).
You can also open it in Google Sheets.</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="learn-read-orientation-model"></a>V. Learning the read orientation model<a class="hash-link" href="#learn-read-orientation-model" title="Direct link to heading">#</a></h4><p>The second step of orientation bias filter is the work
of <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360051305331-LearnReadOrientationModel" target="_blank" rel="noopener noreferrer">GATK LearnReadOrientationModel</a>.
It gets the maximum likelihood estimates of artifact prior probabilities in the orientation bias mixture model filter.
As a result, artifact prior tables are generated for each tumor sample. The resulting file in tar.gz format containing
artifact prior tables can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Somatic SNVs/Indels Discovery</em>&quot; task
details (&quot;<em>Download Read orientation model TAR_GZ</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="filter-row-variants"></a>2. Filter Raw SNVs/Indels<a class="hash-link" href="#filter-row-variants" title="Direct link to heading">#</a></h3><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="get-pileup-summaries"></a>I. Tabulating Pileup metrics<a class="hash-link" href="#get-pileup-summaries" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593451-GetPileupSummaries" target="_blank" rel="noopener noreferrer">GATK GetPileupSummaries</a>
tabulates Pileup metrics for inferring contamination. To do this, it summarizes counts of reads that support
reference, alternate and other alleles for given sites. The resulting table in TXT format can be downloaded in
the &quot;<em>Result files</em>&quot; section in the &quot;<em>Filter Raw SNVs/Indels</em>&quot; task details (&quot;<em>Download Pileup table TXT</em>&quot;).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="calculate-contamination"></a>II. Generating contamination table<a class="hash-link" href="#calculate-contamination" title="Direct link to heading">#</a></h4><p>Based on the data obtained by GetPileupSummaries
in <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#get-pileup-summaries">the previous step</a>,
<a href="https://gatk.broadinstitute.org/hc/en-us/articles/360036888972-CalculateContamination" target="_blank" rel="noopener noreferrer">GATK CalculateContamination</a>
calculates the fraction of reads coming from cross-sample contamination. The resulting table provides the fraction
contamination of each analyzed sample. The table can be downloaded in the &quot;<em>Result files</em>&quot; section in the
&quot;<em>Filter Raw SNVs/Indels</em>&quot; task details (&quot;<em>Contamination table TXT</em>&quot;).</p><h4><a aria-hidden="true" tabindex="-1" class="anchor anchor__h4 anchorWithStickyNavbar_31ik" id="filter-mutect-calls"></a>III. Filtering variants<a class="hash-link" href="#filter-mutect-calls" title="Direct link to heading">#</a></h4><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360036856831-FilterMutectCalls" target="_blank" rel="noopener noreferrer">GATK FilterMutectCalls</a>
filters somatic SNVs and indels called by Mutect2. To do this, it uses:</p><ul><li>the contamination table created
in <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#calculate-contamination">the previous step</a>;</li><li>the Mutect stats file obtained
in <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#merge-mutect-stats">the stats merging step</a>;</li><li>a file containing artifact prior tables obtained
in <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#learn-read-orientation-model">the learn read orientation model step</a>.</li></ul><p>The resulting file is compressed into a GZIP archive
using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>. It can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Filter Raw SNVs/Indels</em>&quot; task details (&quot;<em>Download VCF_GZ</em>&quot;). You can also open this file in
the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link. In addition, the VCF file is indexed
using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>. The resulting index file can be downloaded in the same
section (&quot;<em>Download VCF_TBI</em>&quot;).</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="annotate-variants"></a>Somatic SNVs/Indels annotation<a class="hash-link" href="#annotate-variants" title="Direct link to heading">#</a></h2><p>After <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels#somatic-variants-discovery">the &quot;Somatic SNVs/Indels discovery&quot; stage</a>
has been successfully completed, somatic SNVs/Indels annotation is performed.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="1-annotate-variants"></a>1. Annotate Somatic SNVs/Indels<a class="hash-link" href="#1-annotate-variants" title="Direct link to heading">#</a></h3><ul><li><p>Annotation of SNVs/Indels in the file with data from the <a href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank" rel="noopener noreferrer">RefSeq</a>,
<a href="https://www.internationalgenome.org/" target="_blank" rel="noopener noreferrer">1000 Genomes</a>, <a href="https://sites.google.com/site/jpopgen/dbNSFP" target="_blank" rel="noopener noreferrer">dbNSFP</a>,
<a href="https://www.ncbi.nlm.nih.gov/snp/" target="_blank" rel="noopener noreferrer">dbSNP</a>, <a href="https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/" target="_blank" rel="noopener noreferrer">gnomAD 3</a>,
<a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener noreferrer">gnomAD 4</a>,
<a href="https://preview.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener noreferrer">ClinVar</a>, <a href="https://cadd.gs.washington.edu/" target="_blank" rel="noopener noreferrer">CADD</a>,
<a href="https://spliceailookup.broadinstitute.org/" target="_blank" rel="noopener noreferrer">SpliceAI</a>, <a href="https://enigmaconsortium.org/" target="_blank" rel="noopener noreferrer">ENIGMA</a> databases.</p></li><li><p>Determination of the effect of SNVs/Indels on genes, transcripts, and protein sequence, as well as regulatory
regions, using <a href="https://www.ensembl.org/info/docs/tools/vep/index.html" target="_blank" rel="noopener noreferrer">Ensembl Variant Effect Predictor (VEP)</a>:</p><ul><li><p><a href="http://genetics.bwh.harvard.edu/pph2/" target="_blank" rel="noopener noreferrer">PolyPhen</a> predicts possible impact of an amino acid substitution on
the structure and function of a protein using straightforward physical and comparative considerations.</p></li><li><p>A flag indicating if the transcript is the canonical transcript for the gene is added.</p></li><li><p>Allele number from VCF input is identified.</p></li><li><p>Affected exon and intron numbering is added.</p></li><li><p><a href="https://varnomen.hgvs.org/" target="_blank" rel="noopener noreferrer">HGVS nomenclature</a> based on <a href="https://www.ensembl.org/index.html" target="_blank" rel="noopener noreferrer">Ensembl</a> stable
identifiers is added.</p></li><li><p>Upstream variant consequences are determined if the upstream (5&#x27;) distance between a variant and a transcript
is more than 2000 bp. Downstream variant consequences are determined if the downstream (3&#x27;) distance between
a variant and a transcript is more than 1000 bp.</p><p>   The resulting file with raw annotated SNVs/Indels in TSV format can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Annotate Somatic SNVs/Indels</em>&quot; task details (&quot;<em>Download Raw annotated TSV</em>&quot;).
You can also open it in Google Sheets.<br>
The resulting file with annotated SNVs/Indels without duplicates in TSV format can be downloaded in the same section
(&quot;<em>Download All variants TSV</em>&quot;). You can also open it in Google Sheets.
The same file in CSV format can be downloaded from the same place (&quot;<em>Download All variants CSV</em>&quot;).</p></li></ul></li><li><p>Converting TSV results to VCF.</p></li><li><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037428851-FixVcfHeader-Picard-" target="_blank" rel="noopener noreferrer">GATK FixVCFHeader</a>
replaces or fixes a VCF header.</p></li><li><p>The file is compressed into a GZIP archive using <a href="http://www.htslib.org/doc/bgzip.html" target="_blank" rel="noopener noreferrer">bgzip</a>.
The resulting file can be downloaded in the &quot;<em>Result files</em>&quot; section
in the &quot;<em>Annotate Somatic SNVs/Indels</em>&quot; task details (&quot;<em>Download All variants VCF_GZ</em>&quot;).
You can also open this file in the IGV browser by clicking on the &quot;<em>Open in IGV Browser</em>&quot; link.</p></li><li><p>The file is indexed using <a href="http://www.htslib.org/doc/tabix.html" target="_blank" rel="noopener noreferrer">tabix</a>.
The resulting index file can be downloaded in the same section (&quot;<em>Download All variants VCF_TBI</em>&quot;).</p></li><li><p>Calculating variants&#x27; statistic.</p></li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="save-variants"></a>2. Store annotated variants for SNV Viewer<a class="hash-link" href="#save-variants" title="Direct link to heading">#</a></h3><ul><li>Saving results for display in <a href="/ngsw-docs-en/results/main/snvs-indels/variant-viewer">SNV Viewer</a>, i.e. an embedded service
for viewing and analyzing variants.</li><li>Adding information about <a href="/ngsw-docs-en/results/main/snvs-indels/snv-details-page#occurrences">the variant occurrence</a> in
other samples of the user.</li></ul><p>After the &quot;Somatic SNVs/Indels annotation&quot; stage, the analysis can proceed
with <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">report generation</a>.</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>If you want to add a track with somatic SNVs/Indels discovered by the analysis of the sample you uploaded to
Genomenal to your <a href="https://igv.org/doc/desktop/" target="_blank" rel="noopener noreferrer">desktop IGV</a>, you can do so via a link.
Open the details of the necessary task (&quot;<em>Somatic SNVs/Indels Discovery</em>&quot;, &quot;<em>Filter Raw SNVs/Indels</em>&quot;,
&quot;<em>Annotate Somatic SNVs/Indels</em>&quot;) and do the following:</p><ol><li>Right-click the variant file link (depending on the selected task, this may be the link &quot;<em>Download VCF_GZ</em>&quot;,
&quot;<em>Download All variants VCF_GZ</em>&quot; or &quot;<em>Download Filtered variants VCF_GZ</em>&quot;) and select the &quot;<em>Copy link address</em>&quot; option.</li><li>Upload the track via URL to your desktop IGV, as
described <a href="https://igv.org/doc/desktop/#UserGuide/loading_data/" target="_blank" rel="noopener noreferrer">here</a>.</li><li>Right-click on the download link for the index file corresponding to the annotation file (&quot;<em>Download VCF_TBI</em>&quot;,
&quot;<em>Download All variants VCF_TBI</em>&quot; or &quot;<em>Download Filtered variants VCF_TBI</em>&quot;) and select the &quot;<em>Copy link address</em>&quot;
option.</li><li>Add the index via URL to the corresponding field in IGV.</li><li>Click &quot;<em>OK</em>&quot;. Done! The track with somatic SNVs/Indels discovered in the sample is added to IGV.</li></ol></div></div></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">Â« Calculate coverage</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Germline SNVs/Indels discovery and annotation Â»</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#somatic-variants-discovery" class="table-of-contents__link">Somatic SNVs/Indels discovery</a><ul><li><a href="#1-somatic-variants-discovery" class="table-of-contents__link">1. Somatic SNVs/Indels Discovery</a></li><li><a href="#filter-row-variants" class="table-of-contents__link">2. Filter Raw SNVs/Indels</a></li></ul></li><li><a href="#annotate-variants" class="table-of-contents__link">Somatic SNVs/Indels annotation</a><ul><li><a href="#1-annotate-variants" class="table-of-contents__link">1. Annotate Somatic SNVs/Indels</a></li><li><a href="#save-variants" class="table-of-contents__link">2. Store annotated variants for SNV Viewer</a></li></ul></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">Â© 2019-2025 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.e577558a.js"></script>
<script src="/ngsw-docs-en/assets/js/main.7c50b07b.js"></script>
</body>
</html>