---
sidebar_position: 4
sidebar_label: ACMG Secondary Findings Report
---

# ACMG Secondary Findings Report

The report includes the variants discovered in the sample, which are located in the genes
from [the Secondary Findings (SF) list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/) of
American College of Medical Genetics and Genomics (ACMG). For these genes, specific mutations are known
to be causative of disorders with defined phenotypes that are
clinically actionable by an accepted intervention. The ACMG recommends that pathogenic or likely pathogenic
variants detected in any of these genes should be reported as they are of medical relevance and could be used
in the future to inform clinical treatment.

:::caution
ACMG secondary findings report generation has its limitations due to variant annotation
using [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/).
These limitations are described [below](/results/acmg-sf-report#acmg-sf-limitations).
:::

### Conditions for the report generation {#acmg-sf-conditions}

The report is generated for the sample if the following conditions are met:
1. The sample is uploaded in FASTQ or BAM format or as a VCF file with a single sample.
2. The sample analysis was successfully completed (that is, all stages of the analysis have the status "Complete").
3. For a sample, the variant annotation stage was successfully completed:
"Germline SNVs/Indels annotation" stage if Germline origin was selected when creating a report template,
or "Somatic SNVs/Indels annotation", if Somatic origin.
4. The report template including "SNVs/Indels in ACMG SF genes" block
was [created](/settings/report-templates/report-templates#add-template) before the sample was processed.
5. The report template including "SNVs/Indels in ACMG SF genes" block is active.

### Report content {#acmg-sf-content}

The report includes two sections with lists of variants discovered in the sample, which are filled in according
to the following scheme:

<p align="center">
<img src={require('/img/eng/acmg_sf_scheme.png').default} width="560"/>
</p>

**TIER I**: The most important variants for interpretation for which the following conditions are met:
- The variant is located in a gene from [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/);
- The variant is associated according to [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/) with a
phenotype only of pathogenic or likely pathogenic clinical significance that is
listed in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/) for
the gene in which the variant is located;
- The allele frequency in [gnomAD 3](https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/) is empty or
below the frequency threshold for too frequent pathogenic variants, specified in the report template block
"SNVs/Indels in ACMG SF genes" (recommended default value = 0.03);
- The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)
in the case of a VCF file with a single sample;
- The variant meets all the conditions specified in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/)
for the gene in which the variant is located. These conditions may be associated with:
   - the clinical significance of the phenotype (as noted above, the report primarily includes variants
   associated with a phenotype only with pathogenic or likely pathogenic clinical significance
   according to [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/); this condition is not required only
   for variants in *HFE* gene);
   - type of inheritance;
   - the number of variants discovered in the sample and suitable for the conditions
   that are located in one gene from the ACMG SF list (for example, for some variants with an autosomal
   recessive inheritance to be included in the report, at least two such variants must be discovered in the sample);
   - the variant origin;
   - the genotype (for example, some variants are reported only if they are homozygous in alternative allele);
   - specific nucleotide and amino acid substitution of the variant.

**TIER II**: Variants that require further analysis for which the following conditions are met:
- The variant is located in a gene from [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/);
- The variant is associated with at least one phenotype of pathogenic and/or likely pathogenic clinical
significance according to [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/);
- The allele frequency in [gnomAD 3](https://gnomad.broadinstitute.org/news/2020-10-gnomad-v3-1/) is empty or
below the frequency threshold for too frequent pathogenic variants, specified in the report template block
"SNVs/Indels in ACMG SF genes" (recommended default value = 0.03);
- The variant site has at least one alternative allele (i.e. has the genotype 1/1, 1|1, 0/1, 1/0, 0|1, etc.)
in the case of a VCF file with a single sample;
- The variant does not meet at least one of the conditions in Tier I:
   - the variant is not associated according to [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/) with a
   phenotype that is listed in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/) for the gene
   in which the variant is located;
   - according to [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/), the variant is associated with at
   least one phenotype of benign, likely benign, or uncertain clinical significance, which is listed
   in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/) for the gene in which the variant is located;
   - the variant does not meet at least one of the conditions specified
   in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/) for the gene in which the variant is located.

<p align="center">
<img src={require('/img/eng/acmg73.png').default} width="1000"/>
</p>

Each section contains a table of variants with the following columns:
1. **Gene** is the common name of the gene in which the variant is located.
If you click on a gene, you will see a window with all the gene transcripts:

<p align="center">
<img src={require('/img/vv/transcripts.png').default} width="600"/>
</p>

You can find the description of the transcripts' table
columns [here](/results/main/snvs-indels/snv-details-panel#transcripts).

2. **Position** is a coordinate of the variant in the genome (chromosome + start position).
3. **Genetic variant** is the nucleotide and amino acid substitutions
using [the HGVS notation](https://varnomen.hgvs.org/). Nucleotide substitution: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence) prefix +
genomic position of the substituted nucleotide + reference allele > alternative allele.
Amino acid substitution: “p.” prefix (protein) + reference amino acid + amino acid position in protein + new
amino acid resulting from the substitution.
4. **Transcript** is the main transcript ID from the [RefSeq](https://www.ncbi.nlm.nih.gov/nuccore/) database
(NM_xxxxxx.x). If you click on a transcript, you will see the same window with all the gene transcripts, as when
you click on the "Gene" field.
5. **Consequence** is the effect of the variant on genes. A detailed description of the possible values
can be found [here](/results/main/snvs-indels/variant-viewer#variant-consequences).
6. **Allele frequency** is an alternative allele frequency for the sample (in percentages).
7. **Coverage depth** is a sequencing depth; the total number of reads of the sequence overlapping the variant
position for the sample.
8. **External links** are links to pages with variant information in [dbSNP](https://www.ncbi.nlm.nih.gov/snp/),
[ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/) and [COSMIC](https://cancer.sanger.ac.uk/cosmic/) (if
it was [uploaded as a custom annotation](/settings/custom-annotations#define-annotation-type)),
as well as links to phenotypes in [OMIM](https://www.omim.org/) and [MedGen](https://www.ncbi.nlm.nih.gov/medgen/)
associated with the gene in which the variant is located
in [the ACMG SF list](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/).
9. Links to the variant in embedded services: <img src={require('/img/vv/igv.png').default} width="30"/> is
a module for visualization of variant on the genome, <img src={require('/img/vv/open_vv.png').default} width="20"/> is
a variant details page in SNV Viewer
(["Annotation" tab](/results/main/snvs-indels/snv-details-page/#annotation-tab)).

At the end of the report, there is the version of the ACMG SF list used to generate the report.

### Potential findings {#potential-findings}

Variants included in the ACMG SF report can be used as [potential findings](/results/potential-findings).
To do this, you need
to [turn on the corresponding toggle in the report template](/settings/report-templates/report-templates#block-types).
Potential findings can include Tier I variants
(“*Use variants in ACMG SF genes list associated with the phenotype results as potential findings*”
option) and/or Tier II variants
(“*Use variants in ACMG SF genes list without established phenotype association or genes conditions as potential findings*"
option).

### Report limitations in ClinVar as a database source {#acmg-sf-limitations}

ACMG secondary findings report generation has limitations
of [phenotypic](/results/acmg-sf-report#phenotypic-data)
and [pathogenicity](/results/acmg-sf-report#pathogenicity-data) data
due to variant annotation using [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/) database.

ACMG guidelines for secondary findings in sequencing data provide a list of genes and
corresponding phenotypes along with recommendations for reporting genetic variants
regardless of the reason of genetic testing. The following three conditions should be satisfied:

1. The variant affects a gene
from [the list of secondary findings](https://www.ncbi.nlm.nih.gov/clinvar/docs/acmg/);
2. The variant is reported as associated with the phenotype (condition, disease) listed with
the gene in the list for secondary findings, or there is a possibility to establish such an association.
3. The variant must meet all additional conditions specified in the list of secondary findings, which often
include pathogenic or likely pathogenic clinical significance of the phenotype.

Briefly, it can be described as:

<p align="center">
<img src={require('/img/eng/acmg73_disclaimer.png').default} width="500"/>
</p>

#### Limitations of phenotypic data {#phenotypic-data}

For some of variants in ClinVar, it is impossible to obtain information about phenotype (or condition,
or disease). Missense of phenotypic information in Clinvar can stem from inconsistencies in "Conditions"
field in reports submitted by laboratories. The field can contain values "not reported", "not specified",
or can be filled with all the diseases associated with any of the pathogenic variants in the
gene. In the latter case, correct conclusions can be only by a specialist. Moreover, mutations in
different portions of the gene can result in different genetic diseases, while the list for secondary
findings can contain only a fraction of these phenotypes. Considering these and other factors, the
ACMG Secondary Findings Report by Genomenal includes two lists of variants: those for which the phenotypes
in the ACMG SF list match the condition reported for the variants in ClinVar, and those that are located
in a gene in the list but have a discordant phenotype.

<i>Therefore, the interpretation of the variants in the second section of the report must be approached with
caution.</i>

#### Limitations of pathogenicity data {#pathogenicity-data}

Usually variants are classified according to ACMG guidelines. ClinVar database is populated
by reports of pathogenicity from laboratories assuming that the responsible submitter fully
described submitted variants according to ACMG recommendations. Similarly, Genomenal follows the
same assumptions and variants are included in the report if they are reported as Pathogenic or
Likely Pathogenic without further expert evaluation of pathogenicity.

Additionally, new evidence might emerge in literature about functional effect or lack of it for a
specific variant. It can result in a revision of variant pathogenicity but will not be instantly reflected
in ClinVar. Undoubtedly, systematical review of variants' pathogenicity status is labour-consuming
and can lag compared to emergency of new scientific evidence. These factors can result in
overreporting of variants, or presence of outdated data, or underreporting of variants for
which new data has emerged but it is not reflected in ClinVar. Some variants can be omitted if they
pathogenicity is yet unknown and not detected by anyone.

*Thus, the report can include the following variants:*
- *with imprecise or erroneous pathogenicity status reported by the submitting laboratory;*
- *with pathogenicity revised due to newly emerged data (confirmed* ***pathogenic*** *by new data).*

*The following variants will not be included in the report:*
- *not yet reported in ClinVar or newly discovered;*
- *with pathogenicity revised due to newly emerged data (confirmed* ***benign*** *by new data).*

:::caution
By using ACMG Secondary Findings report, you agree that you understand above mentioned limitations,
and if clarification is needed, you are willing to discuss the result with medical specialist.
:::
