---
sidebar_position: 6
sidebar_label: Bioinformatic Report
---

# Bioinformatic Report

Bioinformatic report has collected the main metrics that will help to control the quality of the source data
and the quality of alignment.

![Bioinformatic report](/img/eng/35-bioinformatic-report.png)

To export a report in PDF format, click on <img src={require('/img/results/export_pdf.png').default} width="150"/>.

## Bioinformatic report sections: {#bioinformatic-report-sections}

### 1. File statistics (only for samples uploaded in FASTQ or BAM formats) {#file-statistics}

<p align="center">
<img src={require('/img/results/statistic_normal.png').default} width="900"/>
</p>

Data on reads count:
- Start: the number of reads in the sample file after it was verified, but before cleanup.
- Final: the number of reads in the sample file counted during the quality check after cleanup.
- Final percent: the percentage of reads that retained in the file after cleanup.

Data on reads length:
- Min length: the minimum read length in the sample file.
- Max length: the maximum read length in the sample file.
- Median length: the average value of the read length in the sample file.

:::caution
A large number of filtered reads and a large decrease in read length indicate a low quality
of the source data. In this case, we recommend you to consider re-sequencing the material.
:::

:::note
Bioinformatic Report of the tumor tissue sample from the tumor/normal sample set provides information
on the tumor and normal samples.
:::

### Quality Report {#quality-report}

This is a report on the sequence quality control metrics calculated at the “Check quality and cleanup”
stage of the pipeline. A detailed report on metrics with visualization can be downloaded in the tab
“**Workflow details**” -> “**Check quality and cleanup**” -> “**Check Quality**” -> “**Result files**”.

<details>
<summary>Click to see the description of metrics and thresholds</summary>

#### Description of metrics and thresholds {#qc-metrics}

<table>
  <tr>
   <td>Metric</td>
   <td>Description</td>
   <td>Threshold</td>
  </tr>
  <tr>
   <td>Total sequences</td>
   <td>Number of sequences</td>
   <td>>200 000</td>
  </tr>
  <tr>
   <td>Length distribution</td>
   <td>Sequence length distribution</td>
   <td>Short sequences percent &lt; 25%, short sequence length ≤ 20</td>
  </tr>
  <tr>
   <td>Tiles sequence quality</td>
   <td>Number of tiles</td>
   <td>Number of tile bad quality cells with maximum deviation 1,165 &lt; 7</td>
  </tr>
  <tr>
   <td>First base sequence quality</td>
   <td>First nucleotides quality</td>
   <td>10% of the lowest quality first bases (3 nucleotides) are of quality > 20</td>
  </tr>
  <tr>
   <td>Middle base sequence quality</td>
   <td>Middle nucleotides quality</td>
   <td>10% of the lowest quality middle bases are of quality > 20</td>
  </tr>
  <tr>
   <td>Last base sequence quality</td>
   <td>Last nucleotides quality</td>
   <td>10% of the lowest quality last bases (3 nucleotides) are of quality > 20</td>
  </tr>
  <tr>
   <td>Overrepresented sequences</td>
   <td>Overrepresented sequences</td>
   <td>Overrepresented sequences percent &lt; 1</td>
  </tr>
  <tr>
   <td>Adapter contaminated</td>
   <td>Adapter sequence contamination</td>
   <td>Adapter contaminated percent &lt; 1</td>
  </tr>
  <tr>
   <td>Base N content</td>
   <td>Number of N among bases</td>
   <td>N content percent &lt; 20</td>
  </tr>
  <tr>
   <td>GC content</td>
   <td>GC content</td>
   <td>Presence of one GC peak</td>
  </tr>
  <tr>
   <td>Base sequence content</td>
   <td>AT/GC ratio</td>
   <td>Average difference between A and T &lt; 1%, maximum difference &lt; 20%. Average difference between
   G and C &lt; 1%, maximum difference &lt; 20%.</td>
  </tr>
</table>

</details>

### Alignment Report {#alignment-report}

Metrics describing the alignment quality. If there are a large number of sequencing errors, incomplete data,
high levels of contamination, or incorrect identification of the sample source (organism), the metrics will
indicate a possible problem.
