---
sidebar_position: 6
sidebar_label: Onco-Genetic Testing Report
---

# Onco-Genetic Testing Report

The report contains selected **[onco-relevant variants](/results/main/snvs-indels/onco-relevance)**, discovered
in the sample, and the corresponding **clinical guidelines**.

:::caution
The results of the report should be interpreted using all available clinical and laboratory data and
should not be used in isolation for diagnosis and treatment.
:::

To get a report, do the following:
1. [Create a report template](/settings/report-templates/report-templates#add-template) that includes
"Onco: Onco relevant SNVs/Indels selected by user for reports" and "Onco: Clinical Guidelines for onco
relevant SNVs/Indels selected by user for reports" blocks.
2. Include the required [onco-relevant variants](/results/main/snvs-indels/onco-relevance) to the report.
You can do this [on SNV Viewer page](/results/main/snvs-indels/variant-viewer#include-to-report)
or [on the variant details page](/results/main/snvs-indels/snv-details-page).

### Report Section with Onco-Relevant Variants {#onco-relevant-variants}

- Corresponds to ["Onco: Onco relevant SNVs/Indels selected by user for reports" template block](/settings/report-templates/report-templates#block-types).
- Contains a table with onco-relevant variants included to report:

<p align="center">
<img src={require('/img/vv/onco_relevant_variant.png').default} width="1000"/>
</p>

The table columns:
1. **Gene** is the common name of the gene in which the variant is located.
If you click on a gene, you will see a window with all the gene transcripts:

<p align="center">
<img src={require('/img/vv/transcripts.png').default} width="600"/>
</p>

You can find the description of the transcripts' table columns
in [the description of "Transcripts" section on the variant details panel](/results/main/snvs-indels/snv-details-panel#transcripts).

2. **Position** is a coordinate of the variant in the genome (chromosome + start position).
3. **Genetic variant** is the nucleotide and amino acid substitutions
using [the HGVS notation](https://varnomen.hgvs.org/). Nucleotide substitution: “c.” (coding; for a
substitution in the coding sequence) or “n.” (non-coding; for a substitution in the non-coding sequence) prefix +
genomic position of the substituted nucleotide + reference allele > alternative allele.
Amino acid substitution: “p.” prefix (protein) + reference amino acid + amino acid position in protein + new
amino acid resulting from the substitution.
4. **Transcript** is the main transcript ID from [RefSeq](https://www.ncbi.nlm.nih.gov/nuccore/) database
(NM_xxxxxx.x). If you click on a transcript, you will see the same window with all the gene transcripts, as when
you click on the "Gene" field.
5. **Consequence** is the effect of the variant on genes. A detailed description of the possible values
can be found [here](/results/main/snvs-indels/variant-viewer#variant-consequences).
6. **Allele frequency** is an alternative allele frequency for the sample (in percentages).
7. **Coverage depth** is a sequencing depth; the total number of reads of the sequence overlapping the variant
position for the sample.
8. **Mutation class** according to the classification of mutations by clinical significance:

<table>
  <tr>
   <td>Mutation class</td>
   <td>Mutation level</td>
   <td>Description</td>
   <td>Mutation type</td>
  </tr>
  <tr>
   <td rowspan="3">I</td>
   <td>A</td>
   <td>Variants with high clinical significance included in clinical guidelines
   (<a href="https://www.rosoncoweb.ru/">RUSSCO</a>, <a href="https://oncology-association.ru/">АОР</a>, etc.)
   for the treatment of a certain type of tumor</td>
   <td rowspan="3">LoF-variants: missense,
   nonsense, insertions/deletions, canonical ±1 or ±2 splice sites, chromosomal rearrangements</td>
   </tr>
	<tr>
   <td>B</td>
   <td>Variants with high clinical significance identified in large-scale studies and with consent of Russian
   experts in this field for certain types of malignant neoplasms</td>
   </tr>
	<tr>
   <td>C</td>
   <td>Variants with high clinical significance, identified in large-scale studies and with consent of
   foreign experts in the field for the treatment of certain types of malignant neoplasms</td>
  </tr>
  <tr>
   <td rowspan="2">II</td>
   <td>D</td>
   <td>Variants with potential clinical significance recommended for other indications or for the treatment
   of other tumor types</td>
   <td rowspan="2">LoF-variants: missense,
   nonsense, insertions/deletions, canonical ±1 or ±2 splice sites, chromosomal rearrangements</td>
   </tr>
	<tr>
   <td>E</td>
   <td>Variants with potential clinical significance identified in preclinical studies for the treatment
   of this oncological disease</td>
  </tr>
  <tr>
   <td>III</td>
   <td></td>
   <td>Somatic variants of uncertain clinical significance: located in genes not associated with the development
   of cancer, or have no evidence to establish their pathogenicity and have low minor allele frequency (MAF) in
   the population</td>
   <td>Mutations with undetermined effect on protein function: mostly missense mutations</td>
  </tr>
  <tr>
   <td>IV</td>
   <td></td>
   <td>Rare benign and likely benign variants with an equitable allele representation (50% for heterozygotes
   and 100% for homozygotes) and high population frequency. Usually germline. They have no clinical
   significance for the treatment of malignant neoplasms.</td>
   <td>Mutations with little or no clear effect on protein function: mostly missense mutations</td>
  </tr>
</table>

You can determine the mutation class in the report section with clinical guidelines.

9. Links to the variant in embedded services: <img src={require('/img/vv/igv.png').default} width="30"/> is
a module for visualization of variant on the genome, <img src={require('/img/vv/open_vv.png').default} width="20"/> is
a variant details page in SNV Viewer
(["Annotation" tab](/results/main/snvs-indels/snv-details-page/#annotation-tab)).

### Report Section with Clinical Guidelines {#clinical-guidelines}

- Corresponds to ["Onco: Clinical Guidelines for onco relevant SNVs/Indels selected by user for reports" template block](/settings/report-templates/report-templates#block-types).
- At the top of the section, there are the following fields:
1. **Oncological disease**: Non-small cell lung carcinoma, Melanoma, Neoplasm of the pancreas, Breast carcinoma,
Ovarian neoplasm, Prostate cancer, Anaplastic thyroid carcinoma, Colorectal carcinoma;
2. **Stage** (of the disease): I-IV;
3. **Stage details**: Relapse, Progressing, Inoperable, Brain metastases;
4. **Therapy line** (used to treat the patient's oncological disease at the time the sequencing sample
material was collected): 1-4 line, Adjuvant or Neoadjuvant therapy, Perioperative, Preoperative or Induction
chemotherapy);
5. **Mutation class** (see description above).

<p align="center">
<img src={require('/img/results/recommendations.png').default} width="900"/>
</p>

- If the fields required to unambiguously find clinical guidelines for the patient (oncological disease,
stage, details) are not selected, then the recommendations will not be displayed.
