---
sidebar_position: 5
sidebar_label: Alignments
---

# Alignments

The "***Alignments***" section contains the sample reads mapped to reference genome and obtained as a result of
[the "Alignment" analysis stage](/results/workflow-details/fastq-analysis/alignment).

<p align="center">
<img src={require('/img/results/alignments.png').default} width="800"/>
</p>

The results of alignment of sample reads are presented as:

1. File with alignments in **BAM (Binary Alignment Map)** format. To download the file,
click on <img src={require('/img/eng/30-bam_file.png').default} width="45"/>.

2. **Index file for alignment in BAI format**. The file is required for some tools.
To download the file, click on <img src={require('/img/eng/31-bai-file.png').default} width="45"/>.

3. **[Integrative Genomics Viewer (IGV)](https://software.broadinstitute.org/software/igv/)** is an embedded
module for visualization of variations on the genome. To open the module,
click on <img src={require('/img/results/igv_btn.png').default} width="45"/>.

***Visualization in IGV: reducing the number of copies on the example of a deletion:***

<p align="center">
<img src={require('/img/eng/32-igv-deletion.jpg').default} width="800"/>
</p>

:::info
If you would like to add an alignment track of the sample analyzed by Genomenal
to [IGV desktop application](https://igv.org/doc/desktop/), you can load it via URL. To do this, do the following:
1. Right-click on <img src={require('/img/eng/30-bam_file.png').default} width="45"/> and select the
"*Copy link address*" option.
2. Load the track via URL to the IGV desktop application as
described [here](https://igv.org/doc/desktop/#UserGuide/loading_data/).
3. Right-click on the index file for alignment <img src={require('/img/eng/31-bai-file.png').default} width="45"/>
and select the "*Copy link address*" option.
4. Add the index file URL in the corresponding field in IGV.
5. Click "*OK*". Done! Sample alignment track has been added to IGV.
:::
