---
sidebar_position: 4
sidebar_label: Copy Number Variations
---

# Copy Number Variations

The "***Somatic Copy Number Variations***" and "***Germline Copy Number Variations***" sections contain
the results of discovery of somatic and germline, respectively, variants in which sections of the genome
are repeated and the number of repeats in the genome varies between individuals.

***Somatic variations*** occur after conception and are not passed on to children. They can cause cancer or other
diseases. Somatic variations discovery is carried out only for ***tumor tissue samples*** and, therefore, the results
are presented only on the tumor tissue sample page.

<p align="center">
<img src={require('/img/results/somatic_cnvs.png').default} width="800"/>
</p>

***Germline variations*** can be discovered in any cells of the body and are passed on from parents to offspring.
Accordingly, germline variations discovery is carried out both for ***tumor tissue samples*** (in the case of
single tumor tissue samples without control) and for ***non-tumor tissue samples***.
For a set of tumor/normal samples, the results of the germline variations discovery
in the normal sample are presented on the pages of both samples of the set.

<p align="center">
<img src={require('/img/results/germline_cnvs.png').default} width="800"/>
</p>

For a single tumor sample without control, the results of discovery of somatic and germline variations
are combined into one section:

<p align="center">
<img src={require('/img/results/somatic_and_germline_cnvs.png').default} width="800"/>
</p>

The detected and annotated copy number variations (CNVs) are presented in all sections as:

1. File in **AnnotSV CSV** format that is a table of copy number variations. They are presented either
like a complete deletion or insertion, or like genes overlapped by a deletion/insertion.
To download the file, click on <img src={require('/img/results/table_btn.png').default} width="120"/>.

<details>
<summary>Click to see the main fields of AnnotSV CSV</summary>

#### The main fields of AnnotSV CSV {#annotsv-fields}

<table>
  <tr>
   <td>Field</td>
   <td>Description</td>
  </tr>
  <tr>
   <td>AnnotSV ID</td>
   <td>Variation ID assigned by the annotator</td>
  </tr>
  <tr>
   <td>SV chrom</td>
   <td>Chromosome</td>
  </tr>
  <tr>
   <td>SV start</td>
   <td>Variation start position in the chromosome</td>
  </tr>
  <tr>
   <td>SV end</td>
   <td>Variation end position in the chromosome</td>
  </tr>
  <tr>
   <td>SV length</td>
   <td>Variation length</td>
  </tr>
  <tr>
   <td>SV type</td>
   <td>Variation type (DEL, DUP)</td>
  </tr>
  <tr>
   <td>AnnotSV type</td>
   <td>Annotation line type (“full” - annotation of all complete variations, “split” - annotation of a variation
   split by gene). Each full annotation line can correspond to several split annotation lines.</td>
  </tr>
</table>

A description of the remaining 54 annotation fields can be found here:
[AnnotSV Manual](https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf).

</details>

2. File in **Google Spreadsheet** (only a limited number of variations can be shown). To open the file,
click on <img src={require('/img/results/google_btn.png').default} width="45"/>.
3. **CNV coverage file** in TSV format that is an unannotated file, convenient for analysing copy number
variations in large regions (chromosome arms, chromosomes). To download the file,
click on <img src={require('/img/results/cnv_tsv.png').default} width="45"/>.

<details>
<summary>Click to see the main fields of CNV coverage file</summary>

#### The main fields of CNV coverage file {#cnv-coverage-fields}

<table>
  <tr>
   <td>Field</td>
   <td>Description</td>
  </tr>
  <tr>
   <td>ID</td>
   <td>Region name (chromosome arm, chromosome)</td>
  </tr>
  <tr>
   <td>Call</td>
   <td>CNV discovery result. Possible values are "+" (increasing the number of copies compared to normal), "-"
   (reducing the number of copies compared to normal), "." (no deviations from the normal chromosome copy number
   were found) and "Not considered" (the locus was not considered at all in the analysis).</td>
  </tr>
  <tr>
   <td>Ampl_cov</td>
   <td>Coverage (in nucleotides) of the effective locus length with segments with a copy number above
   the amplification threshold</td>
  </tr>
  <tr>
   <td>Ampl_cov_frac</td>
   <td>Fraction (in %) of the effective locus length covered by segments with a copy number above
   the amplification threshold</td>
  </tr>
  <tr>
   <td>Chrom</td>
   <td>Chromosome</td>
  </tr>
  <tr>
   <td>Depl_cov</td>
   <td>Coverage (in nucleotides) of the effective locus length with segments with a copy number below
   the depletion threshold</td>
  </tr>
  <tr>
   <td>Depl_cov_frac</td>
   <td>Fraction (in %) of the effective locus length covered by segments with a copy number below
   the depletion threshold</td>
  </tr>
  <tr>
   <td>Effective_end</td>
   <td>Maximum coordinate value of effective length</td>
  </tr>
  <tr>
   <td>Effective_length</td>
   <td>Total effective length</td>
  </tr>
  <tr>
   <td>Effective_start</td>
   <td>Minimum coordinate value of effective length</td>
  </tr>
  <tr>
   <td>End</td>
   <td>End coordinate of the variation</td>
  </tr>
  <tr>
   <td>Length</td>
   <td>Variation length</td>
  </tr>
  <tr>
   <td>Start</td>
   <td>Start coordinate of the variation</td>
  </tr>
</table>

</details>

4. **Genome-wide CNV segments plot** in PNG format. To open the plot,
click on <img src={require('/img/eng/25-plot_file.png').default} width="45"/>. Plot example:

<p align="center">
<img src={require('/img/eng/28-genome-wide-plot.png').default} width="800"/>
</p>

We can see the chromosomes on a horizontal scale and the copy number in chromosomes on a vertical scale.
In the given example, the copy number of chromosome 21 is increased in the sample compared to normal diploid
number of copies.

5. **Karyogram-like graph** with denoted chromosome-level CNVs in PNG format. To open the graph,
click on <img src={require('/img/eng/26_karyotype_file.png').default} width="45"/>. Graph example:

<p align="center">
<img src={require('/img/eng/29-karyotype-plot-example.png').default} width="500"/>
</p>

The karyogram indicates an increased number of copies of chromosome 21 in the sample.

6. **[Integrative Genomics Viewer (IGV)](https://software.broadinstitute.org/software/igv/)** is an embedded
module for visualization of variations on the genome. To open the module,
click on <img src={require('/img/results/igv_btn.png').default} width="45"/>.
