---
sidebar_position: 1
sidebar_label: Variant Viewer
---

# Variant Viewer

### Overview

Variant Viewer is a built-in service for viewing and analyzing variants. Located on the main page of the sample ("**Main**" tab).

On the ***tumor*** sample page, you will find two Variant Viewers: one with ***somatic*** and one with ***germline*** mutations discovered in this sample:

<p align="center">
<img src={require('/img/vv/germline_and_somatic_vv.png').default} width="600"/>
</p>

The ***normal*** sample page presents a Variant Viewer with ***germline*** mutations only:

<p align="center">
<img src={require('/img/vv/germline_only.png').default} width="600"/>
</p>

### The main page

After clicking on the **Variant Viewer** button, you will see the main page of the service:

<p align="center">
<img src={require('/img/vv/main_table.png').default} width="850"/>
</p>

The main object of the Variant Viewer is a table with variants - each variant on a new line:

<p align="center">
<img src={require('/img/vv/snv_row.png').default} width="950"/>
</p>

### Description of table columns

By default, the table has the following columns (the set and order of the columns can be changed):

1. **LOCUS** is the common name of the gene in which the variant is located.
2. **ORIGIN** is a mutation type (somatic or germline).

import HiddenOrigin from '/docs/imported_components/_origin_column.mdx';

<HiddenOrigin />

3. **POSITION** is the coordinates of the variant in the genome (chromosome + start position).
4. **HGVSP** is an amino acid substitution using [the HGVS notation](https://varnomen.hgvs.org/) (“p.” prefix (protein) + reference amino acid + amino acid position in protein + new amino acid resulting from the substitution).
When hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for and HGVSp with amino acids in the three letter code:

<p align="center">
<img src={require('/img/vv/hgvsp_tooltip.png').default} width="400"/>
</p>

4. **ALTERATION** is a nucleotide substitution (reference allele > alternative allele(s)).
5. **HGVSC** is a nucleotide substitution using [the HGVS notation](https://varnomen.hgvs.org/) (genomic position of the substituted nucleotide + reference allele > alternative allele). Starts with a prefix: “c.” (coding; for a substitution in the coding sequence), “n.” (non-coding; for a substitution in the non-coding sequence).
When hovering over the value of the HGVSP column for a variant, you can see the transcript which the substitution was predicted for:

<p align="center">
<img src={require('/img/vv/hgvsc_tooltip.png').default} width="500"/>
</p>

6. **EXON** is the exon number in the transcript which is affected by the variant.
7. **DEPTH (ALT/REF)** is a sequencing depth; the total number of reads of the sequence overlapping the variant position for this sample. ALT is the alternative allele reads count, REF - the reference allele reads count.
8. **GT** is a genotype; allelic values (0 - reference allele, 1 - first alternative allele, 2 - second alternative allele etc.), divided by ”/” (for unphased genotype) or “|” (for phased genotype).
9. **CONSEQUENCE** is the effect of the variant on genes.

<details>
<summary>Click to see a table with the consequences</summary>

#### Consequences of variants in the annotation

<table>
  <tr>
   <td>Consequence
   </td>
   <td>Description
   </td>
   <td>Impact (see description below)
   </td>
  </tr>
  <tr>
   <td>Coding sequence
   </td>
   <td>The variant hits a coding sequence
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Transcript ablation
   </td>
   <td>Deletion of a transcript
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Splice acceptor
   </td>
   <td>The variant hits a splice acceptor site
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Splice donor
   </td>
   <td>The variant hits a splice donor site
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Stop gained
   </td>
   <td>Variant causes a STOP codon
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Frameshift
   </td>
   <td>Insertion or deletion causes a frame shift
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Stop lost
   </td>
   <td>Variant causes stop codon to be mutated into a non-stop codon
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Start lost
   </td>
   <td>Variant causes start codon to be mutated into a non-start codon
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Transcript amplification
   </td>
   <td>A feature amplification of a region containing a transcript
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>Inframe insertion
   </td>
   <td>One or many codons are inserted
   </td>
   <td>MODERATE
   </td>
  </tr>
  <tr>
   <td>Inframe deletion
   </td>
   <td>One or many codons are deleted
   </td>
   <td>MODERATE
   </td>
  </tr>
  <tr>
   <td>Missense
   </td>
   <td>Variant causes a codon that produces a different amino acid
   </td>
   <td>MODERATE
   </td>
  </tr>
  <tr>
   <td>Protein altering
   </td>
   <td>A variant which is predicted to change the protein encoded in the coding sequence
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Splice region
   </td>
   <td>A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron
   </td>
   <td>LOW
   </td>
  </tr>
  <tr>
   <td>Incomplete terminal codon
   </td>
   <td>A sequence variant where at least one base of the final codon of an incompletely annotated transcript is changed
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>Start retained
   </td>
   <td>Variant causes start codon to be mutated into another start codon
   </td>
   <td>LOW
   </td>
  </tr>
  <tr>
   <td>Stop retained
   </td>
   <td>Variant causes stop codon to be mutated into another stop codon
   </td>
   <td>LOW
   </td>
  </tr>
  <tr>
   <td>Synonymous
   </td>
   <td>Variant causes a codon that produces the same amino acid
   </td>
   <td>LOW
   </td>
  </tr>
  <tr>
   <td>Mature miRNA
   </td>
   <td>A transcript variant located with the sequence of the mature miRNA
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>UTR5
   </td>
   <td>A variant hits 5'UTR region
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>UTR3
   </td>
   <td>A variant hits 3'UTR region
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Non coding transcript exon
   </td>
   <td>A sequence variant that changes non-coding exon sequence in a non-coding transcript
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Upstream
   </td>
   <td>Upstream of a gene
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Downstream
   </td>
   <td>Downstream of a gene
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>TF binding site
   </td>
   <td>Transcription factor binding site (TFBS)
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>TFBS ablation
   </td>
   <td>Deletion of a TFBS
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>TFBS amplification
   </td>
   <td>Amplification of a TFBS
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Regulatory region ablation
   </td>
   <td>Deletion of a regulatory region
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Regulatory region amplification
   </td>
   <td>Amplification of regulatory region
   </td>
   <td>HIGH
   </td>
  </tr>
  <tr>
   <td>Feature elongation
   </td>
   <td>A sequence variant that causes the extension of a genomic feature
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>Regulatory region variant
   </td>
   <td>The variant hits a regulatory feature
   </td>
   <td>MODIFIER
   </td>
  </tr>
  <tr>
   <td>Feature truncation
   </td>
   <td>A sequence variant that causes the reduction of a genomic feature
   </td>
   <td>
   </td>
  </tr>
  <tr>
   <td>Intergenic variant
   </td>
   <td>The variant is in an intergenic region
   </td>
   <td>MODIFIER
   </td>
  </tr>
</table>

</details>


10. <b>IMPACT</b> is the predicted effect of the variant on the protein.

<details>
<summary>Click to see a table with impacts</summary>

#### Impacts of variants in the annotation

<table>
  <tr>
   <td>Impact
   </td>
   <td>Description
   </td>
  </tr>
  <tr>
   <td>HIGH
   </td>
   <td>The variant has high (disruptive) impact in the protein, probably causing protein truncation, loss of function or triggering nonsense mediated decay
   </td>
  </tr>
  <tr>
   <td>MODERATE
   </td>
   <td>The variant might change protein effectiveness
   </td>
  </tr>
  <tr>
   <td>LOW
   </td>
   <td>The variant is mostly harmless or unlikely to change protein behavior
   </td>
  </tr>
  <tr>
   <td>MODIFIER
   </td>
   <td>Usually variants affecting non-coding genes, where predictions are difficult or there is no evidence of impact
   </td>
  </tr>
</table>

</details>

11. **EXTERNAL LINKS** are links to pages with variant information in [dbSNP](https://www.ncbi.nlm.nih.gov/snp/), [ClinVar](https://preview.ncbi.nlm.nih.gov/clinvar/) databases.
12. **GNOMAD AF** is the allele frequency according to the [gnomAD](https://gnomad.broadinstitute.org/) database.
13. **ACMG** is the interpretation of the variant by ACMG recommendations. If you have not interpreted the variant, then the column value will be “N/A” (No interpretation).
