---
sidebar_position: 3
sidebar_label: Getting to know the service
---

# Getting to know the service

import Login from '/docs/imported_components/_login.mdx';

<Login />

Next you will see the main page listing the patients. If you have not previously used the service, the page will be empty. There is a search bar at the top to help you navigate through the patient pages.

<p align="center">
<img src={require('/img/eng/1-all-patients-page.png').default} width="1000"/>
</p>

You can see various sections of the service in the navigation bar on the left. The bar contains the **“In process”** section, where there will be only those patients with samples being currently processed; the **“Problems”** section, where there will be only those patients with problems during data processing. You can transfer irrelevant data or data for later deletion to the **“Archive”** section.

Patients can be grouped into **cohorts**, for example, by type of pathology or data source. By selecting a cohort in the navigation bar on the left, you can see only patients belonging to that cohort.

Various global configuration options can be accessed in the
**“Settings”** section. Here you can add sets of genes, which you can
later use to filter the analysis results (**“Gene panels”**
section). When creating a panel, you can select genes from a list of
all genes, or import the prepared gene list as text (for example TP53,
BRCA1, EGFR). Alternatively, you can select a panel from a library of
the most common clinical panels and the genes will be imported from
there automatically. This can significantly speed up further data
interpretation if you are only interested in a small set of genes
relevant for a specific pathology.

Often the data analyzed is prepared by targeted sequencing (WES,
disease-specific gene panels, etc.).  NGS Wizard attempts to detect if
one of the widely adopted commercial kits was used. However, if it's not
the case or detection fails, you can upload sequenced
regions (BED file) in the **“Capture kits”** section (make sure
that you have correctly specified the assembly of the reference genome
when uploading the BED file).
