<!doctype html>
<html class="docs-version-current" lang="en" dir="ltr">
<head>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width,initial-scale=1">
<meta name="generator" content="Docusaurus v2.0.0-beta.6">
<link rel="search" type="application/opensearchdescription+xml" title="Genomenal" href="/ngsw-docs-en/opensearch.xml"><title data-react-helmet="true">Copy number variations discovery | Genomenal</title><meta data-react-helmet="true" name="twitter:card" content="summary_large_image"><meta data-react-helmet="true" property="og:url" content="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery"><meta data-react-helmet="true" name="docsearch:language" content="en"><meta data-react-helmet="true" name="docsearch:version" content="current"><meta data-react-helmet="true" name="docsearch:docusaurus_tag" content="docs-default-current"><meta data-react-helmet="true" property="og:title" content="Copy number variations discovery | Genomenal"><meta data-react-helmet="true" name="description" content="After the &quot;Variant discovery preprocessing&quot; stage"><meta data-react-helmet="true" property="og:description" content="After the &quot;Variant discovery preprocessing&quot; stage"><link data-react-helmet="true" rel="shortcut icon" href="/ngsw-docs-en/img/favicon.png"><link data-react-helmet="true" rel="canonical" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery" hreflang="en"><link data-react-helmet="true" rel="alternate" href="https://genomenal.com/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery" hreflang="x-default"><link data-react-helmet="true" rel="preconnect" href="https://DZAOFCP7KJ-dsn.algolia.net" crossorigin="anonymous"><link rel="stylesheet" href="/ngsw-docs-en/assets/css/styles.ce916e14.css">
<link rel="preload" href="/ngsw-docs-en/assets/js/runtime~main.905320c1.js" as="script">
<link rel="preload" href="/ngsw-docs-en/assets/js/main.d04fc880.js" as="script">
</head>
<body>
<script>!function(){function t(t){document.documentElement.setAttribute("data-theme",t)}var e=function(){var t=null;try{t=localStorage.getItem("theme")}catch(t){}return t}();t(null!==e?e:"light")}()</script><div id="__docusaurus">
<div><a href="#" class="skipToContent_1oUP">Skip to main content</a></div><nav class="navbar navbar--fixed-top"><div class="navbar__inner"><div class="navbar__items"><button aria-label="Navigation bar toggle" class="navbar__toggle clean-btn" type="button" tabindex="0"><svg width="30" height="30" viewBox="0 0 30 30" aria-hidden="true"><path stroke="currentColor" stroke-linecap="round" stroke-miterlimit="10" stroke-width="2" d="M4 7h22M4 15h22M4 23h22"></path></svg></button><a href="https://genomenal.com" target="_blank" rel="noopener noreferrer" class="navbar__brand"><img src="/ngsw-docs-en/img/app-logo.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--light_3UqQ navbar__logo"><img src="/ngsw-docs-en/img/app-logo-white.svg" alt="Genomenal Logo" class="themedImage_1VuW themedImage--dark_hz6m navbar__logo"><b class="navbar__title"></b></a></div><div class="navbar__items navbar__items--right"><div class="react-toggle toggle_3Zt9 react-toggle--disabled"><div class="react-toggle-track" role="button" tabindex="-1"><div class="react-toggle-track-check"><span class="toggle_71bT">🌜</span></div><div class="react-toggle-track-x"><span class="toggle_71bT">🌞</span></div><div class="react-toggle-thumb"></div></div><input type="checkbox" class="react-toggle-screenreader-only" aria-label="Switch between dark and light mode"></div><div class="searchBox_1Doo"><button type="button" class="DocSearch DocSearch-Button" aria-label="Search"><span class="DocSearch-Button-Container"><svg width="20" height="20" class="DocSearch-Search-Icon" viewBox="0 0 20 20"><path d="M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z" stroke="currentColor" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path></svg><span class="DocSearch-Button-Placeholder">Search</span></span><span class="DocSearch-Button-Keys"></span></button></div></div></div><div role="presentation" class="navbar-sidebar__backdrop"></div></nav><div class="main-wrapper docs-wrapper docs-doc-page"><div class="docPage_31aa"><button class="clean-btn backToTopButton_35hR" type="button"><svg viewBox="0 0 24 24" width="28"><path d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z" fill="currentColor"></path></svg></button><aside class="docSidebarContainer_3Kbt"><div class="sidebar_15mo"><nav class="menu thin-scrollbar menu_Bmed menuWithAnnouncementBar_2WvA"><ul class="theme-doc-sidebar-menu menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/">Introduction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/log_in">Login to the System</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Get to Know Genomenal</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-tumor-analysis">Quick Start for Tumor Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/quick-start-for-prenatal-genetic-testing">Quick Start for Prenatal Genetic Testing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/data-for-analysis">Samples for Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/add-patient">Adding Patient</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Uploading Samples</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#">Analysis Results</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/results">Sample Page</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Main</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/potential-findings">Potential Findings</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/sample-interpretation">Sample Interpretation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/acmg-sf-report">ACMG Secondary Findings Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/variants-by-phenotypes">ClinVar Phenotype Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/onco-genetic-testing-report">Onco-Genetic Testing Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/polygenic-report">Report on Polygenic Traits</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/mutations-suitable-for-the-specified-conditions">Mutations Suitable For The Specified Conditions</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/report-with-variants-added-by-user">Report with variants added by the user</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">CNV Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/glioma-report">Glioma Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/bioinformatic-report">Bioinformatic Report</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/coverage-analysis">Coverage Analysis</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Workflow Details</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/workflow-details">Workflow details</a></li><li class="theme-doc-sidebar-item-category menu__list-item"><a class="menu__link menu__link--sublist menu__link--active" href="#" tabindex="0">Analysis of FASTQ or BAM samples</a><ul style="display:block;overflow:visible;height:auto" class="menu__list"><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/workflow-details">Analysis of FASTQ or BAM samples</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/upload-identify-verify">Upload, identify and verify</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/check-quality-and-cleanup">Check quality and cleanup</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/alignment">Alignment</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">Variant discovery preprocessing</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/calculate-coverage">Calculate coverage</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/somatic-snvs-indels">Somatic SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/germline-snvs-indels">Germline SNVs/Indels discovery and annotation</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations">Structural variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link menu__link--active" aria-current="page" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery">Copy number variations discovery</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes">Phenotype prediction</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">Generate reports</a></li></ul></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#" tabindex="0">Analysis of VCF or GT samples</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" tabindex="0" href="/ngsw-docs-en/results/parameters">Parameters</a></li></ul></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/runs">Runs</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/group-analysis">Group Analysis</a></li><li class="theme-doc-sidebar-item-link menu__list-item"><a class="menu__link" href="/ngsw-docs-en/cohorts">Cohorts</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Settings</a></li><li class="theme-doc-sidebar-item-category menu__list-item menu__list-item--collapsed"><a class="menu__link menu__link--sublist" href="#">Examples of Samples</a></li></ul></nav></div></aside><main class="docMainContainer_3ufF"><div class="container padding-top--md padding-bottom--lg"><div class="row"><div class="col docItemCol_3FnS"><div class="docItemContainer_33ec"><article><div class="tocCollapsible_1PrD theme-doc-toc-mobile tocMobile_3Hoh"><button type="button" class="clean-btn tocCollapsibleButton_2O1e">On this page</button></div><div class="theme-doc-markdown markdown"><header><h1>Copy number variations discovery</h1></header><p>After <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/variant-discovery-preprocessing">the &quot;Variant discovery preprocessing&quot; stage</a>
has been successfully completed, copy number variations (CNVs) discovery, which determines how many copies of a
DNA region were, on average, present in the sequenced genome, can be initiated for a sample. To run the stage,
it must be <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#cnv-discovery">included</a> in the analysis workflow
and the sequencing type and capture kit must be defined for the sample (either automatically or manually by the
user). The copy number variations discovery pipeline differs for non-tumor or single tumor samples, and tumor
sample from a tumor/control sample set.</p><p align="center"><img src="/ngsw-docs-en/assets/images/cnvs-9c6786a09c7f6d8d92185dfc2301398c.png" width="650"></p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="normal"></a>Non-tumor or single tumor sample<a class="hash-link" href="#normal" title="Direct link to heading">#</a></h2><p>In case of analysis of a tumor sequencing sample only without control or prenatal diagnosis (analysis of a non-tumor
sequencing sample), <a href="https://cnvkit.readthedocs.io/en/stable/" target="_blank" rel="noopener noreferrer">CNVkit</a> tools are used to discover copy number
variations.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="estimate-read-counts-or-depths"></a>1. Estimate read counts or depths<a class="hash-link" href="#estimate-read-counts-or-depths" title="Direct link to heading">#</a></h3><p>In the first step, <a href="https://cnvkit.readthedocs.io/en/stable/pipeline.html#autobin" target="_blank" rel="noopener noreferrer">CNVkit autobin</a>
estimates read counts or depths in an alignment file to estimate reasonable on- and off-target bin (i.e. a coverage
assessment interval) sizes. In addition, it adds gene names to target regions. Depending on the coverage depth and
the sequencing type, different ways of dividing the genome into bins, within which reads will be counted, are applied:</p><ul><li>If the sample sequencing type is defined as &quot;Panel&quot; (sequencing using a targeted panel) or &quot;WES&quot; (whole-exome
sequencing), autobin is provided with the uniquely mapped genome regions and the capture kit intervals
corresponding to the sample (i.e. the targeted panel used for targeted selection).</li><li>If the sample sequencing type is defined as &quot;WGS&quot; (whole-genome sequencing) or &quot;Low-pass WGS&quot; (whole genome
sequencing with low coverage), autobin is provided with the genomic regions available for sequencing.
First, a target bin size is calculated in the range from 10 to 1000000000 bp. Then, the estimated bin size is used
to limit the given bin size, which is defined in <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#bin-size">parameters</a>.
This size is used for generating an output.</li></ul><p>As a result, autobin generates target and antitarget BED files, and prints a table of estimated average read depths
and recommended bin sizes.</p><p>If a reference samples panel <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#wgs-reference-panel">for WGS</a>
or <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#targeted-capture-reference-panel">for Targeted capture</a>
is selected for sample analysis, the bin size corresponding to the selected panel (it can be customized for some
panels) will be used instead of estimating target bin sizes.</p><p>The resulting files with target and antitarget bins can be downloaded
in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Estimate read counts or depths</em>&quot; task details (&quot;<em>Download target BED</em>&quot;
and &quot;<em>Download antitarget BED</em>&quot;, respectively).</p><div class="admonition admonition-info alert alert--info"><div class="admonition-heading"><h5><span class="admonition-icon"><svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg></span>info</h5></div><div class="admonition-content"><p>If the sample analysis reveals that the sequencing sample has too low coverage with the capture kit (more than 10%
of capture kit regions have zero coverage), then the copy number variations discovery is not further performed
for the sample. This can happen if the capture kit was incorrectly selected or if the sample is of low quality.</p></div></div><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="calculate-coverage"></a>2. Calculate coverage in the given regions<a class="hash-link" href="#calculate-coverage" title="Direct link to heading">#</a></h3><p>At this step, <a href="https://cnvkit.readthedocs.io/en/stable/pipeline.html#coverage" target="_blank" rel="noopener noreferrer">CNVkit coverage</a>
calculates the number of reads in each of the bins determined in the previous step. Coverage is
calculated via mean read depth from a pileup. The resulting files with coverage information in target and
antitarget bins can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Calculate coverage in the given regions</em>&quot;
task details (&quot;<em>Download target coverage CNN</em>&quot; and &quot;<em>Download antitarget coverage CNN</em>&quot;, respectively).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="compile-cn-reference"></a>3. Compile a copy-number reference<a class="hash-link" href="#compile-cn-reference" title="Direct link to heading">#</a></h3><p>To discover copy number variations by read depth, it is necessary to estimate the depth deviation from the copy-number
reference. Normal coverage in regions is calculated taking into account the GC content and repeat-masked proportion of
each region. This is done using the <a href="https://cnvkit.readthedocs.io/en/stable/pipeline.html#reference" target="_blank" rel="noopener noreferrer">CNVkit reference</a>
tool. Both target and antitarget bins are involved in the compilation.</p><p>If a reference samples panel <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#wgs-reference-panel">for WGS</a>
or <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#targeted-capture-reference-panel">for Targeted capture</a>
is selected for sample analysis, then this panel will be used instead of compiling a copy-number reference.</p><p>The resulting file with the expected read depth and the reliability of this estimate can be downloaded in
the &quot;<em>Result files</em>&quot; section in the &quot;<em>Compile a copy-number reference</em>&quot; task details (&quot;<em>Download CNN</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="cnv-discovery"></a>4. CNV discovery<a class="hash-link" href="#cnv-discovery" title="Direct link to heading">#</a></h3><p>This is the key step where the copy number of each genome region is calculated based on the number of reads
mapped to the genome. For this, <a href="https://cnvkit.readthedocs.io/en/stable/" target="_blank" rel="noopener noreferrer">CNVkit</a> tools are used:</p><ol><li><a href="https://cnvkit.readthedocs.io/en/stable/pipeline.html#fix" target="_blank" rel="noopener noreferrer">CNVkit fix</a> combines the uncorrected target
and antitarget coverage tables and correct for biases in regional coverage and GC content, according to the
reference. The resulting table of copy number ratios can be downloaded in
the &quot;<em>Result files</em>&quot; section in the &quot;<em>CNV discovery</em>&quot; task details (&quot;<em>Download CNR</em>&quot;).</li><li><a href="https://cnvkit.readthedocs.io/en/stable/pipeline.html#segment" target="_blank" rel="noopener noreferrer">CNVkit segment</a> infers discrete copy number
segments from the coverage table. If the sample sequencing type is a targeted panel, or if a single tumor sample
is analyzed, then the circular binary segmentation (CBS) is used as the segmentation algorithm. If the sample
sequencing type is WGS, then the experimental method is used as the segmentation algorithm, that is a 3-state
hidden Markov model with fixed amplitude for the loss, neutral, and gain states corresponding to absolute copy
numbers of 1, 2, and 3. The resulting file with segments can be downloaded in the same section (&quot;<em>Download CNS</em>&quot;).</li><li><a href="https://cnvkit.readthedocs.io/en/stable/importexport.html#seg" target="_blank" rel="noopener noreferrer">CNVkit export seg</a> exports the segmentation files
to the standard SEG format to be loaded
in <a href="https://software.broadinstitute.org/software/igv/" target="_blank" rel="noopener noreferrer">the Integrative Genomics Viewer (IGV)</a>.
Called segments in SEG format can be downloaded in the same section (&quot;<em>Download Called segments SEG</em>&quot;).</li></ol><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="evaluate-and-visualize"></a>5. Evaluating and visualizing biological features<a class="hash-link" href="#evaluate-and-visualize" title="Direct link to heading">#</a></h3><p>At this step, the copy number variations are processed depending on the log2FC threshold value, that is
the logarithmic ratio of the detected copy number to the normal copy number. The default threshold value (-0.7)
can be changed in <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#cnv-call-threshold">the parameters</a>. Using this
value as an example, let&#x27;s see how the thresholds for calling deletions and duplications on autosomes and sex
chromosomes will be calculated:</p><ul><li>Copy number variations on autosomes with log2FC ≤ -0.7 are considered deletions. -0.7 is the default
log2FC threshold, below which only one autosome is called instead of the normal two.</li><li>Copy number variations on the X and Y chromosomes with log2FC ≤ -2.11 are considered deletions.</li><li>Copy number variations on autosomes with log2FC ≥ 0.46 are considered duplications.
The threshold for duplication on an autosome is calculated using the following
formula: <em>log2FC = log<sub>2</sub>((2+D)/2)</em>, where <em>D</em> is the “number” of copy number that
is “lost” at log2FC = -0.7, and <em>2</em> is the number of sister chromatids in the autosome.</li><li>Copy number variations on the X and Y chromosomes with log2FC ≥ 0.82 are considered duplications.</li><li>Variations that do not satisfy the thresholds, i.e. variations on autosomes with -0.7 &lt; log2FC &lt; 0.46
and variations on sex chromosomes with -2.11 &lt; log2FC &lt; 0.82 do not pass filtering.</li></ul><details class="details_2Ziz alert alert--info details_1VDD" data-collapsed="true"><summary>Click here to see how log2FC thresholds are calculated for deletions and duplications on autosomes and sex chromosomes</summary><div><div class="collapsibleContent_3OHp">The threshold of calling a single copy instead of two on autosomes is calculated as follows: <i>T<sub>21</sub> = log<sub>2</sub>(C/2)</i>, where <i>C</i> is the number of called copies.<br>Delta of copy number: <i>D = 2 – C = 2 – 2 × 2<sup>T<sub>21</sub></sup></i>.<br>Then the corresponding threshold of calling three copies instead of two on autosomes is calculated as follows: <i>T<sub>23</sub> = log<sub>2</sub>((2+D)/2) = log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>)</i>.<br><b>Threshold of calling deletions on autosomes:</b> <i>lo = T<sub>21</sub></i>.<br><b>Threshold of calling duplications on autosomes:</b> <i>hi = T<sub>23</sub></i>.<br>To calculate the threshold of calling three copies instead of two on the X chromosome, we normalize to XY genotype by dividing it by copy number of 1: <i>T<sub>23X</sub>= log<sub>2</sub>((2+D)/1) = log<sub>2</sub>(2+2-2×2<sup>T<sub>21</sub></sup>) = 1 + log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>) = 1 + T<sub>23</sub></i>.<br><b>Calculating the threshold of calling duplications on the X chromosome:</b><br>- if there is no Y chromosome: <i>X<sub>hi</sub> = 1 + T<sub>23</sub></i>;<br>- if there is Y chromosome: <i>X<sub>hi</sub> = log<sub>2</sub>(3-2×2<sup>T<sub>21</sub></sup>)</i>.<br>The threshold of calling one copy instead of two on the X chromosome: <i>T<sub>21X</sub> = log<sub>2</sub>((2-D)/1) = log<sub>2</sub>(2-2+2×2<sup>T<sub>21</sub></sup>) = 1 + T<sub>21</sub></i>.<br>Threshold of calling no copies instead of one on the X chromosome: <i>T<sub>10X</sub> = log<sub>2</sub>((1-D)/1) = log<sub>2</sub>(1-2+2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br><b>Calculating the threshold of calling deletions on the X chromosome:</b><br>- if there is no Y chromosome: <i>X<sub>lo</sub> = 1 + T<sub>21</sub></i>;<br>- if there is Y chromosome: <i>X<sub>lo</sub> = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br>The threshold of calling one copy instead of none on the Y chromosome in the case of genotype XX: <i>T<sub>01</sub> = log<sub>2</sub>((0+D)/1) = log<sub>2</sub>(2-2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(2×(1-2<sup>T<sub>21</sub></sup>)) =  1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>)</i>.<br><b>Calculating the threshold of calling duplication <i>Y<sub>hi</sub></i> and deletions <i>Y<sub>lo</sub></i> on the Y chromosome:</b><br>- if there is no Y chromosome: <i>Y<sub>hi</sub> = 1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>))</i>; <i>Y<sub>lo</sub> = -∞</i>;<br>- if there is Y chromosome: <i>Y<sub>hi</sub> = X<sub>hi</sub></i>; <i>Y<sub>lo</sub> = X<sub>lo</sub></i>.</div></div></details><p>If <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#germline-aneuploidies-only">the CNV aneuploidies calling only</a> is
enabled for a non-tumor sample, then only aneuploidies (copy number variations on chromosomes) will be presented
in the resulting files.</p><p>As a result of processing the variations, the following files are created, which can be downloaded in the
&quot;<em>Results files</em>&quot; section in the &quot;<em>Evaluating and visualizing biological features</em>&quot; task details:</p><ul><li>The CNV coverage file in TSV format, that is an unannotated file convenient for studying copy number variations
of large regions (chromosome arms, chromosomes). File download link: &quot;<em>Download CNV coverage TSV</em>&quot;.
You can also open this file in Google Sheets.</li><li>A file with copy number bins in BED format (&quot;<em>Download CNV bins BED</em>&quot;).</li><li>A file with copy number segments in BED format (&quot;<em>Download CNV segments BED</em>&quot;).</li><li>A file with log2FC thresholds, i.e. the logarithmic ratio of the detected copy number to the normal copy number,
based on which copy number variations are filtered. The log2FC values at which variations on autosomes, the X
or Y chromosome are considered deletions or duplications are provided, too.
File download link: &quot;<em>Download CNV thresholds JSON</em>&quot;.</li></ul><p>In addition, files with genome-wide visualization and karyogram are generated, which are convenient for studying
large copy number variations. You can open them on <a href="/ngsw-docs-en/results/main/copy-number-variations/cnvs">the &quot;Main&quot; tab</a> or
download in the &quot;<em>Results files</em>&quot; section in the &quot;<em>Evaluating and visualizing biological features</em>&quot; task details:</p><ul><li>Karyogram-like graph with denoted chromosome-level CNVs in JPG format (&quot;<em>Download Karyogram-like graph JPG</em>&quot;).</li><li>Genome-wide CNV segments&#x27; plot in JPG format (&quot;<em>Download JPG</em>&quot;).</li><li>PDF file with CNV segment graphs for all chromosomes (&quot;<em>Download PDF</em>&quot;).</li><li>Files with CNV segment graphs for each chromosome separately (&quot;<em>Download [chromosome] JPG</em>&quot;).</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="annotate-cnv"></a>6. Annotate CNV VCF<a class="hash-link" href="#annotate-cnv" title="Direct link to heading">#</a></h3><p>To completely and effectively interpret the CNV impact on the phenotype, each detected variation is annotated.
Annotation and ranking of discovered copy number variations is performed using the <a href="https://lbgi.fr/AnnotSV/" target="_blank" rel="noopener noreferrer">AnnotSV</a>
program. This tool compiles functionally, regulatory and clinically relevant information and aims at providing
annotations useful to interpret CNV potential pathogenicity and filter out CNV potential false positives.
Segments and bins are annotated separately.</p><p>The resulting files with the annotated copy number variations can be downloaded in
the &quot;<em>Result files</em>&quot; section in the &quot;<em>Annotate CNV VCF</em>&quot; task details:</p><ul><li>A file with all copy number segments in TSV format (&quot;<em>Download Segments all TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with all copy number segments in CSV format (&quot;<em>Download Segments all CSV</em>&quot;).</li><li>A file with filtered copy number segments in TSV format (&quot;<em>Download Segments filtered TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with filtered copy number segments in CSV format (&quot;<em>Download Segments filtered CSV</em>&quot;).</li><li>A file with all copy number bins in TSV format (&quot;<em>Download Bins all TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with all copy number bins in CSV format (&quot;<em>Download Bins all CSV</em>&quot;).</li><li>A file with filtered copy number bins in TSV format (&quot;<em>Download Bins filtered TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with filtered copy number bins in CSV format (&quot;<em>Download Bins filtered CSV</em>&quot;).</li></ul><p>You can find the description of annotation format and fields
in <a href="https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf" target="_blank" rel="noopener noreferrer">the AnnotSV documentation</a>.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="generate-cnv-report"></a>7. Store annotated CNV bins for CNV Viewer<a class="hash-link" href="#generate-cnv-report" title="Direct link to heading">#</a></h3><p>Saving the results of copy number variations discovery in bins for display in the embedded service for viewing
and analyzing copy number variations <a href="/ngsw-docs-en/results/main/copy-number-variations/cnv-viewer">CNV Viewer</a>.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="generate-cnv-report"></a>8. Store annotated CNV segments for CNV Viewer<a class="hash-link" href="#generate-cnv-report" title="Direct link to heading">#</a></h3><p>Saving the results of copy number variations discovery in segments for display in the embedded service for viewing
and analyzing copy number variations <a href="/ngsw-docs-en/results/main/copy-number-variations/cnv-viewer">CNV Viewer</a>.</p><p>After the &quot;Copy number variations discovery&quot; stage, the analysis can proceed
with <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">report generation</a>.</p><h2><a aria-hidden="true" tabindex="-1" class="anchor anchor__h2 anchorWithStickyNavbar_31ik" id="tumor"></a>Tumor sample from a tumor/control sample set<a class="hash-link" href="#tumor" title="Direct link to heading">#</a></h2><p>In case of analyzing a tumor sequencing sample from a tumor and matched normal sample
pair, <a href="https://gatk.broadinstitute.org/" target="_blank" rel="noopener noreferrer">the GATK pipeline</a> is used to to discover copy number variations.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="intervals"></a>1. Preprocess intervals list<a class="hash-link" href="#intervals" title="Direct link to heading">#</a></h3><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/13832754597915-PreprocessIntervals" target="_blank" rel="noopener noreferrer">GATK PreprocessIntervals</a>
prepares bins for coverage collection.</p><p>The way of bin preparation varies depending on the sample sequencing type:</p><ul><li>If the sample sequencing type is defined as &quot;Panel&quot; (sequencing using a targeted panel) or &quot;WES&quot; (whole-exome
sequencing), the input intervals (capture kit) are first checked for overlapping intervals, which are merged.
Then the resulting intervals are padded and binned. Bins containing only N (unknown nucleotides) are filtered out.
Binning (grouping intervals into a smaller number of bins) is not performed. Abutting intervals (i.e. intervals
that are directly side-by-side but do not actually overlap) are not merged, but treated as separate intervals.</li><li>If the sample sequencing type is defined as &quot;WGS&quot; (whole-genome sequencing) or &quot;Low-pass WGS&quot; (whole genome
sequencing with low coverage), each contig is taken as one interval and grouped accordingly. Abutting intervals
are merged into a single continuous interval. This creates bins suitable for whole genome sequencing analysis.
The length of the generated bins is controlled
by <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#bin-size">the corresponding parameter</a>.
Bins containing only N (unknown nucleotides) are filtered out.</li></ul><p>The resulting preprocessed Picard interval-list file can be downloaded in the &quot;<em>Result files</em>&quot; section in the
&quot;<em>Preprocess intervals list</em>&quot; task details (&quot;<em>Download INTERVAL_LIST</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="read-count"></a>2. Collect read counts<a class="hash-link" href="#read-count" title="Direct link to heading">#</a></h3><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037592671-CollectReadCounts" target="_blank" rel="noopener noreferrer">GATK CollectReadCounts</a>
collects read counts at specified intervals. The count for each interval is calculated by counting the number
of read starts that lie in the interval. Abutting intervals (i.e. intervals that are directly side-by-side but
do not actually overlap) are not merged, but treated as separate intervals. The command is executed for the
alignment files of the tumor and matched normal samples. The resulting counts files in HDF5 format (Hierarchical
Data Format, Version 5) can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Collect read counts</em>&quot; task
details (the result for the normal sample is &quot;<em>Download Normal file read counts HDF5</em>&quot;, the result for the tumor
sample is &quot;<em>Download Tumor file read counts HDF5</em>&quot;).</p><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360040510031-CreateReadCountPanelOfNormals" target="_blank" rel="noopener noreferrer">GATK CreateReadCountPanelOfNormals</a>
creates a panel of normals (PoN) for read-count denoising given the read counts for samples in the panel.
It is carried out by counting the number of reads in the normal sample. Genomic intervals with a median (across
samples) of fractional coverage less than or equal to 5th percentile are filtered out. The resulting
panel-of-normals file can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Collect read counts</em>&quot; task
details (&quot;<em>Download Generated panel of normal HDF5</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="normalization"></a>3. Normalization of read counts<a class="hash-link" href="#normalization" title="Direct link to heading">#</a></h3><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360040508731-DenoiseReadCounts" target="_blank" rel="noopener noreferrer">GATK DenoiseReadCounts</a>
denoises read counts to produce denoised copy ratios. A panel of normals produced in the previous step is
used to standardize and denoise read counts in the tumor sample. Standardization includes (a) transforming
to fractional coverage, (b) filtering intervals to those contained in the panel, (c) dividing by interval
medians contained in the panel, (d) dividing by the sample median, and (e) transforming to log2 copy ratio.
The result is then denoised by subtracting the projection onto the specified number of principal components
from the panel. The output is the following files, which can be downloaded in the &quot;<em>Result files</em>&quot; section in
the &quot;<em>Normalization of read counts</em>&quot; task details:</p><ul><li>Denoised-copy-ratios file in TSV format (&quot;<em>Download Denoised copy ratios TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>Standardized-copy-ratios file in TSV format (&quot;<em>Download Standardized copy ratios TSV</em>&quot;).
You can open this file in Google Sheets, too.</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="model-segments"></a>4. Model segments<a class="hash-link" href="#model-segments" title="Direct link to heading">#</a></h3><p><a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225892-ModelSegments" target="_blank" rel="noopener noreferrer">GATK ModelSegments</a>
models segmented copy ratios from denoised read counts. Segmentation is done
using <a href="https://javadoc.io/static/org.broadinstitute/gatk/4.1.4.0/org/broadinstitute/hellbender/tools/copynumber/utils/segmentation/KernelSegmenter.html" target="_blank" rel="noopener noreferrer">kernel segmentation</a>.
Then Markov-chain Monte Carlo (MCMC) is applied to determine posteriors for segmented models for the log2 copy ratio.
Finally, smoothing of the segmented posteriors is performed by merging adjacent segments whose posterior credible
intervals sufficiently overlap sufficiently (number of 10% equal-tailed credible-interval widths must be equal to 2).
Additional rounds of segmentation smoothing are performed until convergence, at which point a final round of MCMC is
performed.</p><p>Copy-ratio segment file can be downloaded in the &quot;<em>Result files</em>&quot; section in the &quot;<em>Model segments</em>&quot; task
details (&quot;<em>Download Copy ration segments SEG</em>&quot;). A file with modeled segments after segmentation smoothing
can be downloaded in the same section (&quot;<em>Download Modeled segments SEG</em>&quot;).</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="evaluate-and-visualize"></a>5. Evaluating and visualizing biological features<a class="hash-link" href="#evaluate-and-visualize" title="Direct link to heading">#</a></h3><p>At this step, the copy number variations are processed depending on the log2FC threshold value, that is
the logarithmic ratio of the detected copy number to the normal copy number. The default threshold value (-0.7)
can be changed in <a href="/ngsw-docs-en/settings/settings/variant-discovery-settings#cnv-call-threshold">the parameters</a>. Using this
value as an example, let&#x27;s see how the thresholds for calling deletions and duplications on autosomes and sex
chromosomes will be calculated:</p><ul><li>Copy number variations on autosomes with log2FC ≤ -0.7 are considered deletions. -0.7 is the default
log2FC threshold, below which only one autosome is called instead of the normal two.</li><li>Copy number variations on the X and Y chromosomes with log2FC ≤ -2.11 are considered deletions.</li><li>Copy number variations on autosomes with log2FC ≥ 0.46 are considered duplications.
The threshold for duplication on an autosome is calculated using the following
formula: <em>log2FC = log<sub>2</sub>((2+D)/2)</em>, where <em>D</em> is the “number” of copy number that
is “lost” at log2FC = -0.7, and <em>2</em> is the number of sister chromatids in the autosome.</li><li>Copy number variations on the X and Y chromosomes with log2FC ≥ 0.82 are considered duplications.</li><li>Variations that do not satisfy the thresholds, i.e. variations on autosomes with -0.7 &lt; log2FC &lt; 0.46
and variations on sex chromosomes with -2.11 &lt; log2FC &lt; 0.82 do not pass filtering.</li></ul><details class="details_2Ziz alert alert--info details_1VDD" data-collapsed="true"><summary>Click here to see how log2FC thresholds are calculated for deletions and duplications on autosomes and sex chromosomes</summary><div><div class="collapsibleContent_3OHp">The threshold of calling a single copy instead of two on autosomes is calculated as follows: <i>T<sub>21</sub> = log<sub>2</sub>(C/2)</i>, where <i>C</i> is the number of called copies.<br>Delta of copy number: <i>D = 2 – C = 2 – 2 × 2<sup>T<sub>21</sub></sup></i>.<br>Then the corresponding threshold of calling three copies instead of two on autosomes is calculated as follows: <i>T<sub>23</sub> = log<sub>2</sub>((2+D)/2) = log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>)</i>.<br><b>Threshold of calling deletions on autosomes:</b> <i>lo = T<sub>21</sub></i>.<br><b>Threshold of calling duplications on autosomes:</b> <i>hi = T<sub>23</sub></i>.<br>To calculate the threshold of calling three copies instead of two on the X chromosome, we normalize to XY genotype by dividing it by copy number of 1: <i>T<sub>23X</sub>= log<sub>2</sub>((2+D)/1) = log<sub>2</sub>(2+2-2×2<sup>T<sub>21</sub></sup>) = 1 + log<sub>2</sub>(2-2<sup>T<sub>21</sub></sup>) = 1 + T<sub>23</sub></i>.<br><b>Calculating the threshold of calling duplications on the X chromosome:</b><br>- if there is no Y chromosome: <i>X<sub>hi</sub> = 1 + T<sub>23</sub></i>;<br>- if there is Y chromosome: <i>X<sub>hi</sub> = log<sub>2</sub>(3-2×2<sup>T<sub>21</sub></sup>)</i>.<br>The threshold of calling one copy instead of two on the X chromosome: <i>T<sub>21X</sub> = log<sub>2</sub>((2-D)/1) = log<sub>2</sub>(2-2+2×2<sup>T<sub>21</sub></sup>) = 1 + T<sub>21</sub></i>.<br>Threshold of calling no copies instead of one on the X chromosome: <i>T<sub>10X</sub> = log<sub>2</sub>((1-D)/1) = log<sub>2</sub>(1-2+2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br><b>Calculating the threshold of calling deletions on the X chromosome:</b><br>- if there is no Y chromosome: <i>X<sub>lo</sub> = 1 + T<sub>21</sub></i>;<br>- if there is Y chromosome: <i>X<sub>lo</sub> = log<sub>2</sub>(-1+2×2<sup>T<sub>21</sub></sup>)</i>.<br>The threshold of calling one copy instead of none on the Y chromosome in the case of genotype XX: <i>T<sub>01</sub> = log<sub>2</sub>((0+D)/1) = log<sub>2</sub>(2-2×2<sup>T<sub>21</sub></sup>) = log<sub>2</sub>(2×(1-2<sup>T<sub>21</sub></sup>)) =  1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>)</i>.<br><b>Calculating the threshold of calling duplication <i>Y<sub>hi</sub></i> and deletions <i>Y<sub>lo</sub></i> on the Y chromosome:</b><br>- if there is no Y chromosome: <i>Y<sub>hi</sub> = 1 + log<sub>2</sub>(1-2<sup>T<sub>21</sub></sup>))</i>; <i>Y<sub>lo</sub> = -∞</i>;<br>- if there is Y chromosome: <i>Y<sub>hi</sub> = X<sub>hi</sub></i>; <i>Y<sub>lo</sub> = X<sub>lo</sub></i>.</div></div></details><p>As a result of processing the variations, the following files are created, which can be downloaded in the
&quot;<em>Results files</em>&quot; section in the &quot;<em>Evaluating and visualizing biological features</em>&quot; task details:</p><ul><li>The CNV coverage file in TSV format, that is an unannotated file convenient for studying copy number variations
of large regions (chromosome arms, chromosomes). File download link: &quot;<em>Download CNV coverage TSV</em>&quot;.
You can also open this file in Google Sheets.</li><li>A file with copy number bins in BED format (&quot;<em>Download CNV bins BED</em>&quot;).</li><li>A file with copy number segments in BED format (&quot;<em>Download CNV segments BED</em>&quot;).</li><li>A file with log2FC thresholds, i.e. the logarithmic ratio of the detected copy number to the normal copy number,
based on which copy number variations are filtered. The log2FC values at which variations on autosomes, the X
or Y chromosome are considered deletions or duplications are provided, too.
File download link: &quot;<em>Download CNV thresholds JSON</em>&quot;.</li></ul><p>In addition, files with genome-wide visualization and karyogram are generated, which are convenient for studying
large copy number variations. You can open them on <a href="/ngsw-docs-en/results/main/copy-number-variations/cnvs">the &quot;Main&quot; tab</a> or
download in the &quot;<em>Results files</em>&quot; section in the &quot;<em>Evaluating and visualizing biological features</em>&quot; task details:</p><ul><li>Karyogram-like graph with denoted chromosome-level CNVs in JPG format (&quot;<em>Download Karyogram-like graph JPG</em>&quot;).</li><li>Genome-wide CNV segments&#x27; plot in JPG format (&quot;<em>Download JPG</em>&quot;).</li><li>PDF file with CNV segment graphs for all chromosomes (&quot;<em>Download PDF</em>&quot;).</li><li>Files with CNV segment graphs for each chromosome separately (&quot;<em>Download [chromosome] JPG</em>&quot;).</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="cnv-discovery-tumor"></a>6. CNV discovery<a class="hash-link" href="#cnv-discovery-tumor" title="Direct link to heading">#</a></h3><p>This is the key step where the copy number of each genome region is calculated based on the number of reads
mapped to the genome. In case of analyzing a tumor sequencing sample from a tumor and matched normal sample
pair, <a href="https://gatk.broadinstitute.org/" target="_blank" rel="noopener noreferrer">the GATK pipeline</a> is used.</p><p>Copy-ratio segments are called as amplified, deleted, or copy-number neutral using
<a href="https://gatk.broadinstitute.org/hc/en-us/articles/360041850791-CallCopyRatioSegments" target="_blank" rel="noopener noreferrer">GATK CallCopyRatioSegments</a>.
This is a relatively naive caller that takes the segments modeled
in <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/cnv-discovery#model-segments">the &quot;Model segments&quot; step</a> and
performs a simple statistical test on the segmented log2 copy ratios to call amplifications and deletions, given
a specified range for determining copy-number neutral segments.</p><p>The output is the following files, which can be downloaded in the &quot;<em>Result files</em>&quot; section in
the &quot;<em>CNV discovery</em>&quot; task details:</p><ul><li>Called copy-ratio-segments file in TSV format (&quot;<em>Download Called segments SEG</em>&quot;).</li><li>CBS-formatted (circular binary segmentation) file with information on segments (&quot;<em>Download Statistics SEG</em>&quot;).
It is compatible with <a href="https://software.broadinstitute.org/software/igv/" target="_blank" rel="noopener noreferrer">the Integrative Genomics Viewer (IGV)</a>.
You can find a description of the file format and its
columns <a href="https://www.genepattern.org/file-formats-guide#CBS" target="_blank" rel="noopener noreferrer">here</a>.</li></ul><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="annotate-cnv"></a>7. Annotate CNV VCF<a class="hash-link" href="#annotate-cnv" title="Direct link to heading">#</a></h3><p>To completely and effectively interpret the CNV impact on the phenotype, each detected variation is annotated.
Annotation and ranking of discovered copy number variations is performed using the <a href="https://lbgi.fr/AnnotSV/" target="_blank" rel="noopener noreferrer">AnnotSV</a>
program. This tool compiles functionally, regulatory and clinically relevant information and aims at providing
annotations useful to interpret CNV potential pathogenicity and filter out CNV potential false positives.
Segments and bins are annotated separately.</p><p>The resulting files with the annotated copy number variations can be downloaded in
the &quot;<em>Result files</em>&quot; section in the &quot;<em>Annotate CNV VCF</em>&quot; task details:</p><ul><li>A file with all copy number segments in TSV format (&quot;<em>Download Segments all TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with all copy number segments in CSV format (&quot;<em>Download Segments all CSV</em>&quot;).</li><li>A file with filtered copy number segments in TSV format (&quot;<em>Download Segments filtered TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with filtered copy number segments in CSV format (&quot;<em>Download Segments filtered CSV</em>&quot;).</li><li>A file with all copy number bins in TSV format (&quot;<em>Download Bins all TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with all copy number bins in CSV format (&quot;<em>Download Bins all CSV</em>&quot;).</li><li>A file with filtered copy number bins in TSV format (&quot;<em>Download Bins filtered TSV</em>&quot;).
You can also open this file in Google Sheets.</li><li>A file with filtered copy number bins in CSV format (&quot;<em>Download Bins filtered CSV</em>&quot;).</li></ul><p>You can find the description of annotation format and fields
in <a href="https://lbgi.fr/AnnotSV/Documentation/README.AnnotSV_latest.pdf" target="_blank" rel="noopener noreferrer">the AnnotSV documentation</a>.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="generate-cnv-report"></a>8. Store annotated CNV bins for CNV Viewer<a class="hash-link" href="#generate-cnv-report" title="Direct link to heading">#</a></h3><p>Saving the results of copy number variations discovery in bins for display in the embedded service for viewing
and analyzing copy number variations <a href="/ngsw-docs-en/results/main/copy-number-variations/cnv-viewer">CNV Viewer</a>.</p><h3><a aria-hidden="true" tabindex="-1" class="anchor anchor__h3 anchorWithStickyNavbar_31ik" id="generate-cnv-report"></a>9. Store annotated CNV segments for CNV Viewer<a class="hash-link" href="#generate-cnv-report" title="Direct link to heading">#</a></h3><p>Saving the results of copy number variations discovery in segments for display in the embedded service for viewing
and analyzing copy number variations <a href="/ngsw-docs-en/results/main/copy-number-variations/cnv-viewer">CNV Viewer</a>.</p><p>After the &quot;Copy number variations discovery&quot; stage, the analysis can proceed
with <a href="/ngsw-docs-en/results/workflow-details/fastq-analysis/report-generation">report generation</a>.</p></div></article><nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation"><div class="pagination-nav__item"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/structural-variations"><div class="pagination-nav__sublabel">Previous</div><div class="pagination-nav__label">« Structural variations discovery</div></a></div><div class="pagination-nav__item pagination-nav__item--next"><a class="pagination-nav__link" href="/ngsw-docs-en/results/workflow-details/fastq-analysis/phenotypes"><div class="pagination-nav__sublabel">Next</div><div class="pagination-nav__label">Phenotype prediction »</div></a></div></nav></div></div><div class="col col--3"><div class="tableOfContents_35-E thin-scrollbar"><ul class="table-of-contents table-of-contents__left-border"><li><a href="#normal" class="table-of-contents__link">Non-tumor or single tumor sample</a><ul><li><a href="#estimate-read-counts-or-depths" class="table-of-contents__link">1. Estimate read counts or depths</a></li><li><a href="#calculate-coverage" class="table-of-contents__link">2. Calculate coverage in the given regions</a></li><li><a href="#compile-cn-reference" class="table-of-contents__link">3. Compile a copy-number reference</a></li><li><a href="#cnv-discovery" class="table-of-contents__link">4. CNV discovery</a></li><li><a href="#evaluate-and-visualize" class="table-of-contents__link">5. Evaluating and visualizing biological features</a></li><li><a href="#annotate-cnv" class="table-of-contents__link">6. Annotate CNV VCF</a></li><li><a href="#generate-cnv-report" class="table-of-contents__link">7. Store annotated CNV bins for CNV Viewer</a></li><li><a href="#generate-cnv-report" class="table-of-contents__link">8. Store annotated CNV segments for CNV Viewer</a></li></ul></li><li><a href="#tumor" class="table-of-contents__link">Tumor sample from a tumor/control sample set</a><ul><li><a href="#intervals" class="table-of-contents__link">1. Preprocess intervals list</a></li><li><a href="#read-count" class="table-of-contents__link">2. Collect read counts</a></li><li><a href="#normalization" class="table-of-contents__link">3. Normalization of read counts</a></li><li><a href="#model-segments" class="table-of-contents__link">4. Model segments</a></li><li><a href="#evaluate-and-visualize" class="table-of-contents__link">5. Evaluating and visualizing biological features</a></li><li><a href="#cnv-discovery-tumor" class="table-of-contents__link">6. CNV discovery</a></li><li><a href="#annotate-cnv" class="table-of-contents__link">7. Annotate CNV VCF</a></li><li><a href="#generate-cnv-report" class="table-of-contents__link">8. Store annotated CNV bins for CNV Viewer</a></li><li><a href="#generate-cnv-report" class="table-of-contents__link">9. Store annotated CNV segments for CNV Viewer</a></li></ul></li></ul></div></div></div></div></main></div></div><footer class="footer footer--dark"><div class="container"><div class="footer__bottom text--center"><div class="footer__copyright">© 2019-2024 Novel Software Systems. All rights reserved.</div></div></div></footer></div>
<script src="/ngsw-docs-en/assets/js/runtime~main.905320c1.js"></script>
<script src="/ngsw-docs-en/assets/js/main.d04fc880.js"></script>
</body>
</html>